<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="">

<title>sc bench benchmark â€“ sc_bench_benchmark_dashboard</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./https://www.sydney.edu.au/etc.clientlibs/corporate-commons/clientlibs/foundation/resources/corporate-frontend/assets/img/favicon/favicon-16x16.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-dashboard/quarto-dashboard.js"></script>
<script src="site_libs/quarto-dashboard/stickythead.js"></script>
<script src="site_libs/quarto-dashboard/datatables.min.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/pdfmake.min.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/vfs_fonts.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/web-components.js" type="module"></script>
<script src="site_libs/quarto-dashboard/components.js"></script>
<link href="site_libs/quarto-dashboard/datatables.min.css" rel="stylesheet" kdttablesentinel="true">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script type="module" src="site_libs/quarto-ojs/quarto-ojs-runtime.js"></script>
<link href="site_libs/quarto-ojs/quarto-ojs.css" rel="stylesheet">
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">

<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>

<script src="site_libs/plotly-binding-4.10.4/plotly.js"></script>

<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>

<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>

<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">

<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>

<link href="site_libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">

<script src="site_libs/plotly-main-2.11.1/plotly-latest.min.js"></script>

<script src="site_libs/d3-4.5.0/d3.min.js"></script>

<script src="site_libs/sankey-1/sankey.js"></script>

<script src="site_libs/sankeyNetwork-binding-0.4/sankeyNetwork.js"></script>

<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">

<script src="site_libs/datatables-binding-0.33/datatables.js"></script>

<link href="site_libs/dt-core-bootstrap-1.13.6/css/dataTables.bootstrap.min.css" rel="stylesheet">

<link href="site_libs/dt-core-bootstrap-1.13.6/css/dataTables.bootstrap.extra.css" rel="stylesheet">

<script src="site_libs/dt-core-bootstrap-1.13.6/js/jquery.dataTables.min.js"></script>

<script src="site_libs/dt-core-bootstrap-1.13.6/js/dataTables.bootstrap.min.js"></script>

<link href="site_libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet">

<script src="site_libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>

<link href="site_libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet">

<script src="site_libs/selectize-0.12.0/selectize.min.js"></script>



</head>

<body class="quarto-dashboard nav-sidebar dashboard-fill fullcontent">

<header id="quarto-dashboard-header">
<nav class="navbar navbar-expand-md slim" data-bs-theme="dark">
  <div class="navbar-container container-fluid">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#dashboard-collapse" aria-controls="dashboard-collapse" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>    


    <div class="navbar-brand-container">       
    <a href="#"><img src="https://www.sydney.edu.au/etc.clientlibs/corporate-commons/clientlibs/foundation/resources/corporate-frontend/assets/img/favicon/favicon-16x16.png" alt="" class="navbar-logo d-inline-block"></a>
      <div class="navbar-title">
        <div class="navbar-title-text"><a href="#">sc bench benchmark</a></div>
        
        
      </div>
    </div>
<div id="dashboard-collapse" class="navbar-collapse collapse"><ul class="navbar-nav navbar-nav-scroll me-auto" role="tablist"><li class="nav-item" role="presentation"><a id="tab-paper" class="nav-link active" data-bs-toggle="tab" role="tab" data-bs-target="#paper" data-scrolling="false" href="#paper" aria-controls="paper" aria-selected="true"><span class="nav-link-text">Paper</span></a></li><li class="nav-item" role="presentation"><a id="tab-data-overview" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#data-overview" data-scrolling="false" href="#data-overview" aria-controls="data-overview" aria-selected="false"><span class="nav-link-text">Data overview</span></a></li><li class="nav-item" role="presentation"><a id="tab-evaluation-summary" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#evaluation-summary" data-scrolling="false" href="#evaluation-summary" aria-controls="evaluation-summary" aria-selected="false"><span class="nav-link-text">Evaluation Summary</span></a></li><li class="nav-item" role="presentation"><a id="tab-general-information" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#general-information" data-scrolling="false" href="#general-information" aria-controls="general-information" aria-selected="false"><span class="nav-link-text">General Information</span></a></li></ul><div class="quarto-dashboard-links"><a class="quarto-dashboard-link" href="https://github.com/SydneyBioX/sc_bench_benchmark_dashboard/" title="GitHub"><i class="bi bi-github"></i></a><a class="quarto-dashboard-link" href="https://twitter.com/intent/tweet?url=|url|&amp;text=sc%20bench%20benchmark" title="Twitter"><i class="bi bi-twitter"></i></a></div></div></div>
</nav>
</header>
<div id="quarto-search-results"></div>
<!-- content -->
<div id="quarto-content" class="quarto-container page-layout-custom quarto-dashboard-content bslib-gap-spacing html-fill-container bslib-page-fill quarto-dashboard-pages">
<!-- sidebar -->
<!-- margin-sidebar -->
<!-- main -->

<div class="cell hidden bslib-grid-item html-fill-item">
<div class="sourceCode cell-code hidden" id="cb1" data-startfrom="20" data-source-offset="-1"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 19;"><span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a>d3 <span class="op">=</span> <span class="pp">require</span>(<span class="st">"d3@7"</span>)</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a>topojson <span class="op">=</span> <span class="pp">require</span>(<span class="st">"topojson"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display hidden">
<div>
<div id="ojs-cell-1-1" data-nodetype="declaration">

</div>
</div>
</div>
<div class="cell-output cell-output-display hidden">
<div>
<div id="ojs-cell-1-2" data-nodetype="declaration">

</div>
</div>
</div>
</div>
<div class="cell hidden bslib-grid-item html-fill-item">
<div class="sourceCode cell-code hidden" id="cb2" data-startfrom="27" data-source-offset="0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 26;"><span id="cb2-27"><a href="#cb2-27" aria-hidden="true" tabindex="-1"></a>data_o <span class="op">=</span> <span class="fu">FileAttachment</span>(<span class="st">"data/data2.csv"</span>)<span class="op">.</span><span class="fu">csv</span>({ <span class="dt">typed</span><span class="op">:</span> <span class="kw">true</span> })</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display hidden">
<div id="ojs-cell-2" data-nodetype="declaration">

</div>
</div>
</div>
<div class="bslib-grid-item html-fill-item tab-content bslib-grid" data-layout="fill" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div id="paper" class="dashboard-page tab-pane show active html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-paper">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" data-width="100%" style="display: grid; grid-template-columns: minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<!-- {{< include ../includes/papers/read_data/index.qmd >}} --><div class="card tabset html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header dashboard-card-no-title bslib-navs-card-title">

<ul class="nav nav-tabs card-header-tabs" role="tablist" data-tabsetid="card-tabset-1"><li class="nav-item" role="presentation"><a href="#card-tabset-1-1" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Category" aria-selected="true" class="nav-link active">Category</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-1-2" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Survey Analysis" aria-selected="false" class="nav-link">Survey Analysis</a></li></ul></div>


<div class="tab-content html-fill-item html-fill-container" data-tabset-id="card-tabset-1"><div class="tab-pane active show html-fill-item html-fill-container" role="tabpanel" id="card-tabset-1-1"><div class="card-body html-fill-item html-fill-container" data-title="Category">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 20fr) minmax(3em, 40fr) minmax(3em, 40fr); grid-auto-columns: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 1fr) minmax(3em, 1fr) minmax(3em, 1fr) minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-warning showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="database-gear" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-database-gear"></i></div>
<div class="value-box-area">
<p class="value-box-title">
Pure benchmark papers
</p>
<p class="value-box-value">
95
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div><div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-info showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="database-gear" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-database-gear"></i></div>
<div class="value-box-area">
<p class="value-box-title">
New method development papers
</p>
<p class="value-box-value">
150
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div><div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-success showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="eyeglasses" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-eyeglasses"></i></div>
<div class="value-box-area">
<p class="value-box-title">
Total number of readers
</p>
<p class="value-box-value">
31
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div><div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-danger showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="book" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-book"></i></div>
<div class="value-box-area">
<p class="value-box-title">
Total number of reading
</p>
<p class="value-box-value">
356
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure benchmark papers" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure benchmark papers</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">

</div>
<div class="cell quarto-layout-panel html-fill-item html-fill-container" title="Pure benchmark papers" data-layout-ncol="2">
<div class="quarto-layout-row html-fill-item html-fill-container">
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-a1dcf845d5e5725cfc82" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-a1dcf845d5e5725cfc82">{"x":{"visdat":{"47647f2d2d73":["function () ","plotlyVisDat"]},"cur_data":"47647f2d2d73","attrs":{"47647f2d2d73":{"ids":{},"labels":{},"parents":{},"values":{},"insidetextorientation":"radial","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"sunburst"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"ids":["Benchmark","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell proteomics","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell mass cytometry","Benchmark - Spatial transcriptomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell multiomics","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Single-cell proteomics - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell mass cytometry - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell RNA-seq - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell proteomics - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell RNA-seq - Data - Benchmarking datasets","Benchmark - Single-cell RNA-seq - Initial analysis - Alignment","Benchmark - Single-cell RNA-seq - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell ATAC-seq - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell RNA-seq - Intermediate analysis - Clustering","Benchmark - Single-cell mass cytometry - Intermediate analysis - Clustering","Benchmark - Spatial transcriptomics - Intermediate analysis - Clustering","Benchmark - Single-cell Hi-C - Intermediate analysis - Clustering","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential expression","Benchmark - Single-cell RNA-seq - Intermediate analysis - Dimension reduction","Benchmark - Single-cell multiomics - Intermediate analysis - Dimension reduction","Benchmark - Single-cell RNA-seq - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Downstream analysis - Cell cell communication","Benchmark - Single-cell RNA-seq - Intermediate analysis - Measure of association","Benchmark - Single-cell RNA-seq - Downstream analysis - Pathway enrichment","Benchmark - Single-cell RNA-seq - Downstream analysis - T/B-cell receptor","Benchmark - Single-cell RNA-seq - Initial analysis - Imputation","Benchmark - Single-cell ATAC-seq - Initial analysis - Imputation","Benchmark - Single-cell multiomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Intermediate analysis - Batch correction","Benchmark - Spatial transcriptomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Intermediate analysis - Data integration","Benchmark - Spatial transcriptomics - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Initial analysis - Standardization","Benchmark - Single-cell RNA-seq - Downstream analysis - Trajectory","Benchmark - Single-cell RNA-seq - Downstream analysis - Velocity","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential analysis - others","Benchmark - Single-cell RNA-seq - Initial analysis - Doublet detection","Benchmark - Single-cell RNA-seq - Initial analysis - Quantification","Benchmark - Single-cell RNA-seq - Data - Simulation","Benchmark - Single-cell RNA-seq - Downstream analysis - Highly variable genes","Benchmark - Single-cell RNA-seq - Intermediate analysis - Subclone detection","Benchmark - Single-cell RNA-seq - Initial analysis - Visualisation"],"labels":["Pure benchmarking paper","Single-cell RNA-seq","Single-cell proteomics","Single-cell ATAC-seq","Single-cell mass cytometry","Spatial transcriptomics","Single-cell Hi-C","Single-cell multiomics","Analysis pipelines","Analysis pipelines","Analysis pipelines","Data","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Intermediate analysis","Initial analysis","Initial analysis","Analysis pipelines","Analysis pipelines","Analysis pipelines","Benchmarking datasets","Alignment","Cell type/state identification","Cell type/state identification","Clustering","Clustering","Clustering","Clustering","Differential expression","Dimension reduction","Dimension reduction","Regulatory network inference","Cell cell communication","Measure of association","Pathway enrichment","T/B-cell receptor","Imputation","Imputation","Data integration","Deconvolution","Batch correction","Data integration","Data integration","Deconvolution","Standardization","Trajectory","Velocity","Differential analysis - others","Doublet detection","Quantification","Simulation","Highly variable genes","Subclone detection","Visualisation"],"parents":["","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell proteomics","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell mass cytometry","Benchmark - Spatial transcriptomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell multiomics","Benchmark - Single-cell ATAC-seq","Benchmark - Spatial transcriptomics","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Single-cell proteomics - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell mass cytometry - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell ATAC-seq - Initial analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis"],"values":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,12,2,2,2,6,14,2,10,4,2,2,16,18,2,12,4,4,6,2,8,2,4,8,13,4,2,4,4,2,2,2,2,2,5,2,2,2],"insidetextorientation":"radial","type":"sunburst","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(255,255,255,1)"}},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="sankeyNetwork html-widget html-fill-item html-fill-container" id="htmlwidget-7e5aa9464e3cfdab24d0" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-7e5aa9464e3cfdab24d0">{"x":{"links":{"source":[0,0,0,1,2,3,3,4,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,6,6,6,7,7,8,8,8,8,8,8,8,8,8,9,9,9,9,9,9,9,9,9,9,9,9,9,9,10,10,10,10,10,10,10,10,10,11,11,11],"target":[8,9,11,9,9,9,9,11,7,7,8,8,8,8,8,8,8,9,9,9,9,9,9,9,10,10,10,10,10,10,10,10,10,11,8,9,9,12,13,14,15,15,16,17,17,18,19,20,21,22,22,23,23,23,23,24,24,24,25,25,26,27,28,29,30,31,32,33,34,35,36,11,11,11],"value":[2,2,2,2,4,2,4,2,2,5,6,8,8,4,2,2,2,14,10,18,4,13,2,2,16,12,4,6,2,2,2,2,2,12,4,2,4,2,5,6,8,4,2,8,2,2,4,2,13,14,2,10,4,2,2,4,4,2,18,2,4,2,4,2,16,2,6,12,2,2,2,12,2,2]},"nodes":{"name":["Single-cell ATAC-seq","Single-cell Hi-C","Single-cell mass cytometry","Single-cell multiomics","Single-cell proteomics","Single-cell RNA-seq","Spatial transcriptomics","Data","Initial analysis","Intermediate analysis","Downstream analysis","Analysis pipelines","Benchmarking datasets","Simulation","Alignment","Deconvolution","Doublet detection","Imputation","Quantification","Standardization","Visualisation","Batch correction","Cell type/state identification","Clustering","Data integration","Dimension reduction","Measure of association","Subclone detection","Cell cell communication","Differential analysis - others","Differential expression","Highly variable genes","Pathway enrichment","Regulatory network inference","T/B-cell receptor","Trajectory","Velocity"],"group":["A","B","C","D","E","F","G","CAT_A","CAT_B","CAT_C","CAT_D","CAT_E","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z"]},"options":{"NodeID":"name","NodeGroup":"group","LinkGroup":null,"colourScale":"d3.scaleOrdinal(d3.schemeCategory20);","fontSize":12,"fontFamily":null,"nodeWidth":15,"nodePadding":10,"units":"","margin":{"top":null,"right":null,"bottom":null,"left":null},"iterations":1,"sinksRight":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development papers" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development papers</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">

</div>
<div class="cell quarto-layout-panel html-fill-item html-fill-container" title="New method development papers" data-layout-ncol="2">
<div class="quarto-layout-row html-fill-item html-fill-container">
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-00c2ea9abcb78758e058" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-00c2ea9abcb78758e058">{"x":{"visdat":{"4764576f123":["function () ","plotlyVisDat"]},"cur_data":"4764576f123","attrs":{"4764576f123":{"ids":{},"labels":{},"parents":{},"values":{},"insidetextorientation":"radial","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"sunburst"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"ids":["Benchmark","Benchmark - Spatial imaging","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell multiomics","Benchmark - Spatial transcriptomics","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell Hi-C","Benchmark - Single-cell genomics","Benchmark - Single-cell sequencing","Benchmark - Single-cell ChIP-seq","Benchmark - Spatial imaging - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell genomics - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Spatial imaging - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell multiomics - Downstream analysis","Benchmark - Single-cell ATAC-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell multiomics - Initial analysis","Benchmark - Single-cell Hi-C - Initial analysis","Benchmark - Single-cell sequencing - Downstream analysis","Benchmark - Spatial imaging - Downstream analysis","Benchmark - Single-cell sequencing - Initial analysis","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell ATAC-seq - Data","Benchmark - Single-cell genomics - Data","Benchmark - Single-cell ChIP-seq - Data","Benchmark - Spatial transcriptomics - Downstream analysis","Benchmark - Spatial imaging - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell RNA-seq - Analysis pipelines - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis - Cell segmentation","Benchmark - Spatial transcriptomics - Initial analysis - Cell segmentation","Benchmark - Single-cell Hi-C - Intermediate analysis - Chromatin conformation","Benchmark - Single-cell RNA-seq - Analysis pipelines - Multi-task","Benchmark - Spatial transcriptomics - Analysis pipelines - Multi-task","Benchmark - Single-cell multiomics - Analysis pipelines - Multi-task","Benchmark - Spatial imaging - Initial analysis - Assembly","Benchmark - Single-cell RNA-seq - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell genomics - Intermediate analysis - Cell type/state identification","Benchmark - Spatial transcriptomics - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell multiomics - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell RNA-seq - Intermediate analysis - Clustering","Benchmark - Spatial imaging - Intermediate analysis - Clustering","Benchmark - Single-cell ATAC-seq - Intermediate analysis - Clustering","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential expression","Benchmark - Single-cell RNA-seq - Intermediate analysis - Dimension reduction","Benchmark - Single-cell multiomics - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Intermediate analysis - Measure of association","Benchmark - Single-cell multiomics - Downstream analysis - Cell cell communication","Benchmark - Single-cell ATAC-seq - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Downstream analysis - T/B-cell receptor","Benchmark - Single-cell RNA-seq - Initial analysis - Imputation","Benchmark - Single-cell multiomics - Initial analysis - Imputation","Benchmark - Spatial transcriptomics - Initial analysis - Imputation","Benchmark - Spatial transcriptomics - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Intermediate analysis - Batch correction","Benchmark - Single-cell multiomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Intermediate analysis - Data integration","Benchmark - Spatial transcriptomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Downstream analysis - Marker/signature identification","Benchmark - Single-cell Hi-C - Initial analysis - Standardization","Benchmark - Single-cell RNA-seq - Initial analysis - Standardization","Benchmark - Single-cell sequencing - Downstream analysis - Trajectory","Benchmark - Single-cell RNA-seq - Downstream analysis - Trajectory","Benchmark - Spatial imaging - Downstream analysis - Trajectory","Benchmark - Single-cell RNA-seq - Downstream analysis - Velocity","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential analysis - others","Benchmark - Single-cell multiomics - Initial analysis - Doublet detection","Benchmark - Single-cell RNA-seq - Initial analysis - Doublet detection","Benchmark - Single-cell sequencing - Initial analysis - Quantification","Benchmark - Single-cell RNA-seq - Initial analysis - Quantification","Benchmark - Single-cell RNA-seq - Intermediate analysis - Rare cell type identification","Benchmark - Single-cell RNA-seq - Data - Simulation","Benchmark - Single-cell ATAC-seq - Data - Simulation","Benchmark - Single-cell genomics - Data - Simulation","Benchmark - Single-cell ChIP-seq - Data - Simulation","Benchmark - Spatial transcriptomics - Downstream analysis - Highly variable genes","Benchmark - Single-cell Hi-C - Intermediate analysis - Subclone detection","Benchmark - Single-cell RNA-seq - Intermediate analysis - Subclone detection","Benchmark - Single-cell RNA-seq - Initial analysis - Visualisation"],"labels":["New method development paper","Spatial imaging","Single-cell ATAC-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell Hi-C","Single-cell genomics","Single-cell sequencing","Single-cell ChIP-seq","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Downstream analysis","Downstream analysis","Initial analysis","Data","Data","Data","Data","Downstream analysis","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Cell segmentation","Cell segmentation","Chromatin conformation","Multi-task","Multi-task","Multi-task","Assembly","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Clustering","Clustering","Clustering","Differential expression","Dimension reduction","Regulatory network inference","Regulatory network inference","Measure of association","Cell cell communication","Regulatory network inference","T/B-cell receptor","Imputation","Imputation","Imputation","Deconvolution","Batch correction","Data integration","Data integration","Data integration","Deconvolution","Marker/signature identification","Standardization","Standardization","Trajectory","Trajectory","Trajectory","Velocity","Differential analysis - others","Doublet detection","Doublet detection","Quantification","Quantification","Rare cell type identification","Simulation","Simulation","Simulation","Simulation","Highly variable genes","Subclone detection","Subclone detection","Visualisation"],"parents":["","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark - Spatial imaging","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell multiomics","Benchmark - Spatial transcriptomics","Benchmark - Single-cell RNA-seq","Benchmark - Spatial imaging","Benchmark - Spatial transcriptomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell genomics","Benchmark - Spatial transcriptomics","Benchmark - Single-cell multiomics","Benchmark - Spatial imaging","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell multiomics","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell multiomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell sequencing","Benchmark - Spatial imaging","Benchmark - Single-cell sequencing","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell genomics","Benchmark - Single-cell ChIP-seq","Benchmark - Spatial transcriptomics","Benchmark - Spatial imaging - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell genomics - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial imaging - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Downstream analysis","Benchmark - Single-cell ATAC-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell multiomics - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell Hi-C - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell sequencing - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Spatial imaging - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell multiomics - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell sequencing - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell ATAC-seq - Data","Benchmark - Single-cell genomics - Data","Benchmark - Single-cell ChIP-seq - Data","Benchmark - Spatial transcriptomics - Downstream analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis"],"values":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2,2,2,2,3,2,1,2,3,1,1,1,20,1,1,1,11,1,1,3,1,7,3,2,1,1,1,5,1,3,3,6,6,3,2,1,7,1,2,1,6,1,1,4,1,1,1,1,4,10,2,1,1,5,1,3,3],"insidetextorientation":"radial","type":"sunburst","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(255,255,255,1)"}},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="sankeyNetwork html-widget html-fill-item html-fill-container" id="htmlwidget-b2ead41d29048225fd4e" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-b2ead41d29048225fd4e">{"x":{"links":{"source":[0,0,0,0,1,2,2,3,3,3,4,4,4,4,4,4,4,4,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,6,6,7,7,7,7,7,8,8,8,8,8,8,8,8,9,9,9,9,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,12,12,12,12,12,12,12,12,12,12,12,12,12,13,13,13,13,13,13,13,13],"target":[9,11,12,13,9,9,11,10,11,11,10,10,11,11,12,12,13,13,9,10,10,10,10,10,10,11,11,11,11,11,11,11,11,12,12,12,12,12,12,12,13,13,10,12,10,10,11,12,13,10,10,10,11,11,12,13,13,14,14,14,14,15,16,16,17,17,18,18,19,19,19,20,20,21,21,22,23,24,24,24,24,25,26,26,26,27,27,27,28,29,30,31,31,32,33,34,35,36,37,37,37,38,39,39,39,40,13,13,13,13,13,41,41,41],"value":[2,1,1,2,1,1,1,1,2,1,1,1,1,6,7,1,2,1,10,5,1,2,1,1,3,20,11,1,2,6,3,4,3,3,3,1,7,6,1,4,3,3,1,1,2,1,1,1,2,1,3,3,1,2,5,2,1,10,2,1,1,1,2,1,3,1,1,1,5,1,3,1,1,1,2,3,6,20,1,1,1,2,11,1,1,6,3,2,1,2,4,1,3,1,4,3,5,7,7,3,1,1,1,6,1,1,2,2,2,2,3,3,1,1]},"nodes":{"name":["Single-cell ATAC-seq","Single-cell ChIP-seq","Single-cell genomics","Single-cell Hi-C","Single-cell multiomics","Single-cell RNA-seq","Single-cell sequencing","Spatial imaging","Spatial transcriptomics","Data","Initial analysis","Intermediate analysis","Downstream analysis","Analysis pipelines","Simulation","Assembly","Cell segmentation","Deconvolution","Doublet detection","Imputation","Quantification","Standardization","Visualisation","Batch correction","Cell type/state identification","Chromatin conformation","Clustering","Data integration","Dimension reduction","Measure of association","Rare cell type identification","Subclone detection","Cell cell communication","Differential analysis - others","Differential expression","Highly variable genes","Marker/signature identification","Regulatory network inference","T/B-cell receptor","Trajectory","Velocity","Multi-task"],"group":["A","B","C","D","E","F","G","H","I","CAT_A","CAT_B","CAT_C","CAT_D","CAT_E","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z","z"]},"options":{"NodeID":"name","NodeGroup":"group","LinkGroup":null,"colourScale":"d3.scaleOrdinal(d3.schemeCategory20);","fontSize":12,"fontFamily":null,"nodeWidth":15,"nodePadding":10,"units":"","margin":{"top":null,"right":null,"bottom":null,"left":null},"iterations":1,"sinksRight":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-1-2"><div class="card-body html-fill-item html-fill-container" data-title="Survey Analysis">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 15fr) minmax(3em, 85fr); grid-auto-columns: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-fill="false" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-body html-fill-item html-fill-container">
<p>Placeholder text for top row (Selection)</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-title="card Title" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">card Title</div></div>
<div class="card-body html-fill-item html-fill-container">
<p>This is a card</p>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div></div><bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div>
<div id="data-overview" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-data-overview">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" data-width="100%" style="display: grid; grid-template-columns: minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Result table" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Result table</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<div class="datatables html-widget html-fill-item html-fill-container" id="htmlwidget-1d17334e9714e159fdeb" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-1d17334e9714e159fdeb">{"x":{"style":"bootstrap","filter":"top","vertical":false,"filterHTML":"<tr>\n  <td><\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Addition - Analysis pipelines&quot;,&quot;Addition - Benchmarking datasets&quot;,&quot;Addition - Cell segmentation&quot;,&quot;Addition - Chromatin conformation&quot;,&quot;Addition - Multi-task&quot;,&quot;Alignment&quot;,&quot;Assembly&quot;,&quot;Classification&quot;,&quot;Clustering&quot;,&quot;Differential Expression&quot;,&quot;Dimensionality Reduction&quot;,&quot;Gene Networks&quot;,&quot;Gene Sets&quot;,&quot;Immune&quot;,&quot;Imputation&quot;,&quot;Integration&quot;,&quot;Marker Genes&quot;,&quot;Normalisation&quot;,&quot;Ordering&quot;,&quot;Perturbations&quot;,&quot;Quality Control&quot;,&quot;Quantification&quot;,&quot;Rare Cells&quot;,&quot;Simulation&quot;,&quot;Variable Genes&quot;,&quot;Variants&quot;,&quot;Visualisation&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;New method development paper&quot;,&quot;Pure benchmarking paper&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"integer\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0\" data-max=\"68\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"72\" data-max=\"4e+07\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Both&quot;,&quot;Experimental data&quot;,&quot;Synthetic data&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"integer\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0\" data-max=\"518\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"integer\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0\" data-max=\"19980\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Bioinformatics (Oxford, England)&quot;,&quot;BMC bioinformatics&quot;,&quot;BMC genomics&quot;,&quot;Briefings in bioinformatics&quot;,&quot;Briefings in functional genomics&quot;,&quot;Cell genomics&quot;,&quot;Cell reports&quot;,&quot;Cell systems&quot;,&quot;Cells&quot;,&quot;Communications biology&quot;,&quot;Computational and structural biotechnology journal&quot;,&quot;Current protocols&quot;,&quot;eLife&quot;,&quot;F1000Research&quot;,&quot;Frontiers in genetics&quot;,&quot;G3 (Bethesda, Md.)&quot;,&quot;Genes&quot;,&quot;Genetic epidemiology&quot;,&quot;Genome biology&quot;,&quot;Genome Biology&quot;,&quot;Genome research&quot;,&quot;Genomics&quot;,&quot;Genomics, proteomics &amp; bioinformatics&quot;,&quot;GigaScience&quot;,&quot;IEEE/ACM transactions on computational biology and bioinformatics&quot;,&quot;iScience&quot;,&quot;Journal of chemical information and modeling&quot;,&quot;Life (Basel, Switzerland)&quot;,&quot;Life science alliance&quot;,&quot;Molecular systems biology&quot;,&quot;NAR genomics and bioinformatics&quot;,&quot;Nature&quot;,&quot;Nature biotechnology&quot;,&quot;Nature communications&quot;,&quot;Nature computational science&quot;,&quot;Nature methods&quot;,&quot;Nucleic acids research&quot;,&quot;Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing&quot;,&quot;Patterns (New York, N.Y.)&quot;,&quot;PLoS computational biology&quot;,&quot;Proceedings of SPIE--the International Society for Optical Engineering&quot;,&quot;Proceedings of the National Academy of Sciences of the United States of America&quot;,&quot;Research square&quot;,&quot;Science advances&quot;,&quot;Scientific data&quot;,&quot;Scientific reports&quot;,&quot;Statistical applications in genetics and molecular biology&quot;,&quot;Statistics in medicine&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Single-cell ATAC-seq&quot;,&quot;Single-cell ChIP-seq&quot;,&quot;Single-cell genomics&quot;,&quot;Single-cell Hi-C&quot;,&quot;Single-cell mass cytometry&quot;,&quot;Single-cell multiomics&quot;,&quot;Single-cell proteomics&quot;,&quot;Single-cell RNA-seq&quot;,&quot;Single-cell sequencing&quot;,&quot;Spatial imaging&quot;,&quot;Spatial transcriptomics&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Analysis pipelines&quot;,&quot;Data&quot;,&quot;Downstream analysis&quot;,&quot;Initial analysis&quot;,&quot;Intermediate analysis&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Alignment&quot;,&quot;Analysis pipelines&quot;,&quot;Assembly&quot;,&quot;Batch correction&quot;,&quot;Benchmarking datasets&quot;,&quot;Cell cell communication&quot;,&quot;Cell segmentation&quot;,&quot;Cell type/state identification&quot;,&quot;Chromatin conformation&quot;,&quot;Clustering&quot;,&quot;Data integration&quot;,&quot;Deconvolution&quot;,&quot;Differential analysis - others&quot;,&quot;Differential expression&quot;,&quot;Dimension reduction&quot;,&quot;Doublet detection&quot;,&quot;Highly variable genes&quot;,&quot;Imputation&quot;,&quot;Marker/signature identification&quot;,&quot;Measure of association&quot;,&quot;Multi-task&quot;,&quot;Pathway enrichment&quot;,&quot;Quantification&quot;,&quot;Rare cell type identification&quot;,&quot;Regulatory network inference&quot;,&quot;Simulation&quot;,&quot;Standardization&quot;,&quot;Subclone detection&quot;,&quot;T/B-cell receptor&quot;,&quot;Trajectory&quot;,&quot;Velocity&quot;,&quot;Visualisation&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n<\/tr>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356"],["Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Benchmarking datasets","Addition - Benchmarking datasets","Addition - Cell segmentation","Addition - Cell segmentation","Addition - Cell segmentation","Addition - Chromatin conformation","Addition - Chromatin conformation","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Alignment","Alignment","Alignment","Alignment","Alignment","Alignment","Assembly","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Immune","Immune","Immune","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Perturbations","Perturbations","Perturbations","Perturbations","Perturbations","Perturbations","Quality Control","Quality Control","Quality Control","Quality Control","Quantification","Quantification","Quantification","Quantification","Rare Cells","Rare Cells","Rare Cells","Rare Cells","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variants","Variants","Variants","Variants","Variants","Variants","Visualisation","Visualisation","Visualisation","Visualisation","Visualisation"],["spicyR: spatial analysis of in situ cytometry data in R","Benchmarking UMI-based single-cell RNA-seq preprocessing workflows","Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments","Interactive analysis of single-cell epigenomic landscapes with ChromSCape","pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools","A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples","The Current State of Single-Cell Proteomics Data Analysis","Assessment of computational methods for the analysis of single-cell ATAC-seq data","EpiScanpy: integrated single-cell epigenomic analysis","TIST: Transcriptome and Histopathological Image Integrative Analysis for Spatial Transcriptomics","Comparison of high-throughput single-cell RNA sequencing data processing pipelines","Interactive analysis of single-cell epigenomic landscapes with ChromSCape","Benchmarking UMI-based single-cell RNA-seq preprocessing workflows","Destin2: Integrative and cross-modality analysis of single-cell chromatin accessibility data","Comparison of high-throughput single-cell RNA sequencing data processing pipelines","The Current State of Single-Cell Proteomics Data Analysis","A systematic evaluation of single cell RNA-seq analysis pipelines","Assessment of computational methods for the analysis of single-cell ATAC-seq data","A systematic evaluation of single cell RNA-seq analysis pipelines","pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools","histocat: analysis of cell phenotypes and interactions in multiplex image cytometry data","Giotto: a toolbox for integrative analysis and visualization of spatial expression data","A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples","TISCH: a comprehensive web resource enabling interactive single-cell transcriptome visualization of tumor microenvironment","Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq","Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments","Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq","A multi-center cross-platform single-cell RNA sequencing reference dataset","A multi-center cross-platform single-cell RNA sequencing reference dataset","Joint cell segmentation and cell type annotation for spatial transcriptomics","Cellpose: a generalist algorithm for cellular segmentation","Cell segmentation in imaging-based spatial transcriptomics","Perspectives for the reconstruction of 3D chromatin conformation using single cell Hi-C data","SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data","High-throughput single-cell RNA-seq data imputation and characterization with surrogate-assisted automated deep learning","High-throughput single-cell RNA-seq data imputation and characterization with surrogate-assisted automated deep learning","Spatially informed clustering, integration, and deconvolution of spatial transcriptomics with GraphST","Joint dimension reduction and clustering analysis of single-cell RNA-seq and spatial transcriptomics data","scGNN is a novel graph neural network framework for single-cell RNA-Seq analyses","Choice of pre-processing pipeline influences clustering quality of scRNA-seq datasets","Comparative analysis of common alignment tools for single-cell RNA sequencing","Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment","Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment","Comparative analysis of common alignment tools for single-cell RNA sequencing","Choice of pre-processing pipeline influences clustering quality of scRNA-seq datasets","Mapping molecular assemblies with fluorescence microscopy and object-based spatial statistics","Influence of Cell-type Ratio on Spatially Resolved Single-cell Transcriptomes using the Tangram Algorithm: Based on Implementation on Single-Cell and MxIF Data","scAB detects multiresolution cell states with clinical significance by integrating single-cell genomics and bulk sequencing data","Spatial-ID: a cell typing method for spatially resolved transcriptomics via transfer learning and spatial embedding","Latent cellular analysis robustly reveals subtle diversity in large-scale single-cell RNA-seq data","Learning Cell Annotation under Multiple Reference Datasets by Multisource Domain Adaptation","scMAGIC: accurately annotating single cells using two rounds of reference-based classification","A multiresolution framework to characterize single-cell state landscapes","scGCN is a graph convolutional networks algorithm for knowledge transfer in single cell omics","scClassify: sample size estimation and multiscale classification of cells using single and multiple reference","Evaluation of Cell Type Annotation R Packages on Single-cell RNA-seq Data","A comparison of automatic cell identification methods for single-cell RNA sequencing data","Adversarial dense graph convolutional networks for single-cell classification","Benchmarking automated cell type annotation tools for single-cell ATAC-seq data","CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing","A comparison of automatic cell identification methods for single-cell RNA sequencing data","Benchmarking automated cell type annotation tools for single-cell ATAC-seq data","Evaluation of single-cell classifiers for single-cell RNA sequencing data sets","scPred: accurate supervised method for cell-type classification from single-cell RNA-seq data","Supervised classification enables rapid annotation of cell atlases","A Systematic Evaluation of Supervised Machine Learning Algorithms for Cell Phenotype Classification Using Single-Cell RNA Sequencing Data","ChrNet: A re-trainable chromosome-based 1D convolutional neural network for predicting immune cell types","scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data","A Systematic Evaluation of Supervised Machine Learning Algorithms for Cell Phenotype Classification Using Single-Cell RNA Sequencing Data","scClassify: sample size estimation and multiscale classification of cells using single and multiple reference","Cross-species cell-type assignment from single-cell RNA-seq data by a heterogeneous graph neural network","Evaluation of some aspects in supervised cell type identification for single-cell RNA-seq: classifier, feature selection, and reference construction","Identifying phenotype-associated subpopulations by integrating bulk and single-cell sequencing data","CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing","Evaluation of some aspects in supervised cell type identification for single-cell RNA-seq: classifier, feature selection, and reference construction","Evaluation of methods to assign cell type labels to cell clusters from single-cell RNA-sequencing data","A comparison of automatic cell identification methods for single-cell RNA sequencing data","scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data","Evaluation of Cell Type Annotation R Packages on Single-cell RNA-seq Data","Evaluation of machine learning approaches for cell-type identification from single-cell transcriptomics data","Evaluation of single-cell classifiers for single-cell RNA sequencing data sets","Integrating multiple references for single-cell assignment","Evaluation of methods to assign cell type labels to cell clusters from single-cell RNA-sequencing data","scGAD: a new task and end-to-end framework for generalized cell type annotation and discovery","Learning for single-cell assignment","CIDR: Ultrafast and accurate clustering through imputation for single-cell RNA-seq data","Benchmarking clustering algorithms onÂ estimating theÂ number ofÂ cell types fromÂ single-cell RNA-sequencing data","ClusterMap for multi-scale clustering analysis of spatial gene expression","Secuer: Ultrafast, scalable and accurate clustering of single-cell RNA-seq data","Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data","A comparison framework and guideline of clustering methods for mass cytometry data","A systematic performance evaluation of clustering methods for single-cell RNA-seq data","HGC: fast hierarchical clustering for large-scale single-cell data","Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis","Benchmarking cell-type clustering methods for spatially resolved transcriptomics data","A review and performance evaluation of clustering frameworks for single-cell Hi-C data","A comparison framework and guideline of clustering methods for mass cytometry data","scMINER: a mutual information-based framework for identifying hidden drivers from single-cell omics data","A spectral clustering with self-weighted multiple kernel learning method for single-cell RNA-seq data","Benchmarking clustering algorithms onÂ estimating theÂ number ofÂ cell types fromÂ single-cell RNA-sequencing data","EDClust: an EM-MM hybrid method for cell clustering in multiple-subject single-cell RNA sequencing","Using single-cell cytometry to illustrate integrated multi-perspective evaluation of clustering algorithms using Pareto fronts","Benchmarking cell-type clustering methods for spatially resolved transcriptomics data","Benchmark and Parameter Sensitivity Analysis of Single-Cell RNA Sequencing Clustering Methods","Clustering methods for single-cell RNA-sequencing expression data: performance evaluation with varying sample sizes and cell compositions","Single-Cell RNA Sequencing Data Interpretation by Evolutionary Multiobjective Clustering","Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data","sc3: consensus clustering of single-cell rna-seq data","A Hybrid Clustering Algorithm for Identifying Cell Types from Single-Cell RNA-Seq Data","Clustering methods for single-cell RNA-sequencing expression data: performance evaluation with varying sample sizes and cell compositions","A systematic performance evaluation of clustering methods for single-cell RNA-seq data","ASTER: accurately estimating the number of cell types in single-cell chromatin accessibility data","Using single-cell cytometry to illustrate integrated multi-perspective evaluation of clustering algorithms using Pareto fronts","A model-based constrained deep learning clustering approach for spatially resolved single-cell data","Benchmark and Parameter Sensitivity Analysis of Single-Cell RNA Sequencing Clustering Methods","A review and performance evaluation of clustering frameworks for single-cell Hi-C data","Benchmarking integration of single-cell differential expression","Comparison of methods to detect differentially expressed genes between single-cell populations","Benchmarking of a Bayesian single cell RNAseq differential gene expression test for doseâ€“response study designs","Benchmarking integration of single-cell differential expression","Comparison of methods to detect differentially expressed genes between single-cell populations","Bias, robustness and scalability in single-cell differential expression analysis","Single-cell RNA-sequencing: assessment of differential expression analysis methods","Benchmarking of a Bayesian single cell RNAseq differential gene expression test for doseâ€“response study designs","Recommendations of scRNA-seq Differential Gene Expression Analysis Based on Comprehensive Benchmarking","Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data","Differential expression of single-cell RNA-seq data using Tweedie models","Recommendations of scRNA-seq Differential Gene Expression Analysis Based on Comprehensive Benchmarking","Bias, robustness and scalability in single-cell differential expression analysis","Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data","Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data","Single-cell RNA-sequencing: assessment of differential expression analysis methods","Reproducibility of Methods to Detect Differentially Expressed Genes from Single-Cell RNA Sequencing","Reproducibility of Methods to Detect Differentially Expressed Genes from Single-Cell RNA Sequencing","Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data","Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics","Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis","A Quantitative Framework for Evaluating Single-Cell Data Structure Preservation by Dimensionality Reduction Techniques","Benchmarking principal component analysis for large-scale single-cell RNA-sequencing","Towards a comprehensive evaluation of dimension reduction methods for transcriptomic data visualization","Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics","Supervised application of internal validation measures to benchmark dimensionality reduction methods in scRNA-seq data","A Comparison for Dimensionality Reduction Methods of Single-Cell RNA-seq Data","Towards a comprehensive evaluation of dimension reduction methods for transcriptomic data visualization","Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer","A Comparison for Dimensionality Reduction Methods of Single-Cell RNA-seq Data","Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer","Tuning parameters of dimensionality reduction methods for single-cell RNA-seq analysis","Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis","A Quantitative Framework for Evaluating Single-Cell Data Structure Preservation by Dimensionality Reduction Techniques","Dimensionality reduction for visualizing single-cell data using UMAP","Dimensionality reduction for visualizing single-cell data using UMAP","Tuning parameters of dimensionality reduction methods for single-cell RNA-seq analysis","Supervised application of internal validation measures to benchmark dimensionality reduction methods in scRNA-seq data","Benchmarking principal component analysis for large-scale single-cell RNA-sequencing","Network embedding-based representation learning for single cell RNA-seq data","Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data","Evaluating the reproducibility of single-cell gene regulatory network inference algorithms","Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data","Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information","Evaluating the reproducibility of single-cell gene regulatory network inference algorithms","Single-cell biological network inference using a heterogeneous graph transformer","NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks","A systematic evaluation of the computational tools for ligand-receptor-based cell-cell interaction inference","Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data","Identifying strengths and weaknesses of methods for computational network inference from single-cell RNA-seq data","Regulatory analysis of single cell multiome gene expression and chromatin accessibility data with scREG","A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data","Evaluating measures of association for single-cell transcriptomics","SRGS: sparse partial least squares-based recursive gene selection for gene regulatory network inference","Performance assessment of sample-specific network control methods for bulk and single-cell biological data analysis","A systematic evaluation of the computational tools for ligand-receptor-based cell-cell interaction inference","Performance assessment of sample-specific network control methods for bulk and single-cell biological data analysis","Impact of similarity metrics on single-cell RNA-seq data clustering","Optimal transport improves cell-cell similarity inference in single-cell omics data","DIRECT-NET: An efficient method to discover cis-regulatory elements and construct regulatory networks from single-cell multiomics data","Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information","IReNA: Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles","A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data","How does the structure of data impact cell-cell similarity? Evaluating how structural properties influence the performance of proximity metrics in single cell RNA-seq data","Functional inference of gene regulation using single-cell multi-omics","maxATAC: Genome-scale transcription-factor binding prediction from ATAC-seq with deep neural networks","Impact of similarity metrics on single-cell RNA-seq data clustering","Single-cell gene regulatory network prediction by explainable AI","Identifying strengths and weaknesses of methods for computational network inference from single-cell RNA-seq data","Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data","Nonparametric single-cell multiomic characterization of trio relationships between transcription factors, target genes, and cis-regulatory regions","Evaluating measures of association for single-cell transcriptomics","Dissecting cell identity via network inference and in silico gene perturbation","How does the structure of data impact cell-cell similarity? Evaluating how structural properties influence the performance of proximity metrics in single cell RNA-seq data","Benchmarking algorithms for pathway activity transformation of single-cell RNA-seq data","Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data","Benchmarking algorithms for pathway activity transformation of single-cell RNA-seq data","Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data","Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data","Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data","Benchmarking computational methods for B-cell receptor reconstruction from single-cell RNA-seq data","Benchmarking computational methods for B-cell receptor reconstruction from single-cell RNA-seq data","Predicting recognition between T cell receptors and epitopes with TCRGP","False signals induced by single-cell imputation","NISC: Neural Network-Imputation for Single-Cell RNA Sequencing and Cell Type Clustering","SpaGE: Spatial Gene Enhancement using scRNA-seq","Comparison of Computational Methods for Imputing Single-Cell RNA-Sequencing Data","scESI: evolutionary sparse imputation for single-cell transcriptomes from nearest neighbor cells","Efficient prediction of a spatial transcriptomics profile better characterizes breast cancer tissue sections without costly experimentation","A systematic evaluation of single-cell RNA-sequencing imputation methods","Region-specific denoising identifies spatial co-expression patterns and intra-tissue heterogeneity in spatially resolved transcriptomics data","Comparison of Computational Methods for Imputing Single-Cell RNA-Sequencing Data","scIMC: a platform for benchmarking comparison and visualization analysis of scRNA-seq data imputation methods","scNPF: an integrative framework assisted by network propagation and network fusion for preprocessing of single-cell RNA-seq data","scIMC: a platform for benchmarking comparison and visualization analysis of scRNA-seq data imputation methods","False signals induced by single-cell imputation","Are dropout imputation methods for scRNA-seq effective for scATAC-seq data?","scHinter: imputing dropout events for single-cell RNA-seq data with limited sample size","A systematic evaluation of single-cell RNA-sequencing imputation methods","Spatial transcriptomics at subspot resolution with BayesSpace","Are dropout imputation methods for scRNA-seq effective for scATAC-seq data?","A novel method for single-cell data imputation using subspace regression","A benchmark study of deep learning-based multi-omics data fusion methods for cancer","SD2: spatially resolved transcriptomics deconvolution through integration of dropout and spatial information","scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets","Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology","Benchmarking of cell type deconvolution pipelines for transcriptomics data","scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning","Benchmarking atlas-level data integration in single-cell genomics","Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","A benchmark study of deep learning-based multi-omics data fusion methods for cancer","DEEPsc: A Deep Learning-Based Map Connecting Single-Cell Transcriptomics and Spatial Imaging Data","A comparison of methods accounting for batch effects in differential expression analysis of UMI count based single cell RNA sequencing","Benchmarking atlas-level data integration in single-cell genomics","Benchmarking of cell type deconvolution pipelines for transcriptomics data","SpatialDWLS: accurate deconvolution of spatial transcriptomic data","Benchmarking atlas-level data integration in single-cell genomics","IBRAP: integrated benchmarking single-cell RNA-sequencing analytical pipeline","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction","Benchmarking and integration of methods for deconvoluting spatial transcriptomic data","Systematic evaluation of transcriptomics-based deconvolution methods and references using thousands of clinical samples","Alignment and integration of spatial transcriptomics data","Multi-omics single-cell data integration and regulatory inference with graph-linked embedding","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","iSMNN: batch effect correction for single-cell RNA-seq data via iterative supervised mutual nearest neighbor refinement","Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology","Deconvolution analysis of cell-type expression from bulk tissues by integrating with single-cell expression reference","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","A comparison of methods accounting for batch effects in differential expression analysis of UMI count based single cell RNA sequencing","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","Fast, sensitive and accurate integration of single-cell data with Harmony","CellMixS: quantifying and visualizing batch effects in single-cell RNA-seq data","Advances in mixed cell deconvolution enable quantification of cell types in spatial transcriptomic data","AIscEA: unsupervised integration of single-cell gene expression and chromatin accessibility via their biological consistency","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","DeepST: identifying spatial domains in spatial transcriptomics by deep learning","BBKNN: fast batch alignment of single cell transcriptomes","Comprehensive evaluation of deconvolution methods for human brain gene expression","scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning","Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction","Benchmarking and integration of methods for deconvoluting spatial transcriptomic data","Single-cell multi-omics integration for unpaired data by a siamese network with graph-based contrastive loss","Comprehensive evaluation of deconvolution methods for human brain gene expression","BBKNN: fast batch alignment of single cell transcriptomes","Integration of single cell data by disentangled representation learning","A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics","IBRAP: integrated benchmarking single-cell RNA-sequencing analytical pipeline","A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets","Systematic evaluation of transcriptomics-based deconvolution methods and references using thousands of clinical samples","Benchmarking spatial and single-cell transcriptomics integration methods","Adversarial domain translation networks for integrating large-scale atlas-level single-cell datasets","A rank-based marker selection method for high throughput scRNA-seq data","Uncovering cell identity through differential stability with Cepo","DrivAER: Identification of driving transcriptional programs in single-cell RNA sequencing data","Triku: a feature selection method based on nearest neighbors for single-cell data","Uncovering cell identity through differential stability with Cepo","De novo gene signature identification from single-cell RNA-seq with hierarchical Poisson factorization","Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq","Normalization and de-noising of single-cell Hi-C data with BandNorm and scVI-3D","Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq","Assessment of Single Cell RNA-Seq Normalization Methods","Normalization Methods on Single-Cell RNA-seq Data: An Empirical Survey","Normalization Methods on Single-Cell RNA-seq Data: An Empirical Survey","Assessment of Single Cell RNA-Seq Normalization Methods","Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq","A lineage tree-based hidden Markov model quantifies cellular heterogeneity and plasticity","Detecting critical transition signals from single-cell transcriptomes to infer lineage-determining transcription factors","A comparison of single-cell trajectory inference methods","Preprocessing choices affect RNA velocity results for droplet scRNA-seq data","A comparison of single-cell trajectory inference methods","PseudoGA: cell pseudotime reconstruction based on genetic algorithm","Predicting cell lineages using autoencoders and optimal transport","psupertime: supervised pseudotime analysis for time-series single-cell RNA-seq data","scShaper: an ensemble method for fast and accurate linear trajectory inference from single-cell RNA-seq data","Preprocessing choices affect RNA velocity results for droplet scRNA-seq data","DeepVelo: Single-cell transcriptomic deep velocity field learning with neural ordinary differential equations","PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells","Density-based detection of cell transition states to construct disparate and bifurcating trajectories","Quantifying Waddingtonâ€™s epigenetic landscape: a comparison of single-cell potency measures","Quantifying Waddingtonâ€™s epigenetic landscape: a comparison of single-cell potency measures","A sparse differential clustering algorithm for tracing cell type changes via single-cell RNA-sequencing data","Transcriptome analysis method based on differential distribution evaluation","Differential abundance testing on single-cell data using k-nearest neighbor graphs","Ultra-fast scalable estimation of single-cell differentiation potency from scRNA-Seq data","Doublet identification in single-cell sequencing data using scDblFinder","Benchmarking computational doublet-detection methods for single-cell RNA sequencing data","Benchmarking computational doublet-detection methods for single-cell RNA sequencing data","Vaeda computationally annotates doublets in single-cell RNA sequencing data","Simulation-based benchmarking of isoform quantification in single-cell RNA-seq","Simulation-based benchmarking of isoform quantification in single-cell RNA-seq","Pheniqs 20: accurate, high-performance Bayesian decoding and confidence estimation for combinatorial barcode indexing","EmptyNN: A neural network based on positive and unlabeled learning to remove cell-free droplets and recover lost cells in scRNA-seq data","CellSIUS provides sensitive and specific detection of rare cell populations from complex single-cell RNA-seq data","Bayesian information sharing enhances detection of regulatory associations in rare cell types","GapClust is a light-weight approach distinguishing rare cells from voluminous single cell expression profiles","Discovery of rare cells from voluminous single cell expression data","A benchmark study of simulation methods for single-cell RNA sequencing data","A benchmark study of simulation methods for single-cell RNA sequencing data","ESCO: single cell expression simulation incorporating gene co-expression","splatPop: simulating population scale single-cell RNA sequencing data","SCAN-ATAC-Sim: a scalable and efficient method for simulating single-cell ATAC-seq data from bulk-tissue experiments","Spearheading future omics analyses using dyngen, a multi-modal simulator of single cells","Simulation, power evaluation and sample size recommendation for single-cell RNA-seq","SCSsim: an integrated tool for simulating single-cell genome sequencing data","powsimR: power analysis for bulk and single cell RNA-seq experiments","Built on sand: the shaky foundations of simulating single-cell RNA sequencing data","simATAC: a single-cell ATAC-seq simulation framework","Built on sand: the shaky foundations of simulating single-cell RNA sequencing data","Simulation, power evaluation and sample size recommendation for single-cell RNA-seq","A Framework for Comparison and Assessment of Synthetic RNA-Seq Data","A statistical simulator scDesign for rational scRNA-seq experimental design","SimCH: simulation of single-cell RNA sequencing data by modeling cellular heterogeneity at gene expression level","scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured","Realistic in silico generation and augmentation of single-cell RNA-seq data using generative adversarial networks","Novel ChIP-seq simulating program with superior versatility: isChIP","SPARK-X: non-parametric modeling enables scalable and robust detection of spatial expression patterns for large spatial transcriptomic studies","SpatialDE: identification of spatially variable genes","SOMDE: a scalable method for identifying spatially variable genes with self-organizing map","SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network","Evaluation of tools for highly variable gene discovery from single-cell RNA-seq data","Evaluation of tools for highly variable gene discovery from single-cell RNA-seq data","SINFONIA: Scalable Identification of Spatially Variable Genes for Deciphering Spatial Domains","Resolving single-cell copy number profiling for large datasets","Delineating Copy Number and Clonal Substructure in Human Tumors from Single-cell Transcriptomes","Systematic comparative analysis of single-nucleotide variant detection methods from single-cell RNA sequencing data","DENDRO: genetic heterogeneity profiling and subclone detection by single-cell RNA sequencing","Systematic comparative analysis of single-nucleotide variant detection methods from single-cell RNA sequencing data","A Variational Algorithm to Detect the Clonal Copy Number Substructure of Tumors from scRNA-seq Data","Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning","Comparison of visualization tools for single-cell RNAseq data","Comparison of visualization tools for single-cell RNAseq data","Generalizable and Scalable Visualization of Single-Cell Data Using Neural Networks","Generalizable and Scalable Visualization of Single-Cell Data Using Neural Networks"],["New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper"],[0,10,27,3,24,21,2,10,11,7,7,3,10,7,7,2,20,10,20,24,5,5,21,0,2,27,2,5,5,1,3,2,2,3,5,5,13,18,9,2,5,2,2,5,2,4,0,5,8,4,2,13,4,4,14,10,22,4,5,3,22,5,9,4,1,13,5,6,13,14,5,9,0,1,9,5,22,2,10,10,9,8,5,7,null,4,14,1,4,12,9,14,20,4,15,8,9,3,7,14,11,3,15,13,11,15,12,5,4,11,14,4,3,7,13,8,46,5,8,46,5,36,6,8,12,11,8,12,36,11,18,6,9,9,18,5,18,11,21,8,5,33,10,8,9,10,9,5,18,11,5,5,5,33,21,3,8,6,8,16,6,4,7,9,12,11,0,14,17,10,16,9,16,4,4,9,16,1,14,17,0,2,4,3,11,12,0,17,4,17,7,7,7,5,5,7,6,6,5,6,14,4,8,9,1,18,8,8,12,8,12,6,7,4,18,7,7,7,16,3,5,7,20,5,68,8,16,16,8,11,68,20,4,68,7,16,10,14,12,3,10,28,28,8,28,4,7,1,14,11,16,4,9,7,7,14,6,6,8,5,10,14,8,8,6,9,18,7,18,14,5,12,16,8,9,9,2,8,9,5,6,8,10,7,8,8,7,10,1,5,45,13,45,6,4,4,6,13,7,4,5,4,4,1,5,4,8,6,9,9,7,5,5,1,4,2,0,4,3,12,12,2,0,2,3,2,2,5,16,0,16,2,5,4,4,8,2,0,3,1,4,5,7,7,3,5,1,7,3,7,2,11,13,13,1,1],["No","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Not sure","No","Yes","No","No","No","No","Yes","Not sure","No","Yes","No","Not sure","No","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","Not sure","No","Yes","Yes","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Not sure","No","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Not sure","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Not sure","Not sure","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Not sure","Yes","Yes","Yes","Yes","No","Yes","Not sure","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","No","Not sure","No","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","Not sure","Yes","Not sure","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","Not sure","Yes","No","Not sure","Not sure","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Not sure","No","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","Yes","No","Yes","Not sure","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes"],["Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","Yes","Yes","Not sure","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No"],["No","Yes","No","No","No","No","No","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","Yes","No","No","Yes","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","No","No","No","Yes","Yes","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No"],[null,10132,4000,null,3918,6425,null,81173,81173,null,null,null,10132,null,null,null,null,81173,null,3918,null,null,6425,null,274182,4000,274182,null,null,null,null,9000,null,null,null,null,100000,33611,null,null,null,800,800,null,null,null,null,null,null,2000000,null,null,null,null,96404,50000,45469,null,43410,null,45469,43410,90000,null,null,8602,null,null,8602,96404,null,null,29888,1300000,null,68579,45469,null,50000,null,90000,null,68579,null,null,10000,10000,null,40000000,8352,80000,6000,400000,null,25551,null,80000,null,null,10000,43603,null,25551,3514,null,null,8352,5000,null,null,6000,68232,null,28000,3514,null,100361,224,null,100361,224,832,200,null,5096,92,null,5096,832,92,null,200,null,null,null,null,10000,null,1306127,1000000,null,27499,50000,1000000,null,50000,null,null,10000,null,200000,200000,null,27499,1306127,1529,null,null,null,null,null,null,null,3005,1000,5520,null,null,null,null,null,3005,null,null,null,null,null,null,null,null,null,null,null,null,5520,1000,null,null,null,null,20000,null,20000,2000,2000,null,72,72,null,null,10000,null,1000,21086,null,100000,null,1000,null,null,null,null,38000,1529,100000,4000,38000,15000,null,null,null,null,10000,1089769,1000000,62337,26252,null,null,null,1000000,10000,null,1000000,null,26252,10000,59651,5052,null,null,null,null,62337,null,null,null,null,833206,null,null,null,80000,null,null,833206,null,524288,null,1089769,10000,59651,null,null,524288,null,53208,null,53208,833206,null,5052,1169213,1100167,null,10000,null,null,10000,9924,null,9230,null,null,2649,2649,null,null,null,null,1000000,null,1000000,null,null,3600,null,null,null,null,null,null,null,50000,null,200000,2000000,25000,10000,10000,null,null,null,null,null,null,null,null,68000,8000,8000,5000,null,1000000,10000,null,null,null,2600,null,2600,null,null,null,1600,null,null,null,10000,null,22860,5665,247,247,null,100000,null,null,null,null,null,68560,2000000,2000000,1283543,1283543],["Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","Yes","No","No","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","No","Yes","No","No","No","No","Yes","No","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","No","No","No"],["Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","Not sure","No","No","No","No","Yes","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Not sure","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","Not sure","No","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Not sure","No","No","Yes","Yes","No","No","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes"],["No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No"],["Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","No","No","No","No","No","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","No","Yes","No","No","No","Yes","No","Yes","No","No","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No"],["Yes","No","No","Yes","No","Not sure","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Not sure","No","Not sure","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","Yes","No","No","Not sure","Yes","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Not sure","No","Yes","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","Yes","No","Not sure","No","No","Not sure","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","Yes","Not sure","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","No","Not sure","Yes","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Not sure","Not sure","No","No","Yes","Yes","Not sure","Yes","No","No","No","Yes","Not sure","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Yes","Not sure","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Not sure","No","Not sure","Yes","No","No","No","No","Not sure","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","Not sure","Yes","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","Yes"],["Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Both","Both","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Experimental data","Both","Both","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Both","Synthetic data","Experimental data","Experimental data","Experimental data","Experimental data","Synthetic data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Synthetic data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Synthetic data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Both","Experimental data","Synthetic data","Synthetic data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Experimental data","Experimental data","Synthetic data","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Both","Both","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data"],[3,12,14,9,7,20,2,3,4,2,8,9,12,4,8,2,6,3,6,7,1,10,20,76,4,14,4,20,20,2,4,9,2,2,11,11,6,5,4,22,4,3,3,4,22,3,1,3,5,18,8,6,4,30,13,6,27,6,5,9,27,5,3,4,6,43,3,9,43,13,1,10,4,5,10,5,27,9,6,20,3,27,5,10,31,3,2,7,15,6,6,4,4,8,7,6,6,11,11,2,4,4,7,17,null,6,6,12,8,null,4,27,4,49,17,6,9,3,1,9,3,9,2,1,2,2,2,2,9,2,0,2,3,3,0,3,30,4,4,10,3,55,5,10,11,5,11,10,30,4,5,5,10,55,4,3,2,6,2,5,6,13,10,15,5,7,1,null,213,4,9,15,9,22,8,4,5,3,null,2,3,518,22,1,7,5,4,213,5,2,32,2,32,10,10,2,3,3,2,18,3,6,2,12,null,6,12,2,2,8,2,18,8,8,6,5,8,25,6,3,14,5,5,5,23,3,45,6,4,1,23,5,2,23,15,45,10,4,2,5,5,4,4,3,4,2,5,3,9,1,45,6,7,5,1,9,5,6,3,5,10,4,5,3,6,11,7,15,7,9,14,2,null,16,4,9,2,23,9,4,3,4,4,120,4,4,120,4,2,6,110,5,110,4,2,1,1,5,6,8,2,11,11,4,12,3,28,16,16,16,16,null,null,1,4,2,3,9,4,35,35,1,3,1,1,11,1,8,12,7,12,11,1,18,13,6,5,2,6,4,6,7,5,5,15,2,6,2,2,2,3,7,1,1,13,13],[1000,null,14,0,2,null,null,10,null,9,null,0,null,null,null,null,19980,10,19980,2,4,4201,null,null,null,14,null,null,null,1,null,0,1,null,null,null,null,2,4,null,null,null,null,null,null,990,1,null,null,3800,null,null,1,null,0,41,null,null,null,1,null,null,null,10,null,3,null,null,3,0,1,null,3,null,null,null,null,null,41,null,null,null,null,1,null,1,null,null,31,null,null,3,null,0,7,null,null,null,null,null,100,null,7,25,700,null,null,null,null,700,3,21,null,6,25,null,7,3,44,7,3,3,10,44,4,10,16,4,3,10,1280,10,1,1,1280,40,2,2,18,2,40,null,30,2,100,30,100,null,2,2,null,null,null,null,18,null,2,0,2,15,0,null,null,null,420,null,null,100,null,null,null,null,null,0,5,0,15,null,100,30,0,null,0,1,null,420,0,null,null,30,null,30,null,null,null,30,74,74,null,60,60,0,5,6,null,1,null,5,6,null,6,60,null,null,1,null,null,116,3,2,0,200,1000,0,2,6,32,3,null,12,2,1000,1,2,10,32,0,3,2,1,39,28,28,6,28,1,200,1,1,12,32,0,91,null,1,1,null,null,100,0,0,3,null,100,null,2,43,10,43,1,0,2,null,0,20,49,null,1,49,null,20,null,10,null,1,1,null,10,5,null,229,0,229,null,null,1,6,0,null,null,5,null,null,7,1,1172,null,1,112,112,null,3,3,1,null,null,null,null,1,null,null,4,null,1,null,40,13,0,null,21,null,14,2,0,5,30,30,null,98,3001,22,null,1,1,null,9,0,1,9,1,500,4,null,null,null,null],["Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Not sure","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Not sure","No","Yes","Yes","Yes","Yes","No","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes"],["Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Not sure","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","Not sure","No","No","Yes","No","Yes","Yes","No","Yes","No","Not sure","Yes","Not sure","Not sure","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Yes","No","Yes","No","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Not sure","Yes","No","Yes","Yes","Not sure","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","Not sure","No","Not sure","Yes","No","No","Not sure","Yes","Yes","No","Yes","No","Not sure","No","Not sure","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Not sure","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Not sure","No","No","No","No","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Not sure","No","No","No","Yes","No","No","No","No","Yes","Not sure","No","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","No","Not sure","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes"],["Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes"],["No","No","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","No"],["Bioinformatics (Oxford, England)","Genome biology","Nature methods","Nature communications","Genome biology","Nature biotechnology","Current protocols","Genome biology","Nature communications","Genomics, proteomics &amp; bioinformatics","Briefings in bioinformatics","Nature communications","Genome biology","Frontiers in genetics","Briefings in bioinformatics","Current protocols","Nature communications","Genome biology","Nature communications","Genome biology","Nature methods","Genome biology","Nature biotechnology","Nucleic acids research","Nature methods","Nature methods","Nature methods","Scientific data","Scientific data","Molecular systems biology","Nature methods","Nature biotechnology","PLoS computational biology","Nature methods","Briefings in bioinformatics","Briefings in bioinformatics","Nature communications","Nucleic acids research","Nature communications","BMC genomics","GigaScience","G3 (Bethesda, Md.)","G3 (Bethesda, Md.)","GigaScience","BMC genomics","Nature communications","Proceedings of SPIE--the International Society for Optical Engineering","Nucleic acids research","Nature communications","Nucleic acids research","Journal of chemical information and modeling","Nucleic acids research","Nature communications","Nature communications","Molecular systems biology","Genomics, proteomics &amp; bioinformatics","Genome biology","Bioinformatics (Oxford, England)","Frontiers in genetics","Nucleic acids research","Genome biology","Frontiers in genetics","Briefings in bioinformatics","Genome biology","Nature methods","Frontiers in genetics","Genomics","iScience","Frontiers in genetics","Molecular systems biology","Genome research","Genome biology","Nature biotechnology","Genome biology","Genome biology","F1000Research","Genome biology","iScience","Genomics, proteomics &amp; bioinformatics","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","F1000Research","Briefings in bioinformatics","Science advances","Genome biology","Genome biology","Nature communications","PLoS computational biology","F1000Research","Genome Biology","F1000Research","Bioinformatics (Oxford, England)","BMC bioinformatics","Briefings in bioinformatics","Briefings in bioinformatics","Genome Biology","Research square","Briefings in bioinformatics","Genome biology","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Frontiers in genetics","Statistical applications in genetics and molecular biology","IEEE/ACM transactions on computational biology and bioinformatics","F1000Research","Nature methods","Genes","Statistical applications in genetics and molecular biology","F1000Research","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Genome research","Frontiers in genetics","Briefings in bioinformatics","Nature communications","Briefings in bioinformatics","Nucleic acids research","Nature communications","Briefings in bioinformatics","Nature methods","Frontiers in genetics","Nucleic acids research","Life (Basel, Switzerland)","BMC bioinformatics","Statistics in medicine","Life (Basel, Switzerland)","Nature methods","BMC bioinformatics","Briefings in bioinformatics","Frontiers in genetics","Frontiers in genetics","Frontiers in genetics","Briefings in bioinformatics","Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","Genome biology","Cell reports","Genome biology","Communications biology","Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","Briefings in bioinformatics","Frontiers in genetics","Communications biology","Nature communications","Frontiers in genetics","Nature communications","Genome biology","Genome biology","Cell reports","Nature biotechnology","Nature biotechnology","Genome biology","Briefings in bioinformatics","Genome biology","Nucleic acids research","BMC bioinformatics","Frontiers in genetics","BMC bioinformatics","Genome biology","Frontiers in genetics","Nature communications","Communications biology","Briefings in functional genomics","Nature methods","G3 (Bethesda, Md.)","Genome biology","Briefings in bioinformatics","Nature methods","BMC genomics","PLoS computational biology","Briefings in functional genomics","PLoS computational biology","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Science advances","Genome biology","iScience","Briefings in bioinformatics","Briefings in bioinformatics","Cell genomics","PLoS computational biology","Briefings in bioinformatics","Nucleic acids research","G3 (Bethesda, Md.)","Nature methods","Cell systems","Nature methods","Nature","Briefings in bioinformatics","Computational and structural biotechnology journal","Genome biology","Computational and structural biotechnology journal","eLife","eLife","Genome biology","NAR genomics and bioinformatics","NAR genomics and bioinformatics","PLoS computational biology","F1000Research","Frontiers in genetics","Nucleic acids research","IEEE/ACM transactions on computational biology and bioinformatics","Briefings in bioinformatics","Scientific reports","Genome biology","Nature communications","IEEE/ACM transactions on computational biology and bioinformatics","Nucleic acids research","BMC genomics","Nucleic acids research","F1000Research","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Genome biology","Nature biotechnology","Briefings in bioinformatics","Scientific reports","Genome biology","Bioinformatics (Oxford, England)","Proceedings of the National Academy of Sciences of the United States of America","Bioinformatics (Oxford, England)","Nature communications","Nature biotechnology","Nature methods","Nucleic acids research","Nature methods","Genome biology","Frontiers in genetics","Computational and structural biotechnology journal","Nature methods","Nature communications","Genome biology","Nature methods","Briefings in bioinformatics","Nature methods","Genome biology","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Nature methods","Nature biotechnology","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","Briefings in bioinformatics","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Genetic epidemiology","Genome biology","Computational and structural biotechnology journal",null,"Nature methods","Life science alliance","Nature communications","Bioinformatics (Oxford, England)","Genome biology","Nucleic acids research","Bioinformatics (Oxford, England)","Nature communications","Nature biotechnology","Genome biology","Bioinformatics (Oxford, England)","BMC bioinformatics","Nature communications","Bioinformatics (Oxford, England)","Nucleic acids research","Nature communications","Briefings in bioinformatics","Nature communications","Genome biology","Proceedings of the National Academy of Sciences of the United States of America","Briefings in bioinformatics","Nature methods","Nature computational science","BMC bioinformatics","Nature computational science","GigaScience","GigaScience","Nature computational science","Molecular systems biology","eLife","Genome biology","Cell systems","G3 (Bethesda, Md.)","Frontiers in genetics","Frontiers in genetics","G3 (Bethesda, Md.)","Cell systems","Communications biology","Nucleic acids research","Nature biotechnology","PLoS computational biology","Nature biotechnology","Nucleic acids research","PLoS computational biology","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","PLoS computational biology","Science advances","Genome biology","Nucleic acids research","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","Briefings in bioinformatics","Nature biotechnology","Bioinformatics (Oxford, England)","F1000Research","Cell systems","Cell systems","Bioinformatics (Oxford, England)","Genome biology","Genome biology","BMC bioinformatics","Patterns (New York, N.Y.)","Genome biology","Bioinformatics (Oxford, England)","Nature communications","Nature communications","Nature communications","Nature communications","Bioinformatics (Oxford, England)","Genome biology","Bioinformatics (Oxford, England)","Nature communications","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Genome biology","Genome biology","Genome biology","Bioinformatics (Oxford, England)","Genes","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Genome biology","Nature communications","Briefings in bioinformatics","Genome biology","Nature methods","Bioinformatics (Oxford, England)","Nature methods","Briefings in bioinformatics","Briefings in bioinformatics","Cells","Briefings in bioinformatics","Nature biotechnology","Genome biology","Genome biology","Genome biology","Nature communications","Nature methods","NAR genomics and bioinformatics","NAR genomics and bioinformatics","Cell systems","Cell systems"],["2022 May 26","2021 Dec 14","2019 May 27","2020 Nov 11","2020 Sep 1","2021 Sep","2023 Jan 12","2019 Nov 18","2021 Sep 1","2022 Dec 19","2021 May 20","2020 Nov 11","2021 Dec 14","2023 Feb 17","2021 May 20","2023 Jan 12","2019 Oct 11","2019 Nov 18","2019 Oct 11","2020 Sep 1","2017 Aug 7","2021 Mar 8","2021 Sep","2021 Jan 8","2020 Jul 27","2019 May 27","2020 Jul 27","2021 Feb 2","2021 Feb 2","2021 May 31","2020 Dec 14","2021 Oct 14","2021 Nov 18","2021 Aug 26","2022 Jan 17","2022 Jan 17","2023 Mar 1","2022 Jul 8","2021 Mar 25","2021 Sep 14","2022 Jan 27","2020 May 4","2020 May 4","2022 Jan 27","2021 Sep 14","2018 Feb 15","2023 Apr 7","2022 Nov 28","2022 Dec 10","2019 Dec 16","2023 Jan 9","2022 May 6","2020 Oct 26","2021 Jun 22","2020 Jun 22","2020 Dec 24","2019 Sep 9","2023 Feb 3","2022 Dec 13","2019 Sep 19","2019 Sep 9","2022 Dec 13","2020 Sep 25","2019 Dec 12","2019 Sep 9","2022 Feb 23","2021 Apr 28","2020 Mar 27","2022 Feb 23","2020 Jun 22","2022 Dec 16","2021 Sep 9","2021 Nov 11","2019 Feb 11","2021 Sep 9","2019 Oct 14","2019 Sep 9","2020 Mar 27","2020 Dec 24","2021 Sep 2","2020 Sep 25","2021 Aug 20","2019 Oct 14","2023 Mar 19","2020 Oct 30","2017 Mar 28","2022 Feb 8","2021 Oct 8","2022 Dec 5","2018 Aug 15","2019 Dec 23","2018 Jul 26","2021 Nov 5","2019 Dec 24","2023 Jan 19","2022 Nov 19","2019 Dec 23","2023 Jan 27","2021 Jul 20","2022 Feb 8","2022 May 13","2021 Jan 28","2023 Jan 19","2019 Dec 11","2019 Aug 14","2020 Sep 1","2018 Aug 15","2017 Mar 27","2019 Jan 29","2019 Aug 14","2018 Jul 26","2023 Jan 1","2021 Jan 28","2022 Oct 5","2019 Dec 11","2022 Nov 19","2023 Mar 21","2017 Sep 1","2022 May 6","2023 Mar 21","2017 Sep 1","2018 Feb 26","2017 May 23","2022 May 6","2022 Jun 7","2019 Jan 18","2022 Aug 15","2022 Jun 7","2018 Feb 26","2019 Jan 18","2022 Sep 20","2017 May 23","2020 Jan 17","2020 Jan 17","2022 Sep 20","2019 Mar 14","2019 Dec 10","2020 May 5","2020 Jan 20","2022 Jul 19","2019 Mar 14","2021 Nov 5","2021 Mar 23","2022 Jul 19","2021 Jan 5","2021 Mar 23","2021 Jan 5","2020 Aug 24","2019 Dec 10","2020 May 5","2018 Dec 3","2018 Dec 3","2020 Aug 24","2021 Nov 5","2020 Jan 20","2017 Nov 2","2018 Jun 19","2021 Mar 22","2018 Jun 19","2022 Oct 17","2021 Mar 22","2023 Feb 21","2022 Nov 23","2022 Sep 16","2020 Jan 6","2023 Mar 9","2022 May 16","2021 May 20","2019 Apr 8","2022 Nov 30","2021 May 6","2022 Sep 16","2021 May 6","2019 Nov 27","2022 Apr 12","2022 Jun 3","2022 Oct 17","2022 Nov 18","2021 May 20","2022 Nov 19","2022 Sep 14","2023 Jan 31","2019 Nov 27","2023 Feb 28","2023 Mar 9","2020 Jan 6","2022 Sep 21","2019 Apr 8","2023 Feb 8","2022 Nov 19","2020 Oct 15","2020 Feb 12","2020 Oct 15","2022 Feb 25","2022 Feb 25","2020 Feb 12","2022 Jul 13","2022 Jul 13","2021 Mar 25","2018 Nov 2","2022 May 3","2020 Oct 9","2020 Mar 1","2022 Sep 20","2022 Mar 8","2020 Aug 27","2022 Nov 14","2020 Mar 1","2022 May 20","2019 May 8","2022 May 20","2018 Nov 2","2022 Jan 17","2020 Feb 1","2020 Aug 27","2021 Jun 3","2022 Jan 17","2022 Feb 17","2022 Aug 9","2022 Oct 31","2019 May 14","2019 Jul 15","2020 Nov 6","2022 Jan 20","2021 Dec 23","2021 Apr 19","2022 May 16","2022 Aug 9","2021 Mar 23","2020 mar 30","2021 Dec 23","2020 Nov 6","2021 May 10","2021 Dec 23","2023 Mar 19","2022 May 16","2022 Sep 5","2023 Jan 1","2021 Nov 5","2022 May 16","2022 May 2","2022 Mar 10","2022 Mar 10","2021 Apr 19","2022 Mar 10","2021 Sep 2","2019 Jul 15","2022 Jul 5","2020 Jan 16","2020 mar 30","2022 May 16","2019 Nov 18","2021 Mar 23","2022 Jan 19","2022 Nov 30","2020 Jan 16","2022 Dec 9","2020 Feb 1","2022 Mar 15","2022 Jan 20","2022 Sep 5","2023 Jan 1","2023 Jan 4","2022 Mar 15","2020 Feb 1","2022 Jan 25","2023 Mar 21","2023 Mar 19","2023 Mar 21","2020 Jan 16","2019 May 14","2021 Nov 5","2022 May 16","2022 May 30","2020 Oct 23","2021 Dec 20","2020 Dec 10","2022 Mar 12","2021 Dec 20","2019 Feb 22","2019 Jul 8","2022 Oct 17","2019 Apr 24","2017 Jul 5","2020 Feb 7","2020 Feb 7","2017 Jul 5","2019 Apr 24","2022 Nov 17","2022 Sep 9","2019 Apr 1","2021 Jan 11","2019 Apr 1","2021 Aug 20","2020 Apr 28","2022 Jun 24","2022 Feb 7","2021 Jan 11","2022 Dec 2","2019 Mar 19","2022 Nov 28","2020 Jan 17","2020 Jan 17","2018 Feb 16","2022 Mar 10","2021 Sep 30","2021 Jul 12","2022 May 16","2021 Feb 17","2021 Feb 17","2023 Jan 1","2018 Nov 7","2018 Nov 7","2021 Jul 2","2021 Aug 13","2019 Jul 17","2021 Jul 12","2021 Jul 7","2018 Nov 9","2021 Nov 25","2021 Nov 25","2021 Aug 25","2021 Dec 15","2021 Jul 19","2021 Jun 24","2020 Dec 8","2020 Feb 15","2017 Nov 1","2023 Mar 29","2021 Mar 4","2023 Mar 29","2020 Dec 8","2022 Dec 14","2019 Jul 15","2023 Jan 19","2021 May 25","2020 Jan 9","2021 Jul 20","2021 Jun 21","2018 Mar 19","2021 Dec 7","2021 Oct 28","2019 Jul 19","2019 Jul 19","2023 Feb 13","2022 Jul 18","2021 Jan 18","2019 Nov 19","2020 Jan 14","2019 Nov 19","2023 Feb 25","2017 Mar 6","2020 Jul 29","2020 Jul 29","2018 Aug 22","2018 Aug 22"],["Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell proteomics","Single-cell ATAC-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell proteomics","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Spatial imaging","Spatial transcriptomics","Single-cell Hi-C","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell genomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell Hi-C","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell mass cytometry","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell multiomics","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell sequencing","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell sequencing","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell genomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ChIP-seq","Spatial transcriptomics","Spatial transcriptomics","Spatial transcriptomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq"],["Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Data","Data","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis"],["Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Benchmarking datasets","Benchmarking datasets","Cell segmentation","Cell segmentation","Cell segmentation","Chromatin conformation","Chromatin conformation","Multi-task","Multi-task","Multi-task","Multi-task","Multi-task","Alignment","Alignment","Alignment","Alignment","Alignment","Alignment","Assembly","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Regulatory network inference","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Measure of association","Cell cell communication","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Measure of association","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","T/B-cell receptor","T/B-cell receptor","T/B-cell receptor","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Data integration","Deconvolution","Batch correction","Deconvolution","Deconvolution","Data integration","Batch correction","Batch correction","Data integration","Data integration","Data integration","Batch correction","Batch correction","Deconvolution","Deconvolution","Batch correction","Data integration","Data integration","Data integration","Deconvolution","Deconvolution","Data integration","Data integration","Batch correction","Batch correction","Batch correction","Batch correction","Batch correction","Deconvolution","Deconvolution","Batch correction","Batch correction","Data integration","Data integration","Batch correction","Deconvolution","Data integration","Batch correction","Data integration","Batch correction","Deconvolution","Data integration","Data integration","Deconvolution","Data integration","Deconvolution","Batch correction","Data integration","Deconvolution","Data integration","Deconvolution","Batch correction","Batch correction","Deconvolution","Data integration","Data integration","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Standardization","Standardization","Standardization","Standardization","Standardization","Standardization","Standardization","Trajectory","Trajectory","Trajectory","Velocity","Trajectory","Trajectory","Trajectory","Trajectory","Trajectory","Velocity","Velocity","Trajectory","Trajectory","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Doublet detection","Doublet detection","Doublet detection","Doublet detection","Quantification","Quantification","Quantification","Quantification","Rare cell type identification","Rare cell type identification","Rare cell type identification","Rare cell type identification","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Visualisation","Visualisation","Visualisation","Visualisation","Visualisation"]],"container":"<table class=\"table table-striped table-condensed\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>luke_topic<\/th>\n      <th>Paper.title<\/th>\n      <th>Paper.category<\/th>\n      <th>Methods.compared<\/th>\n      <th>Selection.criteria<\/th>\n      <th>Variability.of.score<\/th>\n      <th>Memory.measured<\/th>\n      <th>Max.number.of.cells<\/th>\n      <th>Sensitivity.analysis<\/th>\n      <th>Discovery.<\/th>\n      <th>Website<\/th>\n      <th>Data.availability<\/th>\n      <th>Prior.knowledge<\/th>\n      <th>Types.of.data<\/th>\n      <th>Number.of.experimental.datasets<\/th>\n      <th>Number.of.synthetic.datasets<\/th>\n      <th>Recommendation<\/th>\n      <th>Applicability<\/th>\n      <th>Speed.measured.<\/th>\n      <th>Overall.comparison<\/th>\n      <th>journal_name<\/th>\n      <th>publication_date<\/th>\n      <th>data_type<\/th>\n      <th>broader_topic<\/th>\n      <th>finer_topic<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"bPaginate":false,"dom":"<\"top\"flip>rt","language":{"info":"Showing _TOTAL_ entries"},"pageLength":10,"scrollX":true,"searching":true,"autoWidth":true,"columnDefs":[{"width":"150px","targets":[1,23,24,25]},{"width":"200px","targets":2},{"className":"dt-right","targets":[4,8,15,16]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"luke_topic","targets":1},{"name":"Paper.title","targets":2},{"name":"Paper.category","targets":3},{"name":"Methods.compared","targets":4},{"name":"Selection.criteria","targets":5},{"name":"Variability.of.score","targets":6},{"name":"Memory.measured","targets":7},{"name":"Max.number.of.cells","targets":8},{"name":"Sensitivity.analysis","targets":9},{"name":"Discovery.","targets":10},{"name":"Website","targets":11},{"name":"Data.availability","targets":12},{"name":"Prior.knowledge","targets":13},{"name":"Types.of.data","targets":14},{"name":"Number.of.experimental.datasets","targets":15},{"name":"Number.of.synthetic.datasets","targets":16},{"name":"Recommendation","targets":17},{"name":"Applicability","targets":18},{"name":"Speed.measured.","targets":19},{"name":"Overall.comparison","targets":20},{"name":"journal_name","targets":21},{"name":"publication_date","targets":22},{"name":"data_type","targets":23},{"name":"broader_topic","targets":24},{"name":"finer_topic","targets":25}],"order":[],"orderClasses":false,"orderCellsTop":true},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div>
<div id="evaluation-summary" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-evaluation-summary">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card tabset html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header dashboard-card-no-title bslib-navs-card-title">

<ul class="nav nav-tabs card-header-tabs" role="tablist" data-tabsetid="card-tabset-11"><li class="nav-item" role="presentation"><a href="#card-tabset-11-1" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Journal" aria-selected="true" class="nav-link active">Journal</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-11-2" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Publication Year" aria-selected="false" class="nav-link">Publication Year</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-11-3" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Category Frequency" aria-selected="false" class="nav-link">Category Frequency</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-11-4" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Statistics" aria-selected="false" class="nav-link">Statistics</a></li></ul></div>




<div class="tab-content html-fill-item html-fill-container" data-tabset-id="card-tabset-11"><div class="tab-pane active show html-fill-item html-fill-container" role="tabpanel" id="card-tabset-11-1"><div class="card-body html-fill-item html-fill-container" data-title="Journal">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 50fr) minmax(3em, 50fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure Benchmark" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure Benchmark</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - Journal - Pure benchmark-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development paper" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development paper</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - Journal - New method development paper-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-11-2"><div class="card-body html-fill-item html-fill-container" data-title="Publication Year">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 50fr) minmax(3em, 50fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure Benchmark" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure Benchmark</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - Pub year - Pure benchmark-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development paper" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development paper</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - Pub year - New method development paper-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-11-3"><div class="card-body html-fill-item html-fill-container" data-title="Category Frequency">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 50fr) minmax(3em, 50fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure Benchmark" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure Benchmark</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - category freq - Pure benchmark-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development paper" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development paper</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - category freq - New method development paper-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-11-4"><div class="card-body html-fill-item html-fill-container" data-title="Statistics">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-fill="false" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-body html-fill-item html-fill-container flow">
Statistics
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div></div></div><bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="general-information" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-general-information">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, max-content); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-fill="false" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-body html-fill-item html-fill-container">
<!-- ### <center>[**The current landscape and emerging challenges of benchmarking single-cell  methods**](https://www.biorxiv.org/content/10.1101/2023.12.19.572303v1)<center> -->
<div style="text-align: center;" class="html-fill-item html-fill-container">
<h3>
<a href="https://www.biorxiv.org/content/10.1101/2023.12.19.572303v1" target="_blank"> <strong>The current landscape and emerging challenges of benchmarking single-cell methods</strong> </a>
</h3>
</div>
<strong>Introduction</strong>
<p>A comprehensive evaluation strategy is critical for single-cell methodological development to assess the applicability of the method and to examine under what circumstances it works or fails.Informative evaluation results can drive the direction of method refinement. As a result, method development and evaluation should be considered as an iterative process.&nbsp;</p>
<p>Such evaluation should be distinguished from benchmarking purely based on evaluation metrics, as it aims to not only provide the method developers with a better understanding of their own methods but also demonstrate to the biologists how they can use the developed methods to gain new biological insights, which is what all the computational methods should be developed for. Due to the complexity of single-cell omics data, several key aspects should be taken into account while evaluating single-cell computational methods, including:&nbsp;</p>
<strong>Accuracy</strong>
<p>Whether the method does what it is designed to do. For example, data integration methods should remove the dataset effect and other unwanted variations while retaining the biological signals; and cell type classification methods should accurately classify the cell types and reveal the novel cell types that are not in the reference data. This aspect can usually be quantified by evaluation metrics, such as silhouette coefficients, adjusted rand index, Local Inverse Simpsonâ€™s Index (LISI) <em>harmony</em> and kBET<em>buttner2017assessment</em> for data integration methods; and classification accuracy and F1-score for cell type classification methods.</p>
<strong>Scalability</strong>
<p>Whether the method can handle large-scale single-cell data in a computationally efficient manner. This requires benchmarking both the computational time and memory usage of the method.</p>
<strong>Stability</strong>
<ul>
<li><p>Generalisability and Applicability (Stability of input): whether the method can be applied to data with different properties. This involves evaluating the developed method on data from diverse technologies (low- and high-throughput methods), tissues, organisms, stages (developmental and well-differentiated stages), disease, and sample sizes (tissue-specific and atlas-scale).</p></li>
<li><p>Robustness (Stability of model): whether the method is robust to noise in the data and the choice of hyper parameter. It can be evaluated by whether the methodâ€™s performance is significantly impacted when (i) only a subset of data is used; (ii) simulated noise is introduced into the data and (iii) models are run using different hyper parameter settings.</p></li>
<li><p>Reproducibility (Stability of output): technical vs broader. Whether the method produce same result when repeated with the same setting&nbsp;</p></li>
</ul>
<strong>Interpretability</strong>
<ul>
<li><p>Impact on downstream analysis: whether applying the method improves or hinders the performance of downstream analysis that is described in Section <em>downstream</em>. For example, for data integration of developmental data, it is important that the integrated data preserve the continuous signals so that the trajectory analysis can be applied to infer the pseudotime. For example, batch correction methods that return adjusted matrix would be more useful than methods that return embedding.&nbsp;</p></li>
<li><p>Biological impact: whether the developed method can help biologists to gain new biological insights. For example, data integration of multi-omics that is able to reveal rare and novel cell types, which could not be identified using single omics, would allow biologists to examine the cell characteristics of such rare cell types.</p></li>
</ul>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>


<div id="3ade8a4a-fb1d-4a6c-8409-ac45482d5fc9" class="hidden html-fill-item html-fill-container">

</div>
</div>
</div>
</div>


<script type="ojs-module-contents">
eyJjb250ZW50cyI6W3sibWV0aG9kTmFtZSI6ImludGVycHJldCIsImNlbGxOYW1lIjoib2pzLWNlbGwtMSIsImlubGluZSI6ZmFsc2UsInNvdXJjZSI6IlxuZDMgPSByZXF1aXJlKFwiZDNAN1wiKVxudG9wb2pzb24gPSByZXF1aXJlKFwidG9wb2pzb25cIilcblxuIn0seyJtZXRob2ROYW1lIjoiaW50ZXJwcmV0IiwiY2VsbE5hbWUiOiJvanMtY2VsbC0yIiwiaW5saW5lIjpmYWxzZSwic291cmNlIjoiXG5kYXRhX28gPSBGaWxlQXR0YWNobWVudChcImRhdGEvZGF0YTIuY3N2XCIpLmNzdih7IHR5cGVkOiB0cnVlIH0pXG5cbiJ9XX0=
</script>
<script type="module">
if (window.location.protocol === "file:") { alert("The OJS runtime does not work with file:// URLs. Please use a web server to view this document."); }
window._ojs.paths.runtimeToDoc = "../..";
window._ojs.paths.runtimeToRoot = "../..";
window._ojs.paths.docToRoot = "";
window._ojs.selfContained = false;
window._ojs.runtime.interpretFromScriptTags();
</script>
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "î§‹";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/sydneybiox\.github\.io\/sc_bench_benchmark_dashboard\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
<p>&amp;copy Sydney Precision Data Science Centre 2024</p>
</div>
  </div>
</footer>

<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (_event) {
  if (window.bslib.Card) {
    window.bslib.Card.initializeAllCards();
  }
}); 
</script>
  



</body></html>