<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="">

<title>sc bench benchmark dashboard – sc_bench_benchmark_dashboard</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./logo.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-dashboard/quarto-dashboard.js"></script>
<script src="site_libs/quarto-dashboard/stickythead.js"></script>
<script src="site_libs/quarto-dashboard/datatables.min.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/pdfmake.min.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/vfs_fonts.js" kdttablesentinel="true"></script>
<script src="site_libs/quarto-dashboard/web-components.js" type="module"></script>
<script src="site_libs/quarto-dashboard/components.js"></script>
<link href="site_libs/quarto-dashboard/datatables.min.css" rel="stylesheet" kdttablesentinel="true">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script type="module" src="site_libs/quarto-ojs/quarto-ojs-runtime.js"></script>
<link href="site_libs/quarto-ojs/quarto-ojs.css" rel="stylesheet">
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">

<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>

<script src="site_libs/plotly-binding-4.10.4/plotly.js"></script>

<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>

<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>

<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">

<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>

<link href="site_libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">

<script src="site_libs/plotly-main-2.11.1/plotly-latest.min.js"></script>

<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">

<script src="site_libs/datatables-binding-0.33/datatables.js"></script>

<link href="site_libs/dt-core-bootstrap-1.13.6/css/dataTables.bootstrap.min.css" rel="stylesheet">

<link href="site_libs/dt-core-bootstrap-1.13.6/css/dataTables.bootstrap.extra.css" rel="stylesheet">

<script src="site_libs/dt-core-bootstrap-1.13.6/js/jquery.dataTables.min.js"></script>

<script src="site_libs/dt-core-bootstrap-1.13.6/js/dataTables.bootstrap.min.js"></script>

<link href="site_libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet">

<script src="site_libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>

<link href="site_libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet">

<script src="site_libs/selectize-0.12.0/selectize.min.js"></script>

<script type="ojs-define">

{"contents":[{"name":"paper_data","value":{"luke_topic":["Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Benchmarking datasets","Addition - Benchmarking datasets","Addition - Cell segmentation","Addition - Cell segmentation","Addition - Cell segmentation","Addition - Chromatin conformation","Addition - Chromatin conformation","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Alignment","Alignment","Alignment","Alignment","Alignment","Alignment","Assembly","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Immune","Immune","Immune","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Perturbations","Perturbations","Perturbations","Perturbations","Perturbations","Perturbations","Quality Control","Quality Control","Quality Control","Quality Control","Quantification","Quantification","Quantification","Quantification","Rare Cells","Rare Cells","Rare Cells","Rare Cells","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variants","Variants","Variants","Variants","Variants","Variants","Visualisation","Visualisation","Visualisation","Visualisation","Visualisation"],"Timestamp":["2023-04-24 00:05:36","2023-01-23 22:23:45","2023-01-30 22:14:33","2023-01-09 12:28:07","2023-02-17 12:55:17","2023-07-17 14:50:57.654","2023-07-15 21:57:09.662","2023-02-26 21:44:30","2023-07-18 11:04:38.939","2023-07-17 00:18:48.963","2023-01-19 01:30:00","2023-02-27 23:46:10","2023-01-25 15:47:29","2023-07-18 14:00:03.941","2023-02-01 17:14:43","2023-07-18 14:25:35.944","2023-02-06 11:04:39","2023-01-26 22:01:51","2023-01-26 21:38:40","2023-01-31 10:50:01","2023-04-22 17:14:30","2023-04-20 11:45:48","2023-07-21 18:26:26.999","2023-07-18 10:13:49.874","2023-07-17 10:30:46.552","2023-02-14 15:52:59","2023-07-18 09:36:15.527","2023-02-09 13:09:52","2023-03-02 13:36:35","2023-04-08 15:45:43","2023-04-08 16:59:23","2023-04-05 13:39:35","2023-07-18 10:55:17.058","2023-07-18 10:59:50.283","2023-07-17 16:36:50.483","2023-07-23 13:50:11.492","2023-04-19 16:28:31","2023-04-19 14:02:02","2023-07-18 17:20:42.943","2023-01-29 01:49:53","2023-02-10 12:14:41","2023-02-12 15:15:29","2023-02-16 12:42:37","2023-02-06 16:48:45","2023-02-13 00:23:37","2023-04-23 23:44:09","2023-07-17 13:58:41.343","2023-04-10 16:35:02","2023-07-17 23:59:10.961","2023-04-19 12:22:43","2023-07-18 08:38:59.932","2023-07-18 09:01:48.864","2023-04-19 10:11:30","2023-07-18 08:48:01.742","2023-02-14 13:17:27","2023-01-29 01:32:33","2023-02-07 15:42:59","2023-07-17 18:38:48.866","2023-02-26 20:44:26","2023-04-13 10:36:10","2023-02-13 23:15:27","2023-01-17 10:21:56","2023-02-16 11:17:59","2023-04-12 10:06:47","2023-04-12 10:08:41","2023-07-18 10:47:04.472","2023-07-17 21:07:21.129","2023-04-17 12:40:47","2023-07-17 11:15:47.273","2023-03-02 15:57:10","2023-07-17 18:21:14.643","2023-02-09 14:33:45","2023-04-11 19:47:53","2023-04-17 11:07:41","2023-02-14 18:09:15","2023-02-07 17:10:44","2023-02-15 11:46:02","2023-07-17 22:45:18.585","2023-02-20 01:02:30","2023-07-22 12:52:14.372","2023-02-15 00:02:52","2023-07-17 22:29:04.848","2023-02-19 21:49:01","2023-07-17 21:53:04.737","2023-07-17 21:31:29.131","2023-04-19 11:30:45","2023-02-09 21:52:15","2023-04-08 16:32:49","2023-07-17 23:46:26.884","2023-02-10 11:25:40","2023-07-17 13:12:34.289","2023-02-07 18:00:15","2023-07-21 19:39:58.927","2023-04-19 09:43:06","2023-01-29 00:45:58","2023-07-22 18:58:03.495","2023-07-18 17:18:42.342","2023-07-17 12:05:53.272","2023-07-21 12:02:19.027","2023-02-08 10:37:30","2023-07-20 17:37:30.532","2023-07-17 18:02:54.974","2023-01-17 09:26:29","2023-02-08 10:23:49","2023-07-17 12:10:20.871","2023-07-19 16:01:06.739","2023-02-08 10:01:35","2023-04-19 16:51:32","2023-07-17 17:15:45.085","2023-07-17 16:34:15.895","2023-02-10 12:16:19","2023-07-19 22:43:07.078","2023-07-24 17:13:34.231","2023-04-19 15:51:42","2023-02-10 16:28:53","2023-07-17 11:58:19.412","2023-07-17 13:41:25.553","2023-02-06 16:20:51","2023-03-06 13:35:10","2023-07-24 14:42:12.57","2023-01-26 13:12:23","2023-02-08 22:09:24","2023-01-25 21:46:16","2023-01-25 23:37:04","2023-07-20 14:03:36.144","2023-01-26 12:26:21","2023-07-17 15:59:06.997","2023-07-17 12:51:09.834","2023-02-06 15:38:10","2023-02-16 11:16:38","2023-01-25 21:41:08","2023-01-27 13:33:10","2023-01-25 14:39:30","2023-01-27 09:10:46","2023-02-14 16:04:29","2023-07-24 11:38:04.193","2023-04-06 20:21:32","2023-02-27 23:07:26","2023-01-26 13:54:15","2023-07-17 11:34:27.443","2023-07-17 18:58:38.08","2023-02-27 23:06:21","2023-07-21 14:37:45.466","2023-07-18 11:21:54.211","2023-01-17 23:07:10","2023-07-17 13:05:43.693","2023-01-25 17:32:11","2023-01-27 16:07:32","2023-03-06 12:26:24","2023-01-26 14:40:42","2023-02-07 15:49:39","2023-01-13 12:56:36","2023-02-27 23:01:18","2023-01-26 22:09:16","2023-02-04 20:26:42","2023-07-17 17:18:09.171","2023-01-06 16:58:07","2023-01-30 17:39:32","2023-02-28 13:09:39","2023-07-21 11:52:48.328","2023-01-06 16:21:24","2023-07-17 11:47:20.672","2023-07-16 12:16:10.077","2023-07-23 14:09:23.832","2023-02-06 11:57:55","2023-07-18 15:30:44.422","2023-04-06 13:37:43","2023-01-06 19:05:40","2023-01-13 17:22:45","2023-07-17 11:34:11.436","2023-07-18 12:06:08.432","2023-07-17 12:21:49.758","2023-07-17 11:51:30.378","2023-02-14 16:40:24","2023-07-18 18:07:51.554","2023-04-06 13:31:02","2023-07-17 10:12:23.66","2023-04-06 13:51:15","2023-01-10 10:57:41","2023-07-18 14:55:58.358","2023-04-06 13:13:10","2023-07-17 12:24:08.992","2023-03-03 17:12:45","2023-07-18 16:40:25.902","2023-07-17 12:25:17.851","2023-02-08 23:16:41","2023-04-06 13:21:06","2023-02-16 20:51:12","2023-04-05 09:55:20","2023-07-15 23:04:24.036","2023-02-01 21:25:16","2023-01-29 00:25:11","2023-02-10 13:26:16","2023-01-27 15:53:28","2023-02-02 14:03:16","2023-02-26 12:52:51","2023-02-03 15:50:18","2023-02-16 12:15:17","2023-07-25 13:52:07.984","2023-02-10 11:02:56","2023-07-20 17:58:17.768","2023-04-23 20:04:45","2023-04-06 16:02:00","2023-07-16 17:10:21.82","2023-04-17 15:33:28","2023-01-24 23:07:31","2023-07-17 23:40:19.921","2023-02-01 16:08:16","2023-07-18 10:38:01.161","2023-07-19 19:59:43.884","2023-07-22 13:07:51.759","2023-02-01 13:43:46","2023-07-18 10:22:41.578","2023-07-16 12:13:00.103","2023-01-11 15:40:52","2023-04-19 15:25:55","2023-07-17 10:51:15.114","2023-07-19 20:47:40.252","2023-02-01 17:00:01","2023-07-16 16:34:35.937","2023-02-13 22:25:21","2023-07-23 15:05:27.921","2023-02-14 14:50:27","2023-02-15 16:03:57","2023-02-09 12:12:25","2023-02-08 11:27:35","2023-03-05 18:09:31","2023-03-06 14:25:56","2023-07-16 16:12:24.374","2023-03-07 10:24:47","2023-02-08 13:50:04","2023-02-01 15:10:48","2023-07-16 10:58:53.995","2023-04-16 19:32:46","2023-07-17 15:17:15.55","2023-02-09 16:56:15","2022-12-16 10:26:32","2023-01-25 17:58:36","2023-07-17 15:32:31.666","2023-04-11 17:43:11","2023-07-17 14:57:43.497","2023-04-06 17:16:53","2023-02-07 17:12:47","2023-02-13 20:57:45","2023-03-06 11:20:04","2023-07-17 10:04:22.341","2023-07-17 11:12:12.671","2023-07-14 21:55:14.189","2023-03-08 10:58:07","2023-02-03 21:07:13","2023-07-17 15:01:03.837","2023-04-21 11:34:57","2023-04-11 07:23:11","2023-07-18 17:47:07.548","2023-07-18 11:06:06.415","2023-01-24 23:44:40","2023-07-17 23:16:50.183","2023-07-17 10:20:21.13","2023-07-17 14:36:43.993","2023-03-02 16:26:55","2023-02-03 13:13:11","2023-02-09 15:35:48","2023-07-14 22:38:31.779","2023-07-24 16:04:50.869","2023-07-17 11:02:12.833","2023-07-17 11:59:47.064","2023-07-17 15:12:11.002","2023-07-22 12:37:07.715","2023-07-18 16:37:07.364","2023-04-16 20:46:24","2023-03-02 15:30:10","2023-07-24 16:29:42.908","2023-07-22 12:16:12.191","2023-05-31 11:09:39","2023-07-17 15:26:14.257","2023-02-14 15:31:20","2023-07-17 15:59:25.094","2023-07-16 11:33:01.766","2023-03-03 16:50:03","2023-07-17 15:40:07.148","2023-07-25 14:56:26.068","2023-07-17 11:48:18.689","2023-02-06 13:40:28","2023-01-31 12:44:47","2023-04-06 15:30:35","2023-01-27 08:16:40","2023-02-01 11:11:58","2023-01-24 22:46:41","2023-07-18 15:33:18.466","2023-07-18 21:35:37.201","2023-02-06 16:45:03","2023-02-03 12:15:34","2023-02-08 21:18:30","2023-07-18 22:09:07.892","2023-07-18 21:56:25.315","2023-07-18 17:23:09.872","2023-07-21 14:19:23.448","2023-02-07 22:00:11","2023-07-17 14:45:41.874","2023-07-18 16:25:46.731","2023-07-18 14:07:43.259","2023-02-01 20:50:51","2023-02-17 11:41:14","2023-04-11 21:59:00","2023-07-25 12:44:17.488","2023-04-10 16:39:07","2023-07-18 22:33:46.505","2023-07-17 15:01:10.852","2023-02-08 15:01:05","2023-02-09 12:43:44","2023-07-17 15:42:18.237","2023-02-13 02:15:22","2023-02-16 18:46:08","2023-07-17 13:51:24.745","2023-07-17 14:46:25.571","2023-07-18 14:21:43.312","2023-07-18 17:11:34.934","2023-07-18 15:11:40.144","2023-04-19 11:13:29","2023-02-09 16:30:12","2023-02-08 15:28:46","2023-07-18 11:15:59.64","2023-07-18 20:59:37.03","2023-07-18 15:41:55.797","2023-07-18 19:43:10.468","2023-04-12 20:35:36","2023-07-18 14:37:32.451","2023-04-11 21:25:23","2023-02-08 17:55:49","2023-07-18 12:08:07.473","2023-03-02 13:17:49","2023-07-17 21:21:47.262","2023-07-22 13:27:53.683","2023-04-11 22:34:51","2023-07-17 20:55:33.271","2023-07-17 22:46:22.566","2023-04-12 22:36:34","2023-07-18 16:51:31.085","2023-04-20 13:41:11","2023-04-24 01:15:23","2023-04-24 01:59:06","2023-04-24 01:36:01","2023-01-20 12:01:35","2023-02-16 19:33:38","2023-07-17 22:49:09.377","2023-07-18 15:46:32.182","2023-04-15 20:10:22","2023-02-01 16:38:10","2023-07-18 11:06:56.498","2023-02-01 16:35:52","2023-04-15 20:45:12","2023-07-17 19:51:48.078","2023-02-10 08:56:32","2023-01-10 11:27:04","2023-07-24 16:45:26.807","2023-07-17 17:36:18.025"],"Paper_title":["spicyR: spatial analysis of in situ cytometry data in R","Benchmarking UMI-based single-cell RNA-seq preprocessing workflows","Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments","Interactive analysis of single-cell epigenomic landscapes with ChromSCape","pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools","A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples","The Current State of Single-Cell Proteomics Data Analysis","Assessment of computational methods for the analysis of single-cell ATAC-seq data","EpiScanpy: integrated single-cell epigenomic analysis","TIST: Transcriptome and Histopathological Image Integrative Analysis for Spatial Transcriptomics","Comparison of high-throughput single-cell RNA sequencing data processing pipelines","Interactive analysis of single-cell epigenomic landscapes with ChromSCape","Benchmarking UMI-based single-cell RNA-seq preprocessing workflows","Destin2: Integrative and cross-modality analysis of single-cell chromatin accessibility data","Comparison of high-throughput single-cell RNA sequencing data processing pipelines","The Current State of Single-Cell Proteomics Data Analysis","A systematic evaluation of single cell RNA-seq analysis pipelines","Assessment of computational methods for the analysis of single-cell ATAC-seq data","A systematic evaluation of single cell RNA-seq analysis pipelines","pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools","histocat: analysis of cell phenotypes and interactions in multiplex image cytometry data","Giotto: a toolbox for integrative analysis and visualization of spatial expression data","A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples","TISCH: a comprehensive web resource enabling interactive single-cell transcriptome visualization of tumor microenvironment","Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq","Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments","Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq","A multi-center cross-platform single-cell RNA sequencing reference dataset","A multi-center cross-platform single-cell RNA sequencing reference dataset","Joint cell segmentation and cell type annotation for spatial transcriptomics","Cellpose: a generalist algorithm for cellular segmentation","Cell segmentation in imaging-based spatial transcriptomics","Perspectives for the reconstruction of 3D chromatin conformation using single cell Hi-C data","SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data","High-throughput single-cell RNA-seq data imputation and characterization with surrogate-assisted automated deep learning","High-throughput single-cell RNA-seq data imputation and characterization with surrogate-assisted automated deep learning","Spatially informed clustering, integration, and deconvolution of spatial transcriptomics with GraphST","Joint dimension reduction and clustering analysis of single-cell RNA-seq and spatial transcriptomics data","scGNN is a novel graph neural network framework for single-cell RNA-Seq analyses","Choice of pre-processing pipeline influences clustering quality of scRNA-seq datasets","Comparative analysis of common alignment tools for single-cell RNA sequencing","Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment","Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment","Comparative analysis of common alignment tools for single-cell RNA sequencing","Choice of pre-processing pipeline influences clustering quality of scRNA-seq datasets","Mapping molecular assemblies with fluorescence microscopy and object-based spatial statistics","Influence of Cell-type Ratio on Spatially Resolved Single-cell Transcriptomes using the Tangram Algorithm: Based on Implementation on Single-Cell and MxIF Data","scAB detects multiresolution cell states with clinical significance by integrating single-cell genomics and bulk sequencing data","Spatial-ID: a cell typing method for spatially resolved transcriptomics via transfer learning and spatial embedding","Latent cellular analysis robustly reveals subtle diversity in large-scale single-cell RNA-seq data","Learning Cell Annotation under Multiple Reference Datasets by Multisource Domain Adaptation","scMAGIC: accurately annotating single cells using two rounds of reference-based classification","A multiresolution framework to characterize single-cell state landscapes","scGCN is a graph convolutional networks algorithm for knowledge transfer in single cell omics","scClassify: sample size estimation and multiscale classification of cells using single and multiple reference","Evaluation of Cell Type Annotation R Packages on Single-cell RNA-seq Data","A comparison of automatic cell identification methods for single-cell RNA sequencing data","Adversarial dense graph convolutional networks for single-cell classification","Benchmarking automated cell type annotation tools for single-cell ATAC-seq data","CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing","A comparison of automatic cell identification methods for single-cell RNA sequencing data","Benchmarking automated cell type annotation tools for single-cell ATAC-seq data","Evaluation of single-cell classifiers for single-cell RNA sequencing data sets","scPred: accurate supervised method for cell-type classification from single-cell RNA-seq data","Supervised classification enables rapid annotation of cell atlases","A Systematic Evaluation of Supervised Machine Learning Algorithms for Cell Phenotype Classification Using Single-Cell RNA Sequencing Data","ChrNet: A re-trainable chromosome-based 1D convolutional neural network for predicting immune cell types","scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data","A Systematic Evaluation of Supervised Machine Learning Algorithms for Cell Phenotype Classification Using Single-Cell RNA Sequencing Data","scClassify: sample size estimation and multiscale classification of cells using single and multiple reference","Cross-species cell-type assignment from single-cell RNA-seq data by a heterogeneous graph neural network","Evaluation of some aspects in supervised cell type identification for single-cell RNA-seq: classifier, feature selection, and reference construction","Identifying phenotype-associated subpopulations by integrating bulk and single-cell sequencing data","CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing","Evaluation of some aspects in supervised cell type identification for single-cell RNA-seq: classifier, feature selection, and reference construction","Evaluation of methods to assign cell type labels to cell clusters from single-cell RNA-sequencing data","A comparison of automatic cell identification methods for single-cell RNA sequencing data","scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data","Evaluation of Cell Type Annotation R Packages on Single-cell RNA-seq Data","Evaluation of machine learning approaches for cell-type identification from single-cell transcriptomics data","Evaluation of single-cell classifiers for single-cell RNA sequencing data sets","Integrating multiple references for single-cell assignment","Evaluation of methods to assign cell type labels to cell clusters from single-cell RNA-sequencing data","scGAD: a new task and end-to-end framework for generalized cell type annotation and discovery","Learning for single-cell assignment","CIDR: Ultrafast and accurate clustering through imputation for single-cell RNA-seq data","Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data","ClusterMap for multi-scale clustering analysis of spatial gene expression","Secuer: Ultrafast, scalable and accurate clustering of single-cell RNA-seq data","Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data","A comparison framework and guideline of clustering methods for mass cytometry data","A systematic performance evaluation of clustering methods for single-cell RNA-seq data","HGC: fast hierarchical clustering for large-scale single-cell data","Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis","Benchmarking cell-type clustering methods for spatially resolved transcriptomics data","A review and performance evaluation of clustering frameworks for single-cell Hi-C data","A comparison framework and guideline of clustering methods for mass cytometry data","scMINER: a mutual information-based framework for identifying hidden drivers from single-cell omics data","A spectral clustering with self-weighted multiple kernel learning method for single-cell RNA-seq data","Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data","EDClust: an EM-MM hybrid method for cell clustering in multiple-subject single-cell RNA sequencing","Using single-cell cytometry to illustrate integrated multi-perspective evaluation of clustering algorithms using Pareto fronts","Benchmarking cell-type clustering methods for spatially resolved transcriptomics data","Benchmark and Parameter Sensitivity Analysis of Single-Cell RNA Sequencing Clustering Methods","Clustering methods for single-cell RNA-sequencing expression data: performance evaluation with varying sample sizes and cell compositions","Single-Cell RNA Sequencing Data Interpretation by Evolutionary Multiobjective Clustering","Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data","sc3: consensus clustering of single-cell rna-seq data","A Hybrid Clustering Algorithm for Identifying Cell Types from Single-Cell RNA-Seq Data","Clustering methods for single-cell RNA-sequencing expression data: performance evaluation with varying sample sizes and cell compositions","A systematic performance evaluation of clustering methods for single-cell RNA-seq data","ASTER: accurately estimating the number of cell types in single-cell chromatin accessibility data","Using single-cell cytometry to illustrate integrated multi-perspective evaluation of clustering algorithms using Pareto fronts","A model-based constrained deep learning clustering approach for spatially resolved single-cell data","Benchmark and Parameter Sensitivity Analysis of Single-Cell RNA Sequencing Clustering Methods","A review and performance evaluation of clustering frameworks for single-cell Hi-C data","Benchmarking integration of single-cell differential expression","Comparison of methods to detect differentially expressed genes between single-cell populations","Benchmarking of a Bayesian single cell RNAseq differential gene expression test for dose–response study designs","Benchmarking integration of single-cell differential expression","Comparison of methods to detect differentially expressed genes between single-cell populations","Bias, robustness and scalability in single-cell differential expression analysis","Single-cell RNA-sequencing: assessment of differential expression analysis methods","Benchmarking of a Bayesian single cell RNAseq differential gene expression test for dose–response study designs","Recommendations of scRNA-seq Differential Gene Expression Analysis Based on Comprehensive Benchmarking","Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data","Differential expression of single-cell RNA-seq data using Tweedie models","Recommendations of scRNA-seq Differential Gene Expression Analysis Based on Comprehensive Benchmarking","Bias, robustness and scalability in single-cell differential expression analysis","Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data","Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data","Single-cell RNA-sequencing: assessment of differential expression analysis methods","Reproducibility of Methods to Detect Differentially Expressed Genes from Single-Cell RNA Sequencing","Reproducibility of Methods to Detect Differentially Expressed Genes from Single-Cell RNA Sequencing","Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data","Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics","Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis","A Quantitative Framework for Evaluating Single-Cell Data Structure Preservation by Dimensionality Reduction Techniques","Benchmarking principal component analysis for large-scale single-cell RNA-sequencing","Towards a comprehensive evaluation of dimension reduction methods for transcriptomic data visualization","Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics","Supervised application of internal validation measures to benchmark dimensionality reduction methods in scRNA-seq data","A Comparison for Dimensionality Reduction Methods of Single-Cell RNA-seq Data","Towards a comprehensive evaluation of dimension reduction methods for transcriptomic data visualization","Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer","A Comparison for Dimensionality Reduction Methods of Single-Cell RNA-seq Data","Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer","Tuning parameters of dimensionality reduction methods for single-cell RNA-seq analysis","Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis","A Quantitative Framework for Evaluating Single-Cell Data Structure Preservation by Dimensionality Reduction Techniques","Dimensionality reduction for visualizing single-cell data using UMAP","Dimensionality reduction for visualizing single-cell data using UMAP","Tuning parameters of dimensionality reduction methods for single-cell RNA-seq analysis","Supervised application of internal validation measures to benchmark dimensionality reduction methods in scRNA-seq data","Benchmarking principal component analysis for large-scale single-cell RNA-sequencing","Network embedding-based representation learning for single cell RNA-seq data","Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data","Evaluating the reproducibility of single-cell gene regulatory network inference algorithms","Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data","Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information","Evaluating the reproducibility of single-cell gene regulatory network inference algorithms","Single-cell biological network inference using a heterogeneous graph transformer","NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks","A systematic evaluation of the computational tools for ligand-receptor-based cell-cell interaction inference","Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data","Identifying strengths and weaknesses of methods for computational network inference from single-cell RNA-seq data","Regulatory analysis of single cell multiome gene expression and chromatin accessibility data with scREG","A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data","Evaluating measures of association for single-cell transcriptomics","SRGS: sparse partial least squares-based recursive gene selection for gene regulatory network inference","Performance assessment of sample-specific network control methods for bulk and single-cell biological data analysis","A systematic evaluation of the computational tools for ligand-receptor-based cell-cell interaction inference","Performance assessment of sample-specific network control methods for bulk and single-cell biological data analysis","Impact of similarity metrics on single-cell RNA-seq data clustering","Optimal transport improves cell-cell similarity inference in single-cell omics data","DIRECT-NET: An efficient method to discover cis-regulatory elements and construct regulatory networks from single-cell multiomics data","Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information","IReNA: Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles","A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data","How does the structure of data impact cell-cell similarity? Evaluating how structural properties influence the performance of proximity metrics in single cell RNA-seq data","Functional inference of gene regulation using single-cell multi-omics","maxATAC: Genome-scale transcription-factor binding prediction from ATAC-seq with deep neural networks","Impact of similarity metrics on single-cell RNA-seq data clustering","Single-cell gene regulatory network prediction by explainable AI","Identifying strengths and weaknesses of methods for computational network inference from single-cell RNA-seq data","Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data","Nonparametric single-cell multiomic characterization of trio relationships between transcription factors, target genes, and cis-regulatory regions","Evaluating measures of association for single-cell transcriptomics","Dissecting cell identity via network inference and in silico gene perturbation","How does the structure of data impact cell-cell similarity? Evaluating how structural properties influence the performance of proximity metrics in single cell RNA-seq data","Benchmarking algorithms for pathway activity transformation of single-cell RNA-seq data","Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data","Benchmarking algorithms for pathway activity transformation of single-cell RNA-seq data","Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data","Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data","Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data","Benchmarking computational methods for B-cell receptor reconstruction from single-cell RNA-seq data","Benchmarking computational methods for B-cell receptor reconstruction from single-cell RNA-seq data","Predicting recognition between T cell receptors and epitopes with TCRGP","False signals induced by single-cell imputation","NISC: Neural Network-Imputation for Single-Cell RNA Sequencing and Cell Type Clustering","SpaGE: Spatial Gene Enhancement using scRNA-seq","Comparison of Computational Methods for Imputing Single-Cell RNA-Sequencing Data","scESI: evolutionary sparse imputation for single-cell transcriptomes from nearest neighbor cells","Efficient prediction of a spatial transcriptomics profile better characterizes breast cancer tissue sections without costly experimentation","A systematic evaluation of single-cell RNA-sequencing imputation methods","Region-specific denoising identifies spatial co-expression patterns and intra-tissue heterogeneity in spatially resolved transcriptomics data","Comparison of Computational Methods for Imputing Single-Cell RNA-Sequencing Data","scIMC: a platform for benchmarking comparison and visualization analysis of scRNA-seq data imputation methods","scNPF: an integrative framework assisted by network propagation and network fusion for preprocessing of single-cell RNA-seq data","scIMC: a platform for benchmarking comparison and visualization analysis of scRNA-seq data imputation methods","False signals induced by single-cell imputation","Are dropout imputation methods for scRNA-seq effective for scATAC-seq data?","scHinter: imputing dropout events for single-cell RNA-seq data with limited sample size","A systematic evaluation of single-cell RNA-sequencing imputation methods","Spatial transcriptomics at subspot resolution with BayesSpace","Are dropout imputation methods for scRNA-seq effective for scATAC-seq data?","A novel method for single-cell data imputation using subspace regression","A benchmark study of deep learning-based multi-omics data fusion methods for cancer","SD2: spatially resolved transcriptomics deconvolution through integration of dropout and spatial information","scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets","Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology","Benchmarking of cell type deconvolution pipelines for transcriptomics data","scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning","Benchmarking atlas-level data integration in single-cell genomics","Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","A benchmark study of deep learning-based multi-omics data fusion methods for cancer","DEEPsc: A Deep Learning-Based Map Connecting Single-Cell Transcriptomics and Spatial Imaging Data","A comparison of methods accounting for batch effects in differential expression analysis of UMI count based single cell RNA sequencing","Benchmarking atlas-level data integration in single-cell genomics","Benchmarking of cell type deconvolution pipelines for transcriptomics data","SpatialDWLS: accurate deconvolution of spatial transcriptomic data","Benchmarking atlas-level data integration in single-cell genomics","IBRAP: integrated benchmarking single-cell RNA-sequencing analytical pipeline","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction","Benchmarking and integration of methods for deconvoluting spatial transcriptomic data","Systematic evaluation of transcriptomics-based deconvolution methods and references using thousands of clinical samples","Alignment and integration of spatial transcriptomics data","Multi-omics single-cell data integration and regulatory inference with graph-linked embedding","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","iSMNN: batch effect correction for single-cell RNA-seq data via iterative supervised mutual nearest neighbor refinement","Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology","Deconvolution analysis of cell-type expression from bulk tissues by integrating with single-cell expression reference","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","A comparison of methods accounting for batch effects in differential expression analysis of UMI count based single cell RNA sequencing","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","Fast, sensitive and accurate integration of single-cell data with Harmony","CellMixS: quantifying and visualizing batch effects in single-cell RNA-seq data","Advances in mixed cell deconvolution enable quantification of cell types in spatial transcriptomic data","AIscEA: unsupervised integration of single-cell gene expression and chromatin accessibility via their biological consistency","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","DeepST: identifying spatial domains in spatial transcriptomics by deep learning","BBKNN: fast batch alignment of single cell transcriptomes","Comprehensive evaluation of deconvolution methods for human brain gene expression","scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning","Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction","Benchmarking and integration of methods for deconvoluting spatial transcriptomic data","Single-cell multi-omics integration for unpaired data by a siamese network with graph-based contrastive loss","Comprehensive evaluation of deconvolution methods for human brain gene expression","BBKNN: fast batch alignment of single cell transcriptomes","Integration of single cell data by disentangled representation learning","A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics","IBRAP: integrated benchmarking single-cell RNA-sequencing analytical pipeline","A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets","Systematic evaluation of transcriptomics-based deconvolution methods and references using thousands of clinical samples","Benchmarking spatial and single-cell transcriptomics integration methods","Adversarial domain translation networks for integrating large-scale atlas-level single-cell datasets","A rank-based marker selection method for high throughput scRNA-seq data","Uncovering cell identity through differential stability with Cepo","DrivAER: Identification of driving transcriptional programs in single-cell RNA sequencing data","Triku: a feature selection method based on nearest neighbors for single-cell data","Uncovering cell identity through differential stability with Cepo","De novo gene signature identification from single-cell RNA-seq with hierarchical Poisson factorization","Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq","Normalization and de-noising of single-cell Hi-C data with BandNorm and scVI-3D","Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq","Assessment of Single Cell RNA-Seq Normalization Methods","Normalization Methods on Single-Cell RNA-seq Data: An Empirical Survey","Normalization Methods on Single-Cell RNA-seq Data: An Empirical Survey","Assessment of Single Cell RNA-Seq Normalization Methods","Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq","A lineage tree-based hidden Markov model quantifies cellular heterogeneity and plasticity","Detecting critical transition signals from single-cell transcriptomes to infer lineage-determining transcription factors","A comparison of single-cell trajectory inference methods","Preprocessing choices affect RNA velocity results for droplet scRNA-seq data","A comparison of single-cell trajectory inference methods","PseudoGA: cell pseudotime reconstruction based on genetic algorithm","Predicting cell lineages using autoencoders and optimal transport","psupertime: supervised pseudotime analysis for time-series single-cell RNA-seq data","scShaper: an ensemble method for fast and accurate linear trajectory inference from single-cell RNA-seq data","Preprocessing choices affect RNA velocity results for droplet scRNA-seq data","DeepVelo: Single-cell transcriptomic deep velocity field learning with neural ordinary differential equations","PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells","Density-based detection of cell transition states to construct disparate and bifurcating trajectories","Quantifying Waddington’s epigenetic landscape: a comparison of single-cell potency measures","Quantifying Waddington’s epigenetic landscape: a comparison of single-cell potency measures","A sparse differential clustering algorithm for tracing cell type changes via single-cell RNA-sequencing data","Transcriptome analysis method based on differential distribution evaluation","Differential abundance testing on single-cell data using k-nearest neighbor graphs","Ultra-fast scalable estimation of single-cell differentiation potency from scRNA-Seq data","Doublet identification in single-cell sequencing data using scDblFinder","Benchmarking computational doublet-detection methods for single-cell RNA sequencing data","Benchmarking computational doublet-detection methods for single-cell RNA sequencing data","Vaeda computationally annotates doublets in single-cell RNA sequencing data","Simulation-based benchmarking of isoform quantification in single-cell RNA-seq","Simulation-based benchmarking of isoform quantification in single-cell RNA-seq","Pheniqs 20: accurate, high-performance Bayesian decoding and confidence estimation for combinatorial barcode indexing","EmptyNN: A neural network based on positive and unlabeled learning to remove cell-free droplets and recover lost cells in scRNA-seq data","CellSIUS provides sensitive and specific detection of rare cell populations from complex single-cell RNA-seq data","Bayesian information sharing enhances detection of regulatory associations in rare cell types","GapClust is a light-weight approach distinguishing rare cells from voluminous single cell expression profiles","Discovery of rare cells from voluminous single cell expression data","A benchmark study of simulation methods for single-cell RNA sequencing data","A benchmark study of simulation methods for single-cell RNA sequencing data","ESCO: single cell expression simulation incorporating gene co-expression","splatPop: simulating population scale single-cell RNA sequencing data","SCAN-ATAC-Sim: a scalable and efficient method for simulating single-cell ATAC-seq data from bulk-tissue experiments","Spearheading future omics analyses using dyngen, a multi-modal simulator of single cells","Simulation, power evaluation and sample size recommendation for single-cell RNA-seq","SCSsim: an integrated tool for simulating single-cell genome sequencing data","powsimR: power analysis for bulk and single cell RNA-seq experiments","Built on sand: the shaky foundations of simulating single-cell RNA sequencing data","simATAC: a single-cell ATAC-seq simulation framework","Built on sand: the shaky foundations of simulating single-cell RNA sequencing data","Simulation, power evaluation and sample size recommendation for single-cell RNA-seq","A Framework for Comparison and Assessment of Synthetic RNA-Seq Data","A statistical simulator scDesign for rational scRNA-seq experimental design","SimCH: simulation of single-cell RNA sequencing data by modeling cellular heterogeneity at gene expression level","scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured","Realistic in silico generation and augmentation of single-cell RNA-seq data using generative adversarial networks","Novel ChIP-seq simulating program with superior versatility: isChIP","SPARK-X: non-parametric modeling enables scalable and robust detection of spatial expression patterns for large spatial transcriptomic studies","SpatialDE: identification of spatially variable genes","SOMDE: a scalable method for identifying spatially variable genes with self-organizing map","SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network","Evaluation of tools for highly variable gene discovery from single-cell RNA-seq data","Evaluation of tools for highly variable gene discovery from single-cell RNA-seq data","SINFONIA: Scalable Identification of Spatially Variable Genes for Deciphering Spatial Domains","Resolving single-cell copy number profiling for large datasets","Delineating Copy Number and Clonal Substructure in Human Tumors from Single-cell Transcriptomes","Systematic comparative analysis of single-nucleotide variant detection methods from single-cell RNA sequencing data","DENDRO: genetic heterogeneity profiling and subclone detection by single-cell RNA sequencing","Systematic comparative analysis of single-nucleotide variant detection methods from single-cell RNA sequencing data","A Variational Algorithm to Detect the Clonal Copy Number Substructure of Tumors from scRNA-seq Data","Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning","Comparison of visualization tools for single-cell RNAseq data","Comparison of visualization tools for single-cell RNAseq data","Generalizable and Scalable Visualization of Single-Cell Data Using Neural Networks","Generalizable and Scalable Visualization of Single-Cell Data Using Neural Networks"],"Paper_category":["New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper"],"Methods_compared":[0,10,27,3,24,21,2,10,11,7,7,3,10,7,7,2,20,10,20,24,5,5,21,0,2,27,2,5,5,1,3,2,2,3,5,5,13,18,9,2,5,2,2,5,2,4,0,5,8,4,2,13,4,4,14,10,22,4,5,3,22,5,9,4,1,13,5,6,13,14,5,9,0,1,9,5,22,2,10,10,9,8,5,7,null,4,14,1,4,12,9,14,20,4,15,8,9,3,7,14,11,3,15,13,11,15,12,5,4,11,14,4,3,7,13,8,46,5,8,46,5,36,6,8,12,11,8,12,36,11,18,6,9,9,18,5,18,11,21,8,5,33,10,8,9,10,9,5,18,11,5,5,5,33,21,3,8,6,8,16,6,4,7,9,12,11,0,14,17,10,16,9,16,4,4,9,16,1,14,17,0,2,4,3,11,12,0,17,4,17,7,7,7,5,5,7,6,6,5,6,14,4,8,9,1,18,8,8,12,8,12,6,7,4,18,7,7,7,16,3,5,7,20,5,68,8,16,16,8,11,68,20,4,68,7,16,10,14,12,3,10,28,28,8,28,4,7,1,14,11,16,4,9,7,7,14,6,6,8,5,10,14,8,8,6,9,18,7,18,14,5,12,16,8,9,9,2,8,9,5,6,8,10,7,8,8,7,10,1,5,45,13,45,6,4,4,6,13,7,4,5,4,4,1,5,4,8,6,9,9,7,5,5,1,4,2,0,4,3,12,12,2,0,2,3,2,2,5,16,0,16,2,5,4,4,8,2,0,3,1,4,5,7,7,3,5,1,7,3,7,2,11,13,13,1,1],"Selection_criteria":["No","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Not sure","No","Yes","No","No","No","No","Yes","Not sure","No","Yes","No","Not sure","No","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","Not sure","No","Yes","Yes","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Not sure","No","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Not sure","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Not sure","Not sure","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Not sure","Yes","Yes","Yes","Yes","No","Yes","Not sure","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","No","Not sure","No","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","Not sure","Yes","Not sure","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","Not sure","Yes","No","Not sure","Not sure","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Not sure","No","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","Yes","No","Yes","Not sure","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes"],"Trajectory_inference":[null,null,null,"prediction accuracy of the terminal cell fates (lineages)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"correlation between reference and inferred \"pseudo\" times",null,null,null,"others (please specify in the next question)",null,null,null,null,"prediction accuracy of cell fates",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,"prediction accuracy of cell fates",null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"prediction accuracy of cell fates, quality of cell assignment","prediction accuracy of cell fates, quality of cell assignment","performance under different trajectory types, topological similarity between the reference model and the inferred topology, quality of assigning cells to branch, the correlation between the reference and inferred “pseudo” times",null,"performance under different trajectory types, topological similarity between the reference model and the inferred topology, quality of assigning cells to branch, the correlation between the reference and inferred “pseudo” times, others (please specify in the next question)","correlation between reference and inferred \"pseudo\" times","quality of cell assignment","correlation between reference and inferred \"pseudo\" times","different trajectory types, topological similarity, prediction accuracy of cell fates",null,"different trajectory types, topological similarity, prediction accuracy of cell fates, correlation between reference and inferred \"pseudo\" times","different trajectory types, prediction accuracy of cell fates, quality of cell assignment","different trajectory types, topological similarity, prediction accuracy of cell fates",null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"performance under different trajectory types",null,null],"Trajectory_inference___others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"pseudotemporal trajectories",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Pseudo- time trajectory reconstruction",null,null,null,null,null,"reconstruction of cellular trajectory",null,null,null,null,null,null,null,"constructs the epigenome pattern of human hematopoiesis and characterizes the epigenome heterogeneity within the phenotypic classification group",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"The agreement between trajectory DE features from the known trajectory and the predicted trajectory.",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"Gene_regulatory_network":[null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"recovering interactions, others (please specify in the next question)","performance under multiple network types, others (please specify in the next question)","recovering interactions",null,"others (please specify in the next question)","others (please specify in the next question)","others (please specify in the next question)",null,"performance under multiple network types, recovering activating edges, recovering inhibitory edges, recovering feedforward loops, recovering feedback loops, recovering interactions",null,"others (please specify in the next question)","recovering interactions",null,null,null,null,null,null,null,"others (please specify in the next question)",null,"others (please specify in the next question)","others (please specify in the next question)",null,null,"others (please specify in the next question)",null,"recovering interactions","others (please specify in the next question)",null,"others (please specify in the next question)",null,"others (please specify in the next question)",null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"recovering activating edges, recovering inhibitory edges, recovering interactions",null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null],"Gene_regulatory_network___others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"similarity of network measured by ranking each edge and applying PCA--similarity of network structure measured by distribution of degrees","testing reproducibility",null,null,"stability","Comparison of the number of unique TFs in GRNs that showed significantly enriched biological functions, Comparison of the number of cell-type-specific regulons in GRNs significantly enriched in only one biological function/pathway","comparison between the competing algorithms in terms of average rankings of gold standard TFs for each target gene",null,null,null,"reconstruction of GRN",null,null,null,null,null,null,null,null,"Reconstruction of GRN",null,"reconstructing GRNs","comparison of the inferred GRNs with the simulated GRNs",null,null,"Transcription Factor Binding site prediction",null,null,"- AUPR \n- F-score\n- predictable TFs\n- effect of imputations on inferred networks\n- effect of addition of priors and transcription factor activities",null,"Reconstruction of GRN",null,"recovering TF-gene regulations",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"Doublet_detection":[null,null,null,"performance in datasets with various doublet rate","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"removing cells that are true doublets, removing cells that are not doublets, retaining cells that are true doublets",null,null,null,null,null,"others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"various doublet rate, datasets with different structure, removing cells that are true doublets, removing cells that are not doublets, retaining cells that are true doublets","performance in datasets with various doublet rate, performance in datasets with different structure, removing cells that are true doublets, removing cells that are not doublets, retaining cells that are true doublets","performance in datasets with various doublet rate, performance in datasets with different structure, removing cells that are true doublets, removing cells that are not doublets, retaining cells that are true doublets","various doublet rate, datasets with different structure, removing cells that are true doublets, removing cells that are not doublets, retaining cells that are true doublets",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null],"Doublet_detection___others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"Simulation":[null,null,null,"purpose of each simulation method","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"purpose of each simulation method",null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"purpose of each simulation method",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"purpose of each simulation method",null,null,null,null,"purpose of each simulation method",null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"purpose of each simulation method, method can capture the distributions for first-order statistics, method can capture the distribution for second-order statistics, method can simulate multiple cell types, method can simulate DE expression","purpose of each simulation method, method can capture the distributions for first-order statistics, method can capture the distribution for second-order statistics, method can simulate multiple cell types, method can simulate DE expression","distributions for first-order statistics","distributions for first-order statistics, distribution for second-order statistics","simulation of multiple cell types","distributions for first-order statistics, distribution for second-order statistics","distributions for first-order statistics","others (please specify in the next question)","others (please specify in the next question)","purpose of each simulation method, method can simulate multiple cell types","distributions for first-order statistics, distribution for second-order statistics","purpose of each simulation method, method can capture the distributions for first-order statistics, method can capture the distribution for second-order statistics, method can simulate multiple cell types","distributions for first-order statistics, simulation of DE expression",null,"distributions for first-order statistics, distribution for second-order statistics, simulation of DE expression","distributions for first-order statistics, distribution for second-order statistics","distributions for first-order statistics, distribution for second-order statistics","distributions for first-order statistics, distribution for second-order statistics","distributions for first-order statistics",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"method can capture the distributions for first-order statistics, others (please specify in the next question)",null,null],"Simulation___others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"overall stability of each method",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"The evaluation of structural network control methods",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"simulate a variety of datasets",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"- SNV sensitivity\n- Allele dropout Rate\n- Genome coverage","goodness of fit of model--Akaike Information Criterion--observed versus predicted dropouts per gene--Likelihood Ratio Test",null,null,"performance of batch simulators for batch correction",null,null,null,null,null,"expression of marker genes--correlation between variable genes--classification accuracy between real and generated cells",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"Differential_expression":[null,null,null,"others (please specify in the next question)","others (please specify in the next question)","multiple-group DE comparison",null,null,null,null,null,null,"others (please specify in the next question)",null,"detection of truly differentially expressed genes",null,"detection of truly differentially expressed genes, others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,"detection of true DE genes, others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"detection of true DE genes, covariates","detection of truly differentially expressed genes","detection of truly differentially expressed genes, avoiding false positives under null conditions, method can model multiple-group DE comparison, others (please specify in the next question)",null,"detection of truly differentially expressed genes, avoiding false positives under null conditions","detection of truly differentially expressed genes, avoiding false positives under null conditions, method can deal with covariates, others (please specify in the next question)","detection of truly differentially expressed genes, avoiding false positives under null conditions","detection of truly differentially expressed genes, avoiding false positives under null conditions","detection of true DE genes, covariates, others (please specify in the next question)","detection of truly differentially expressed genes, avoiding false positives under null conditions","detection of true DE genes, null case with no DE genes, covariates","detection of true DE genes","detection of truly differentially expressed genes, avoiding false positives under null conditions","detection of truly differentially expressed genes, avoiding false positives under null conditions, others (please specify in the next question)","detection of truly differentially expressed genes, avoiding false positives under null conditions","detection of truly differentially expressed genes","detection of truly differentially expressed genes, avoiding false positives under null conditions","others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,"detection of true DE genes, multiple-group DE comparison, others (please specify in the next question)",null,null,null,null,null,null,null,null,"detection of true DE genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,"detection of true DE genes",null,null,null,null,null,"detection of true DE genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"multiple-group DE comparison",null,null,null,null,null,null,null,null,"detection of truly differentially expressed genes, avoiding false positives under null conditions",null,null,null,null,null,null,null,null,null,"detection of true DE genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"detection of true DE genes, multiple-group DE comparison",null,"detection of true DE genes, null case with no DE genes, multiple-group DE comparison",null,null,"detection of true DE genes, multiple-group DE comparison",null,null,null,null,"detection of true DE genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,"detection of true DE genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"detection of true DE genes, null case with no DE genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null],"Differential_expression___others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"DE results between Asymmetric and Symmetric expression distribution",null,null,null,null,null,null,null,"detect cluster-specific markers by differential expression analysis between cells within and outside a cluster",null,null,null,null,null,null,null,null,null,null,"selected marker genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"impact of changing the number of cells--similarity between the methods","number of differentially expressed genes called--concordance between gene rankings--false discovery rate (FDR) control and statistical power of the methods","Considered DE, DP, DM and DB--Examined separately for low and highly expressed genes","tested Cell abundance equally distributed among dose groups--tested varying cell numbers in each dose group--tested percent DE genes-- tested proportion of down regulated DE genes--tested fold-change location--tested fold-change scale--tested dropout rate--tested agreement between the methods","FC bias","effect of sample size on detecting DE genes--agreement among the methods",null,null,null,"Effect of sample size","evaluation of DS analysis on DB, DE, DM and DP--impact of the number of cells on performance--impact of the number of samples on performance--impact of imbalanced distribution of cells across samples",null,null,"quality evaluation","Identification of Differential state changes between conditions from multi subject scRNA-seq data",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"The over lap of the LRpairs in the 23 resources.\nI consider DE as LR-pairs",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"- Classification accuracy of phenotypes based on detected genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"check if the interactive visualisation methods deal with differential expression",null,null,null],"Batch_correction":[null,null,null,"others (please specify in the next question)","others (please specify in the next question)","preservation of biology when batch and biology are associated",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,"preservation of biology when biology is not associated with unwanted factors",null,"preservation of biology when batch and biology are associated","others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,"preservation of biology when batch and biology are associated, others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"preservation of biology when library size and biology are associated",null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"method can preserve biology when batch and biology are associated",null,null,null,"method can preserve biology when batch and biology are associated, method can preserve biology when biology is not associated with unwanted factors, others (please specify in the next question)","others (please specify in the next question)",null,null,null,"method can preserve biology when biology is not associated with unwanted factors, others (please specify in the next question)","method can preserve biology when library size and biology are associated, method can preserve biology when batch and biology are associated, method can preserve biology when biology is not associated with unwanted factors",null,null,"when cell types are unknown",null,null,null,null,null,"others (please specify in the next question)",null,null,null,"method can preserve biology when batch and biology are associated, others (please specify in the next question)","method can preserve biology when library size and biology are associated, method can preserve biology when batch and biology are associated, method can preserve biology when biology is not associated with unwanted factors, cases when the cell types are not entirely the same between the batches","when cell types are different between batches, when cell types are unknown",null,null,"method can preserve biology when batch and biology are associated, method can preserve biology when biology is not associated with unwanted factors, cases when the cell types are not entirely the same between the batches","others (please specify in the next question)",null,"preservation of biology when batch and biology are associated, preservation of biology when biology is not associated with unwanted factors, when cell types are different between batches, when cell types are unknown","others (please specify in the next question)",null,null,"method can preserve biology when library size and biology are associated, method can preserve biology when biology is not associated with unwanted factors, cases when the cell types are not entirely the same between the batches, others (please specify in the next question)",null,"preservation of biology when batch and biology are associated, when cell types are different between batches",null,null,null,null,null,null,"preservation of biology when batch and biology are associated, preservation of biology when biology is not associated with unwanted factors, when cell types are different between batches, when cell types are unknown","preservation of biology when library size and biology are associated, preservation of biology when batch and biology are associated, preservation of biology when biology is not associated with unwanted factors, when cell types are different between batches",null,null,null,null,"method can preserve biology when batch and biology are associated, cases when the cell types are not entirely the same between the batches",null,null,"preservation of biology when batch and biology are associated, when cell types are different between batches",null,null,null,null,null,null,null,"preservation of biology when batch and biology are associated, preservation of biology when biology is not associated with unwanted factors, when cell types are different between batches",null,null,null,null,null,"method can preserve biology when library size and biology are associated, method can preserve biology when batch and biology are associated",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null],"Multi_omics_integration__matched_":[null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)","integrating more than two modes, when the cell types are unknown",null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,"method can integrate more than two modes",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,"preservation of biology when library size and biology are associated",null,null,"integrating more than two modes, others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"method can preserve biology when batch and biology are associated",null,null,"others (please specify in the next question)","method can integrate more than two modes",null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"method can preserve biology when batch and biology are associated",null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null],"Clustering":[null,null,null,"others (please specify in the next question)","others (please specify in the next question)","concordance with ground truth",null,null,"concordance with ground truth",null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"concordance with the ground truth",null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,"concordance with ground truth","concordance with ground truth","correct number of cell types, concordance with ground truth","concordance with ground truth",null,null,null,null,null,null,null,null,null,"correct number of cell types, concordance with ground truth","concordance with ground truth",null,null,"concordance with ground truth",null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,"correct number of cell types",null,null,null,"others (please specify in the next question)",null,"others (please specify in the next question)",null,"correctly estimating the true number of cell types, concordance with the ground truth, performance with varying number of cells, others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,null,"concordance with ground truth","correctly estimating the true number of cell types, concordance with the ground truth, cases when there are imbalance cell types, performance with varying number of cells",null,"concordance with ground truth, varying number of cells","correctly estimating the true number of cell types, concordance with the ground truth, others (please specify in the next question)","correct number of cell types, concordance with ground truth, others (please specify in the next question)","concordance with the ground truth, others (please specify in the next question)","concordance with ground truth","concordance with ground truth","concordance with the ground truth, others (please specify in the next question)","concordance with ground truth","correct number of cell types","concordance with ground truth","concordance with ground truth","correctly estimating the true number of cell types, concordance with the ground truth, cases when there are imbalance cell types, performance with varying number of cells","concordance with ground truth","concordance with ground truth","concordance with the ground truth, others (please specify in the next question)","correctly estimating the true number of cell types, concordance with the ground truth, cases when there are imbalance cell types","concordance with ground truth","concordance with ground truth","concordance with the ground truth","concordance with ground truth","concordance with ground truth","correct number of cell types, varying number of cells, stratified performance on difficulties","concordance with the ground truth","correct number of cell types, examine if there is imbalance cell types, varying number of cells, stratified performance on difficulties","concordance with ground truth","concordance with ground truth, others (please specify in the next question)","correctly estimating the true number of cell types, concordance with the ground truth","others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,"others (please specify in the next question)",null,null,null,null,"stratified performance on difficulties, others (please specify in the next question)",null,null,"concordance with ground truth",null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"correctly estimating the true number of cell types, concordance with the ground truth, performance with varying number of cells","concordance with the ground truth",null,null,"concordance with the ground truth",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"correct number of cell types, concordance with ground truth, examine if there is imbalance cell types, varying number of cells",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,"concordance with ground truth",null,"concordance with ground truth, others (please specify in the next question)",null,null,null,null,null,null,"concordance with ground truth","concordance with ground truth, others (please specify in the next question)",null,"performance under easy separation and hard separation",null,null,"concordance with ground truth",null,null,null,null,null,null,null,null,null,null,null,null,"correct number of cell types, concordance with ground truth",null,null,null,null,"correct number of cell types, varying number of cells",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"concordance with ground truth",null,null,null,null,null,null,null,null,null,null,"correct number of cell types, concordance with ground truth",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","concordance with ground truth",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"concordance with ground truth",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"concordance with ground truth",null,null,null,"correct number of cell types, concordance with ground truth",null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null],"Clustering___others":[null,null,null,"stability of clusters--consensus score for each partition",null,null,null,null,null,null,null,null,null,null,"average pairwise ARI",null,null,null,null,null,null,null,null,null,"community detection",null,null,null,"clustering consistency with inferCNV",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"unsupervised cell searching",null,"Performance with varying number of marker genes (gradual removal)",null,null,null,null,null,null,null,null,null,null,null,null,null,"stability of the clustering methods with filtering of cells, genes and use of different aligners--Factors influencing clustering solutions","- ability of each tool to identify the inner structure of data sets\n- ability to find refined sub-clusters of manual labels","The optimal number of clusters can differ from the ”true” one--Inconsistent degree of similarity between methods",null,null,"robustness to clustering parameters--robustness to variation in histology images--robustness to sequencing depth",null,null,null,null,null,null,null,"Comparison of robustness to sequencing depth--Comparison of robustness to variation in histology images",null,null,null,null,null,null,null,"deviation of number of clusters from true one--consistency between methods",null,null,"predicted label’s spatial heterogeneity","Effect of Data Basic Preprocessing on Accuracy--Effect of Parameter Settings on Accuracy","visualization and clustering performance with evaluation metrics",null,null,null,null,null,null,null,null,null,null,null,"classify the DE methods into three groups in power medians",null,null,null,null,null,null,null,null,"NMI--ARI",null,null,null,null,null,null,null,null,"low-dimensional representation of scRNA-seq data",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"NMI--Jaccard index--Purity--ARI",null,null,null,null,null,"cell clustering",null,null,null,null,null,null,null,"subpopulation distance",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"correctness--compactness--robustness",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"accuracy of detecting marker genes--different sparsity of marker genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Visualisation of clustering and ability to highlight cell types in as dimension reduction interactive visualisation",null,null,null],"Classification":[null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,"classification accuracy",null,null,null,null,null,null,null,null,null,null,"classification accuracy",null,null,null,"classification accuracy, easy separation and hard separation, imbalanced cell types","classification accuracy, when testing set contain cell types not in training, easy separation and hard separation, varying number of cells, imbalanced cell types",null,"classification accuracy, when testing set contain cell types not in training, easy separation and hard separation, varying number of cells, imbalanced cell types","classification accuracy, when testing set contain cell types not in training, performance with varying number of cells","classification accuracy, performance under easy separation and hard separation","classification accuracy, when testing set contain cell types not in training, performance under easy separation and hard separation, performance with varying number of cells, others (please specify in the next question)","classification accuracy, when testing set contain cell types not in training, easy separation and hard separation, varying number of cells, imbalanced cell types","classification accuracy, when testing set contain cell types not in training, performance under easy separation and hard separation, performance with varying number of cells","classification accuracy","classification accuracy, performance under easy separation and hard separation, performance with varying number of cells","classification accuracy, when testing set contain cell types not in training, performance with varying number of cells, others (please specify in the next question)","classification accuracy, when testing set contain cell types not in training, cases when there are imbalanced cell types","classification accuracy, varying number of cells, imbalanced cell types","classification accuracy","classification accuracy","classification accuracy","classification accuracy","classification accuracy","classification accuracy, when testing set contain cell types not in training, performance under easy separation and hard separation, performance with varying number of cells","classification accuracy, when testing set contain cell types not in training","classification accuracy, performance with varying number of cells",null,"classification accuracy","classification accuracy, when testing set contain cell types not in training, performance with varying number of cells, others (please specify in the next question)",null,"classification accuracy, others (please specify in the next question)","classification accuracy","classification accuracy, when testing set contain cell types not in training, performance under easy separation and hard separation, performance with varying number of cells","classification accuracy, easy separation and hard separation, varying number of cells","classification accuracy, performance with varying number of cells, cases when there are imbalanced cell types","classification accuracy, easy separation and hard separation","classification accuracy, others (please specify in the next question)","classification accuracy, when testing set contain cell types not in training, easy separation and hard separation, varying number of cells, imbalanced cell types","classification accuracy, when testing set contain cell types not in training, easy separation and hard separation, varying number of cells, imbalanced cell types",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,"classification accuracy",null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"classification accuracy",null,"classification accuracy",null,null,"classification accuracy, others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"classification accuracy, varying number of cells",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"classification accuracy",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,"classification accuracy","varying number of cells","classification accuracy, others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null],"Classification___others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"marker detection",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"whether it allows unknown labels--effect on gene filtering--number of cell types--different proportions of DE genes per cell type","Performance under intra dataset--Performance under inter dataset--Performance under sorted datasets--Performance under low and high--Performance under number of genes-- Performance under low and high number of cells--Performance under different protocols--Performance under with and without alignment",null,"performance across different mislabeling proportions--the consistency of predicted labels among different methods--performance across different downsampling proportions",null,"impact of prior knowledge--impact of selection of marker genes","mislabeling proportions--sequencing depths--number of unique cell types",null,null,null,null,null,null,null,"performance using joint classification--impact of capture efficiency--impact of sequencing depth",null,null,null,null,"Impact of number of cell types--Impact of cell type annotations with real experimentally annotation-Impact of data preprocessing",null,"percentage of unclassified cells--computation time",null,"Rare cell type detection",null,"batch effect",null,"influence of number of genes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"consistent with the annotation using manually label",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Treating deconvolution as the same as estimation of cell types",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"RMSE--PCC--JSD",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"efficiency",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"If interactive visualisation addresses cell-type prediction",null,null,null],"Variability_of_score":["Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","Yes","Yes","Not sure","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No"],"Memory_measured":["No","Yes","No","No","No","No","No","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","Yes","No","No","Yes","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","No","No","No","Yes","Yes","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No"],"Max_number_of_cells":[null,10132,4000,null,3918,6425,null,81173,81173,null,null,null,10132,null,null,null,null,81173,null,3918,null,null,6425,null,274182,4000,274182,null,null,null,null,9000,null,null,null,null,100000,33611,null,null,null,800,800,null,null,null,null,null,null,2000000,null,null,null,null,96404,50000,45469,null,43410,null,45469,43410,90000,null,null,8602,null,null,8602,96404,null,null,29888,1300000,null,68579,45469,null,50000,null,90000,null,68579,null,null,10000,10000,null,40000000,8352,80000,6000,400000,null,25551,null,80000,null,null,10000,43603,null,25551,3514,null,null,8352,5000,null,null,6000,68232,null,28000,3514,null,100361,224,null,100361,224,832,200,null,5096,92,null,5096,832,92,null,200,null,null,null,null,10000,null,1306127,1000000,null,27499,50000,1000000,null,50000,null,null,10000,null,200000,200000,null,27499,1306127,1529,null,null,null,null,null,null,null,3005,1000,5520,null,null,null,null,null,3005,null,null,null,null,null,null,null,null,null,null,null,null,5520,1000,null,null,null,null,20000,null,20000,2000,2000,null,72,72,null,null,10000,null,1000,21086,null,100000,null,1000,null,null,null,null,38000,1529,100000,4000,38000,15000,null,null,null,null,10000,1089769,1000000,62337,26252,null,null,null,1000000,10000,null,1000000,null,26252,10000,59651,5052,null,null,null,null,62337,null,null,null,null,833206,null,null,null,80000,null,null,833206,null,524288,null,1089769,10000,59651,null,null,524288,null,53208,null,53208,833206,null,5052,1169213,1100167,null,10000,null,null,10000,9924,null,9230,null,null,2649,2649,null,null,null,null,1000000,null,1000000,null,null,3600,null,null,null,null,null,null,null,50000,null,200000,2000000,25000,10000,10000,null,null,null,null,null,null,null,null,68000,8000,8000,5000,null,1000000,10000,null,null,null,2600,null,2600,null,null,null,1600,null,null,null,10000,null,22860,5665,247,247,null,100000,null,null,null,null,null,68560,2000000,2000000,1283543,1283543],"Sensitivity_analysis":["Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","Yes","No","No","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","No","Yes","No","No","No","No","Yes","No","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","No","No","No"],"Sensitivity_analysis_detail":["Exploring how the method performs when parameters of simulations are changed (changing sigma and range of radii)",null,null,"The paper compared ARI scores for different methods",null,null,null,null,null,null,null,null,"To compare computational performance, they created subsets of the data of varying sizes\nranging from 8M to 100M for plate-based datasets and 75M to 600M for droplet-based\ndatasets.",null,null,null,"In order to further investigate the effect of imputation on sparse data, we downsampled the Smart-seq2 and SCRB-seq data, which were originally based on 1 million reads/cell, to make them more comparable to the 10X-HGMM data with on average of 60,000 reads/cell.",null,"subsampling the cells",null,null,null,null,null,null,"The kNN-smoothing and DrImpute methods show similar results with different normalization methods.",null,null,null,null,null,"different RNG seeds, varying parameters of the method, downsampling","\"The reconstruction accuracy depends on the polymer volume fraction\"","Subsampling the cells",null,null,null,null,"\" We simulated the dropout effects by randomly flipping a number of the non-zero entries to zeros.\"",null,null,null,null,null,null,"Using repeated simulations and changing specific parameters to examine sensitivity of method to those parameters.",null,null,null,null,null,null,null,null,"perturbed each training dataset in the pancreas data collection by randomly subsampling 80%","downsample genes, simulated datasets generated using different seeds",null,null,null,"CHETAH performing at a similar level of 92% concordance as analyzed by the cross-validation method in the SuperCT study","1. Performance sensitivity to the input features\n2. Performance sensitivity to the number of cells",null,"Seurat and CaSTLe showed lower performances in the class-imbalanced group versus the balanced group, whereas others had nearly equal MCC values between the two groups.",null,null,null,null,null,null,"\"perturbed each training dataset in the pancreas data collection by randomly subsampling 80%\"","they conducted a large-scale benchmarking study to compare the performance of CAME with five classical methods in terms of cell-type assignment and model robustness to insufficiency and inconsistency of sequencing depths.","In terms of classifiers, MLP is the best overall, but SVM with both linear and RBF kernels provides comparable results.","random subsampling of bulk samples","CellFishing.jl and scmap-cell are roughly comparable for cell mapping accuracy across different species","They do downsampling analysis, but what they want is separate the impact of sample size or polling effect.\n\nIn order to distinguish whether the performance gain/loss comes from the increased reference data size or the pooling effect, we also perform down-sampling on the reference data to make a fair comparison.","Subsampling the marker gene set. I'm unsure if marker genes are considered as input data or not, though.","Some methods are clearly overtrained when the number of features increases",null,"The effect of gene filtering on performance stability","We employed the HVG method to evaluate the importance and select the subsets of features, then used the top 100, 300, 500, 1000, 3000 and 5000 genes on all the artificial classification tasks for evaluation.","Bootstrapping test and 10-fold CVs of tools in self-projection",null,"The robustness of each method was assessed by randomly subsampling 10% to 100% of the genes from cell type gene expression signatures and repeating the cell type prediction, and ROC and PR curve analyses for each subsample",null,"The paper evaluated the performance of the benchmarked methods and the proposed scLearn framework on datasets containing cell types that are either very similar or very distinct. The paper also evaluated the ability of these methods to identify rare cell types with less than 10 cells.",null,"\"sampling 10 times per method per setting\"",null,null,"stability of the clustering methods by examining three different features: (i) filtering of cells Figure 5), (ii) filtering of genes (Figure 6 and Supplementary Figure 4), and (iii) use of different aligners (Supplementary Figure 5).","(1) given a fixed sample size, but with different subsampling datasets, for testing; (2) directly given different subsampling sizes, ranging from 5000 cells to 80,000 cells, for testing.","In our study, we evaluated the methods with respect to their sensitivity to random starts.",null,null,"subsample sequencing depth per cell",null,"DEPECHE and FlowSOM tend to group similar clusters into meta-clusters",null,null,"I sampled from Atlas datasets and repeated each sampling 10 times",null,"Under V-measure, all three algorithms scored similarly",null,null,null,null,"To test the robustness of different clustering methods we pursued a sampling strategy in terms cells.","The robustness of the 4–7% range was supported by a simulation experiment in which the reads from the six gold-standard datasets were downsampled by a factor of ten",null,null,"\"the average performance of each method across the five runs on each data set, for the true number of clusters.\"",null,null,null,"\" Each simulation setup has been repeated 5 times choosing 5 different values of the seed.\"","scHiCluster considers local and global information of the contact map. The method achieves good clustering performance on all datasets","pseudobulk methods formed a separate cluster; edgeR- and limmatrend-based methods were close to each other; and DESeq2- and limmavoom-based methods were also close",null,"Batch parameters were used to include sample variation associated with data obtained from 3 individuals in each dose group. A total of 5000 genes were simulated for 4500 cells (500 per dose group) using the same doses as the real dataset. Sensitivity analyses varied each of the following parameters according to values in Supplementary Tables S2 and S3: cell abundance equally distributed among dose groups, varying cell numbers in each dose group, percent DE genes, proportion of downregulated DE genes, fold-change location or scale, and dropout rate.",null,"\"These two fake populations were compared with each other and the procedure was repeated 10 times\"",null,"When simulating : The procedure was repeated 10 times in order to produce 10 independent synthetic replicates.","Simulated datasets were generated varying diverse parameters 10 times",null,"By employing random sampling from the 80 samples, we generated 10 datasets to obtain the statistical characteristics of the results.",null,null,null,null,"repeated the downsampling of the replicates 50 times for both the balanced and imbalanced data sets, generating in total 100 data sets.","All methods performed similarly in ranking DEGs, with the exception of Monocle for Simulated datasets","We focus on the reproducibility of the methods in terms of rediscovery rate (RDR) (Ganna et al., 2014) of top-ranking genes. RDR is defined as the proportion of top-ranking findings detected from a training sample that are replicated in a validation sample",null,null,null,null,null,null,null,"We performed k-means clustering for 50 times to get a stable measurement",null,null,"PaCMAP and UMAP achieve comparable results on datasets with sample size over 10000",null,"The evaluation metrics of t-SNE and SIMLR increased when their hyperparameters increased from 2 to 5, after that ARI and NMI tend to be stable.",null,null,null,null,null,"Data subsamples of sizes ranging from 20 to 200,000 cells",null,null,null,"We repeated the process 20 times and got the final accuracy rate.",null,"methods tested on two independent datasets from the same biological condition","including/excluding drop out for bulk GRN methods on the same data (they were tested on sc data)",null,"Using different data sets under same condition. Also subsets the cells of the same data set.",null,null,"effect of subsampling","Stability across multiple runs. We executed each algorithm ten times on a dataset to ask if the inferred networks changed from one run to another. We represented every result by the corresponding ranked list of edges. For every pair of results, we computed the Spearman’s correlation of the ranked lists. We performed this analysis for curated models and for experimental datasets.","We ran LARS-EN in a stability selection procedure similar to MERLIN",null,"tests with different levels of drop-out (30%, 50%, 70%, 80%, 90%)","comparing measures of association using the same data","subsampling the cells","MMS has a similar ability to identify “dark genes” as Differential Expression Genes (DEG) selection method for a given network construction method and a given dataset","iTALK, ICELLNET, NicheNet and NATMI provide a relatively more consistent number of LR interaction predictions with each other than others",null,"\"A repeated stratified subsampling procedure was introduced to account for the variability in each clustering run).",null,null,"CytoTalk evaluation method gave out a similar result to our DES evaluation method",null,null,null,null,null,"\"A repeated stratified subsampling procedure was introduced to account for the variability in each clustering run\"",null,null,"\"We executed each algorithm ten times on a dataset to ask if the inferred networks changed from one run to another. \"",null,"“To assess reproducibility, we analyzed five scRNA-seq datasets of alpha, beta and delta cells from the human pancreas, for a total of 30 pairwise comparisons, and quantified the degree of reproducibility of network pairs. ”","\"downsampling to 400, 200, 100, 50, 25, and 10 cells\"",null,null,"subsampling genes to be removed","Pagoda2, PLAGE, and Vision gained a slightly better performance in UMI-based dataset than that in nonUMI-based datasets, while AUCell and GSVA outperformed their performances in nonUMI-based datasets compared with in UMI-based datasets.\nThe z-score, ssGSEA, Pagoda2, and GSVA had the moderate correlations across 13 technologies (median r = 0.77, 0.68, 0.61, and 0.5, respectively)","We next generated gold standard up- and down-regulated gene sets at various sizes to benchmark detection sensitivity. To simulate real-world scenario, we added noises to each gene set so that higher noise levels attenuate more of their signals (Method).\n\n",null,"\"To account for stochasticity effects during sampling, we repeat this analysis 25 times for each parameter combination.\"",null,null,null,"SAVER and scImpute are very similar to the unimputed data with high specificity but relatively low sensitivity.\nBoth scImpute and DrImpute are designed specifically to only impute excess zeros.",null,null,null,null,null,"subsampling the cells",null,"simulated an observed count matrix with 1000 genes and 100 cells, and set the dropout.shape parameter","The similar results were also observed by DrImpute and scIGANs. Moreover, the NMI, ARI, Si score and Purity of different methods in different datasets can be found in Sup- plemental Materials (Supplementary Figures S40–S43).","SC3 clustering is repeated for 10 times. Each dot represents an individual SC3 clustering run",null,"\"Ten such expression matrices were independently simulated. Each imputation method was run on each replicate with a range of parameter values\"","AVER, deepImpute, PRIME and bayNorm did achieve a similar distribution of all three types of cells, but more dispersed",null,null,null,null,null,null,null,"\"This procedure is repeated 10 times.\"",null,null,"\"accuracy rates of scJoint, Seurat and Conos using 20%, 50% and 80% of cells from scRNA-seq data as training data. Ten random subsamplings were performed for each setting to generate the variance.\"",null,null,null,null,null,null,null,null,"we divided the simulated spots into subsets where the cell type was present\nor absent and evaluated the RMSE errors for each subset",null,null,"subsampling to a very sparse scenario, as well as testing raw versus normalised input and how that affects method performance.","perturbation?","varying \"sequence depth\"","CIBERSORT and GEDIT produce the lowest absolute errors across all datasets",null,"\"Thus, we repeated the evaluations using subsampled datasets of various sizes\"","Cells in batch were down-sampled to have highly variable library sizes.",null,null,null,null,"Most methods show near-perfect correlations on CD8+ T-cells, B cells, NK, fibroblasts and endothelial cells",null,null,null,null,null,"The paper discusses \"sensitivity\" , but in a sense different from that here.",null,"Subsampling the cells.",null,null,null,null,"\"Ten random subsamplings were performed for each setting to generate the variance.\"",null,"They used different downsamping rates for different sequencing depths. They also used tested 4 different normalisation methods.",null,null,null,null,"section on robustness",null,null,null,"\"Stratified sampling is performed to randomly subset 80% of cells from the datasets, repeated 10 times to examine stability. \"",null,"We assessed the accuracy of each integration method in predicting the spatial distribution of transcripts, including for sparse spatial transcriptomics data that were down-sampled from the original datasets",null,null,"\" we devised an experiment to evaluate the reproducibility of CIG detection in subsampled data\"",null,null,"\"we undertook an evaluation of the reproducibility of CIG detection for rare cell types (Methods). Briefly, for the cell type of interest, we subsampled the cell type to 5% of its original size while keeping the other cell types constant.\"",null,null,null,null,null,null,null,null,"by subsampling the cells",null,"subsampled dataset","We subsampled each dataset 10 times with 95% of the original cells and 95% of the original features.",null,"\"To test the stability of each method, we executed each method on ten different subsamples of the datasets (95% of the cells, 95% of the features), and calculated the average similarity between each pair of models using the same scores used to assess the accuracy of a trajectory\"",null,null,null,null,"Different counting schemes for genes with multiple transcripts and ambiguity which regions are exons and which are introns (might be exon in one isoform but intron in another).",null,"sampling paths with different parameters","by generating multiple data using different seeds",null,"Comparing hESCs to NPCs at day 26 and separately hESCs to differentiated neurons at day 54 revealed that SCENT and MCE could discriminate cells with almost perfect accuracy","100 simulation for each scenario",null,null,"robustness of CCAT",null,"We randomly downsampled two large real scRNA-seq datasets—pbmc-ch and pbmc-2ctrl-dm—into 20 data subsets with 90% droplets and 90% genes.","\"We randomly down- sampled two large real scRNA-seq datasets—pbmc-ch and pbmc-2ctrl-dm—into 20 data subsets with 90% droplets and 90% genes.\"",null,null,null,null,null,null,null,null,"A simulation study was designed to analyze the robustness and sensitivity of FiRE score with respect to the number of differentially expressed genes.",null,null,null,null,null,null,"\"The simulation is repeated 50 times and illustrated in the\"",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"two independent experiments were simulated by randomly choosing cells from the Klein and Islam data sets without replacements and putting them into two groups, Group1 and Group2; then, the percentage of HVGs from Group1 that were rediscovered in Group2 was calculated. This analysis was repeated 30 times for the Klein data set and 100 times for Islam data set.","\"To investigated the rediscovery rate [22] of each method, two in- dependent experiments were simulated by randomly choosing cells from the Klein and Islam data sets without replacements\"",null,null,"Bootstrap resampling 1000 times.",null,null,null,null,"varying numbers of latent dimensions",null,null,null,"An analogous analysis we performed, based on agglomerative clustering instead of k-means, leads to similar concordance between net-SNE and t-SNE"],"Tuning":["Default setting","Default setting","Default setting","Parameter tuning","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Parameter tuning","Default setting","Parameter tuning","Parameter tuning","Default setting","Default setting","Parameter tuning","Parameter tuning","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Parameter tuning","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Parameter tuning","Parameter tuning","Parameter tuning","Default setting","Parameter tuning","Parameter tuning","Default setting","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Parameter tuning","Parameter tuning","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Parameter tuning","Default setting","Parameter tuning","Default setting","Parameter tuning","Default setting","Default setting","Default setting","Default setting","Default setting","Default setting"],"Downstream_analysis_details":["distance between cells","highly variable genes,clustering",null,"clustering,DE method,pathway enrichment",null,"clustering, DE method, NA","NA, NA",null,"NA, NA","identification of marker genes, spatial mapping, NA","highly variable genes,clustering,DE method","pathway enrichment","clustering","NA, NA","highly variable genes,clustering,DE method","clustering, cell type identification, batch effects, imputation, normalisation, NA",null,"clustering","DE method",null,"clustering,cell type identification distance between cells,spatial mapping","cell type identification","NA, NA","clustering, DE method, pathway enrichment, cell type identification, identification of marker genes, batch effects, data integration, normalisation, NA","clustering, DE method, trajectory inference, HVG, identification of marker genes, classification, batch effects, dimensionality reduction, doublet detection, feature selection, normalisation, NA","trajectory inference,clustering","HVG, NA","highly variable genes,copy number variation analysis",null,"cell type identification classification,spatial mapping",null,"clustering,cell type identification classification","NA, NA","DE method, DE method","clustering, cell type identification, NA","pathway enrichment, cell type identification, identification of marker genes, NA",null,null,"clustering, gene regulatory network, NA","clustering,DE method","clustering,DE method","clustering","clustering,cell type identification",null,"clustering","Others","NA, NA","pathway enrichment,cell type identification identification of marker genes,disease states classification,survival analysis,Others,motif enrichment","clustering, cell type identification, distance between cells, NA","cell type identification","NA, NA","NA, NA","Others","NA, NA","cell type identification",null,null,"NA, NA",null,"classification",null,null,null,"classification","classification","classification, NA","NA, NA","clustering,cell type identification","cell type identification, classification, NA","cell type identification","trajectory inference, cell type identification, others, cell type identification, trajectory inference","classification","pathway enrichment,cell type identification identification of marker genes,disease states classification,survival analysis","DE method,cell type identification classification,dimensionality reduction,feature selection","cell type identification",null,null,"NA, NA",null,"NA, NA",null,"NA, NA",null,"others, clustering","NA, NA",null,null,"clustering,cell type identification classification,spatial mapping","NA, NA",null,"NA, NA",null,"NA, NA",null,null,"clustering, NA","clustering, NA","NA, NA","NA, NA",null,"NA, NA","clustering, NA",null,null,"clustering, NA","feature selection, NA",null,null,"NA, NA","NA, NA",null,"NA, NA","NA, NA",null,null,"clustering, NA","DE method, NA","DE method",null,"identification of marker genes, NA",null,null,null,"pathway enrichment","NA, NA","pathway enrichment","NA, NA","clustering, DE method, NA",null,"pathway enrichment",null,"DE method",null,"DE method",null,"clustering, NA","clustering,trajectory inference","clustering","clustering","classification, dimensionality reduction, NA","clustering, cell type identification, NA",null,"NA, NA","NA, NA","clustering","clustering, NA","pathway enrichment","clustering","trajectory inference,clustering",null,"cell type identification",null,null,"clustering",null,"clustering, pathway enrichment, cell type identification, NA",null,null,null,"gene regulatory network, distance between cells, spatial mapping, NA",null,"NA, NA","NA, NA","others, cell cell communication",null,"clustering, imputation, NA","gene regulatory network",null,"clustering,gene regulatory network","NA, NA","NA, NA","clustering, classification, NA","DE method, others, network structural control",null,"clustering, NA","gene regulatory network","distance between cells, NA","gene regulatory network",null,"clustering, dimensionality reduction, NA","gene regulatory network","NA, NA",null,"pathway enrichment, identification of marker genes, gene regulatory network, NA","NA, NA",null,"gene regulatory network","pathway enrichment,gene regulatory network,clustering","gene regulatory network,perturbation prediction","NA, NA",null,"clustering","pathway enrichment",null,null,"clustering,highly variable genes,pathway enrichment",null,null,"NA, NA",null,"NA, NA","clustering,identification of marker genes,spatial mapping","clustering,DE method,trajectory inference,dimensionality reduction","clustering, DE method, trajectory inference, cell type identification, identification of marker genes, NA",null,"DE method,clustering,trajectory inference","NA, NA","DE method","clustering, DE method, trajectory inference, imputation, NA","clustering, imputation, NA","clustering, DE method, trajectory inference, NA","DE method","clustering, imputation, NA","clustering, NA","DE method,clustering,trajectory inference",null,"clustering, trajectory inference, cell type identification, classification, distance between cells, imputation, normalisation, NA","clustering, NA",null,"NA, NA","trajectory inference","cell type identification, identification of marker genes, NA",null,"cell type identification",null,"clustering,DE method",null,null,"NA, NA",null,null,null,"spatial mapping, NA","clustering,batch effects","clustering, NA",null,"trajectory inference,classification",null,"deconvolution, NA","Others","clustering, DE method, gene regulatory network, NA",null,null,"clustering",null,"DE method, batch effects, NA","cell type identification, identification of marker genes, NA","NA, NA","DE method","DE method","NA, NA","identification of marker genes",null,"identification of marker genes, NA","NA, NA","DE method","clustering, cell type identification, batch effects, NA","NA, NA","DE method, NA","cell type identification","classification",null,"NA, NA","NA, NA","NA, NA","NA, NA","NA, NA","clustering, NA","cell type identification, deconvolution, NA","clustering,DE method,highly variable genes,batch effects","trajectory inference,DE method,clustering","NA, NA","NA, NA",null,"NA, NA","clustering,classification,trajectory inference,pathway enrichment","clustering, DE method, trajectory inference, NA","clustering, classification, NA","classification,trajectory inference,pathway enrichment","clustering, NA","NA, NA","clustering, clustering",null,"clustering","DE method,classification","DE method","clustering",null,"NA, NA","clustering, NA",null,"velocity",null,"NA, NA","NA, NA","NA, NA","NA, NA","velocity","DE method, NA","NA, NA","clustering, NA","gene regulatory network",null,"identification of marker genes","clustering, pathway enrichment, NA","cell type identification identification of marker genes","trajectory inference, NA","NA, NA","DE method,highly variable genes,clustering,trajectory inference","DE method,highly variable genes,clustering,trajectory inference","NA, NA",null,null,"NA, NA","clustering, DE method, identification of marker genes, doublet detection, NA","NA, NA","cell type identification, NA","NA, NA",null,"DE method","DE method","others, Gene-gene co-expression network","DE method, others, eQTL","clustering, NA","trajectory inference, gene regulatory network, velocity, NA","Others","others, single nucleotide variation detection","DE method","clustering,DE method,trajectory inference","clustering, NA",null,"DE method, NA","NA, NA","DE method,dimensionality reduction","clustering, DE method, trajectory inference, imputation, NA","clustering, others, rare cell population identification","trajectory inference,classification,gene regulatory network","others, peak calling","pathway enrichment,feature selection","identification of marker genes","identification of marker genes",null,"highly variable genes","clustering","clustering, others, Domain Resolution, Latent Representation","NA, NA","copy number variation analysis",null,"pathway enrichment, others, Phylogenetic tree inference, number of neoantigens estimation",null,"copy number variation analysis","clustering, pathway enrichment, rare population identification",null,null,"NA, NA","clustering, cell type identification, NA"],"Discovery_":["Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","Not sure","No","No","No","No","Yes","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Not sure","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","Not sure","No","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Not sure","No","No","Yes","Yes","No","No","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes"],"Wet_lab_validation":["No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No"],"Recommendation_evidence":["The spicyR package has many advantages compared to other dif- ferential co-localization strategies used in high parameter image ana- lysis.","Overall, Cell Ranger, alevin-fry performed slightly better on droplet-based datasets on average. Although slightly lower correlations were observed between zUMIs and other workflows, and it uniquely considers intron reads by default, we did not observe improved clustering results by applying it.","\"However, consistently satisfactory results were observed for scran, Linnorm, DrImpute and SAVER for normalization and imputation; Seurat for cluster- ing; Monocle2 and Slingshot for trajectory analysis; and MNNs for data integration. \"","We compared ChromSCape to methods specifically designed for single-cell epigenomic datasets(scATAC-seq) for their ability to identify cell identities. Based on a recent scATAC-seq benchmark9, we selected the topperforming methods, namely Cusanovich201812, SnapATAC10, and CisTopic11. We also benchmarked EpiScanpy15, a recent analysis pipeline for various single-cell epigenomic data (scATAC-seq, scDNA methylation, …) developed in Python. We applied hierarchical clustering on the reduced feature space\nobtained by each method and used an ARI metric to evaluate their ability to identify cell phenotypes. ChromSCape with default parameters manages to separate almost perfectly the 4 cell types, with an ARI of 0.998 (Fig. 2a), as Cusanovich2018 and CisTopic (both an ARI of 0.996, Fig. 2b), followed closely by EpiScanpy (ARI of 0.940, Fig. 2b). ChromSCape, EpiScanpy, and SnapATAC were all run on 50 kbp bins, but SnapATAC had noisier clusters\nand a slightly poorer ARI (0.822).","Filtering:Doublet detection and removal is advised and can be performed at little computing cost with software such as scDblFinder or scds\nNormalisation: sctransform offered the best overall performance in terms of the separability of the subpopulations\nFeature selection: Deviance [40] offered the best ranking of genes for feature selection\nImputation: We found especially ALRA (with prior normalization), DrImpute (with prior processing), and DCA to offer the best performances\nPCA: we recommend the irlba-based PCA using weighting of the components, as implemented in Seurat\nClustering: scran-based walktrap clustering provided reasonable estimates, although at greater computational costs","We summarize below 11 best practice recommendations for the\ncommunity based on our analysis.",null,"\"Our benchmarking results highlight SnapATAC, cisTo- pic, and Cusanovich2018 as the top performing scATAC-seq data analysis methods to perform cluster- ing across all datasets and different metrics.\"","\"EpiScanpy was benchmarked against other 11 scATAC-seq methods, and it consistently scores among the top tools on its ability to discriminate cell types.\"",null,null,"ChromSCape","Overall, zUMIs followed by salmon alevin and scPipe provided slightly higher silhouette widths across all normalization methods, while dropSeqPipe, kallisto bustools, and Cell Ranger return relatively lower silhouette widths on the cell line data.\nOverall, kallisto bustools and Optimus rank among the top 2 for performance as assessed by the coefficients from the linear model analysis.\nOverall, Cell Ranger, alevin-fry performed slightly better on droplet-based datasets on average.","\"Destin2 utilizes three statistically rigorous and computationally efficient methods—CPCA, MultiCCA, and WNN—and shows its outperformance and robustness.\"","Cell Ranger shows the highest algorithm complexity and parallelization, whereas scPipe, umis and zUMIs have lower complexity that is suitable for large-scale scRNA-seq integration analysis.\n\nUMI-tools show the highest transcript quantification accuracy on ERCC datasets from three scRNA-seq platforms.\n\nIntegration of expression matrices from different pipelines will introduce confounding factors akin to batch effect.\n\nzUMIs and dropEst have higher sensitivity to detect more genes for single cells, which may also bring unwanted factors.\n\nFor most downstream analysis, Drop-seq-tools, Cell Ranger and UMI-tools show high consistency, whereas umis and zUMIs show inconsistent results compared with the other pipelines.","scp/scpdata and sceptre represent strong foundations that can support computational benchmarking efforts","Due to good performance across DE-settings and its speed (Supplementary Figs. 22 and 27) we would recommend scran with prior clustering as the best choice for normalisation (Fig. 5f).","Despite variation across methods and datasets, SnapATAC, Cusanovich2018, and cisTopic outperform other methods in separating cell populations of different coverages and noise levels in both synthetic and real datasets.",null,"Practical recommendations On the basis of our findings, we can make a number of practical analysis recommendations, also summarized in Additional File 1: Figure S35:",null,null,"Box 1 We summarize below 11 best practice recommendations for the community based on our analysis...",null,null,"The performance of methods varied across different datasets, with no clear winners in all situations.","Cumulus provides the community with a cloud-based, scalable, cost-effective, comprehensive and easy-to-use platform for sc/snRNA-seq research",null,"scRNA-seq data batch effect correction: \nThe Harmony method was applied to the combined data, and two clusters corresponding to the different cell types were clearly apparent, BBKNN batch correction also generated two clusters representing each cell type.\nEstimation of copy number variation and clustering analysis: \nThe CNV analysis and expression-based clustering analysis were highly consistent","We found that JSTA was more accurate at assigning mRNA molecules\nto cells than pciSeq","Cellpose substantially outperformed previous methods at all\nmatching thresholds, when trained and tested on the specialized\ndata.","Overall, we find that Baysor outperformed other methods\nWe demonstrate that Baysor performs well on data acquired using five different protocols, making it a useful general tool for analysis of imaging-based spatial transcriptomics.","\"Benchmarking on artificial data shows that DPDchrom significantly outperforms CMD in reconstruction tasks. It also outperforms the Stevens method in the globule reconstruction at low numbers of contacts per bead\"","\"SnapHiC consistently identified more loops and achieved greater F1 scores than the other methods\"","Comprehensive studies show that SEDIM significantly improves the imputation and clustering performance compared with other benchmark methods","Comprehensive studies show that SEDIM significantly\nimproves the imputation and clustering performance\ncompared with other benchmark methods.",null,null,"scGNN integrates three iterative multi-modal autoencoders and outperforms existing tools for gene imputation and cell clustering on four benchmark scRNA-Seq datasets.","kallisto potentially provides a better entry point to analyze scRNA-seq data, resulting in increased data quality and clustering quality.","In general, we could show that STARsolo is an ideal substitute for Cell Ranger 6 because it is faster but otherwise performs similarly.",null,"\"In summary, based on the datasets used in this study, we conclude that Kallisto’s use of computing resources is much less demanding than STAR when only cDNA sequences are used as the reference; however, such efficiency gain is at the cost of loss of information. The users should make decisions based on their preferences in accuracy vs. computing resources.\"","\"In general, we could show that STARsolo is an ideal substitute for Cell Ranger 6 because it is faster but otherwise performs similarly.\"",null,"These results demonstrate that SODA is a versatile and effec- tive tool to statistically map large data sets of multi-colour molecular assemblies in cellulo with high spatial resolution.\n",null,"scAB shows superior performance over the widely used survival models such as Cox regression model.","Compared with the control methods, Spatial-ID could effectively\nidentify the cell types (Fig. 2c) and achieve better performance (Fig. 2f).\nOn all 12 MOP samples, Spatial-ID achieves the highest mean accuracy\n92.75% (Fig. 2f), and the differences with the control methods are very\nsignificant (Wilcoxon test p-value≪0.001 for all other methods).",null,null,null,null,null,"Recommend scClassify","In summary, we found that methods such as Seurat, SingleR, CP, RPC, and SingleCellNet performed relatively well overall, with Seurat being the best-performing methods in annotating major cell types.","\"Taken together, we recommend the use of the general- purpose SVMrejection classifier (with a linear kernel) since it has a better performance compared to the other classifiers tested across all datasets. \"",null,"\"Across all the experimental scenarios, we found Bridge integration was consistently the best performer in terms of overall accuracy and F1 (macro),\"","In the cancer datasets, CHETAH generally outperforms other methods (Figure ​(Figure3A)3A) in terms of combined true positives (correct assignments) and true negatives (cells correctly left unclassified)","Taken together, we recommend the use of the generalpurpose SVMrejection classifier (with a linear kernel) since it has a better performance compared to the other classifiers tested across all datasets.","Based on the findings in our benchmarking study, we have the following recommendations. If all data are from 10x and multimodal data from the same tissue are available, Bridge integration is likely the best method for label transfer; otherwise, scJoint is the to-go method. For scJoint, the caveat is that it tends to misclassify ATAC-specific cell types to the biologically similar cell types in RNA. If one cares about ATACspecific cell types, a better strategy might be using scGCN or Seurat v3 and another method in two separate rounds. For scGCN or Seurat v3, manual annotations can be performed on cells that have high entropy and low enrichment.","Results showed that Seurat based on random forest, SingleR based on correlation analysis and CaSTLe based on XGBoost performed better than others.","These results show that scPred yields higher accuracy than our baseline models and outperforms predictions based on differentially expressed genes",null,"The results showed that ElasticNet with interactions performed the best for small and medium-sized datasets. The NaiveBayes classifier was found to be another appropriate method for medium-sized datasets. With large datasets, the performance of the XGBoost algorithm was found to be excellent.",null,"We not only validated the extremely high feasibility of ES protocol in scCATCH but also demonstrated the superiority of scCATCH over other methods of identifying marker genes, including Seurat, the cell-based annotation method CellAssign, Garnett, SingleR, scMap, and CHETAH, through three scRNA-seq validation datasets\n","Finally, we discovered that the linear algorithms were better than the non-linear algorithms for classifying the different-sized datasets.","scClassify","they recommended the use of the CAME model for cross-species cell-type assignment and gene module extraction from scRNA-seq data, especially for nonmodel species with poorly annotated genomes and limited known biomarkers.","Based on our results, we make the following main recommendations. First, applying F-test on the reference dataset to select features and using MLP as a classifier is the best performer overall when the reference data is reasonably large (e.g., > 5000 cells)",null,"Our approach outperforms scmap-cell, the state-of-the-art method for cell searching, in both accuracy and throughput using real scRNA-seq data sets","We suggest combining all individuals from available datasets to construct the reference dataset and use multi-layer perceptron (MLP) as the classifier, along with F-test as the feature selection method.","\"Considering overall performance, robustness to incomplete cell type signatures, and computing times, we found that GSVA offers one of the best options to label cell clusters from scRNA-seq datasets.\"","Our results highlight the general-purpose support vector machine (SVM) classifier as the best performer overall.",null,"we found that methods such as Seurat, SingleR, CP, RPC, and SingleCellNet performed relatively well overall, with Seurat being the best-performing methods in annotating major cell types.","Results showed that most classifiers perform well on a variety of datasets with decreased accuracy for complex datasets, while the Linear Support Vector Machine (linear-SVM) and Logistic Regression classifier models have the best overall performance with remarkably fast computation time.","Results showed that Seurat based on random forest, SingleR based on correlation analysis and CaSTLe based on XGBoost performed better than others.",null,"GSVA was the overall top performer and was more robust in cell type signature subsampling simulations,",null,"the paper recommended the use of scLearn for single-cell assignment tasks as it provides a well-generalized performance on different single-cell types and can identify novel cell types that are absent in the reference datasets.",null,"\"Fig.8 Summary of clustering method performance across all major evaluation criteria\"","Compared to the\nconventional watershed method, ClusterMap accurately identified\ncells, more precisely outlined cell boundary and illustrated cell\nmorphology",null,"\"Figure 9. Summary of the performance of each method across all evaluations\"","Lower panel: summary of characteristics of each tool and suggested decision pipeline to choose the right tool","We found substantial differences in the performance, run time and stability between the methods, with SC3 and Seurat showing the most favorable results.",null,null,"irst, in terms of clustering accuracy, Seurat-LVM, SpaGCN and Seurat-LV are overall the most accurate clustering methods.","In clustering performance, MRscHiC achieves the best clusteringp erformance on Flyamer dataset, Collombet dataset, and Lee dataset. Higashi can achieve the best clustering performance on Ramani dataset and 4DN dataset.","we recommended FlowSOM, PhenoGraph, and DEPECHE as preferred tools for the task of capturing inner structure of CyTOF data.",null,null,"While it is hard to pinpoint the factors contributing to the performance difference among different categories of clustering algorithms, we suspect that methods based on community detection and clustering stability may share a similar implementation strategy whereas the implementation strategies for methods based on inter- and intra-cluster similarities are more diverse and hence may have contributed to the wider range of performance.",null,"FlowSOM was consistently competitive, but not always optimal","First, in terms of clustering accuracy, Seurat-LVM, SpaGCN and Seurat-LV are overall the most accurate clustering method; Second, given decreased sequencing depth to 50%, Seurat-based methods are the most robust methods. Third, among methods that require users to specify the number of clusters, SpaGCN, SpaGCN+ and Giotto-H maintain the highest average clustering accuracy when given mis-specified parameter values. Fourth, for clustering methods that take histology images as input (stLearn, SpaGCN+, SpaCell and SpaCell-I), they do not demonstrate obvious improvement when images of better quality are supplied. Fifth, Seurat-based and SpaGCN-based methods have the best computational efficiency, and Seurat, SpaCell and stLearn have the best software support.",null,"the SC3 method outperformed all other ten clustering methods and ranked highly in most of the settings.",null,"This summary suggests that Seurat provides the best clustering solutions for 10x Genomics scRNA-seq data in terms of running time, robustness and accuracy.",null,null,null,"\"SC3 and Seurat, two clustering methods developed specifically for single-cell RNA-seq data, delivered the overall best performance, and were the only ones to properly recover the cell types in the droplet- based data sets.\"",null,"Under our Pareto front methodology, FlowSOM emerged unequivocally as the superior algorithm, whereas evaluations based on single metrics alone prompted only a marginal and non-unanimous preference for FlowSOM",null,"\"In particular, we found that some of the methods performed relatively well across most of the datasets we have considered and with respect to the different choices of the parameter settings (i.e., CIDR, and several combinations of Linnorm, SC3 and sscClust), or often poorly, as, sincell (with many parameter settings) and ascend.\"",null,"or a low depth (depth-10), DESeq2, limmatrend and meta-analysis that used log-normalized data (LogN_FEM) showed a good performance, and covariate modeling still improved the results","ROTS is the most general method among the tested methods, as it is not designed specifically for sequencing data (bulk or single-cell). However, it performed well in all the comparisons in this study.","our findings suggest that scBT and LRT linear fit-for-purpose tests are better suited for the differential expression analysis of dose–response studies and when false positives are of greater concern than false negatives","For a moderate depth (depth-77), single-cell-dedicated methods showed a good performance.\nFor less sparse data (zero rate ≈40%), the situation changed much; many BEC methods considerably improved DE analysis and covariate modeling clearly improved DE analysis for both moderate and low depths.","\"ROTS performed well in all the comparisons and MAST had good statistical properties (false positives, precision and recall) as well.\"","\"Figure 5 | Summary of DE method performance across all major evaluation criteria.\"","\"considering our test design, none tool emerged as the best one. \"","fit-for-purpose tests scBT and LRT linear consistently ranked higher fol- lowed by multiple groups tests such as KW and LRT mul- tiple.","For robustness, we would recommend NEBULA-HL unless the computation time is unrealistic.","\"Among all of the tools, DEsingle and SigEMD, which are designed for the scRNAseq, tend to show a better trade-off between TPRs and precision.\"",null,"cell-level DE methods originating from negative binomial mixed models such as glmmTMB and NEBULA-HL outperformed MAST.cdr and pseudo-bulk DE methods on average.",null,null,"\"Our results suggest that the pseudobulk methods and the mixed models that model subjects as a random effect were superior compared to the naïve single-cell methods that do not model the subjects in any way\"",null,"The results indicate that some widely used methods, such as edgeR and monocle, have worse RDR performances compared to the other methods, especially for the top-ranked genes.","Overall, the scRNA-seq based method BPSC performs well against the other methods, particularly when there is a sufficient number of cells.","Overall, the pseudobulk methods outperformed the mixed models. If the user wants to achieve high specificity and precision at the risk of losing some true positives, we recommend the pseudobulk ROTS with the sum aggregation. If sensitivity is more important than the false-positive results, then we recommend the pseudobulk methods Limma, DESeq2 or edgeR combined with the sum aggregation. We recommend that scRNA-seq analysis pipeline developers should begin to include pseudobulk methods and mixed models in their pipelines.","At the current time, we recommend that authors who which to apply these methods expect to perform parameter tuning to achieve acceptable performance, which is likely to require substantially more compute time than is often reported because many manuscripts report only the compute time to train the final model.\nWe recommend that authors developing these methods refrain from emphasizing comparisons unless methods are equally tuned and/or some sort of blinded design is used to control researcher degrees of freedom.","As a re- sult, several dimensionality reduction methods, most notably FA, can achieve similar or better performance as compared to UMAP.",null,"Section on \"Guidelines for users\"","Finally, some DR methods are substantially more efficient than others and can handle much larger datasets, which is an important practical consideration, particularly for conducting some of the other evaluations that require repeated runs of an algorithm.","At the current time, we recommend that authors who which to apply these methods expect to perform parameter tuning to achieve acceptable performance, which is likely to require substantially more compute time than is often reported","however, if preservation of global structure is a priority, linear methods such as GRP or PCA are better candidates.","we observed that the summarized performance of t-SNE outperformed the performance of other methods","TriMap and ForceAtlas2 have better performance across all datasets by achieving relatively high centroid distance correlation score on most datasets","Hence, intNMF should be prioritized by researchers focusing on clustering samples. ... Our results rather suggest that for the study of factor-level information, such as associations with clinical annotation or survival, MCIA, JIVE, MOFA, and RGGCA are the best algorithms to use. When focusing on the underlying biology of the metagenes, tICA and MCIA should be prioritized.","We found that t-SNE achieved the best overall performance score with the highest accuracy and computing cost.","we observe that intNMF performs best in clustering, while MCIA offers an effective behavior across many contexts","\"We find that principal component analysis(PCA)-based methods like scran and Seurat are competitive with default parameters but do not benefit much from parameter tuning, while more complex models like ZinbWave, DCA, and scVI can reach better performance but after parameter tuning.\"","recommend the use of a small number of low-dimensional components with Monocle3","\"we assert that input data structure is highly variable across single-cell technologies and biological samples (Figure S3D), and we recommend evaluating dimensionality reduction tools in the context of their in- tended application.\"","\"On the basis of its ease of use and results of our benchmarking analyses, we anticipate that UMAP will be a valuable tool that can be rapidly adopted by the single-cell analysis community.\"","We find that UMAP provides the fastest run times, highest reproducibility and the most meaningful organization of cell clusters. The work highlights the use of UMAP for improved visualization and interpretation of single-cell data.","We find that principal component analysis (PCA)-based methods like scran and Seurat are competitive with default parameters but do not benefit much from parameter tuning, while more complex models like ZinbWave, DCA, and scVI can reach better performance but after parameter tuning.\nBoth PCA-based methods (scran and Seurat) are nevertheless outperformed by ZinbWave and both autoencoder-based methods (DCA and scVI), particularly in terms of silhouette. On the other hand, we also found that autoencoder-based methods have more parameters that require careful tuning than ZinbWave and PCA-based methods.","\"Our results suggest that several overlooked dimensional- ity reduction methods may provide separation of cell types than currently popular methods like UMAP or t-SNE\"","Based on all the benchmarking results and our implementation in this work, we propose some user guidelines (Fig. 8). Considering that bioinformatics studies combine multiple tools to construct a user’s specific workflow, the programming language is an important factor in selecting the right PCA implementation.",null,"Recommendation is like *everything sucks*\n\n\" Comparisons demonstrate that most of these assessed network methods are not able to predict network structures from single cell expression data accurately, even if they are specifically developed for single cell methods. Also, single cell methods, which usually depend on more elaborative algorithms, in general have less similarity to each other in the sets of edges detected.\"","GENIE 3 emerged again as the best performing method in all cases.","Out of all the methods, SCODE obtained the highest score in general for the simulated datasets (ranked as the best performer for both Sim1 and Sim2, Figs. 2 and 4). Significantly, we found that SCODE did not perform well when applied to single cell experimental data. This result further emphasizes the data-specific performance for each method, and that the utility of the methods de- pends on the study of interest. In particular, the strong performance of SCODE for simulated data was not sur- prising, considering the similarity between the mechan- ism in which the simulated data was generated using GNW, and the prediction algorithm used in SCODE (i.e., pseudotime is inferred from the static single cell data, and then the algorithm uses ODEs to describe the molecular dynamics based on the inferred time series data).","Our study presents a comprehensive evaluation of cell-cell interaction\ntools for scRNA-seq. CellChat, CellPhoneDB, NicheNet, and ICELLNET show overall\nbetter performance than other tools in terms of consistency with spatial tendency and\nsoftware scalability.","Once taking into account networks with up to 100,000 links, GENIE3 results to be the most reproducible algorithm and, together with GRNBoost2, show higher intersection with ground-truth biological interactions. These results are independent from the single-cell sequencing platform, the cell type annotation system and the number of cells constituting the dataset. Finally, GRNBoost2 and CLR show more reproducible performance once a more stringent thresholding is applied to the networks (1,000–100 links).",null,null,null,"Based on these observations, we make specific recommendations for users seeking to apply these methods.","Based on F-score, Pearson, MERLIN and PIDC had the best median rank of 3, followed by SCENIC (rank of 4) and LEAP (rank of 5.5). Based on AUPR, Pearson, SCENIC, and MERLIN were the top three ranking methods.",null,"\"In conclusion, regarding the accuracy, SCENIC is the best method with the highest AUROC values in most simulation studies. Regarding time complexity, LEAP and NLNET are the best methods that can finish every single analysis in minutes (even for datasets with 3000 genes or 1000 samples). Regarding sensitivity to sparsity, SCOUP is the most stable method. It produces AUROC values that are consistently above 0.5 with very low variability across different sparsity levels.\"","Across a large number of analytical tasks, two measures of proportionality, ρp and φs, consistently per- formed well, reconstructing gene networks with high functional coherence and significant overlap with other cellular interaction networks, yielding reproducible models of cellular organization across datasets generated using distinct experimental protocols and clustering transcriptomic profiles by their cell type of origin with high accuracy","On DREAM3 datasets, SRGS is the best, and then GENIE3 and PIDC; on simulated single-cell data without dropout, GENIE3 is the best and then SRGS and PIDC.","In Table 1, we summarized all recommended methods in different evaluations. On one hand, for sample-specific network construction methods (i.e., the first step of SSC analysis), CSN or SSN are suggested as the preferred methods to try at first due to their competitive results compared to other methods. On the other hand, for the network structural control methods (i.e., the second step of SSC analysis), MDS and NCUA could achieve higher efficiency, F-measure, and AUC scores than other methods on many datasets, although they do not currently use edge direction information for the identification of network driver nodes;","iTALK, ICELLNET and NATMI, have shown good overlap with each other. They also predicted a larger percentage of well-annotated LR pairs.","On the other hand, for the network structural control methods (i.e., the second step of SSC analysis), MDS and NCUA could achieve higher efficiency, F-measure, and AUC scores than other methods on many datasets, although they do not currently use edge direction infor- mation for the identification of network driver nodes","Recommend Pearson correlation as the similarity measure","In all cases, the use of OT distance resulted in improved clustering results.",null,"Our results suggested that CellChat has the best performance in consistencies with spatial information",null,"In conclusion, regarding the accuracy, SCENIC is the best method with the highest AUROC values in most simulation studies. Regarding time complexity, LEAP and NLNET are the best methods that can finish every single analysis in minutes (even for datasets with 3000 genes or 1000 samples).","our evaluation framework is available as a python package, scProximitE, to allow users to evaluate the performance of proximity metrics for their own datasets and structural properties of interest",null,null,"Pearson correlation","scGeneRAI is able to predict the network topologies of individual cells and significantly outperforms the method LIONESS.","the methods PIDC, SCENIC, MERLIN, and Pearson are the top methods","\"Based on these observations, we make specific recommendations for users seeking to apply these methods\"",null,"\"Across a large number of analytical tasks, two measures of proportionality, ρp and φs, consistently per- formed well, reconstructing gene networks with high functional coherence and significant overlap with other cellular interaction networks,\"",null,"Several clustering methods that use alternative metrics\nhave already been shown to perform well for scRNA-seq data.","\"Compared with other tools, Pagoda2 showed the best overall performance. AUCell and PLAGE achieved a relatively better overall performance. \"","In terms of TF activity inference, DoRothEA performed best on the simulated single cells followed by D-AUCell and then metaVIPER.","Pagoda2 yielded the best overall performance with the highest accuracy, scalability, and stability.","Bulk-sample-based methods were not designed to deal with excessive dropouts, which may mislead analysis especially if cell biology underpins different dropout rates. We further show that among single-cell methods, JASMINE and SCSE showed better concordance, and JASMINE outperforms SCSE in down sampling tests.","“ We show that single-cell- based methods are more robust at identifying bona fide up- and down regulated gene signatures from scRNAseq data.”","\"In terms of TF activity inference, DoRothEA performed best on the simulated single cells followed by D-AUCell and then metaVIPER.\"","\"Our recommendation tree for method selection, according to the research question and type of the dataset to be analyzed.\"",null,null,"Of the methods we considered, SAVER was the least likely to generate false or irreproducible\nresults, thus should be favoured over alternatives if imputation is necessary.",null,"Using five dataset-pairs, SpaGE outperformed previously published methods and showed scalability to large datasets. Moreover, SpaGE predicted new spatial gene patterns that are confirmed independently using in situ hybridization data from the Allen Mouse Brain Atlas.",null,"The results demonstrate that scESI has a significant improvement in not only predicting true gene expression profiles but also clustering and visualizing cell type accurately.",null,"Overall, MAGIC, kNN-smoothing, and SAVER were found to outperform the other methods most consistently.","MIST consistently outperformed the other methods across all datasets, with higher PCC and lower RMSE scores during holdout value evaluation.",null,"Therefore, deep learning-based methods are more advan- tageous for learning the features of data than model-based methods. Finally, deep learning algorithms are good at constructing gene-gene and cell-cell relationship networks in scRNA-seq data. With these prior information, the over- all structure of data can be recovered through continuous optimization.",null,"it can be concluded that DCA and DeepImpute significantly outperform other methods for maintaining true expression distribution.","\"If imputation is used, combining SAVER with an effect size threshold is the best option to avoid irreproducible results.\"","MAGIC performed better than the other methods most consistently across all assessments","Generally, scHinter is a desirable choice for imputing missing\nvalues in scRNA-seq data, which not only performs well for various\ndatasets with sufficient samples but also provides high and stable\nperformance for imbalanced data or data with limited samples.","The model-based methods SAVER and SAVER-X were among the top per- formers with UMI-based datasets, and they outperformed no imputation in 3 out of the 4 UMI-based test settings",null,"Overall, MAGIC has the best performance. Of course, many external factors also affect the performance of these methods, such as the design of the experimental scheme, so these methods should be carefully selected.",null,"moGAT achieves the best performance","Our comprehensive\nexperiments demonstrate that SD2 reaches great quantitative\nand visible performance under different resolutions, ST techniques\nand metrics. The comparison with three state-of-the-art methods\nalso shows the superior performance of SD2.","\"scMerge Outperforms Existing Integration Methods\"","table 2","In conclusion, when performing a deconvolution task, we\nadvise users to: (a) keep their input data in linear scale; (b) select\nany of the scaling/normalization approaches described here with\nexception of row scaling, column min-max, column z-score or\nquantile normalization; (c) choose a regression-based bulk\ndeconvolution method (e.g., RLR, CIBERSORT or FARDEEP)\nand also perform the same task in parallel with DWLS, MuSiC or\nSCDC if scRNA-seq data is available; (d) use a stringent marker\nselection strategy that focuses on differences between the first and\nsecond cell types with highest expression values; (e) use a com-\nprehensive reference matrix that include all relevant cell types\npresent in the mixtures.","Recommend scJoint","\"Fig. 5 | Guidelines to choose an integration method.\"","Our results highlight Seurat as the top performer as it correctly merges batches while maintaining the separation of distinct cell populations.","We found that Tangram, gimVI, and SpaGE outperformed other integration methods for predicting the spatial distribution of RNA transcripts, whereas Cell2location, SpatialDWLS, and RCTD are the top-performing methods for the cell type deconvolution of spots","Overall, GNN-based methods should be prioritized by researchers focusing on classification tasks.","DEEPsc consistently performs at or above the level\nof several existing methods across all four biological systems\nstudied herein, including systems for which existing methods\nwere originally developed","Recommendations: Because of the robustness of SVA under different scenarios, we recommend using SVA to adjust for latent batch effects. When users are confident that the group information is highly accurate, cate is also a good candidate for adjusting for latent batch effects. More details about the advantages, and limitations of the methods, along with our recommendations are summarized in Table 4.","We find that Scanorama and scVI perform well, particularly on complex integration tasks. If cell annotations are available, scGen and scANVI outperform most other methods across tasks, and Harmony and LIGER are effective for scATAC-seq data integration on window and peak feature spaces.","\"In conclusion, when performing a deconvolution task, we advise users to\"","Taken together, these analyses suggest spatialDWLS is more accurate\nand computationally efficient than these other methods","In general, deep learning methods showed variable performance: while scANVI, scGen and scVI were top performers, trVAE, DESC and SAUCIE performed poorly.","In terms of integration, we discovered that Scanorama, followed by Harmony and Seurat CCA, led to the most accurate cell-type label annotations on average, but Seurat CCA eliminated the most batch effects on average.","We  found  that  Tangram,  gimVI, and SpaGE  outperformed  other  integration  methods  for  predicting  the  spatial  distribution  of  transcripts,  whereas  Cell2location, SpatialDWLS, and RCTD were superior to other integration methods for cell type deconvolution of spots in histological sections. Our  study  helps  researchers  to  choose  appropriate  tools  and  to  optimize data-analysis workflows to accurately and efficiently integrate  spatial  transcriptomics  data  with  scRNA-seq  data.","While intermediate integration approaches such as Seurat v4, SNF, Grassmann joint embedding, integrated diffusion, and PRECISE facilitate increased performance on inferring biological trajectories, simple integration of spliced and unspliced counts through concatenation, sum, or PRECISE achieves increased trajectory inference correlation scores, perturbation classification accuracy, and disease state classification accuracy across most datasets.","Regarding the deconvolution-based methods, cell2location and RCTD performed best among all probabilistic model-based deconvolution methods. However, the usability of cell2location was worse than that of RCTD.","CIBERSORT and GEDIT most consistently produce accurate results","We found that translating the spatial coordinates using PASTE gives qualitatively better positioning of the neocortical layers on top of each other","\"Compared to other methods, GLUE achieved high level of biology conservation and omics mixing simultaneously (Fig. 2a, each quantified by three separate metrics as shown in Extended Data Fig. 1), and was consistently the best method across all benchmark datasets in terms of overall score \"","Seurat distorts the data when there is severe cell composition imbalance due to dimensional reduction by CCA. In this case, reciprocal PCA instead of CCA is recommended.","Section \"A guideline to select suitable procedures\"","\"Our results highlight Seurat as the top performer as it correctly merges batches while maintaining the separation of distinct cell populations. \"","In integrating multiple batches, where different magnitude of batch effects and cell population imbalance are generally involved, NN-based methods except scMerge are recommended.","Our SMNN method shows clear advantages over three state-of-the-art batch effect correction methods and can better mix cells of the same cell type across batches and more effectively recover cell-type specific features, in both simulations and real datasets.","In summary, our results demonstrate that computational deconvolution is feasible at high accuracy, given well-defined, high-quality signatures.","In practice, we recommend to use the mixed models since the mixed models perform better than the fixed models.","Based on our results, Harmony, LIGER, and Seurat 3 are the recommended methods for batch integration. Due to its significantly shorter runtime, Harmony is recommended as the first method to try, with the other methods as viable alternatives.","\"Table 4\nSummary of evaluated methods.\"","We found that Tangram, gimVI, and SpaGE outperformed other integration methods for predict- ing the spatial distribution of transcripts, whereas Cell2location, SpatialDWLS, and RCTD were superior to other integration meth- ods for cell type deconvolution of spots in histological sections.","\"We confirmed that these new factors were positive markers for at least one neuronal subtype. Benchmarking this analysis, (Supplementary Results), we found that except for MNN Correct, other algorithms failed to maintain smoothness or incorrectly mixed distinct progenitor populations \"","\" Taking all our results together, cms and lisi are the top performing methods for quantifying batch effects.\"","We propose an algorithm harnessing log-normal regression and modelling background, outperforming classical least-squares methods.","\"AIscEA outperforms the competing methods yielding the lowest FOSCTTM score as it strength.\"","Based on our results, Harmony, LIGER, and Seurat 3 are the recommended methods for batch integration. Due to its significantly shorter runtime, Harmony is recommended as the first method to try, with the other methods as viable alternatives.","Here, we present DeepST, an accurate\nand universal deep learning framework to identify\nspatial domains, which performs better than the\nexisting state-of-the-art methods on benchmarking\ndatasets of the human dorsolateral prefrontal cortex.",null,"CIBERSORT generally performing best based on both r and normalised mean absolute error (nmae) values","scJoint","While intermediate integration approaches such as Seurat v4, SNF, Grassmann joint embedding, integrated diffusion, and PRECISE facilitate increased performance on inferring biological trajectories, simple integration of spliced and unspliced counts through concatenation, sum, or PRECISE achieves increased trajectory inference correlation scores, perturbation classification accuracy, and disease state classification accuracy across most datasets.","The major new findings in this study are as follows: (i) cell2loction, RCTD and spatialDWLS are more accurate than other ST deconvolution methods, based on the evaluation of RMSE, PCC and JSD; (ii) cell2location and spatialDWLS are more robust to the variation of sequencing depth than RCTD; (iii) the accuracy of the existing methods tends to decrease\nas the spot size becomes smaller; (iv) most deconvolution methods only perform well when they normalize ST data using the method described in their original publications; and (v) the ensemble learning-based deconvolution method, EnDecon, achieves more accurate deconvolution of the ST data.","Since it can be trained in multiple batches and in loss design, the model can\ndistinguish batch variation from actual biological variation and generate a better coembedding space while mixing batches well","Our results show that partial deconvolution methods, particularly CIBERSORT, outperform complete deconvolution methods on human brain data.",null,"Benchmark results show that SCIDRL outperforms other methods in most cases and exhibits excellent performances in two common situations: (i) effective integration of batch-shared rare cell types and preservation of batch-specific rare cell types; (ii) reliable integration of datasets with different cell compositions","section on guidelines","IBRAP presents the community with an attractive, valid and easy-to-use analytic platform, not only for single sample analysis, but also for multi-sample and cross study integration.","In terms of performance, CARD, Cell2location, and Tangram are the best methods for conducting the cellular deconvolution task.","Among the top performing methods of LIGER, Harmony, and Seurat 3, Harmony was the fastest method, requiring less than 1 h for the big datasets. This makes Harmony a good choice for initial data exploration. If the results are not satisfactory, LIGER is the best alternative. Seurat 3, due to its long runtime, can then be employed if both Harmony and LIGER fail to deliver good results.","scMerge","Although no single tool produces the best results across all types of datasets, CIBERSORT (both modes), DeconRNASeq and GEDIT are able to produce reliable results (average error <0.15, correlation >0.6) across all four datasets.","Methods constructed on the basis of probabilistic models combined with negative binomial or Poisson distributions, such as gimVI, Cell2location, and RCTD, generally perform better at predicting the spatial distribution of transcripts or deconvolving cell types of spots. A deep learning algorithm was also applied in several integration methods, among which Tangram is one of the best-performing methods in predicting spatial distribution of the undetected transcripts.","\"We have compared Portal with several representative deep-learning-based integration methods, showing the advantages of Portal.\"","According to the metrics introduced in this work, RANKCORR is consistently one of most optimal marker selection methods on scRNA-seq data.","Recommend Cepo",null,"Triku outperforms other feature selection\nmethods on benchmarking and artificial datasets, using\nunbiased evaluation metrics such as normalized mutual\ninformation (NMI) or Silhouette.","Cepo","Applied to scRNA-seq of the core and margin of a high-grade glioma, scHPF uncovers marked differences in the abundance of glioma subpopulations across tumor regions and regionally associated expression biases within glioma subpopulations. scHFP revealed an expression signature that was spatially biased toward the glioma-infiltrated margins and associated with inferior survival in glioblastoma.",null,"BandNorm yields leading performances in a time and memory efficient manner for cell-type separation, identification of interacting loci, and recovery of cell-type relationships",null,"for methods not considering spike-in ERCCs, RUVr showed higher Rand index and lower Jaccard index than FPKM, UQ, DESeq, and TMM",null,null,"normalization methods considering ERCC significantly outperforms methods that not.",null,"The tHMM consistently outperformed this approach",null,"For method users, we created an interactive set of guidelines (available at guidelines.dynverse.org), which gives context-specific recommendations for method usage.","\"Table 4. Summary of the evaluated quantification methods. The first group of methods are recommended for use with droplet scRNA-seq data, owing to their good performance on the experimental data sets used in this study. The last group of methods are not recommended for use with droplet scRNA-seq data. \"","\"Fig. 5 | Practical guidelines for method users. \"","PseudoGA turns out to be more robust than other methods. Thus, performance of PseudoGA under PhenoPath and Splat simulation indicates that it maintains its accuracy and robustness in presence of outliers and highly variable genes.","Overall, we found that ImageAEOT outperforms the baselines, demonstrating the quantitative gains of ImageAEOT over applying optimal transport directly to cell measurements or to standard linear features.","We show that psupertime outperforms benchmark classifiers in terms of identifying time-varying genes and provides better individual cell orderings than popular unsupervised pseudotime techniques.","The results showed that scShaper outperformed the principal curves algorithm in terms of all three metrics that were used in the evaluation.","\"Among the counting strategies contrasted in this manuscript, alevin_sep_gtr, kallisto|bus_sep_excl, starsolo and alevin_coll_gtr provided velocity embeddings most in line with expectations in the three real data sets.\"","Benchmarking results on six datasets show that our method can reduce the RNA velocity prediction error by at least 50%, indicating that DeepVelo can provide a more accurate representation of velocity vector fields and compute future cell states with better numerical precision","PAGA is robust and qualitatively outperforms previous lineage reconstruction algorithms","DBCTI outperforms recent tools for cell trajectory inference\n","\"The observation that MCE and SCENT performed best and that both make use of an interaction network suggests that such information is valuable for improving the estimates of differentiation potency. \"","in Bulk RNA-seq, SCENT and MCE currently provide the more robust measures of cell potency","\"Through simulation, it is clear to us that the TVC algorithm is not suitable for DC analysis via scRNA-seq data, either in the sense of computational speed or performance. Comparatively, on both of the above datasets (1000 and 100 genes), SparseDC finished within 15 s and achieved a classification rate no less than 98%.\"",null,"A strength of our approach is that it is applicable to a wide range of datasets with different topologies, including gradual state transitions, thus removing the need for time-consuming iterative subclustering and identifying subtle differences in differential abundance that would otherwise be obscured",null,"scDblFinder and the bcds method (Bais and Kostka 2020) are both based on a boosted classifier trained on artificial\ndoublets, however scDblFinder performs considerably better.","In summary, based on the results of two clustering algorithms, we would recommend DoubletDetection and DoubletFinder as the top two choices for removing spurious cell clusters in cell clustering analysis, and we identified Scrublet and DoubletFinder as the bestperforming algorithms for removing homotypic doublets before the identification of celltype-specific genes.","\"Figure 5. A Graphical Summary of Benchmark Results\"","In summary, we have shown that vaeda is a state-of-the-art\nmethod for computationally annotating doublets in scRNA-seq\ndata, and that it is the top choice for python workflows.","For high-quality simulated scRNA-seq datasets with a high number of reads/cell, RSEM, Kallisto, Salmon, and Sailfish can accurately detect and quantify isoform expression.","“For high-quality simulated scRNA-seq datasets with a high number of reads/cell, RSEM, Kallisto, Salmon, and Sailfish can accurately detect and quantify isoform expression.“","We provide a multithreaded software package with comprehensive input validation, Pheniqs, that is faster and more scalable than existing tools.","In summary, EmptyNN outperformed competing\nmethods based on the accuracy inferred from genetic variation.","In summary, these results indicate superior performance with regard to specificity and sensitivity of CellSIUS compared to other approaches when applied to the complex and heterogeneous data generated here and demonstrate the algorithm’s ability to identify rare populations within major cell types that differ by their metabolic state, cell cycle phase, or migratory state.",null,"GapClust consistently performs better than existing approaches in extensive simulation experiments, including five scenarios of simulated datasets with varying numbers of rare cells, different levels of differential expression between the rare cell type and other cell types by Splatter, 99 subsampled datasets from the 68 k PBMC dataset, ten subsampled datasets from the 293T-Jurkat dataset and 140 subsampled datasets from the 68 k PBMC dataset for doublets identification.",null,"\"Fig. 2: Ranking of methods across key aspects of evaluation criteria.\"","Fig. 2: Ranking of methods across key aspects of evaluation criteria.",null,null,null,null,"\"Overall, POWSC significantly outperforms scDesign and Splat in imitating the real scRNA-seq data in this comparison.\"",null,"\"the negative binomial was most often the best fitting one for both non-UMI (57%) and UMI-methods (66%)\"","Taken together, while a set of performant methods to generate synthetic scRNA-seq data exist",null,"“ZINB-WaVE and muscat were also the most performant among type b and k simulators”",null,"SimSeq, seqgendiff and potentially SPsimSeq are best suited for preserving the feature-feature correlation structure while compcodeR appears to be the better choice if the research task is geared towards classification or clustering studies.","\"We expect scDesign to be an effective bioinformatic tool that assists rational scRNA-seq experiment design and benchmarks competing scRNA–seq computational methods based on specific research goals.\"",null,null,"\"The results from the t-SNE visualization, marker gene correlation, MMD, and classification corroborate that the scGAN generates realistic data from complex distributions, outperforming existing methods for in silico scRNA-seq data generation. \"",null,"The benefits of SPARK-X over SPARK on detecting SE genes can also be observed in other small data sets collected from other technologies.",null,"On high sparse data, the performance of SOMDE was better than other methods.","Our results consistently showed that SpaGCN can identify spatial domains with coherent gene expression and histology, and detect SVGs and meta genes that have much clearer spatial expression patterns and biological interpretations than genes detected by SpatialDE and SPARK","Overall, all methods, except scran, raised issues of concern in the results. In comparison with the other methods, scran has a good performance in clustering, stable number of detected HVGs, good independency from the mean and good running time.","\"Brennecke was shown to have stable good performances in a wide range of data sets.\"","SINFONIA again deciphered the spatial domains better than baseline methods. Additionally,\nSINFONIA provided overall better performance than its two variants and was more robust\nthan the variants.\nAs an ensemble-based method, SINFONIA can provide the overall best performance based on both the global autocorrelation (Moran’s I) and the local spatial autocorrelation (Geary’s C), highlighting the contribution of the ensemble strategy in SINFONIA for SVG identification.","\"SeCNV outperformed other tools on the nines imulated scDNA-seq datasets.\"",null,"\"Fig. 7 The flowchart demonstrating the recommending process for the choices of suitable SNV-calling methods in scRNA-seq\"",null,"Fig7 in the paper shows a decision tree for how to choose method.","SCEVAN is more accurate and faster than state-of-the-art methods.","In summary, SIMLR can determine which cells are similar in a way that generalizes across different single-cell data sets for dimension reduction, clustering and visualization applications.","From our personal experience and partly supported by benchmarking, we currently recommend using cellxgene for publishing and sharing scRNAseq data.","From our personal experience and partly supported by benchmarking, we currently recommend using cellxgene for publishing and sharing scRNAseq data.",null,"net-SNE can accurately position entire new subtypes of cells from previously unseen datasets and can also be used to reduce the runtime of visualizing 1.3 million cells by 36-fold"],"Applicability_evidence":["It is important to acknowledge that spicyR should not be used in isolation to other analysis techniques, with the package being a key step in a broader analysis pipeline for high parameter imaging data. Firstly, spicyR is contingent on single-cell segmentation and cell-type classification (unsupervised or manually gated) being suffi- ciently accurate to elicit biologically relevant results.","Our analysis revealed that kallisto bustools (v0.39.3) was unsuitable for use on CEL-Seq2 data due to its UMI handling strategy, which limits the dynamic range that can be observed in the presence of short UMIs (6bp in the case of these CEL-Seq2 datasets).","\"By evaluating the results from method combinations across different tasks, we observed a number of interesting trends related to the suitability of different preprocessing methods in downstream analysis. For example, we found that although imputation generally improved the results of clustering and trajectory analyses, it can lead to poor mixing of data from dif- ferent batches in data integration analyses \"","ChromSCape accepts multiple samples to allow comparisons of cell populations between and within samples. It can determine cell identities from single-cell histone modification\nprofiles, whatever the technology, as well as scATAC-seq datasets.",null,"In many scenarios, Seurat version 3, Harmony, BBKNN and fastMNN allowed for correct classification of the two cell types. However, when samples containing large frac- tions of biologically distinct cell types were compared, Seurat version 3 overcorrected the batch effect and misclassified the cell types (that is, breast cancer cells and B lymphocytes clustered together), whereas limma and ComBat failed to remove batch effects.","we do not expect a single computational workflow to perform optimally for all SCP\ndatasets, as different workflows may focus on different characteristics in the data such as the pattern of missing values.","\" SnapATAC is the only method able to analyze a large dataset (> 80,000 cells).\"","\"Its flexible data structure is ready to handle other new types of single-cell omic data, such as Hi-C or NOME-seq, as well as multi-omics single-cell data.\"",null,"On the other hand, scPipe, umis and zUMIs demonstrated lower algorithm complexity and good compatibility for different scRNA-seq platforms, suggesting their usability for large-scale scRNA-seq integration analysis when computational resources are limited.",null,"scRNA-seq preprocessing workflows varied in their detection and quantification of lowly expressed genes across datasets.","\"For motif deviation, Destin2, by its default, resorts to chromVAR, which can be computationally intensive and infeasible to handle atlas-scale scATAC-seq data.\"","umis and zUMIs show inconsistent results compared with the other pipelines.",null,null,"SnapATAC is the only method able to analyze a large dataset (> 80,000 cells). Cusanovich2018 is the method that best balances analysis performance and running time.",null,null,null,"In this evaluation, we excluded trendsceek since its speed inhibited its use in large-scale simulation studies.\n\nPAGE, RANK, and RCTD all had high accuracy (Fig. S6A) and were robust to changes in number of transcripts (UMIs) (Fig. S6B). However, RCTD ran much slower (Fig. S6C), making it difficult to analyze large datasets.",null,null,null,"However, consistently satisfactory results were observed for scran, Linnorm, DrImpute and SAVER for normalization and imputation; Seurat for clustering; Monocle2 and Slingshot for trajectory analysis; and MNNs for data integration. Some normalization and imputation method combinations were also found to give good results in most downstream analysis tasks, such as Linnorm and SAVER.",null,null,null,null,null,null,"\"Using DPD and CMD, we can create conformations, not just reconstruct them, whereas the Stevens method has been developed only to reconstruct conformations\"",null,null,null,null,null,"Some limitations can still be found in scGNN. (i) It is prone to achieve better results with large data sets, compared to relatively small data sets (e.g., <1000 cells), as it is designed to learn better representations with many cells from scRNA-Seq data,",null,null,null,null,"\"We generally recommend STARsolo or Alevin-fry for most end-users as an alternative to Cell Ranger because these tools’ performance was very stable over all datasets.\"",null,null,null,null,"the Spatial-ID demonstrates well robustness against the variation of gene dropout, but it is inevitably influenced by the high dimensionality and biological noise associated with SRT sequencing technologies.",null,null,null,null,null,null,"Overall, we recommend using Seurat for general annotation tasks for cell types that are relatively separable and without rare cell type identification as the objective. However, for datasets containing cell types with high similarities or rare cell populations, if a reference dataset with clean annotations is available, SingleR, RPC, and CP are preferable.","\"Fig. 8 Summary of the performance of all classifiers during different experiments. \"",null,"\" If one cares about ATAC- specific cell types, a better strategy might be using scGCN or Seurat v3 and another method in two separate rounds. \"",null,"scPred (which is based on an SVM with a radial kernel), LDA, ACTINN, and singleCellNet performed well on most datasets, yet the computation time is long for large datasets. singleCellNet also becomes slower with a large number of cell populations.","Conos was the most time and memory efficient method but performed the worst in terms of prediction accuracy. scJoint tended to assign cells to similar cell types and performed relatively poorly for complex datasets with deep annotations but performed better for datasets only with major label annotations. The performance of scGCN and Seurat v3 was moderate, but scGCN was the most time-consuming method and had the most similar performance to random classifiers for cell types unique to scATAC-seq.","When the cell number is small or cell types are extremely imbalanced in the reference, SingleR is the best choice, based on the cluster-level similarities.",null,null,"The NB algorithm can also be considered as an appropriate classifier for medium-sized datasets. However, when the sample size of the datasets was large, the XGB algorithm was found to be more suitable","The authors benchmarked their model with several other models and showed that their model outperformed the others with an accuracy of over 90%. They also suggested that their model can be used for other cell type classification methods and can become a reference architecture.",null,"In summary, the investigation of the performance of ten algorithms revealed that XGB is the most practical method for large datasets. Additionally, ensemble-weighted.F1 can also be used as an appropriate method for large datasets.",null,null,"When the reference data is small (< 1000 cells), using a correlation-based method such as scmap is a better strategy.",null,"This makes our method even more useful because it is widely applicable to any scRNA-seq data with no cell annotations",null,"\"CIBERSORT and METANEIGHBOR were also comparable or even superior to GSVA in datasets where the number of cell clusters matched the number of cell types expected.\"",null,null,null,null,null,null,null,null,null,null,null,null,null,"\"While Seurat performed slightly better than the next best methods, in our opinion, the choice of clustering method should be informed by the user’s familiarity with statistical concepts and R programming. Many methods, including Seurat, require the user to make informed parameter choices and occa- sionally troubleshoot code. Methods requiring no parameter choices, like Cell Ranger, may offer a better choice for non-experts.\"","Upper panel: relative advantages of each tool.",null,null,null,null,null,"FlowSOM had the highest precision. An apparent limitation of FlowSOM is that it requires the number of clusters to be predetermined",null,"Some limitations can still be found in SMSC. (i) It is prone to achieve better results with small datasets, compared to relatively large datasets,as it is designed to learn better representations with many genes and not so many cells from scRNA-Seq data, as shown in the benchmark results and (ii) For large-scale datasets (e.g. Chen,more than 14 000 cells), compared with statistics model-based methods, the computing and fusing of multiple kernel matrices framework need more computational resources, which is more time-consuming. In the future, we will investigate creating a more efficient SMSC model with a lighter and more compressed architecture.","Together, these results suggest that methods such as SC3, ACTIONet, and Seurat tend to over-estimate the number of cell types, especially when the number of cells is large in each cell type, and also having higher variability in their estimation, unveiling a limitation of these clustering methods when dealing with datasets with a relatively large number of cells",null,"Under F1-score, ChronoClust generated the best solutions for two datasets and the worst solutions for the other two datasets",null,null,"In such cases, sequencing of up to 1000 cells could not be sufficient for the clustering methods evaluated herein; reaching even higher cell numbers would also pose a major challenge. However, the SC3 method is still far from perfect when applied to relatively small numbers of cells and, also, in cases of high dataset complexity.",null,"However, it should be noted that RCA performed particularly poorly on the gold standard dataset. This highlights that RCA’s performance hinges on the studied cell types being represented in the reference used during the supervised clustering approach.",null,null,null,null,"However, ASTER does not model the batch variation specifically. Since technical variation may be large in some scCAS datasets, we recommend performing batch correction before estimating the number of cell types by ASTER.",null,"in the introduction section, the author mentioned that 'However, BayesSpace has a limited scope of application as it is primarily designed for decomposing the data with low resolution from sequencing-based technologies, such as 10x Visium.'",null,"MRscHiC achieves the best clustering performance on Flyamer dataset, Collombet dataset and Lee dataset. Higashi can achieve the best clustering performance on Ramani dataset and 4DN dataset.","For a moderate depth (depth-77), single-cell-dedicated methods showed a good performance. For example, MAST which used zero-inflation model incorporating the sparsity information and edgeR combined with the observation weights for zero-inflation model (ZINB-WaVE) performed the best.","In addition to statistical properties, other features are also relevant when deciding which method to use [6, 19]. For example, SCDE, DESeq, Limma and ROTS require count data, whereas MAST prefers transcripts per million [20]. DESeq, Limma and ROTS are rather user-friendly and SCDE has a detailed tutorial available (http://hms-dbmi.github.io/scde/tutorials.html).","our findings suggest that scBT and LRT linear fit-for-purpose tests are better suited for the differential expression analysis of dose–response studies and when false positives are of greater concern than false negatives","Thus, we suggested suitable DE workflows under different conditions based on our simulation and real data analysis results",null,"\"many real studies require a more complex experimental design, which not all of the evaluated methods can accommodate.\"",null,null,null,null,null,"Pseudobulk DE methods tend to be too conservative in the results we studied, as observed in their deflated FDP and empirical FPR. Consequently, pseudobulk DE methods may lose power to detect DEGs, and we observe some evidence of that in our real data application.","many real studies require a more complex experimental design, which not all of the evaluated methods can accommodate.",null,null,null,null,null,null,null,"Among the methods that are directly applicable to large-scale data sets, we found that UMAP consistently outperforms the remaining dimensionality reduction methods across a range of low-dimensional components by a large margin.\nIn the case of a small number of low-dimensional components, we found that the performance of both FA and NMF are reasonable and follow right after UMAP.",null,"Section on \"Guidelines for users\" \nAlso figure 8: User guidelines. Recommended PCA implementations categorized based on written language and matrix size. The recommended parameter of each PCA implementation is also described (red)","If the dataset is large, the user should consider algorithms that have better scalability according to Supplementary Table 4. Scalability is important especially when one would like to choose different parameters for investigation and thus needs to run the algorithm repeatedly.",null,"Furthermore, even without correcting for multiple comparisons, there were few significant differences in methods nearby in rank; these rankings should therefore be considered as a general guide and not absolute.",null,null,"Hence, intNMF should be prioritized by researchers focusing on clustering samples. However, intNMF is not effective when assessing the quality of individual factors and metagenes, as observed in the bulk cancer sub-benchmark.",null,"As mentioned earlier, intNMF, representative of the Non-negative Matrix Factorization (NMF) approaches, performs well for the clustering tasks, i.e., for detecting substantial patterns of variation across the omics datasets. This is observed for both simulated bulk data clustering and single-cell data clustering. Hence, intNMF should be prioritized by researchers focusing on clustering samples. However, intNMF is not effective when assessing the quality of individual factors and metagenes, as observed in the bulk cancer sub-benchmark.",null,"Louvain method is recommended as the initial step for lineage inference and a small number of low-dimensional components there is often sufficient for accurate results.","\"we assert that input data structure is highly variable across single-cell technologies and biological samples (Figure S3D), and we recommend evaluating dimensionality reduction tools in the context of their in- tended application.\"","Implies that it is applicable to biologists who can't use command line tools: \"The popular commercial software platform for flow cytometry analysis FlowJo recently released a plug-in to run UMAP.\"",null,null,"\"Methods such as LDA or PHATE, despite their relative obscurity in scRNA-seq anal- ysis pipelines, are promising alternatives to traditionally used or recommended methods of cell type differentiation; however, if preservation of global structure is a priority, linear methods such as GRP or PCA are better candidates.\"","If the data matrix is not too large (e.g., GC ≤107), the data matrix can be loaded as a dense matrix, and full-rank SVD in LAPACK is then accurate and optimal (in-memory and dense matrix). In such a situation, the wrapper functions for the full-rank SVD written in each language are suitable.",null,null,null,"Therefore, in both the DREAM challenge and this study, it is clear that GENIE3 is a suitable approach for the simulated datasets, when bulk sample gene ex- pression data is considered for network reconstruction. In our comparison, however, with induced drop-out in- cluded, the performance of GENIE3 was severely af- fected in ways that were apparent for the Sim2 dataset, where the AUROC score fell below 0.5 (AUROC= 0.425, Fig. 2). This is not surprising, given the fact that methods like GENIE3, although robust for bulk sample gene expression datasets, were not specifically designed for datasets with such high rates of zero values, as in the case of single cell data",null,null,null,null,null,"GRNBoost2 was less sensitive to the presence of dropouts.","Our observations also showed that SCENIC was less susceptible to dataset size change than MERLIN, which would indicate that the non-linear, non-parametric model of Random Forests regression is favorable for modeling scRNA-seq profiles",null,"\"In the next parts of this section, we describe the methods in each category, together with their strengths and weaknesses. We also provide a detailed description and analysis procedure of each method in ...\"",null,null,null,"We observed that the number of iTALK-predicted LR interactions dropped from 108 (the lowest for mouse skin sample) in the mouse skin sample to 44 (mouse cortex sample). At the same time, ICELLNET, NicheNet and NATMI still predicted over 100 LR interactions in the mouse cortex sample.","Of note, when adopting a reference network as the prior-known part of CSN, SSN, SPCC, or LIONESS for obtaining high-confidence sample- specific state transition networks, it is suggested a few better-performing network structural control algorithms should be evaluated for analysis sensitivity.",null,null,null,"SingleCellSignalR can generate relatively robust results, considering the time and memory usage, CellChat and CellPhoneDB are more recommended to estimate CCIs as they will scale well when the cell numbers increased, especially for those consortium projects which usually generate millions of cells.",null,"When the number of genes in the network increases from 20 to 3000, eight methods are not able to finish the analysis: BTR does not converge for datasets with 30 genes or more, SCNS cannot analyze more than 64 genes; Boolean Pseudotime and SINCERA do not converge when analyzing datasets with 500 genes; SCOUP, SCINGE, SINCERITIES and Empirical Bayes crash when analyzing datasets with 500, 1000, 2000 and 3000 genes, respectively. Only six methods are able to analyze all datasets: SCODE, Information Measures, NLNET, SCENIC, LEAP and SCIMITAR",null,null,null,null,null,null,"Fig. 6 | summary of properties of GRN inference algorithms and results obtained from BeeLINe",null,"\"The gene co-dependency index18 accurately predicted disease genes and clustered cell types, particularly across batches, but performed more poorly in gene network analyses. The zero-inflated Kendall correlation22 yielded reproducible and functionally coherent gene coexpression net- works, but performed only moderately well on the network overlap and clustering tasks.\"",null,"Given the complexity of scRNA-seq datasets, it is unlikely for a single metric to\nperform best in all situations.",null,"As one could expect, the single-cell tools D-AUCell metaVIPER performed better on single cells than on the original bulk samples.","AUCell, PLAGEA, and z-score were remarkably affected by normalization with distinct methods","Outputs from these methods can be conveniently tested for association with complex sample characteristics.","“”Bulk-sample-based methods were not designed to deal with excessive dropouts, which may mislead analysis especially if cell biology under- pins different dropout rates.",null,"\"Our recommendation tree for method selection, according to the research question and type of the dataset to be analyzed.\"","we provide clear guidance to select the best\nmethod (Figure 7C) according to the data type and research\nquestion the user has at the start of the BCR reconstruc-\ntion experiment to facilitate the research",null,"Model-based imputation methods typically generated fewer false-positives but this varied\ngreatly depending on the diversity of cell-types in the sample.",null,"method that integrates spatial and scRNA-seq datasets to predict whole-transcriptome expressions in their spatial configuration. Using five dataset-pairs, SpaGE outperformed previously published methods and showed scalability to large datasets","LLSimpute designed for the bulk-RNAseq data performs well in a homogenous cell population, but it fails when the data shows large heterogeneity and sparsity, which are two key character- istics of scRNA-seq data.\nBayesian methods such as BISCUIT, SAVER and scUnif should be applied to the data with relative larger number of cells, which will need longer time. scImpute and DrImpute are required to detect similar cells. Thus, these two methods are more suitable to the case that data in the same subpopulation have large collinearity while data in different subpopulations have small collinear- ity. MAGIC can robustly restore complex non-linear struc- tures, and it may be more robust to data structures. However, it is hard to choose proper parameters.",null,"DeepSpaCE as well as super-resolution and section imputation methods aim to maximize the value of existing datasets and provide foundations for subsequent experiments from at least a single dataset without additional experimental costs. This attribute is an important difference from the recently proposed ST-Net study where trained spatial transcriptome data (not from the Visium platform) was obtained from as many as 23 individuals",null,"Another limitation is that MIST does not allow cross-sample analyses in one run, which might improve the analyses by mutual referencing. Although the current version of MIST is\nperformed at a slide-by-slide level, enabling integrative analyses is of great interest in the future improvement of the algorithm.","1. no on method performs the best in all situations; and 2. the performance of imputation methods depends on the analysis task.","In addition, we found scIGANs performed bet- ter when zero expression rate became lower, accounting for the application of GAN in scRNA-seq analysis.",null,null,"\"Of the methods we tested, SAVER was the least likely to generate false positives, but its performance was variable when tested on real data.\"","scImpute was not the best performer, but it was moderate and consistent","However, MAGIC performed the worst on imbalanced data\n(Table 1) and its performance also dropped dramatically for data with\nsmall number of cells",null,"uture direction of our research. While our work focused on the ST and Visium platforms from 10x Genomics, BayesSpace should be applicable to other platforms in which spots are arranged on a lattice. Slight modifications may be needed so that our spatial model can be used with a different neighborhood structure.","Although some imputation methods applicable to scRNA-seq are effective for scATAC-seq data, they also have some limitations that hinder algorithm performance.","scImpute has good results in small datasets but is unable to impute datasets with more than 10,000 cells.",null,null,null,null,null,null,"\"Fig. 5 | Guidelines to choose an integration method.\"","The second set, Harmony and fastMNN, instead operate on a low-dimensional embedding of the original expression matrices. As such their output cannot be used for downstream analyses which require the expression matrix, limiting their use for some applications.",null,null,null,null,"Fig. 5 | Guidelines to choose an integration method",null,null,null,"For example, SCTransform normalization and Louvain clustering performed well for most pancreas samples, while SCTransform and PAM clustering performed well for most cell line samples.","Another potential application of spatial transcriptomics is to pre- dict ligand–receptor interactions between two cell types that are spa- tially close to each other. Many analytical tools have been developed  for  this  task,  such  as  SpaOTsc21,  Giotto71,  CellChat72,  NicheNet73,  ICELLNET74, and SingleCellSignalR75. However, the vast discrepan- cies  in  the  results  from  different  methods  make  informative  com- parison difficult. For instance, only a small proportion (< 5%) of the  predicted ligand–receptor interactions were shared in the results for  > 3 methods",null,"DSTG was omitted for benchmarking since its deconvoluted result had no annotation information and could not be compared with the ground truth.",null,null,null,null,"Table 1. Summary of performance of batch correction methods in different scenarios","\"Our attempt to identify marker genes from the corrected dataset demonstrates the difficulty of using the merged ex- pression matrix for downstream analyses. As none of the methods considered in our study performed adequately in this benchmark, we highlight this as an area where improve- ments are required.\"","Seurat, scMerge, ZINB-WaVE, limma and ComBat are recommended for gene-based downstream analysis.",null,"Most methods show near-perfect correlations on CD8+ T-cells, B cells, NK, fibroblasts and endothelial cells. Currently, only MCP-counter specifically addresses mDCs.","In real data  analysis, the mixed‐effect models detect biologically meaningful high beta‐cell expressions while CIBERSORTx and the two fixed‐effect models fail.","To improve recovery of DEGs in batch-corrected data, we\nrecommend scMerge for batch correction.","\"Table 4\nSummary of evaluated methods.\"","Methods constructed on the basis of probabilistic models combined with negative binomial or Poisson distributions, such as gimVI, Cell2location, and RCTD, generally perform better at predicting the spatial distribution of transcripts or deconvolving cell types of spots.","\"Hence, Harmony should be applied first before a full analysis pipeline is employed. Downstream analyses, such as clustering, trajectory analysis and visualization, can then use adjusted Harmony embeddings.\"",null,"The least-squares methods NNLS and v-SVR both had genes with high influence on deconvolution results, while DWLS and the log-normal method were not subject to outsize influence from any genes","\"AIscEA assumes the input single-cell data to consist of separable cell clusters and it is not designed to perform an integrative analysis of single-cell trajectories datasets that do not fulfill this.\"","Based on our results, we found LIGER, Harmony, and Seurat 3 to be the top batch mixing methods. Harmony performed well on datasets with common cell types, and also different technologies. The comparatively low runtime of Harmony also makes it suitable for initial data exploration of large datasets. Likewise, LIGER performed well, especially on datasets with non-identical cell types. The main drawback of LIGER is its longer runtime than Harmony, though it is acceptable for its performance. Seurat 3 is also able to handle large datasets, however with 20–50% longer runtime than LIGER. Due to its good batch mixing results with multiple batches, it is also recommended for such scenarios.To improve recovery of DEGs in batch-corrected data, we recommend scMerge for batch correction.",null,null,"We found that CIBERSORT best handled collinearity, and deconvolution of brain cell sub-types",null,null,"They provide practical guidelines for users, combining accuracy, stability and user-friendliness of the methods. This is in Fig 5 in the form of a flowchart.","BindSC was better at mixing batches, but MinNet distinguished the\ncell type variance from batch variance to provide better cell type separation",null,"One major limitation is that CLAIRE needs MNNs as signals to merge batches. However, computing MNNs between every pair of batches is time consuming, which is the major bottleneck of CLAIRE’s computation consumption.","exhibits excellent performances in two common situations: (i) effective integration of batch-shared rare cell types and preservation of batch-specific rare cell types; (ii) reliable integration of datasets with different cell composition",null,"ASAP, a web-based analytical pipeline, sought to integrate a range of analytical methods but is missing a large variety of single-cell-specific methods for single-cell data","Tangram, stereoscope, and DestVI performed steadily with all 36 datasets. However, on these datasets, DSTG and SpiceMix did not perform well",null,null,null,null,null,"Most methods show similar overall performance, however; thus, the speed of the algorithm is the most important consideration for large data sets (and comparing the markers selected by several methods can be fruitful). RANKCORR is fast enough to easily handle the largest data sets and, as such, it is a useful tool to add into computational pipelines when dealing with high throughput scRNA-seq data",null,"Therefore, DrivAER captures complex, non-linear expression patterns commonly observed in scRNA-seq data. An additional benefit of DrivAER is its ability to visualize the gene set–specific data manifold. These visualizations promote discovery of transcriptional regulation that may otherwise be hidden in the summary statistics generated by other methods including gene set enrichment analysis or VISION and PAGODA",null,null,"scHFP effectively models the nuanced features of scRNA-seq data while identifying highly variable gene signatures, unconstrained by predefined structures such as clusters or trajectories.",null,", while scVI-3D exhibits advantages for rare cell types and under high sparsity scenarios.",null,null,"BASiCS showed noticeable improvement over the raw data across spike-in real data sets and simulated sets, performing best among all methods on the mouse embryonic and simulated data sets.\nSCnorm was the most effective method for both the mouse lung data set and the human embryonic data set, in addition to being one of the best methods in terms of visualization.\nAs may be expected, no single method will be the clear choice for every data set, but the strengths and weaknesses of each are useful knowledge when deciding which approach to employ.","we compared the three normalization approaches (Linnorm, SCnorm, scran) that do not require spike-in genes on a real data set consisting of human embryonic stem cells in various stages of the cell cycle, as well as on a 10X Genomics real data set of peripheral blood mononuclear cells (PBMCs).",null,"We demonstrate that the scone methodology is generally applicable to different scRNA-seq protocols and study designs. The modularity of the Bioconductor R software package scone allows researchers to tune and compare a set of default normalizations as well as to include user-defined methods, providing a useful framework for both practitioners and method developers.",null,null,null,null,"\"Fig. 5 | Practical guidelines for method users. \"",null,null,null,null,"\"Fourth, for 3’ tag data such as the 10x Genomics data we have considered in this study, quantifying the spliced and unspliced transcripts (rather than spliced transcripts and introns only) implies that for a large fraction of the reads, it is difficult to resolve whether they stem from the spliced or unspliced target. Thus, this type of reference may be more suitable for full-length scRNA-seq protocols, where reads are sampled across the entire length of the transcript.\"",null,null,null,null,"SCENT/MCE are equally applicable to nondirectional networks such as the PPI networks","\"SparseDC is applicable to many other forms of single-cell data such as single-cell qRT-PCR data, and applicable to bulk-based RNA-seq/microarray data\"","However, DE methods can find DE and DEV genes, and generally cannot detect DV and DM genes whereas DV methods can find DV and DEV genes, and generally cannot detect DE and DM genes.","Although we have addressed several challenges, several qualifiers should be considered when performing differential abundance analysis with Milo.","CCAT exhibited a higher robustness to an increase in the dropout rate compared to CytoTRACE and GCS",null,"Figure 5",null,null,null,null,"The modular architecture of Pheniqs allows for the addition of alternative error models that may be better suited for sequencing platforms with different error characteristics and could be tested similarly.",null,null,null,null,"While RaceID15 and GiniClust16 offer binary predictions, FiRE gives a rareness score to every individual expression profile. We demonstrated how these scores might help the users focus their downstream analyses on a small fraction of the input scRNAseq profiles. A score is particularly helpful since a number of complex techniques such as pseudo-temporal analysis30, shared nearest neighborhood-based topological clustering31 et cetera are applicable only on a few hundreds of cells.","\"Fig. 2: Ranking of methods across key aspects of evaluation criteria.\"","Fig. 2: Ranking of methods across key aspects of evaluation criteria.",null,null,null,null,null,null,null,"most current simulators are rather limited in their applicability.",null,"“As a result, most current simulators are rather limited in their applicability.”",null,null,null,"Besides, scDesign2 can only be used to evaluate cell clustering methods without other functions, which will limit its use.",null,"\" It is worth noting that SUGAR can be used, like here, to generate cells that reflect the original distribution of the data. It was, however, originally designed and optimized to specifically sample cells belonging to regions of the original dataset that have a low density, which is a different task than what is covered by this manuscript. \"","However, isChIP modelling was not suitable to test any genomic variations such as SNPs, indels, DNA re-arrangements or mutations as well as to simulate enhancers and multiple interactions.",null,null,null,null,null,null,null,"\"SeCNV successfully processes large datasets (>50 000 cells) within 4 min, while other tools fail to finish within the time limit, i.e. 120 hours.\"","\"A limitation of these previous methods (inferCNV and HoneyBadger) is that they are not compatible with 3′ and 5′ scRNA-seq methods (10X Genomics, Drop-Seq, InDrop) that are now widely used in the field of single-cell genomics but were instead developed for first-generation scRNA-seq methods, such as Fluidigm and SMART-seq2. In contrast, CopyKAT is compatible with high-throughput scRNA-seq methods that generate sparse data on thousands of cells that are sequenced in parallel and is also compatible with data from first-generation scRNA-seq methods. \"","\"Fig. 7 The flowchart demonstrating the recommending process for the choices of suitable SNV-calling methods in scRNA-seq\"","Don't use this method on other protocols than Smart-seq.\n\"We do not recommend applying DENDRO to tag-based scRNA-seq (e.g 10X, Drop-seq) sequencing design because of two reasons: (1) limited number of variants can be detected with tag-based methods as they only profile a small fraction of the transcript (3-prime or 5-prime end); and (2) the sequencing depth of tag-based methods are critically low (< 0.1X), resulting in unreliable variant calling\"",null,"\"SCEVAN has been evaluated here with different single-cell technology and recently used in a large study integratingmillions of single cells from 538 samples and 309 patients across 29 datasets using the most commonly applied platforms such as 10x Chromium, Smartseq2, GEXSCOPE, inDrop and Drop-Seq.\"",null,"Our results show that each tool has particular advantages and disadvantages and as such a simple ranking cannot be achieved. We looked specifically at the tools suitable for sharing an interactive visualization of results via a web interface (the Web Sharing row in Table 1). Again, in this case there is not one tool that stands out as significantly better than the rest in all categories.","Figure 1 summarizes the benchmarking results. iSEE-loom, SCope, scSVA and loom-viewer all enable efficient integration with the hierarchical data format (HDF5) from which loom and h5ad formats are derived. HDF5 format allows for on-demand loading, i.e. the necessary data is only loaded inRAMwhen it is needed by the application. In this case, at the start the tools only use the coordinates of the cells without loading other input data into memory. scSVA and loom-viewer are the most efficient HDF5-backed tools with SCope being slightly slower.",null,"We found that for all of the datasets net-SNE is able to learn an embedding that closely matches the output of t-SNE"],"Trade_off_evidence":["Thirdly, there are trade-offs to using small versus large radii when applying spicyR. Large radii may provide a better overall summary of cell-type co-localization within an image, but can have decreased sensitivity, particularly if the co-localization occurs strongly only over short distances.",null,"\"By evaluating the results from method combinations across different tasks, we observed a number of interesting trends related to the suitability of different preprocessing methods in downstream analysis. For example, we found that although imputation generally improved the results of clustering and trajectory analyses, it can lead to poor mixing of data from dif- ferent batches in data integration analyses \"",null,"scran-based walktrap clustering provided reasonable estimates, although at greater computational costs",null,"Therefore, strengths and weaknesses of computational pipelines need to be identified and documented.","\"However, in exploring the best combination of modules to build such a method, it is important to consider the balance between clustering performance, running time, and memory requirements. \"","\"Interestingly, compared to epiScanpy, all other top methods assessed have less robust performance across datasets, performing well in some scenarios but dropping in performance in the other datasets.\"",null,null,null,"scPipe assigned systematically more reads to pseudogenes, whereas dropSeqPipe, salmon alevin, alevin-fry, and kallisto bustools assigned many fewer reads to this feature class. In addition, for the workflows that use pseudoalignment or selective alignment, kallisto bustools provided more pseudogenes yielding a limited number of counts; alevin-fry and salmon alevin recovered more biological noise in pseudogenes.",null,null,null,"n contrast, the pseudo-alignment method kallisto24 appears to assign reads precisely, but looses a lot of reads leading to a lower mean expression.",null,null,null,null,null,"CCA/Seurat version 3 had superior mixability for biologically similar samples but overcorrected batch effects and misclassified cells (that is, poor clusterability/cell type classification) when large proportions of distinct cell types were present. However, Seurat version 3 performed well both for clusterability and mixability for datasets when only a small fraction of dissimilar cells (for example, 5–10%) was present",null,"Each of the five methods evaluated showed a trade-off between the two rates, and none was a clear best performer (Fig. 2a). Pegasus’ L/S method is the fastest (Extended Data Fig. 2a), while maintaining a good balance between the two rates (Fig. 2a and Extended Data Fig. 2b–g).","Imputation generally improved the results of clustering and trajectory analyses, it can lead to poor mixing of data from different batches in data integration analyses.\nclustering and trajectory analyses, it can lead to poor mixing of data from different batches in data integration analyses .\nSlingshot and Monocle2 showed robust results according to both metrics and generated meaningful representations of the trajectory, while Slingshot sometimes gave an extra trajectory path (Fig. 4a and Supplementary Fig. 7). In contrast, SLICER places all cells in the correct path but was unable to order them correctly or recover the expected structure induced by the mixture designs.",null,null,null,null,null,"Not all of the downstream analyses require cell segmentation.\nFor instance, region segmentation or tissue cell-type composition\nmay be inferred directly from molecular data","\"DPDchrom demonstrates accuracy of 35% for capture-C data and of 50% for oligopaint data\"",null,null,null,null,null,null,null,null,"we concludethatKallisto’suseof computing resources is much less demanding than STAR when only cDNA sequences are used as the reference; however, such efficiency gain is at the cost of loss of information.","\"In summary, based on the datasets used in this study, we conclude that Kallisto’s use of computing resources is much less demanding than STAR when only cDNA sequences are used as the reference; however, such efficiency gain is at the cost of loss of information. The users should make decisions based on their preferences in accuracy vs. computing resources.\"","\"For very large projects with a high number of samples, pseudo-alignment tools such as Kallisto can be advantageous in terms of runtime and storage efficiency, at the cost of a slight reduction in accuracy.\"",null,null,null,"These results showed that the increase of the sample size in the bulk training set can improve the prognostic performance of scAB model, and that few number of samples may reduce the prediction accuracy.",null,null,null,null,null,null,null,"In summary, we found that methods such as Seurat, SingleR, CP, RPC, and SingleCellNet performed relatively well overall, with Seurat being the best-performing methods in annotating major cell types. Seurat, SingleR, RPC, and CP were more robust against downsampling of features and read depth. However, Seurat had a major drawback at predicting rare cell types, as well as minor issues at differentiating highly similar cell types and coping with increased cell type classes, compared to SingleR and RPC.",null,null,"\"The performance of scGCN and Seurat v3 was moderate, but scGCN was the most time-consuming method \"",null,null,null,"Although scMCA also performed relatively better, it is time-consuming",null,null,"Although the computation time of XGB was slightly longer, its performance was relatively higher than the other methods.",null,null,null,null,null,"MLP can be replaced by SVM with either linear or RBF kernel, which produces comparable results. However, due to the computational burden of training SVM, especially in large datasets or having many cell types, we recommend using MLP.",null,"scmap-cell selects better features, it has limited effects on the performance of CellFishing.jl",null,"\"A caveat of METANEIGHBOR is that in addition to the typical inputs it requires a training phase based on known cell type gene markers to compute an AUC ROC as its prediction\nscores, but known cell type markers are not available for several scRNA-seq datasets.\"","scPred and scmapcell performed better in the intra-dataset setup, and scVI and ACTINN had a better performance in the inter-dataset setup",null,"Similar to other study [24], we also found that the method that incorporates the prior knowledge (e.g., Garnett and SCINA) does not improve the classification performance over other methods without such requirements. This prior knowledge is limited when cell–cell similarity is large.","but it should be noted that the number of cell clusters did influence the complexity of the same dataset even though it doesn’t make much difference (for example, AMB datasets)",null,null,null,null,null,null,"\" Most importantly, we found that better performance on cell clustering does not necessarily imply accuracy in estimating the number of cell types. \"",null,null,null,"semi-supervised tools resulted in better performance in terms of precision and stability than unsupervised approaches.performance of semi-supervised tools was beaten by unsupervised tools with respect to coherence (internal evaluation),",null,null,null,null,"They focus on sparsity as a reason for performance difference. \"Finally, the best model for single-cell Hi-C data analysis is recommended based on the sparsity of the target dataset.\"",null,null,null,"CIDR is reputable for its ultrafast clustering time, but uses the highest amount of peak memory among all parallel methods, highlighting a trade-off between memory usage and computational efficiency for this method.",null,"Phenograph producing notably poor solutions on the Levine_32dim dataset and yet the best performances on the Levine_13dim and Samusik_all datasets.",null,"We have demonstrated that computational cost does not always trade for empirical accuracy and some configurations are just unpractical for specific protocols.","It is also worth mentioning that there is a trade-off among the performance metrics that were used for the evaluation of the clustering algorithms. Although accuracy is one of the most commonly used selection metrics for the best model or method, it may be insufficient in various cases. For instance, the cost of having misclassified actual positives as false negatives can be high, since a small number of rare cells of interest can be annotated falsely.",null,null,null,null,"Although accuracy is one of the most commonly used selection metrics for the best model or method, it may be insufficient in various cases. For instance, the cost of having misclassified actual positives as false negatives can be high, since a small number of rare cells of interest can be annotated falsely.",null,null,null,null,null,"Higashi makes it possible to exploit the underlying correlations between cells to improve the state of bin-level imputation. This method can achieve the highest clustering performance on the dataset with higher sparsity. However, Higashi needs to construct a dataset as a hypergraph, which requires a large amount of memory for computation. The running time of this method is the longest among all evaluated methods.","For a low depth (depth-10), DESeq2, limmatrend and meta-analysis that used log-normalized data (LogN_FEM) showed a good performance, and covariate modeling still improved the results. However, the use of observation weights deteriorated edgeR and DESeq2 from this depth, as the low depth made it difficult for zero-inflation model to discriminate biological zeros from technical zeros","It is based on the application whether a high or low number of detections is more favorable. The conclusions are summarized in Table 4.",null,null,null,null,"\"Some of the scRNA-seq tools (MAST and SCDE) performed best in terms of precision but had a drop in performance in terms of recall. Others (Monocle and D3E) had an average trade-off between precision and recall but did not reach the desired cut-offs for any of these measures\"","While the importance of controlling type I error rates is acknowledged, a balance must be struck with type II error rates. The tradeoff should be determined based on the individual research question being investigated.",null,"\"Methods with higher true positive rates tend to show low precision due to their introducing false positives, whereas methods with high precision show low true positive rates due to identifying few DE genes.\"",null,null,null,"In general, the methods that can\ncapture multimodality (non-parametric methods), per-\nform better than do the model-based methods designed\nfor handling zero counts. However, a model-based method\nthat can model the drop-out events well, can perform bet-\nter in terms of true positive and false positive","\" If the user wants to achieve high specificity and precision at the risk of losing some true positives, we recommend the pseudobulk ROTS with the sum aggregation. \"","Some of the scRNA-seq tools (MAST and SCDE) performed best in terms of precision but had a drop in performance in terms of recall. Others (Monocle and D3E) had an average trade-off between precision and recall but did not reach the desired cut-offs for any of these measures.",null,"But for the lowly expressed genes performance varies substantially; edgeR and monocle are too liberal and have poor control of false positives, while DESeq2 is too conservative and consequently loses sensitivity compared to the other methods.","Overall, the pseudobulk methods outperformed the mixed models. If the user wants to achieve high specificity and precision at the risk of losing some true positives, we recommend the pseudobulk ROTS with the sum aggregation. If sensitivity is more important than the false-positive results, then we recommend the pseudobulk methods Limma, DESeq2 or edgeR combined with the sum aggregation. We recommend that scRNA-seq analysis pipeline developers should begin to include pseudobulk methods and mixed models in their pipelines.",null,"Overall, the computational cost of three methods, ZINB-WaVE, ZIFA, and pCMF, is substantially heavier than that of the remaining methods. Their computation time increases substantially with both increasingly large number of low-dimensional compo- nents and increasingly large number of cells in the data.",null,"\"there is a trade-off between these properties and memory usage\"",null,"Wolpert and Macready [28] reported a No Free Lunch theorem that states that improved performance of an optimizer on one problem is paired with a decrease in performance in some other area.","This benchmark focused on the separation of different cell types, which may be too simplistic for some scRNA-seq applications such as trajectory analysis.",null,"t-SNE performs well on Duo 4Eq but performs poorly on Muraro and Kang","Hence, intNMF should be prioritized by researchers focusing on clustering samples. However, intNMF is not effective when assessing the quality of individual factors and metagenes, as observed in the bulk cancer sub-benchmark.","For example, SIMLR and PCA performed better than UMAP based on accuracy, while SIMLR showed weaker computing cost and PCA showed weaker stability.","As mentioned earlier, intNMF, representative of the Non-negative Matrix Factorization (NMF) approaches, performs well for the clustering tasks, i.e., for detecting substantial patterns of variation across the omics datasets. This is observed for both simulated bulk data clustering and single-cell data clustering. Hence, intNMF should be prioritized by researchers focusing on clustering samples. However, intNMF is not effective when assessing the quality of individual factors and metagenes, as observed in the bulk cancer sub-benchmark.","\"On the other hand, we also found that autoencoder-based methods have more parameters that require careful tuning than ZinbWave and PCA-based methods\"","the performance of pCMF decreases with increasing number of low-dimensional components in UMI data but increases in non-UMI data",null,"UMAP can't append data points but scVis can: \"Although this will also be possible with UMAP, the autoencoder scvis currently offers the advantage of being able to append new data points to an existing embedding.\"","scvis and UMAP both appeared to better preserve large-scale distances. scvis appeared marginally better than UMAP in that regard.","An interesting result of our study is the large drop in performance, for all methods, on\nthe immune cell mix data compared to the cell lines. This shows that good performances\nin the later does not necessarily translate to good performances in the former.",null,null,"However, just as every coin has two sides, the gene pair information that IntPath has provided only includes a subset of the full reference genes, so the low representations learnt for the genes are incomplete. This incompleteness could reduce the power of SCRL for finding marker genes and significant pathways.",null,null,"As above","Most\nstatistical-based tools only focused on the expression of ligands and receptors, so the\nmain time costs are from the iterations in the statistical test step. In this case, they are\nless time-consuming and have less memory usage than those network-based tools (if the\npermutation times in statistical tests are not too many).  In the \"Running time and maximum memory usage\" section","GRNBoost2 and CLR show more reproducible performance once a more stringent thresholding is applied to the networks (1,000–100 links).",null,null,null,null,"SILGGM and knnDREMI which had a relatively lower performance on most datasets with the exception of Tran A2S/FBS Perturb where it ranked highly",null,"\"Therefore, methods in this category are most suitable for the inference of small-size networks.\"","The zero-inflated Kendall correlation22 yielded reproducible and functionally coherent gene coexpression net- works, but performed only moderately well on the network overlap and clustering tasks.","PIDC shows its advantage compared with GENIE3 which is not robust enough from data without dropouts to with dropouts, while SRGS can still demonstrate a competitive performance.",null,"PyMINEr predicted a decent number of LR interaction pairs for our test data, but few of PyMINEr’s predictions are in the C-LRI database",null,null,null,null,"CellChat and CellPhoneDB will generate high-confidence results with scaled computational resources, while SingleCellSignalR showed a good performance but consumes much time and memory",null,null,null,null,null,null,null,null,"\"Since these methods do not require pseudotime- ordered cells, they are immune to any errors in pseudotime computation. As the quality of pseudotime inference improves, SINCERITIES may become a good choice, especially since it is stable across multiple runs and in the presence of dropouts.\"",null,null,"We note some limitations of the method. First, CellOracle visualizes the simulation vector within the existing trajectory space; thus, cell states that do not exist in the input scRNA-seq data cannot be analysed.",null,null,null,"Vision showed a moderate performance than Pagoda2, AUCell, and PLAGE, it is an annotation toolkit that could capture biological variation with or without priori labeling of cells.",null,null,"\"In summary, DoRothEA subsetted to confidence levels A and B performed the best on real scRNA-seq data but at the cost of the TF coverage. The results of the in silico and in vitro benchmark are in agreement. \"",null,null,null,"Model-based methods which only impute low expression values, generated fewer false positives but had minimal improvements to sensitivity",null,null,"MAGIC is designed based on a Markov affinity-based graph, enabling it impute missing values with similar cells. However, it is sensi- tive to the diffusion time with the adaptive kernel.\nExtensive studies highlight that there is no one method performs the best in all situations.",null,null,null,null,null,null,null,null,"\"We found that different imputation methods favour either sensitivity or specificity but that none of them result in an over- all improvement for detecting differential expression \"","SAVER performed well on Forebrain, but failed on GM12878HEK and GM12878HL",null,"While MAGIC was one of the top performers overall (Fig. 6), it performed worse than many other methods when identifying differentially expressed genes in hypothesis testing type settings that take into account cell variability (Fig. 3).",null,null,null,null,null,"\"the trade-off between separating cell types and merging batches\"",null,"The discussion section on Page 10",null,"\"tradeoffs between batch integration and conservation of biological variance.\"",null,null,null,"DEEPsc attains a balance between robustness and\nprecision (Figure 3). This balance is especially important when\ninvestigating the impact of cellular spatial neighborhood on\ncell fate acquisition. It is desired to narrow down the inferred\nspatial neighborhood (precision) and at the same time reduce\nthe sensitivity to noise (robustness).","zinbwave\tUseful for modeling non-UMI based scRNA-seq\tLarge inflated FDR or reduced power in DE analysis with latent batches","Particularly in more complex integration tasks, we observed a tradeoff between batch effect removal and bio-conservation",null,null,null,null,null,null,"Regarding the deconvolution-based methods, cell2location and RCTD performed best among all probabilistic model-based deconvolution methods. However, the usability of cell2location was worse than that of RCTD.","Most tools return results within minutes, but on large datasets runtimes for CIBERSORT can exceed hours or even days.",null,null,null,"Section \"A guideline to select suitable procedures\"\n\"The performance of scVI is robust to cell populational imbalance but sensitive to magnitude of batch effects. When there is a strong batch effect, scVI leads to under- correction\"",null,"The normalization method Scran increased batch mixing (high ASW_batch and scUnifrac_batch scores) at the cost of losing cell group structures slightly (low ASW_group and scUnifrac_group scores), especially when combined with Seurat, limma or ComBat (Figure 3).",null,"xCell and MCP-counter are based on marker genes, CIBERSORT, quanTIseq and EPIC use a signature matrix and TIMER depends on multiple reference samples for each cell type.",null,"We found that each batch-effect removal method\nhas its advantages and limitations, with no clearly superior\nmethod.",null,null,null,"\"except that lisi exhibits less stability toward cell type composition imbalance and\ncms has higher CPU and memory requirements.\"",null,"\"state-of-the-art approaches showed high coverage but poor FOSCTTM score.\"",null,null,null,"CIBERSORT (CIB), a highly cited deconvolution method initially optimised for immune cell-types18; DeconRNASeq (DRS), which implements the non-negative least-squares approach employed by the PsychENCODE consortium15; MuSiC (MUS), which is a single-cell-based deconvolution approach accounting for individual- and cell-specific expression variability in the signature; and dtangle (DTA)","\"Overall, scJoint has the highest median F1 score of silhouette coefficients, achieving a better trade-off between removing the technological variations in modalities and maintaining the cell-type signals \"",null,"They provide figures on the accuracy, stability, and usability of the compared methods. The figure shows that methods do better/worse on different criteria.",null,null,"contrastive-learning-based methods exhibit noticeable ad hoc trade-off between batch mixing and preservation of cellular heterogeneities (mix-heterogeneity trade-off)","SCIDRL also has limitations. Since the design of SCIDRL to achieve disentanglement is based on classifier and discriminator, it can remove batch effect effectively for most cases. However, the classifier may not always perform well for highly unbalanced data (dataset with unbalanced number of cells of different batches), which can be solved by up-samplings or down-samplings. Another limitation is the running time for large datasets using a single GPU machine, which is a common issue for deep learning models",null,"In contrast, fully supervised (reference-based) methods such as singleR are more capable of identifying these minor populations together with other cellular compositions even with low cell numbers. Our results further highlighted the advantages of reference-based cell annotation; however, such analysis heavily relies on the resolution and bandwidth of cell types annotated within the reference.",null,null,"\"scMerge manages the trade-off between separating cell types and merging batches well (Fig. 3A) across multiple data collections in comparison with other methods.\"",null,null,null,null,null,null,null,null,null,"For the foreseeable future, there may be a trade-off between the flexibility of these models and the difficulty in training them and interpreting their output.","while scVI-3D exhibits advantages for rare cell types and under high sparsity scenarios.","Taken together, these observations suggest that there is no single normalization method that uniformly outperforms the others and that “scone” is able to identify appropriate normalization procedures in a data-dependent fashion.","Considering ERCC, such as in the models of RUVg and GRM, significantly improved performance. Spike-in ERCCs would reduce the sequencing depth of mRNAs of interest, and there is also a concern about whether the synthesized ERCC molecules behave the same as mRNAs from the cel","Linnorm and scran were extremely quick to execute for their quality of normalization, though neither was exclusively the top method for any data set.","BASiCS showed noticeable improvement over the raw data across spike-in real data sets and simulated sets, performing best among all methods on the mouse embryonic and simulated data sets. However, it was not very effective on the mouse lung data set, and had a longer execution time than many other methods.",null,"Through graphical summaries and quantitative reports, scone summarizes trade-offs and ranks large numbers of normalization methods by panel performance.","While we have identified states that represent phenotypically distinct subpopulations of cells, we currently cannot comment on the molecular factors leading to these phenotypes.",null,null,null,"\"‘good-yet-not-the- best’ methods46 can still provide a very valuable contribution to the field, especially if they make use of novel algorithms, return a more scalable solution or provide a unique insight in specific use cases.\"",null,null,null,null,"\"Second, not considering the strandedness of the reads from 10x Genomics (here exemplified by the kallisto|bustools variants, dropest, and by explicitly running alevin in unstranded mode) implies that many reads in regions where genes on different strands overlap each other (exonically or intronically) are considered ambiguous. Depending on the method, these reads may consequently be excluded from the quantifications.\"",null,null,"DBCTI may lose some precision when handling more complicated trajectories. As demonstrated in Figure 4, a relatively complicated bifurcating pattern for instance, has a lower similarity score comparing to the simpler topologies.",null,"MCE and SCENT showed the expected gradual decrease of potency with differentiation time point, StemID exhibited a less significant pattern, while SLICE did not exhibit a decrease",null,null,"Although we have addressed several challenges, several qualifiers should be considered when performing differential abundance analysis with Milo.",null,"Although the doublet scores given by scDblFinder can be directly\ninterpreted as probabilities, simplifying their interpretation, the method also includes a trade-off thresholding procedure\nincorporating doublet rate expectations with classification optimization, thereby facilitating its usage","In summary, DoubletFinder was the best method in terms of accuracy, yet, its computational efficiency and stability were not among the best.","\"In summary, DoubletFinder was the best method in terms of accuracy, yet, its computational efficiency and stabil- ity were not among the best\"","Such metrics are reported in Table 2, and we see that vaeda outperforms doubletFinder and performs comparable so scDblFinder in\nterms of mcc, precision, and accuracy; in terms of f1 score and recall\nscDblFinder performs a bit better than vaeda. Therefore, we conclude that vaeda performs comparable to other state-of-the-art\nmethods (scDblFinder and doubletFinder), but note that\nscDblFinder has slightly better average performance in terms of f1\nscore and recall.",null,null,null,null,null,null,"n principle, GapClust does not apply to the occasional situation where only the rare cell-type-specific genes distinguish the rare cell population from the abundant cells, as these features cannot be captured by vst feature selection in Seurat due to low variance.",null,"\"Finally, we found there exists various trade-offs between the four aspects of criteria and having a well-rounded approach could be more important than a framework that performs best on one aspect but limiting in the other aspects.\"","However, some top performing methods, such as SPsimSeq and ZINB-WaVE revealed poor scalability (Fig. 2c). This highlights the potential trade-off between computational efficiency and complexity of modelling framework",null,null,null,null,null,null,null,"simulators that better mimic reference datasets do not necessarily yield more similar method comparison results",null,"“For example, muscat was among the highest ranked simulators in our study, but integration and clustering method ranking obtained from muscat simulations were rather inconsistent with those from real data. ”",null,null,"\"These results demonstrate the trade-off between the cell number and the cell-wise library size in scRNA-seq experiments. \"","scDesign2 presented competitive performance in comparison to SimCH based on its flexible model-selection mechanism, which could dramatically improve the fitting results by selecting an optimum model from a set of models for each gene. However, its higher performance in the eight metrics we specified sacrifices its running time",null,"\" While Splatter models some marginal distribution of the read counts well (Supplementary Fig. 5), it struggles to learn the joint distribution of these counts\"",null,null,null,null,null,null,null,null,"\"The methods based on objective function are more suitable for joint segmentation since the information from multiple samples can be collected in one function. However, choosing suitable objective functions and efficient optimization algorithms are challenging.\"",null,"\"Generally, we observed the precision-recall trade-offs. In brief, adjusting parameters were important for SNV-calling tools to achieve best performance, and users should choose the most suitable parameters according to the preference of sensitivities or specificities.\"","\"\"As expected, the numbers of cells, the depth of sequencing, the actual number of subclonal mutations, and the phylogenetic tree structure all influence the power of DENDRO.\"\"",null,null,null,null,"For four tools (iSEE-SCE, Single Cell Explorer, UCSC Cell Browser and cellxgene) the preprocessing memory and preprocessing time grow exponentially with the number of input\ncells. Among these four tools cellxgene has the shortest preprocessing times. Cellxgene performs well in terms of both memory and preprocessing times (Figure 1) and leads by the UX scores (Table 2). It also has a thriving community, is the most supported (Table 1) with 24 contributors and has the highest developer activity in the last 2 months.",null,null],"Future_evidence_":["There may be further scope for producing biologically relevant ground truths with which spicyR can be tested against, as well as more complicated simulation studies to elicit the effective- ness of the package.","Extending our benchmarking to cover additional protocols, new preprocessing methods, and other data analysis tasks is left as future work",null,null,"With respect to the methods themselves, we believe there is still space for improvement in some of the steps. Concerning cell filtering, we noticed that the current approach based on whole-population characteristic (e.g., MAD cut-off) can be biased against certain subpopulations, suggesting that more refined methods expecting multimodal distributions should be used rather than relying on whole-population characteristics.",null,"In this section, we provide several recommendations that we hope will help practitioners to improve and facilitate future SCP data analyses.",null,null,"so we are also working to add an automatic alignment method to mapping ST data with histopathological images for other platforms such as Seq-Scope and Stereo-seq. Collectively, it may be possible to extend TIST to realize accurate single-cell ST analysis given that TIST provides an excellent multimodal feature extraction and data fusion method.",null,null,"Extending our benchmarking to cover additional protocols, new preprocessing methods, and other data analysis tasks is left as future work.\nFurther work could explore the performance of preprocessing workflows on datasets that include a more diverse range of cell types.\nWe did not delve further into the individual steps within a workflow, which include alignment, deduplication","\"For CCA, its variants and extensions, such as sparse CCA (Witten et al., 2009) and decomposition-based CCA, can potentially further boost performance.\"",null,null,null,null,null,"Although most top-ranking methods in our comparison performed well on both Smart-seq and 10x datasets that we tested, future benchmarking efforts should strive to include less represented technologies.","Relationships and cellular crosstalk between CD68+ cells and other cells, including phospho-S6+/vimentin+ stromal cells (PC 6) and E-cadherin+/phospho-S6+/Twist+ tumor cells (PC 22), were identified, suggestive of distinct tumor microenvironments for future study.",null,null,"In the future, we will continue to pay efforts to improve TISCH. We will maintain the web resources regularly to integrate new datasets. We will also provide novel functions in TISCH, such as inferring gene–gene co-expression and cell–cell interactions based on expression correlations at the single-cell level. As the increasing numbers of public TME scRNA-seq data are available, we anticipate continued development and maintenance of the TISCH web resource will benefit the broader cancer research community.",null,"Our benchmarking platform will benefit future package developers as it allows new methods to be evaluated on the same standards, avoiding ambiguity caused by cherry-picking evaluation datasets. We hope that this study will reinvigorate interest in the important area of benchmark data generation and analysis, providing new insights into current best practice and guiding the development of better scRNA-seq algorithms in the future to ensure the biological insights derived from single cell technology stand the test of time.",null,null,null,"JSTA currently does not split or merge cells, but this postprocessing\nstep could be added to further improve segmentation","Other extensions of\nCellpose are possible, for example to cell tracking40, which could be\naddressed by adding a ‘temporal gradient’ dimension to the spatial\ngradient dimensions. Finally, further algorithmic improvements\ncould be targeted toward low convexity cells, which were the least\nwell segmented by all algorithms (Fig. 5d).","Although the Baysor algorithm performed well on most of the\npublished protocols, a number of potential improvements could be\nintroduced, such as improving modeling of cell shapes28. Further\nimprovements could be gained by extending the hierarchical\nBayesian model to introduce cell-type-specific shape and composition\ncharacteristics, ideally incorporating explicit models of\ncell-type-specific transcript compartmentalization structure.",null,null,null,null,"We also plan to further improve performance through distributed computing and batch training implementations.","Third, DR-SC essentially performs unsupervised clustering, but with the availability of labels for some cells/spots, it would be interesting to perform semisupervised clustering of those data. We will investigate these issues in future work.","In the future, we will investigate creating a more efficient scGNN model with a lighter and more compressed architecture.",null,"Future mapping tools might, e.g., consider the likelihood of a gene to be expressed in a certain cell type.",null,null,null,null,null,null,"further improvements could be done for linking single cells and bulk samples and for reducing the redundant fine-grain cell subsets with similar cancer hallmarks",null,null,"It suggests that the proposed methods, scMDR and scMultiR, can be further improved by incorporating additional information, such as cell morphology and spatial information. The paper also mentions that the proposed methods can be extended to other omics data types, such as single-cell proteomics and metabolomics data.","scMAGIC can be further improved by incorporating additional features such as cell morphology, spatial information, and prior knowledge of cell type-specific gene regulatory networks. Moreover, the method can be extended to other omics data such as single-cell ATAC-seq and single-cell proteomics data. These extensions can potentially improve the accuracy of scMAGIC in annotating single cells using scRNA-seq data.",null,"The authors suggest that future work can focus on improving the scGCN model by incorporating more advanced graph neural network architectures and exploring the use of transfer learning techniques. They also suggest that scGCN can be extended to other types of omics data beyond single-cell RNA sequencing, such as single-cell epigenomics and proteomics.",null,null,"\"The inter-dataset comparison requires extensive development in order to deal with technical differences between protocols, batches, and labs, as well as proper matching between different cell population annotations. \"","in future studies, they plan to explore cellular and gene expression data about the interconnections between different biological tissues to increase multiple source information for cellular analysis.",null,"Although the full range of single cell genome-wide approaches can be expected to increase further in the near future, the need for methods such as CHETAH that improve the ease and precision of the analysis of the resulting data is evident.",null,null,"In the future, feature selection and enhancement of novel cell prediction are worth further exploration to improve the accuracy and functionality of single-cell classification tools.",null,null,null,"The authors suggested that their ChrNet model can be further improved by incorporating more information, such as epigenetic information, and by optimizing the model architecture. They also mentioned that their model can be applied to other datasets and cell types beyond immune cells.","The present results would greatly benefit studies on scRNA-seq data through the elucidation of the cell composition in complex tissues and provide novel insights into mechanisms underlying disease pathogenesis and progression.",null,null,"their hypergraph strategy can potentially be adopted to expand the current model if one wants to adopt some prior knowledge about cells or genes (e.g., GO)","How to assess and select high-quality references and use supervised methods to identify new cell types.",null,null,null,"\"As more scRNA-seq datasets become available and authors provide gold standard\nannotations of their cell types, our benchmark can be expanded.\"",null,"The authors suggested that it would be interesting to evaluate the effects of multiple clustering algorithms on cell type annotations in the future. They also mentioned that scCATCH could be further improved by incorporating more comprehensive and accurate reference databases, as well as by optimizing the scoring system for marker gene selection. Additionally, the authors suggested that scCATCH could be extended to other omics data types, such as single-cell ATAC-seq and single-cell proteomics, to enable more comprehensive and accurate cell type identification.",null,"In the future, we will provide a comprehensive assessment of the influence of different cluster numbers on dataset complexity by integrating more datasets.","In the future, feature selection and enhancement of novel cell prediction are worth further exploration to improve the accuracy and functionality of single-cell classification tools.","the paper mentions that the two characteristics of mtSC (the flexible integration of multiple references and the ability to handle data heterogeneity) have great potential utility when much more sequencing data on different species have accumulated in the future.","Studying how cluster parameters and data structure (e.g. cluster density, fuzzy vs. hard clusters) affects our results should also be considered in future work.","The authors suggest that scGAD can be applied to time series data to explore the trajectory path of cell development. Additionally, they plan to adapt this new annotation task and the proposed framework to multiomics data study in the future, considering the rapid growth of high-throughput single-cell multi-omics sequencing technologies.","The paper mentioned that efficient cell assignment and detection of rare cell types remain future challenges. The paper excluded rare cell types whose cell number is less than 10 because they contain limited information and are unreliable for subsequent assignment.",null,"\" Thus, developing frameworks that are capable of benchmarking multi-resolution or multi-scale clustering on datasets with large numbers of cell types is a critical future research direction. \"","In the future, we\nenvision that ClusterMap can also be extended by combining other\ntypes of biological features (e.g., subcellular organelles, cell shapes,\netc.) to uncover the basic principles of how gene expression shapes\ncellular architecture and tissue morphology","As the rapid development of droplet-based single cell technologies, we expect our framework can eventually be applied to identify cell clusters in large-scale omics data other than scRNA-seq, such as scATAC-seq, CyTOF and image-based spatial data.",null,null,"In addition to the number of clusters, most methods rely on other hyperparameters. In this study, we have fixed these to reasonable values. However, additional investigations into the effect of these hyperparameters on the results would be an interesting direction for future research.",null,"An additional direction for future research may include exploring other methods of basic dimension reduction, such as weighted gene selection based on variability or other metrics; this may be more useful in capturing cell type-specific characteristics by retaining genes containing more biological signal related to cell type.","We anticipate that the comparison can be extended to spatial data with real gene expression levels after curated spatial transcriptomics atlas becomes available.","Very short, limit in the last 2 sentances. \"In the future, we can continue to study ...\"","in the future, this benchmarking framework can be applied to make a thorough comparison.",null,"In the future, we will continue to enhance SMSC by incorporate deep neural networks to obtain more useful features. It may improve both the accuracy and robustness of predictions regarding cell–cell interactions. We plan to develop a more userfriendly software system from our SMSC model, together with interactive visualizations.","Thus, developing frameworks that are capable of benchmarking multi-resolution or multi-scale clustering on datasets with large numbers of cell types is a critical future research direction.",null,null,"In addition, we would like to discuss two future directions. First, in this study, we utilized semi-synthetic datasets with real spatial locations and synthetic gene expression data generated by the scRNA-seq simulator scDesign2, since real spatial transcriptomics datasets with high-quality cell-type labels are very rare. We anticipate that the comparison can be extended to spatial data with real gene expression levels after curated spatial transcriptomics atlas becomes available. Second, this is a fast evolving field, and we have noticed several new clustering methods during the preparation of this manuscript, to support further comparison, we have uploaded the data analyzed in this study to a publicly available repository at Github","Such points are probably less relevant and could be solved with future releases of the methods.","However, the SC3 method is still far from perfect when applied to rel- atively small numbers of cells and, also, in cases of high dataset complexity. Therefore, further studies of the issue would be valuable.","In the future, we are interested in integrating other cluster validity indices into multiobjective evolutionary algorithm.",null,null,null,null,"\"Additional investigations into the effect of these hyperparameters on the results would be an interesting direction for future research.\"",null,null,"If spatial information could be accommodated in this GAN architecture, we may expect similar promising improvement in the analysis of sp-scRNA-seq data. We leave such exploration to future work.","\"Hence, there is a need for developing novel and more accurate methods, in particular for those datasets containing a very large and heterogeneous amount of cells. \"","In the future, we can continue to study the application of PCA in the following profiles: (i) interchromosomal interactions profile, (ii) TAD profile and (iii) compartment score profile.",null,null,null,null,null,null,"\"would be interesting to consider aspects such as different preprocessing strategies and normalization techniques, studying the effects of these steps on the accuracy of single-cell differential expression analysis.\"",null,"For the simulation effort, we would like to include additional parameters to simulate FC distributions and an imbalanced contrast group design as opposed to requiring simulation of equal numbers of disease and control samples as in the present study. We would also like to enable the simulation of real covariates from input data for disease samples rather than using the current control covariate subject-mapping approach. Correlation structures, both between cell types and between individual cells, could also be modelled in future simulations. We would also like to incorporate into our simulator and DE pipeline codes the ability to ingest other single-cell sequence data formats to broaden the datasets capable of being studied using this simulation workflow.","\"he lack of agreement in finding DE genes by these tools and their limitations in detecting true DE genes and biologically relevant gene sets indicate the need for devel- oping more precise methods for differential expression\"","it may be possible to borrow information across features to obtain shrinkage estimates","Depending on the goal of the DE analysis, the choice of genes to analyze in a single-cell experiment will rely on careful evaluation of the experimental data",null,"The lack of agreement in finding DE genes by these\ntools and their limitations in detecting true DE genes and\nbiologically relevant gene sets indicate the need for devel-\noping more precise methods for differential expression\nanalysis of scRNAseq data. Multimodality, heterogeneity,\nand sparsity (many zero counts) are the main characteris-\ntics of scRNAseq data that all need to be addressed when\ndeveloping new methods.","\"Our findings opened many interesting research questions that require further investigation in future studies. \"","In future works it would be interesting to consider aspects such as different preprocessing strategies and normalization techniques, studying the effects of these steps on the accuracy of single-cell differential expression analysis.","Finally, as shown the figures, the number of lowly expressed genes is not trivial, so our results also highlight the need for further development of methods to deal with these genes.\n",null,null,null,"Evaluating the benefits of different methods and criteria for selecting informative genes for different downstream tasks is another important future direction.\nMany existing dimensionality reduction methods, in particular those that require the computation and memory storage of a covariance or dis- tance matrix among cells, will no longer be applicable there. We have examined a particular sub-sampling strategy to scale all dimensionality reduction methods to large data sets. However, while the sub-sampling strategy is computationally efficient, it unfortunately reduces the performance of many dimensionality reduction methods by a substantial margin. Therefore, new algorithmic innovations and new efficient computational approximations will likely be needed to effectively scale many of the existing dimensionality reduction methods to millions of cells.",null,"Section on \"Future perspective\"",null,"We recommend that authors developing these methods refrain from emphasizing comparisons unless methods are equally tuned and/or some sort of blinded design is used to control researcher degrees of freedom. It may be most practical to rely primarily on disinterested third parties or challenge-based frameworks for comparisons between methods.",null,null,null,"In the future, it would be interesting to extend our benchmark to evaluate jDR methods on discrete omics data.","Therefore, new algorithms will likely be needed to effectively deal with dropout rate and millions of cells.","In the future, it would be interesting to extend our benchmark to evaluate jDR methods on discrete omics data. Indeed our current benchmark focuses on continuous data (e.g., expression, methylation), whereas many omics and annotations can be formalized as discrete data (e.g., copy number, mutation, drug response). Further extensions of our benchmark could also investigate the impact of different variables on the jDR methods, such as the stability of the methods with respect to variations in the structure of omics data (e.g., imbalance in variability or number of features); or optimal performances according to different combinations of omics data (e.g., are three omics more informative than two?).","\"It would be interesting to investigate in further studies how well our metrics translate to downstream applications: in particular which metric out of the AMI and silhouette correlates best with downstream performances.\"","investigating the performance of dimensionality reduction methods in these different scRNA-seq downstream analyses is an important future research direction.",null,null,"The scRNAseq analysis toolkits scanpy20 and Seurat21 recently implemented UMAP\nas a possible tool for dimensionality reduction, and the popular commercial software platform for flow cytometry analysis FlowJo recently released a plug-in to run UMAP. Altogether, on the basis of its ease of use and results of our benchmarking analyses, we anticipate that UMAP will be a valuable tool that can be rapidly adopted by the single-cell analysis community.","It would be interesting to investigate in further studies how well our metrics translate to downstream applications: in particular which metric out of the AMI and silhouette correlates best with downstream performances.",null,"In this benchmarking study, we found that PCA implementations based on full-rank SVD are accurate but cannot be scaled for use with high-throughput scRNA-seq datasets such as the BrainSpinalCord and Brain datasets, and alternative implementations are thus required.","Further experiments would be required to narrow down and validate any predicted marker genes and significant pathways.","\" Therefore, evaluating the performance of existing algorithms and understanding their limitations when applied to a newer wave of data, such as single cell gene expression data, is extremely helpful to facilitate further development of new methodologies that are specific for single cells. Such models would further provide new opportunities to understand cell-to-cell heterogeneity, and other biologically interesting questions such as stem cell differentiation and cancer development.\"","Future extensions of this study could include pseudotime-based network inference methods, once adequate datasets will become available. To date, available independent datasets relevant for pseudotime- based network inference algorithms (e.g., cells profiled during development stimulation) present too many experimental variations to be employed for a reliable evaluation of reproducibility. Of note, such extensions will be greatly facilitated by taking advantage of the Jupyter notebook (scNET)","Therefore, evaluating the performance of exist- ing algorithms and understanding their limitations when applied to a newer wave of data, such as single cell gene expression data, is extremely helpful to facilitate further development of new methodologies that are specific for single cells. Such models would further provide new op- portunities to understand cell-to-cell heterogeneity, and other biologically interesting questions such as stem cell differentiation and cancer development.","In the future evaluation, it is better to\nfurther split some cell type clusters into mini-clusters according to their local structures\nand evaluate respectively.","Future extensions of this study could include pseudotime-based network inference methods, once adequate datasets will become available.","While there are advantages and improved performances for analyzing scMulti-omics data, there is still room to improve the power of DeepMAPS further. First, the computational efficiency for super-large datasets (e.g., more than 1 million cells) might be a practical issue considering the complexity of the heterogeneous graph representation (which may contain billions of edges). Moreover, DeepMAPS is recommended to be run on GPUs, which leads to a potential problem of reproducibility. Different GPU models have different floating-point numbers that may influence the precision of loss functions during the training process. For different GPU models, DeepMAPS may generate slightly different cell clustering and network results. Lastly, the current version of DeepMAPS is based on a bipartite heterogeneous graph with genes and cells. Separate preprocessing and integration steps are required to transfer different modalities to genes for integration into a cell-gene matrix",null,"Last 10 lines but very weak discussion",null,"this highlights the need for improved methods and better gold standards for regulatory network inference from scRNAseq datasets.",null,"\"Most importantly, the lack of standard benchmarks in assessing the performance of these mathematical models raises questions about the reliability of the constructed GRN. As the field develops and more information becomes available to validate the networks, GRN inference methods using single-cell data will be promising tools not only in discovering missing components of GRNs in the cell level but also in a wide range of real-world applications such as identifying disease biomarker and pathways, designing drugs and precision medicine.\"",null,null,"Throughout this study, we have observed a number of issues that deserve to be points of focus in future investigations.","additional information such as those from proteomics data and experiments on the effect of receptor gene knockouts can be used for further testing and benchmarking when available.","Throughout this study, we have observed a number of issues that deserve to be points of focus in future investigations.",null,"The good performances provided by OT in this work suggest that further efforts should be devoted to design DR methods for single-cell data based on other metrics.",null,"Our benchmark could guide the development of future computational tools in resolving the CCIs using multimodal datasets such as ST and scRNA-seq",null,"Future methods will need to accurately sep- arate cell stages from single-cell expression data to generate a comprehensive network.\n",null,"Last, further “ground truth” experimental data confirming enhancer-gene or TF-gene relationships will be invaluable to the field in resolving the accuracy of GRN methods, including FigR.\n\n",null,null,"it may in the future be necessary to design panels of relevant network patterns, thus saving computational cost and retaining statistical power.","A direction of future work would be to examine the effect of metacells on scalability and accuracy of network inference algorithms.","\"To avoid such indirect interactions, it may be important to inte- grate additional types of data such as known TF binding sites or ChIP–seq measurements15. Finally, a target gene’s expression level may change even if the regulating TF does not vary in abundance. Recent approaches that interrogate single cells along multiple modalities33,34 may be important for the next generation of GRN inference algorithms.\"",null,"\" Our analysis suggests that substituting conventional mea- sures of association with measures of proportionality could lead to an increase in the accuracy, and therefore interpretability, of diverse computational analyses for scRNA-seq beyond those evaluated here\"",null,"Therefore, future extensions to the framework design to study the inf luence\nof pre-processing could be explored","\"Benefiting from the establishment of Human Cell Atlas [52], Mouse Cell Atlas [53], and Tubula Muris [54], researchers could unbiasedly investigate the ability of PASs on supervised cell type annotation across technologies, organs and species, which might further facilities the automatic functional annotation of single cells from transcriptomics perspective.\"","Across technologies, the TF activities from SCENIC correlated less well than those calculated with the other tools, which is consistent with overfitting by SCENIC, but further analysis is required.","Further benchmarking studies are warranted to systematically compare the performance between GRN tools and PAS tools or investigate which combinations of these state-of-art tools could provide more efficient and robust transcriptional heterogeneity","In conclusion, our results caution against using bulk-sample-based signature-scoring methods to score single cells. Our study also suggests that it is important to consider cellular contexts when benchmarking methods for scRNA-seq data analysis, particularly when bulk-sample-based methods are involved.","“Our study also suggests that it is important to consider cellular contexts when benchmarking methods for scRNA-seq data analysis, particularly when bulk-sample- based methods are involved.”","\"This hypothesis could be tested in the future by decoupling functional analysis tools into gene sets and statistics. Benchmarking all possible combinations of gene sets and statistics (i.e., DoRothEA gene sets with a linear model or PROGENy gene sets with VIPER) would shed light on this question which we believe is of high relevance for the community.\"","\"However, the unfeasibility to obtain the ground truth for 10x data and the scarcity of tools to process them made us exclude this type of data from our benchmark.Nevertheless, such comparisons should be addressed in the future studies.\"",null,null,null,null,null,"Some defects of these methods such as scalability, robustness, and unavailability in some situations need to be addressed in future studies.","Some more complex and robust models can be used to measure local similarity between cells, like autoencoder and manifold learning.",null,"In the future, it is worthwhile to continue investigating how conclusions presented in this study may translate to applications in a diseased setting such as cancer tissues.","we will develop further analysis on them to help understand the communications between different regions in future work",null,null,null,null,"\"reference- based imputation is limited by the completeness of the exter- nal dataset. Alternatively, models could be developed to use gene-gene correlations derived from large external databases of expression data (Obayashi et al., 2008), while more generalizable such methods may not capture cell-type specific relationships\"","It is hoped that this study will be of benefit to researchers and provide new ideas for future scATAC-seq data imputation tasks","However, the open option\nprovided in scHinter for assigning weights allows users to test\ntheir own strategies if they have a better solution to determine weights\nor facilitates the evaluation of the effect of different combinations of\nmetrics. Additional work will be carried out in this direction for a better\nintegration of different distance metrics. Moreover, the current version\nof scHinter is implemented in Matlab, we set out to develop a\ncorresponding R package available for the broader community.",null,"Finally, it may also be possible to extend BayesSpace to jointly cluster spots from multiple samples given appropriate data normalizations.","In addition, combining the scRNA-seq and scATAC- seq data for further analysis is also an idea worth exploring. It is hoped that this study will be of benefit to researchers and provide new ideas for future scATAC-seq data imputation tasks.",null,null,"Second, SD2 could be further improved by utilizing the distribution\nof all scRNA-seq data for assistance.",null,"An open question is if profiles derived from one tumor type also\napply to another.","Finally, as more scRNA-seq datasets become available in the\nnear future, its aggregation (while carefully removing batch\neffects) will increase the robustness of the reference matrices\nbeing used in the deconvolution and will fuel the development of\nmethodologies similar to SCDC, which allows direct usage of\nmore than one scRNA-seq dataset at a time.","\"Extending scJoint to handle epigenomic data directly at locus level will require designing a separate encoder that is suitable for the high dimensionality and remains easy to train. With the increasing availability of paired scRNA-seq and scATAC-seq data and other single-cell multimodal technologies, methods such as BABEL41, which trains on paired data to impute gene-level signals from scATAC-seq data, can potentially be used to provide more comprehensive gene-level features than gene activity scores alone. We will pursue this for future work.\"",null,null,null,"In the future, different combinations of omics data and the selection of a less-redundant set of omics data will be considered in this benchmark study.","Other improvements might include\nsimplifying the calculation of the robustness term to require fewer\nintensive calculations.",null,null,"\"Finally, as more scRNA-seq datasets become available in the near future, its aggregation (while carefully removing batch effects) will increase the robustness of the reference matrices being used in the deconvolution and will fuel the development of methodologies similar to SCDC, which allows direct usage of more than one scRNA-seq dataset at a time\"",null,"Instead, dimensionality reduction approaches designed for scATAC-seq data in combination with an MNN approach as implemented in FastMNN or Scanorama may represent a promising avenue for future integration approaches for this modality.",null,"((Methods   constructed   on   the   basis   of   probabilistic   models    combined  with  negative  binomial  or  Poisson  distributions,  such   as  gimVI,  Cell2location,  and  RCTD,  generally  perform  better  at   predicting  the  spatial  distribution  of  transcripts  or  deconvolving   cell  types  of  spots.  A  deep  learning  algorithm  was  also  applied  in   several  integration  methods,  among  which  Tangram  is  one  of  the   best-performing  methods  in  predicting  spatial  distribution  of  the   undetected  transcripts.  Technically,  Tangram  employs  non-convex   optimization  in  the  model  and  selects  only  the  optimal  subset  of   scRNA-seq  observations  in  the  loss  function.  A  combination  of   these measures may help improve the predictive power of the tools.))\n\nand maybe also:\n\n((Benchmarking analysis may thereby rely on more experimental validation of cell–cell interactions in the future.))","However, future work could focus on evaluating temporal gene expression integration for its ability to gain increased biological insight according to a wider range of tasks, such as unsupervised cell population identification [66], characterizing phenotypic-related cells [42], characterizing differentially abundant cell populations [67, 68], or gene regulatory network inference [69].","In future studies, new methods could be developed for ST deconvolution by considering the constraint of adjacent spots.",null,"Second, the running time of PASTE could be further improved to support future generations of ST technology that have a larger number of spots.","\"Beyond canonical graph convolution, more advanced graph neural network architectures59,60,61 may also be adopted to extract richer information from the regulatory graph.\"","Revealing limitations of existing methods provides guidance and directions how to improve. Our study also pointed out the urgent need of new metrics for evaluation since cell types are unknown in real applications.","\"To automate the process, new quantitative metrics are needed for the evaluation of batch correct methods, which not only measure batch mixing but also evaluate imperceptible cell-type mixing.\"\n\"Our study also pointed out the urgent need of new metrics for evaluation since cell types are unknown in real applications\"","\"Our attempt to identify marker genes from the corrected dataset demonstrates the difficulty of using the merged ex- pression matrix for downstream analyses. As none of the methods considered in our study performed adequately in this benchmark, we highlight this as an area where improve- ments are required.\"","One feasible way is to analyze each dataset separately to gain some idea about cell types in each batch and then explore the batch-corrected results manually to decide whether there is any suspicious mixing. But it is time-consuming and requires a strong background. To automate the process, new quantitative metrics are needed for the evaluation of batch correct methods, which not only measure batch mixing but also evaluate imperceptible cell-type mixing.","SMNN can not only better rescue biological features and thereof provide improved cluster results but also facilitate the identification of biologically relevant DEGs. Therefore, we anticipate that our SMNN is valuable for integrated analysis of multiple scRNA-seq datasets, accelerating genetic studies involving single-cell dynamic","We suggest that future efforts should be dedicated to refining cell population definitions and finding reliable signatures.",null,null,null,"Advances in spatial transcriptomics technologies, such as new versions of 10X Visium and BGI Stereo-seq76, may enable detection of transcriptomic information for spots with diameters much smaller than cell size. However, in the near future, each spot still might not correspond exactly to a single cell when using these technologies.","\"First, it can be extended to accurately model gene counts. This will allow users to apply Harmony preprocessing for methods that require full gene expression profiles, rather than low-dimensional cell embeddings, such as RNA velocity. Next, we envision specializing Harmony to quickly map cells to a multi-billion cell reference that covers a comprehensive set of tissues, organisms and clinical conditions.\"",null,null,"\"it also provides a stepping stone for integrating other single-cellmeasurements.\"",null,null,null,null,"\"With the increasing availability of paired scRNA-seq and scATAC-seq data and other single-cell multimodal technologies, methods such as BABEL41, which trains on paired data to impute gene-level signals from scATAC-seq data, can potentially be used to provide more comprehensive gene-level features than gene activity scores alone. We will pursue this for future work.\"","future work could focus on evaluating temporal gene expression integration for its ability to gain increased biological insight according to a wider range of tasks, such as unsupervised cell population identification [66], characterizing phenotypic-related cells [42], characterizing differentially abundant cell populations [67, 68], or gene regulatory network inference; extended to the analysis of other extrapolated regulatory modalities, including RNA velocity in-situ [14], protein velocity [15], or chromatin velocity [70].","In future studies, new methods could be developed for ST deconvolution\nby considering the constraint of adjacent spots. More specifically,\nfor the ST data, we can consider synthesizing the adjacent spots\ninto new spots containing more cells, which satisfies the linear constraint\nthat the proportion of cell types in the simulated big spot is\nequal to the weighted sum of the proportion of cell types in the respective\nsmall spots.","Another future improvement involves the gene activity score.",null,"What’s more, considering CLAIRE’s excellent performance in integrating scRNA-seq datasets, we believe it has a great potential to migrate to multi-omics data integration problems.","As more scRNA-seq data become available, we believe SCIDRL will be a valuable tool for the comprehensive single cell heterogeneity analysis. This framework may be extended to integrate multimodal datasets such as scATAC-seq, spatial transcriptome etc., which is specially promising in future single cell multi-omics data analysis.","We also present possible future directions of the field to shed light on the development in the field.",null,"We also present possible future directions of the field to shed light on the development in the field.",null,null,null,null,"\"Currently, Portal uses shared highly variable genes among datasets. Improvements can be made if an effective way of leveraging the whole transcriptome of all datasets is developed, which is left for future work to address.\"","\nFinally, in the way that data processing pipelines are currently set up, researchers will often be forced to select markers without the knowledge of a ground truth set of markers. Thus, it may be valuable to consider metrics such as the ones discussed in this work when performing marker selection. Combining the values of several of the metrics may help to aid researchers in deciding when they have selected enough markers to adequately describe their cell types (so that they are not considering genes that were chosen at random), for example. The question of how to stop selecting markers is another important consideration for future work.",null,"We envision that users will apply DrivAER to infer activity of regulators not generally detected in scRNA-seq data such as microRNAs and long noncoding RNAs.",null,null,"We anticipate that scHFP will be a complementary tool for dissecting the transcriptional underpinnings of cellular identity and state.",null,null,"The present article also focuses on scRNA-seq, but the methodology and software are general and applicable to other types of assays, including microarray, bulk RNA-seq, and adductomic and metabolomic assays.",null,null,"The limitations of these normalization methods on the 10X data set depicted also highlights the need for a method that can appropriately adjust to this new format and its properties.",null,null,null,null,"In the future, this framework could be extended to allow additional input data, such as spatial and RNA velocity information45, and easier downstream analyses.","\"Going forward, we expect that improvements in counting strategies for scRNA-seq data, specifically tailored for RNA velocity preprocessing, will likely come alongside an increased understanding of the read generation process and the biases underlying specific scRNA-seq library preparation protocols, and that different counting strategies may be optimal for differ- ent types of scRNA-seq data. An increased understanding of the read generation process will also enable realistic simulation of sets of spliced and unspliced scRNA-seq reads, which in turn will provide an improved platform for objective evaluation of the performance of counting strategies.\"","\"we also highlight several ongoing challenges, which should be addressed before TI can be a reliable tool for analyzing single-cell omics datasets with complex trajectories. Foremost, new methods should focus on improving the unbiased inference of tree, cyclic graph and disconnected topologies, as we found that meth- ods repeatedly overestimate or underestimate the complexity of the underlying topology, even if the trajectory could easily be identified using a dimensionality reduction method\"",null,null,"This demonstrates the potential of ordinal regression models for further methodological developments.",null,"\"Going forward, we expect that improvements in counting strategies for scRNA-seq data, specifically tailored for RNA velocity preprocessing, will likely come alongside an increased understanding of the read generation process and the biases underlying specific scRNA-seq library preparation protocols, and that different counting strategies may be optimal for different types of scRNA-seq data.\"","Potential directions for future improvement include incorporating other modalities for regulation relationship inference and formulating the model as neural stochastic differential equations. With the exponential growth of scRNA-seq data, DeepVelo can be a useful tool for the community to illuminate the rules governing cell state dynamics and cell fates.",null,"As a future work, it is expected that novel and effective feature selection methods combined with dimension reduction techniques, as well as methods able to detect the differentiation ways of cells will improve DBCTI’s performance, especially for the bifurcating patterns presented in Supplementary Figures S1E, S2E and S3E.","\" we have identified the most robust single-cell potency measures, revealing that their robustness is driven by a correlation between mRNA expression and network connectivity. This study thus provides a foundation to further improve upon the analysis of single-cell RNA-Seq data, by direct in silico estimation of single-cell potency.\"","This study thus provides a foundation to further improve upon the analysis of single-cell RNA-Seq data, by direct in silico estimation of single-cell potency.","“In the future, we will extend it to data from more than two conditions, which can be done by modifying the target function”","To avoid the impact on the klDD model, we controlled the target FDR at 0.1% to identify DD genes in real data, and also it is our future topic to improve the klDD model for handling strong noisy data.\n",null,null,"scDblFinder additionally provides utilities for identifying the origins of doublets (in terms of composing cell types) and\ntesting for different forms of doublet enrichment. At present, however, the value of such tests is limited by the difficulty of\naccurately identifying doublet origins. Further research will be needed to assess to what extent this can be improved.","Regarding the future development and benchmark of computational doublet-detection methods, here we list five open questions we deem important for computational scientists.","Regarding the future development and benchmark of compu- tational doublet-detection methods, here we list five open ques- tions we deem important for computational scientists:","We note that\nvaeda, as well as other methods (see Fig. 3), do not perform particularly well on these data, indicating that improving our ability to\nsimulate doublets might be a promising approach to improve\nmethod performance.","Future isoform quantification tools designed explicitly for scRNA-seq could improve on the performance of existing tools by being more conservative in calling isoforms as expressed, and by explicitly modeling the technical noise inherent to scRNA-seq.","“Future isoform quantification tools designed explicitly for scRNA-seq could improve on the performance of exist- ing tools by being more conservative in calling isoforms as expressed, and by explicitly modeling the technical noise inherent to scRNA-seq.”","To address this, we advocate for working toward common benchmarks for barcode decoding quality, which would be tremendously useful for the development of community standards for evaluation and reporting conventions.","However, we\nconsider our algorithm robust to various choices of k and T\nand plan to explore hyperparameter selection approaches in\nfuture work.",null,"The integration of these multi-modal datasets (Singh et al., 2020) in a manner similar to ShareNet would serve as an exciting next step for characterizing the intricate relationship between gene regulatory relationships and cellular identity in greater detail.",null,null,"\"Based on the experiments conducted, we identified several areas of exploration for future researchers. \"","Future method development should consider this factor as an aspect of evaluation when assessing model performance.","The efficient estimation of the hyperparameters from the real data in the tree-structured cell group and continuous cell trajectories scenario still needs improvement.",null,null,"While dyngen can generate non-linear trajectories (e.g. cyclic or branching), both aligning non-linear trajectories and constructing a quantitative accuracy metric for non-linear trajectory alignment is not trivial and an avenue for future work.","\"In the near further, we will extend the power evaluations for DE in more complex experimental design (multiple-group comparison, with continuous covariates, etc.). \"",null,null,"Thus, future work should be focused on the development of flexible, faithful simulation frameworks to fill this gap, especially in scenarios where an experimental ground truth is challenging or infeasible to establish.",null,"“Thus, future work should be focused on the development of flexible, faithful simulation frameworks to fill this gap, especially in scenarios where an experimental ground truth is challenging or infeasible to establish.”","For that, we will develop new statistical model to characterize the data, which is our future research plan.",null,"\"To extend scDesign’s ability to evaluate experimental designs for cell states whose scRNA-seq data are not yet publicly available, a future direction is to incorporate bulk RNA-seq data of the same type as a surrogate to estimate the gene expression parameters.\"","Moreover, the impact of detailed experimental characteristics (e.g. different protocols) specific to the simulators will be evaluated systematically in the future.","For future methodological improvement, there are other ways to address this high-dimensional estimation problem.","\"It is easy to envision, for example, how cscGAN variants could generate realistic (small) RNA-seq or proteomic data. \"","This is a challenging task, which needs to be explored in the future.","Future methodological development for exploring the use of other covariance functions as well as other transformations in a computationally efficient manner may help improve the power of SPARK-X further.","Other areas of future work might include the incorporation of information about tissue makeup or local differences in cell density.","SOMDE can also be extended to 3D spatial sequencing data in the future, since the Gaussian Process and SOM have no restriction on dimensionality.","Further, cell type-specific gene expression needs to be estimated to tease out the contribution of cell types and spatial location in gene expression variation. We anticipate that methods development along these directions is warranted for future research.",null,null,null,"\"we plan to devise approaches for phasing genomes and profiling allele-specific copy numbers.\"","\"We anticipate that this tool will be applied widely to many types of solid tumors in addition to the cancer types analyzed in this study. These studies will greatly improve understanding of the malignant gene expression programs of tumor cells by providing a pure tumor cell signal, whereas previous bulk RNA-seq methods have been challenged by the intermixing of stromal and immune cells with the tumor cells.\"","\"Our results not only provide a guideline for variant detection, but also highlight the necessity of improving detection sensitivity in compre- hensive conditions when developing variant-calling methods for scRNA-seq.\"","\"However, we do anticipate that emerging technologies, such as long-read full-transcript scRNA-seq technologies and transcriptome-based deep targeted sequencing will overcome these limitations of tag-based scRNA-seq.\"",null,null,null,"Additionally, there are several efforts to enrich visualizations by either using a 3D plot instead of 2D or even by using virtual reality.","One way of optimization is to add integration with the HDF5 format as supported by our results in Figure 1. A simpler approach is to down-sample the data by selecting a small subset of cells in a random manner.\nAdditionally, there are several efforts to enrich visualizations by either using a 3Dplot instead of 2D (18,33) or even by using virtual reality (34). The third dimension may allow\nresolution of cell-types not visible in two dimensions. Virtual reality can support multiple embeddings in the same VR space so that they can be directly compared.","While embedding all cell types in a space with as few as two or three dimensions is unlikely to be successful given the complexity of the problem, we believe that insights from net-SNE and its future extensions may lead to an effective approach for learning compact vector space representations of all human cells that can be readily plugged into new and existing computational methods to further advance our understanding of biology.","we believe that insights from net-SNE and its future extensions may lead to an effective approach for learning compact vector space representations of all human cells that can be readily plugged into new and existing computational methods to further advance our understanding of biology."],"Website":["No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No"],"Data_availability":["Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","No","No","No","No","No","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","No","Yes","No","No","No","Yes","No","Yes","No","No","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No"],"Package_availability":["Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes"],"Prior_knowledge":["Yes","No","No","Yes","No","Not sure","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Not sure","No","Not sure","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","Yes","No","No","Not sure","Yes","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Not sure","No","Yes","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","Yes","No","Not sure","No","No","Not sure","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","Yes","Not sure","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","No","Not sure","Yes","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Not sure","Not sure","No","No","Yes","Yes","Not sure","Yes","No","No","No","Yes","Not sure","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Yes","Not sure","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Not sure","No","Not sure","Yes","No","No","No","No","Not sure","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","Not sure","Yes","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","Yes"],"Types_of_data":["Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Both","Both","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Experimental data","Both","Both","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Both","Synthetic data","Experimental data","Experimental data","Experimental data","Experimental data","Synthetic data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Synthetic data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Synthetic data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Both","Experimental data","Synthetic data","Synthetic data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Experimental data","Experimental data","Synthetic data","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Both","Both","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data"],"Number_of_experimental_datasets":[3,12,14,9,7,20,2,3,4,2,8,9,12,4,8,2,6,3,6,7,1,10,20,76,4,14,4,20,20,2,4,9,2,2,11,11,6,5,4,22,4,3,3,4,22,3,1,3,5,18,8,6,4,30,13,6,27,6,5,9,27,5,3,4,6,43,3,9,43,13,1,10,4,5,10,5,27,9,6,20,3,27,5,10,31,3,2,7,15,6,6,4,4,8,7,6,6,11,11,2,4,4,7,17,null,6,6,12,8,null,4,27,4,49,17,6,9,3,1,9,3,9,2,1,2,2,2,2,9,2,0,2,3,3,0,3,30,4,4,10,3,55,5,10,11,5,11,10,30,4,5,5,10,55,4,3,2,6,2,5,6,13,10,15,5,7,1,null,213,4,9,15,9,22,8,4,5,3,null,2,3,518,22,1,7,5,4,213,5,2,32,2,32,10,10,2,3,3,2,18,3,6,2,12,null,6,12,2,2,8,2,18,8,8,6,5,8,25,6,3,14,5,5,5,23,3,45,6,4,1,23,5,2,23,15,45,10,4,2,5,5,4,4,3,4,2,5,3,9,1,45,6,7,5,1,9,5,6,3,5,10,4,5,3,6,11,7,15,7,9,14,2,null,16,4,9,2,23,9,4,3,4,4,120,4,4,120,4,2,6,110,5,110,4,2,1,1,5,6,8,2,11,11,4,12,3,28,16,16,16,16,null,null,1,4,2,3,9,4,35,35,1,3,1,1,11,1,8,12,7,12,11,1,18,13,6,5,2,6,4,6,7,5,5,15,2,6,2,2,2,3,7,1,1,13,13],"Number_of_synthetic_datasets":[1000,null,14,0,2,null,null,10,null,9,null,0,null,null,null,null,19980,10,19980,2,4,4201,null,null,null,14,null,null,null,1,null,0,1,null,null,null,null,2,4,null,null,null,null,null,null,990,1,null,null,3800,null,null,1,null,0,41,null,null,null,1,null,null,null,10,null,3,null,null,3,0,1,null,3,null,null,null,null,null,41,null,null,null,null,1,null,1,null,null,31,null,null,3,null,0,7,null,null,null,null,null,100,null,7,25,700,null,null,null,null,700,3,21,null,6,25,null,7,3,44,7,3,3,10,44,4,10,16,4,3,10,1280,10,1,1,1280,40,2,2,18,2,40,null,30,2,100,30,100,null,2,2,null,null,null,null,18,null,2,0,2,15,0,null,null,null,420,null,null,100,null,null,null,null,null,0,5,0,15,null,100,30,0,null,0,1,null,420,0,null,null,30,null,30,null,null,null,30,74,74,null,60,60,0,5,6,null,1,null,5,6,null,6,60,null,null,1,null,null,116,3,2,0,200,1000,0,2,6,32,3,null,12,2,1000,1,2,10,32,0,3,2,1,39,28,28,6,28,1,200,1,1,12,32,0,91,null,1,1,null,null,100,0,0,3,null,100,null,2,43,10,43,1,0,2,null,0,20,49,null,1,49,null,20,null,10,null,1,1,null,10,5,null,229,0,229,null,null,1,6,0,null,null,5,null,null,7,1,1172,null,1,112,112,null,3,3,1,null,null,null,null,1,null,null,4,null,1,null,40,13,0,null,21,null,14,2,0,5,30,30,null,98,3001,22,null,1,1,null,9,0,1,9,1,500,4,null,null,null,null],"Experimental_diversity":["different tissue types--different cell types--different biological complexity--different time point--different number of cell types--NA","different sequencing protocol--different organisms--different cell lines--different tissue types",null,"different cell types",null,"different sequencing protocol--different cell types--others--different preprocessing pipelines","NA--NA",null,"different tissue types--different cell types--others--Different epigenomic characteristics--different epigenomic characteristics","different organisms--different sequencing protocol--different tissue types--different cancer types--different biological complexity--different time point--NA","different sequencing protocol--different organisms--different tissue types","different organisms--different tissue types--different sequencing protocol","different organisms--different sequencing protocol","different organisms--different tissue types--NA","different organisms--different sequencing protocol","different cell types--NA","different sequencing protocol--different tissue types--different cell lines",null,"different sequencing protocol","different organisms--different tissue types--different sequencing protocol--different cell lines","NA--NA","different organisms--different sequencing protocol--different tissue types--different cell types--different cancer types--NA","different sequencing protocol--different tissue types--NA","different organisms--different tissue types--different cell types--different cancer types--different number of cell types--NA","different cell types--NA","different tissue types","different organisms--different sequencing protocol--NA","different cell lines","different cell lines","different sequencing protocol--NA","different organisms--different sequencing protocol--NA","different sequencing protocol--NA","NA--NA","NA--NA","different cell lines--different cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell types--NA","different organisms--different sequencing protocol--different tissue types--NA","different organisms--different tissue types--NA","different organisms--different sequencing protocol--different tissue types--different cell types--different biological complexity--different number of cell types--NA","different organisms--different tissue types","different organisms--different sequencing protocol","different organisms--different tissue types--different sequencing protocol--different cell lines",null,"different organisms","different organisms--different cell types--different tissue types",null,"NA--NA","different sequencing protocol--different cancer types--NA","different organisms--different sequencing protocol--NA","different organisms--different sequencing protocol--different tissue types--NA","different organisms--different sequencing protocol--different cell types--NA","different sequencing protocol--different tissue types--different cell types--NA","different tissue types--NA","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--NA","different organisms--different tissue types--different cell types--different sequencing protocol","different organisms","different organisms--different sequencing protocol","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--NA","different organisms","different sequencing protocol--different tissue types--different cancer types--different number of cell types--NA","different organisms","different sequencing protocol--different tissue types--different organisms",null,"different sequencing protocol--different cell lines--different cell types--different cancer types--different number of cell types--NA","different organisms--different sequencing protocol--different cell types--different biological complexity--NA","different cell types--NA","different tissue types--different cell lines--different cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell types--different cancer types--different number of cell types--NA","different cell types--NA","different organisms","different organisms--different cell types--NA","different organisms--different tissue types","different cancer types--NA","different organisms--different sequencing protocol--NA","different organisms--different tissue types--different sequencing protocol--different cell lines","different organisms--different tissue types--different sequencing protocol","different cancer types--different cell lines--different cell types","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--NA","different organisms--different tissue types--different sequencing protocol","different organisms--different sequencing protocol--different tissue types--different cell types--NA","different cell lines--different tissue types","different sequencing protocol--different tissue types--different cell lines--different cell types--NA","different organisms--different tissue types--different sequencing protocol--different cell lines","different organisms--different sequencing protocol--different tissue types--different cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--NA","different organisms--different tissue types--NA","different organisms--different cell types","different sequencing protocol--NA","different organisms--different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","different cell lines","different organisms--different tissue types--different cell lines--different number of cell types--NA","different sequencing protocol--different organisms","different organisms--different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell types--NA",null,"different organisms--different tissue types--different cell lines--NA","different sequencing protocol--NA","different sequencing protocol--different tissue types--different cell lines--different cell types--different number of cell types--NA","different organisms--different tissue types--different cell types--different number of cell types--NA","different organisms","different organisms--different tissue types--NA","NA--NA","different sequencing protocol","different sequencing protocol--different tissue types--different cell lines--different organisms",null,"different organisms--different tissue types--different number of cell types--NA","different cell lines--different tissue types",null,"different organisms--different tissue types--different number of cell types--NA",null,null,"different organisms--different sequencing protocol--different tissue types--different cell types--different biological complexity--NA","different organisms--different tissue types--different number of cell types--NA","different organisms--different sequencing protocol--different tissue types--NA","different organisms","different organisms--different cell lines--different cell types--NA","different organisms--different sequencing protocol--NA","different organisms",null,"different tissue types--others--different number of batches",null,null,"different sequencing protocol",null,"different tissue types--different cancer types--NA",null,"NA--NA","NA--NA","different sequencing protocol--different organisms--different cell lines",null,"different organisms","different cell types","different cell types--different cell lines--different sequencing protocol","different cell lines",null,"different tissue types--different cell types--NA","different sequencing protocol--different number of cells--NA",null,"different cell types","NA--NA","NA--NA","different cell types--different tissue types--different sequencing protocol","different organisms--different cell lines--different time point--NA","different organisms--different sequencing protocol--NA","different cancer types--different sequencing protocol","different organisms--different cell types--NA",null,"different organisms",null,"different cell types","different organisms--different sequencing protocol","different tissue types","different organisms",null,"different tissue types","NA--NA","different cell types","different tissue types",null,"different organisms--different tissue types--different cell types--NA","different cell types","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--different number of cell types--NA","different organisms--different sequencing protocol--NA","different sequencing protocol--different tissue types--different time point--different number of cell types--NA","different organisms--different cell lines","different organisms--different cell types--NA","NA--NA",null,null,"different organisms--different sequencing protocol--different tissue types--different cell types--different time point--NA","different cell lines--different cancer types--NA","NA--NA","different cell types--different time point--others--different sample types (normal vs disease samples)","different organisms--different tissue types--different sequencing protocol","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--different cancer types--different number of cell types--NA","NA--NA","different organisms--different sequencing protocol--different cell types--NA","different sequencing protocol--different cell lines--different cell types--NA",null,"different cell lines--different cell types--NA","different sequencing protocol--different biological complexity--NA","NA--NA","different organisms--different tissue types--different sequencing protocol","different cell types--NA","different organisms--different cell types--NA","different organisms--different cell types","NA--NA","different sequencing protocol","different organisms--different sequencing protocol--different tissue types--different cell types--different biological complexity--different time point--different number of cell types--NA","NA--NA","different tissue types--different sequencing protocol--different cell types--different cell lines","different tissue types","different organisms--different tissue types","different cancer types","different cancer types--different sequencing protocol",null,null,"different sequencing protocol","NA--NA",null,"different organisms--different tissue types--different number of cell types--NA","different organisms--different tissue types--different cell types--different biological complexity--different number of cell types--NA",null,"different organisms--different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","different organisms--NA","different cell lines--different tissue types--different sequencing protocol","different organisms--different cancer types--NA","different cell lines","different sequencing protocol--different time point--NA","different organisms--different sequencing protocol--different tissue types--different cell types--NA","different cell lines--different time point--NA","different tissue types--different sequencing protocol","different organisms--different sequencing protocol--different cell lines--different cell types--NA","different sequencing protocol--different tissue types--different cell types--NA","different cell lines",null,"different sequencing protocol--different cell lines--different cell types--NA","different organisms--different sequencing protocol--others--different sparsity","different cancer types","different organisms--different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","different organisms--different tissue types--different cell types","different cell types--different cancer types--different number of cell types--NA","different tissue types","different organisms--different tissue types--different sequencing protocol","different organisms--different tissue types","different sequencing protocol--different cell lines--different organisms","different organisms",null,"different organisms--NA",null,"different organisms--different cell lines","different sequencing protocol--different cell types","different organisms--different sequencing protocol--different tissue types--different cell types--different time point--different number of cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell types--different biological complexity--different time point--different number of cell types--NA","different cell lines--different cell types--others--different sequencing protocol","different organisms--different sequencing protocol--different cell lines","different organisms--different sequencing protocol","different organisms","different tissue types--different cell types--NA","different tissue types--different cell lines--different cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell types--different biological complexity--different number of cell types--NA","different cell types--NA","different organisms--different tissue types","different organisms",null,"different cell types--different number of cell types--NA","different cell types--NA","different sequencing protocol--NA","different sequencing protocol",null,"different sequencing protocol--different cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--different biological complexity--different time point--different number of cell types--NA","different sequencing protocol--different tissue types--different cell types--different number of cell types--Different preservation protocols--NA","others--differential proportion of cell types","NA--NA","different organisms--different cell lines--different tissue types","different organisms--different sequencing protocol--NA","different tissue types--different cell types--different number of cell types--NA","different tissue types--NA","different organisms--different tissue types--different sequencing protocol","different cell lines","different organisms--different sequencing protocol","different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","different tissue types--different biological complexity--others--different number of samples","different organisms--different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--different biological complexity--different number of cell types--NA","different organisms--different sequencing protocol--different tissue types--NA","different sequencing protocol--different tissue types--different cell lines--NA","different organisms--different sequencing protocol--different cell types--NA","different organisms--different tissue types--different cell lines--different cell types--different number of cell types--NA","different organisms--different tissue types--different sequencing protocol","different sequencing protocol--different cell types--others--different number of samples","different organisms--different tissue types--different cancer types--NA","different organisms--different tissue types--different biological complexity--NA","different organisms--different cell types--NA","different organisms--different tissue types--different cell types","NA--NA","different organisms--different sequencing protocol--different tissue types--NA","different organisms--different tissue types--different sequencing protocol","different organisms--different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","NA--NA","different organisms--different sequencing protocol--different tissue types--different cell lines--different cell types--different number of cell types--NA",null,"different sequencing protocol","different cell types--different number of cell types--NA",null,null,"different tissue types--different sequencing protocol","different organisms--NA","different organisms--different cell types--NA","different organisms--different sequencing protocol",null,"different organisms--different sequencing protocol","different organisms--different cell types--NA","different organisms--different cell types--NA","different organisms--different cell lines--different cell types--NA","NA--NA","different tissue types","different organisms--different tissue types--different cell types--different time point--NA","different organisms--different tissue types--different cell lines--different cell types--NA","NA--NA","different cell types--different sequencing protocol--different cell lines",null,"different organisms--different tissue types--different cell types--NA","different cell types--different biological complexity--NA","different organisms--different tissue types--different biological complexity--different time point--NA","different organisms--different sequencing protocol--NA","different tissue types--different cell types--different biological complexity--NA","different sequencing protocol--different cell types","different cell types","different organisms--different tissue types--different cell types--different biological complexity--NA",null,null,"NA--NA","different organisms--different tissue types--different cell types--NA","NA--NA","different organisms--NA","different organisms--different tissue types--different cell types--NA","different organisms--different tissue types--NA","different sequencing protocol--different tissue types--different cell lines--different organisms","different sequencing protocol--different tissue types--different organisms","NA--NA","NA--NA","NA--NA","NA--NA","different organisms--different sequencing protocol--different cell types--NA","NA--NA","different organisms--different sequencing protocol--different cell types--NA","different sequencing protocol--different organisms--different cell lines--different cell lines","different organisms--different sequencing protocol--different tissue types--different cell lines--NA","different tissue types--different sequencing protocol","different organisms--different sequencing protocol--different tissue types--NA","others--different sequencing protocol","different organisms--different sequencing protocol--different cell types--NA","different organisms--different sequencing protocol--different tissue types--different cell types--different number of cell types--NA","different organisms--different sequencing protocol--different tissue types--NA","NA--NA","NA--NA","different organisms--different sequencing protocol--different tissue types--NA",null,"different sequencing protocol--different tissue types--NA","different organisms--different sequencing protocol--different cell types--NA","different cell lines","different cell types--different organisms","different organisms--different sequencing protocol--NA","different sequencing protocol--NA","different sequencing protocol--different tissue types--different cancer types--different number of cell types--NA",null,"different organisms--different tissue types--different cancer types--different sequencing protocol",null,"different sequencing protocol--different tissue types--different cancer types--different number of cell types--NA","different organisms--different sequencing protocol--different tissue types--different number of cell types--different number of cells",null,null,"different sequencing protocol--different cell types--different number of cell types--NA","different cell types--NA"],"Diversity_of_experimental_data":["Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes",null,"Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes",null,"Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes",null,"Yes","Yes",null,"Yes",null,"No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","No","Yes","Yes","Yes","Yes","No","No","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Not sure","Yes","Not sure","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes",null,null,"Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes",null,"Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes",null,null,"Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Not sure","Yes","Yes","Not sure","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes",null,null,"No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes",null,"Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes"],"Details_on_discovery":["However, using the mixed-effects model there was a decrease in co-localization between delta cells and beta cells (from beta to delta: P 1⁄4 0.008, Supplementary Table S2) suggesting a pattern to the way that beta cells degrade in the islet.",null,null,null,null,null,null,null,null,null,null,"\"In addition, the H3K27me3 signature of potential immune cells is restricted to cells from the TNBC model (cluster C3), suggesting that these immune cells are absent from the luminal tumor.\"",null,null,null,null,null,null,null,null,"Potentially identified distinct tumour microenvironments in breast cancer tumours.",null,null,null,null,null,null,null,null,"JSTA revealed a non-uniform distribution of subtypes in the CA3 region.",null,null,null,"Association between disease SNPs to APOE gene: \"disease-associated GWAS SNPs (rs112481437 and rs138137383) in astrocyte. Our results indicated that APOE was the putative target gene of these two GWAS SNPs specifically in astrocytes.\"",null,null,null,null,"\"Such findings provide a direction for the discovery of SP3 function in AD studies.\"",null,null,null,null,null,null,"Described the arrangement of synapses in the hippocampus",null,"scAB identifies previously unrecognized cell subsets with high metabolism associated with metastasis and poor prognosis in liver cancer",null,"We applied LCA to 1169 malignant cells from eight tumors and to 2588 stromal cells. LCA identified 18 clusters with a clear biological interpretation. LCA further revealed subtle diversity among the infiltrating T cells.",null,null,null,null,"\"Remarkably, scClassify was able to identify a group of only six cells that were originally labelled as leucocytes and dendritic cells but express high levels of the three top basophil markers, \"",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"\"The new classifications were supported by the high expression level of marker genes for each cell type\"","They identified a new rare subtype of excitatory neurons in the human cortex that expresses the gene RORB. They also identified a new rare subtype of inhibitory neurons in the mouse hippocampus that expresses the gene Npy2r. These new cell types and subtypes were not previously identified using traditional methods, and their discovery can provide new insights into the cellular diversity and evolutionary mechanisms of different species.",null,"revealed a PDCD1/CTLA4 low and TCF7 high T cell subpopulation whose distinct transcriptome was essential to favorable response to immunotherapy.",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"novel drivers of celltypes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Visualization of significantly enriched terms identified enriched functions associated with growth factor and immune cell signaling in addition to expected terms such as xenobiotic metabolism and nuclear receptors in- volved in lipid metabolism\n\nPortal fibroblasts are impli- cated in proliferation of cholangiocytes and the secretion of growth factors during development. Enrichment of these terms suggests a functional role consistent with the induc- tion of bile duct proliferation by TCDD",null,null,"most of these uniquely detected genes have potential to reveal important functional mechanisms",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"We also identify the activator ZNF467, a completely novel regulator, which regulates 86 genes","novel TF binding motifs",null,"\"The highest proportion of tumor cells with T2 network activity was found in the two patients who had a TP53 mutation (p023 and p027). Mutation of TP53 might therefore be associated with a high activity of this network that is associated to cell cycle activity.\"",null,null,"Here, TRIPOD\n110 identifies known trio relationships, as well as putative novel regulatory crosstalk between\n111 neuronal TFs and glial-lineage genes","\" The specific coexpression of CVD genes in these cells may reflect a previously unappreciated role in CVD pathophysiology.\"","\"we simulate and experimentally validate a previously unreported phenotype that results from the loss of noto, an established notochord regulator\"",null,null,null,null,null,null,null,null,null,"TCRGP was able to identify a T cell cluster that was enriched with HBV-targeting clonotypes, which was the most exhausted and least functional. This could provide a mechanism for loss of immunological control against HBV after chronic antigen stimulation which could aid in pathogenesis towards HCC.",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"\"facilitates reannotation of cell types in scATAC-seq and discovery of new subtypes not present in training data\"",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"\"The GLUE alignment helped improve the effects of cell typing in all omics layers, including the further partitioning of the scRNA-seq ‘MGE’ cluster into Pvalb+ (‘mPv’) and Sst+ (‘mSst’) subtypes\"",null,null,null,null,null,null,null,null,null,null,"\"We confirmed that these new factors were positive markers for at least one neuronal subtype.\"",null,null,null,null,null,null,null,"\"In the combined analysis of stimulation and control, we find that the joint embedding generated by scJoint contains further information that allows for the identification of a cellular subtype.\"",null,null,"For example, in monocytes,\nEIF4B and its remote peaks chr12-52884739–52885239 are positively correlated in\nhealthy control and moderately infected patients but are negatively correlated in severely\ninfected patients (Fig. 5D). Such findings indicate that the peak’s positive regulation is\ndestroyed in severely infected patients.",null,null,null,null,null,null,null,null,null,null,"\"By taking the intersection of those genes across three species, we identified 228 highly variable genes that are shared in spermatogenesis across all three species. Such comparisons could give insight into shared and divergent features of spermatogenesis across different species.\"",null,null,null,null,null,"Survival analysis revealed significantly shorter overall survival (~ 1 year median difference) for patients enriched for a margin-biased scHPF astrocyte-like signature (Fig 4G and H), which included astrocytic markers ALDOC, CLU, and SPARCL1 (Bachoo et al, 2004; Zhang et al, 2014, 2016), as well as cystatin super-family members CST1 though CST5 (Figs 3C and EV4A). Cystatin C (CST3) is highly expressed in mature human astrocytes (Bachoo et al, 2004; Zhang et al, 2016) and is induced in Alzheimer's disease and epilepsy (Steinhoff et al, 2001; Pirttilä et al, 2005; Gauthier et al, 2011), raising the possibility that astrocyte-like glioma cells may be responding to the same cues or stresses that reactive astrocytes encounter in these disorders","cNMF was further able to refine cell types by disentangling the contributions of identity and activity programs to the gene expression of cells.",null,null,null,null,null,null,null,null,"new cell transition states",null,null,null,null,null,null,null,null,null,null,"cell subpopulation",null,null,null,"Moreover, we consider that five subtypes may be more suitable than four for KIRC in this dataset. This means that the klDD-clustering method could find new subtypes of cancers and thus has potential applications in medicine or personalized therapy.","identify progenitor state that is depleted with age; Milo identified multiple groups of neighborhoods within the same pre-defined subclusters that showed opposing directions of differential abundance between the control and cirrhotic liver experimental samples, Milo also identified strong differential abundance between healthy and cirrhotic cells in lineages that were unexplored in the original study",null,null,null,null,null,null,null,null,null,"rare cell types","identify gene changes in cell population","rare cell type",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Unlike t-SNE, net-SNE is able to identify the newly added cells as a separate cluster"],"Wet_lab_details":[null,null,"cell culture was done to generate the benchmark data",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"high-throughput RNAi treatment against all 465 expressed TFs in the cell lin",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"we simulate and experimentally validate a previously unreported phenotype that results from the loss of noto, an established notochord regulator",null,null,null,null,null,null,null,"\"To validate the specificity of the BCR sequences assembled by the different algorithms, we produced three mAbs using productive BCR sequences that were assigned to three dif- ferent clonotypes by BRACER using the SMART-seq2 Lei- den dataset obtained in this study.\"",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"\"To validate the reliability of these SNPs, we further generated bulk RNA-seq data of tumor tissue for patient P0411 and detected SNPs from bulk RNA-seq data by aligning reads with STAR and calling SNPs with SAMtools. We found that of all the 5861 sufficiently expressed (read depths > 5 in RNA-seq data) benchmark SNPs called from bulk WES, 97.8% (5827/5861) could also be independently identified from bulk RNA-seq data, supporting the reliability of our benchmark SNPs.\"",null,null,null,null,null,null,null,null],"Recommendation":["Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Not sure","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Not sure","No","Yes","Yes","Yes","Yes","No","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes"],"Applicability":["Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Not sure","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","Not sure","No","No","Yes","No","Yes","Yes","No","Yes","No","Not sure","Yes","Not sure","Not sure","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Yes","No","Yes","No","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Not sure","Yes","No","Yes","Yes","Not sure","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","Not sure","No","Not sure","Yes","No","No","Not sure","Yes","Yes","No","Yes","No","Not sure","No","Not sure","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Not sure","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Not sure","No","No","No","No","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Not sure","No","No","No","Yes","No","No","No","No","Yes","Not sure","No","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","No","Not sure","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes"],"Trade_offs":["Yes","No","Yes","No","Yes","Not sure","Yes","No","Yes","No","No","No","Yes","No","No","No","Yes","No","No","No","No","No","Yes","No","Yes","Yes","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","Yes","No","No","Yes","No","No","Not sure","No","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","No","No","No","No","Yes","No","No","No","Yes","No","No","No","No","Yes","No","No","No","Yes","No","Yes","No","Yes","Yes","No","No","No","No","Yes","No","No","No","No","No","Yes","Yes","Yes","Not sure","No","No","No","Yes","Yes","No","Yes","No","Not sure","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Not sure","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Not sure","No","No","No","Yes","No","Not sure","No","No","No","No","No","No","Yes","Not sure","No","Yes","No","No","No","Yes","Not sure","No","Yes","No","No","No","Yes","No","Not sure","Yes","No","No","No","No","No","No","No","No","Yes","Yes","No","Yes","No","No","No","No","No","Yes","Not sure","Yes","No","Yes","No","No","Not sure","Not sure","Yes","Yes","No","No","Not sure","No","No","No","Yes","Yes","Not sure","No","Not sure","Yes","No","Yes","Not sure","Yes","No","Yes","No","No","No","Yes","No","Yes","No","No","No","Yes","Yes","No","Not sure","No","No","Yes","Yes","No","Yes","No","Not sure","Yes","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Not sure","No","No","Yes","No","No","Not sure","No","Yes","No","No","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","No","No","No","No","Not sure","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","Yes","No","Yes","No","No","No","Not sure","No","No","No","No","Yes","No","Yes","Yes","No","No","No","No","Yes","No","No"],"Future_directions":["Yes","Yes","No","No","Yes","Not sure","Yes","No","No","Yes","No","No","Yes","Yes","No","No","No","No","No","Yes","Yes","No","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","No","No","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","No","No","Yes","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","No","No","Yes","Yes","No","No","Yes","No","Yes","No","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","Yes","No","Yes","No","No","Yes","No","Not sure","Yes","No","No","Yes","No","No","No","Not sure","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes"],"Diversity_of_synthetic_datasets":["Yes",null,null,null,"No",null,null,"Yes",null,"Yes",null,null,null,null,null,null,"Yes","No","Yes","Yes","No","No",null,null,null,null,null,null,null,"No",null,null,"No",null,null,null,null,"Yes","Yes",null,null,null,null,null,null,"Yes","No",null,null,"Yes",null,null,"No",null,null,"Yes",null,null,"Yes","Yes",null,null,"Yes","Yes",null,"No",null,null,"Not sure",null,"Yes","Yes","Yes",null,null,null,null,null,"Yes",null,null,null,null,"No",null,null,null,null,"Yes",null,null,"No",null,null,"Yes",null,null,null,null,null,"Yes",null,"Yes","Yes","Yes",null,null,null,null,"Yes","Yes","No",null,"Yes","Yes",null,"Yes",null,"Not sure","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Not sure","Yes","Yes","Not sure","Yes","Yes","No","Yes",null,"Yes","Yes","Yes","Yes","No",null,"Not sure","Yes",null,null,null,null,"Yes",null,"No",null,"Not sure","Yes",null,null,null,null,"Yes",null,null,"Yes",null,"Yes",null,null,null,null,"Yes",null,"Yes",null,"Yes","Yes",null,null,null,"Yes",null,"Yes",null,null,null,"Yes",null,"Yes",null,null,null,"Yes","Yes","Yes",null,"Yes","Yes",null,"Yes","Yes",null,"No",null,"Not sure","Yes",null,"Yes","No",null,null,"Not sure",null,null,"Yes","Not sure","Yes",null,"No","Yes",null,"Yes","Not sure","Yes","Not sure",null,"Not sure","Not sure","Yes","No","Yes","No","Yes",null,"Yes","Yes","Not sure","Yes","Yes","Yes","No","Not sure","No","Yes","Yes","Yes","No","Yes",null,"Yes",null,"No","No",null,"Not sure","Yes",null,null,"Yes",null,"Yes","Not sure","Yes","No","Yes","Yes","Yes",null,"Yes","Yes",null,"Yes","Yes",null,"Yes","Yes",null,"No",null,"Yes",null,"Not sure","Not sure",null,"No","Yes",null,"Yes",null,"Yes",null,null,"No","Yes",null,null,null,"Yes",null,null,"No","No","Yes",null,"No","Yes","Yes",null,"Yes","Yes","No",null,null,null,null,"No",null,null,"Yes",null,"No",null,"Yes","No",null,null,"Yes",null,"Yes","Yes",null,"Yes","Yes","No",null,null,"Yes","Yes",null,"Yes","No",null,"Yes",null,"No","Yes","No","Yes","Yes",null,null,null,null],"Synthetic_diversity":[null,null,null,null,null,null,null,null,null,"different sparsity--NA",null,null,null,null,null,null,null,null,"different library size",null,null,null,null,null,null,null,null,null,null,"NA--NA",null,null,"NA--NA",null,null,null,null,null,"different number of cell types--different sparsity--different organisms--NA",null,null,null,null,null,null,null,"NA--NA",null,null,"different number of cells--different number of cell types--different cell type composition--different number of batches--NA",null,null,"NA--NA",null,null,"different number of cells--different number of cell types--different number of genes",null,null,"different number of cells--different mislabeled proportion--different sequencing protocols--different number of cell types","different number of cells--different number of genes--different sequencing protocols--NA",null,null,null,null,null,"NA--NA",null,null,"NA--NA",null,"different cell type composition--different organisms--NA","different number of cells","different DE setups--different batch effect--NA",null,null,null,null,null,"different number of cells",null,null,null,null,"NA--NA",null,"NA--NA",null,null,"different number of cells--NA",null,null,null,null,null,"different spread of the genes--different sparsity",null,null,null,null,null,"different number of cells--different batch effect--NA",null,"different mislabeled proportion","balanced and unbalanced","different number of cells--different number of cell types--different DE setups--others--different degree of cluster separability--different degrees of rareness for individual types of cell",null,null,null,null,"different number of cells--different number of cell types--different cell lines--different organisms--different tissues--NA","different spread of the genes--different sparsity","NA--NA",null,null,"different number of cells--different number of cell types--balanced and unbalanced",null,"different number of cells--different sparsity--different number of batches--NA",null,null,"different sequencing protocols--different sparsity--different number of batches--NA",null,null,null,null,"different number of cell types--different tissues--NA",null,"different number of cells--different number of genes--different distribution--NA","different number of cells--different number of genes--different organisms--NA","different cell lines",null,"different number of cells","different spread of the genes",null,null,"different sequencing protocols","different number of cells--different number of genes--different number of cell types--others--different probabilities of expression outliers","NA--NA","discrete and continuous","different number of cells","NA--NA","different number of cells--different number of genes--different number of cell types--different distribution--NA",null,"different number of cells--different number of genes--different number of cell types--different sparsity--NA","different number of samples--NA","different number of clusters","different number of cells--different number of genes--different sparsity--others--different probabilities of expression outliers",null,null,null,"discrete and continuous",null,null,null,null,"different number of cells",null,null,null,null,"different number of cells--different number of genes--different number of cell types--different sparsity--different distribution--others--different long and short range interaction with different spatial distance.",null,null,null,null,"different organisms--different cell lines",null,null,"different spread of the genes--different sparsity",null,"different number of cells--different number of genes--different sparsity--NA",null,null,null,null,"different number of cells--different degrees of cluster separability--NA",null,"different number of cells--different number of cell types--different organisms--NA",null,"different spread of the genes","discrete and continuous--NA",null,null,null,"different number of cell types--NA",null,null,null,null,null,"different number of genes--different sparsity--discrete and continuous--NA",null,"different number of cells",null,null,null,"different number of cells","different library size","different cell lines",null,"different spread of the genes--different number of cells","different number of cells--different number of genes--different number of cell types--different sparsity--NA",null,"different number of cells--different number of genes--NA","different number of cells--different number of genes--NA",null,null,null,null,"different sparsity--NA",null,"different sparsity--NA",null,null,null,null,null,null,"different sparsity--different distribution--different degrees of cluster separability--NA",null,"different number of cells--different number of genes--different tissues--NA",null,"NA--NA","different tissue types",null,"different number of cells",null,null,null,null,null,null,"different sparsity","NA--NA","different number of cells--different number of genes--different number of cell types--different batch effect--different cell type composition--different number of batches--NA","NA--NA","healthy and diseased",null,"different sequencing protocols","different number of cells--different number of cell types--NA","NA--NA","different number of cells--different sparsity--different gene regulatory network--NA","different cell type composition--NA","different library size",null,null,"NA--NA","different number of cells--NA","different cell type composition--NA","different sequencing protocols--different number of cells",null,"different cell type composition--others--different integration methods",null,"different DE setups--different batch effect--NA",null,"NA--NA",null,null,"NA--NA","different number of cells--NA",null,null,"different sequencing protocols",null,"different cell type composition--different spread of DS genes--NA","NA--NA","different number of cells--different number of cell types--NA","NA--NA","different number of cells--different number of cell types--NA","different number of cell types--NA","different number of cells--different DE setups--balanced and unbalanced--different batch effect--NA",null,"different number of cell types--different cell type composition--NA","different sparsity--different tissues--healthy and diseased--NA",null,"different DE setups--different spread of DS genes--NA","different spread of the genes",null,"different DE setups--different cell type composition--NA","different spread of the genes--different sparsity",null,"NA--NA",null,null,null,"NA--NA",null,null,null,"different number of cells--different number of cell types--NA",null,"different spread of the genes",null,"different sequencing protocols",null,null,"NA--NA","different number of cells--different number of cell types--NA",null,null,null,"NA--NA",null,null,"NA--NA","NA--NA",null,null,"NA--NA",null,"different sequencing protocols--different number of cell types",null,"different sparsity","different simulation methods","NA--NA",null,null,null,null,"NA--NA",null,null,"different number of cell types--different dynamic processes--NA",null,"NA--NA",null,"different number of cells--different cell type composition--different sparsity--NA","NA--NA",null,null,"different sparsity--different noise levels--NA",null,"different number of cells--different sequencing protocols--NA","others--different variance in each cell class",null,"different number of cell types--different sparsity--different DE setups--different dynamic processes--NA","different number of cells--different sequencing protocols--NA","NA--NA",null,null,"different distribution--different noise levels--NA","different sparsity--different DE setups--different noise levels--NA",null,"different number of genes",null,null,"different number of cell types--different noise levels--NA",null,null,"different number of cells--different sequencing protocols--different cell type composition--different degrees of cluster separability--NA",null,"different number of cells--different cell type composition--different sparsity--NA","different number of cells--different distribution--different noise levels--NA",null,null,null,null],"Details_on_selection_criteria":[null,null,"normalisation - eight popular normalization methods , clustering -  five representative clustering methods ,  data integration - state-of-the-art methods including xxx","They included all methods",null,null,null,null,null,"several classical and recently published methods",null,"\" Based on a recent scATAC-seq benchmark, we selected the top- performing methods, namely Cusanovich, SnapATAC, and CisTopic. \"","Cell Ranger, Optimus, salmon alevin, alevin-fry, kallisto bustools, dropSeqPipe, scPipe, zUMIs, celseq2, and scruff","One method for each feature extraction category: \"no integrative methods are available for a cross-modality analysis of scATAC-seq data, yet it has been shown that the peaks, genes, and motifs all contain signals to separate the different cell types/states.\"",null,"Five studies supplemented their publication with material allowing to repeat, at least partially, their computational analysis.",null,null,null,null,"Paper included current analysis tools",null,null,null,null,"8 normalisations, 3 imputations, 7 clustering, 5 trajectory, 6 data integrations",null,null,"7 pre-processing, 8 normalisation, 7 batch correction",null,"We included in the comparisons\nseveral other state-of-the-art methods",null,null,null,"we compared SEDIM with five other state-of-the-art imputation methods including DeepImpute, DCA, MAGIC, SAVER and PRIME","They wrote \"Compare with five other state-of-the-art imputation methods ...\" To me this statement is too weak to considered \"selection criteria\".","To showcase the effectiveness of GraphST in spatial clustering, we compared GraphST with seven state-of-the-art methods, the nonspatial method Seurat7 and spatial methods Giotto8, SpaGCN9, SpaceFlow14,conST15, BayesSpace11, and STAGATE12.\n\nTo evaluate the performance of multiple tissue slice integration, we compared GraphST with Harmony16,scVI17, and STAGATE for vertical integration and STAGATE and SpaGCN for horizontal integration.\n\n\nFor ST cell-type deconvolution with scRNA-seq data, GraphST was compared with cell2location, which had been benchmarked to be the best existing method41,aswellasSeurat,RCTD,SPOTlight,andNNLS.",null,null,null,null,null,"\"Recently, two methods have gained popularity in the single cell field:\"","\"5 most common alignment tools\"",null,null,null,"hree methods designed for scRNA-seq data (Scissor (9), scPrognosis (7) and DEGAS (10)), one baseline method (marker) and using clinical variable (I grouped all different clinical as one method)","state-of-the-art methods",null,"Currently, only two approaches incorporate multiple reference datasets to the method level, namely, mtSC and scMRA.",null,null,null,null,"Eight of them are popular methods developed specifically for single-cell research, including Seurat, scmap, SingleR, CHETAH, SingleCellNet, scID, Garnett, and SCINA. The other two methods were repurposed from deconvoluting DNA methylation data, i.e., linear constrained projection (CP) and robust partial correlations (RPC).  We only considered pre-printed or published methods with detailed documentation on installation and execution. We excluded any methods that required extensive runtime, and where we were unable to customize the reference dataset, or inconsistent predictions were produced. we only selected packages in R with good documentation, as R is still the most popular bioinformatics platform for open-source scRNA-seq analysis packages.",null,"on their popularity and state-of-the-art performance in single-cell classification",null,"It compared to the state-of-the-art methods",null,null,"To evaluate all tools in the classification category of the website scrna-tools.org",null,null,"n this study, 13 popular supervised machine learning algorithms for cell phenotype classification were evaluated",null,"Seurat, a widely used software package for scRNA-seq analysis","In this study, 13 popular supervised machine learning algorithms for cell phenotype classification were evaluated using published real and simulated datasets with diverse cell sizes.","comphrensive for single cell classification","they selected five classical methods for comparison based on the following criteria:\nThe methods should be widely used in the field of scRNA-seq data analysis.\nThe methods should be able to handle cross-species data.\nThe methods should be able to perform cell-type assignment and gene module extraction.","Three off-the-shelf supervised learning, two supervised cell typing, two supervised deep learning and two semi-supervised deep learning methods.",null,"Code availability","But according to a recent comparison [20], SVM with rejection, scmap, and CHEAH are among the best performers, so we decide not to include more such methods.","One or two methods from each category of method. Different generations.",null,null,"To leverage existing deconvolution methods for bulk omics data, we also included two modified methods: linear constrained projection (CP) and robust partial correlations (RPC) that are popular in DNA methylation analysis","We employed 10 general-purpose classifiers from the scikit-learn [15] library in Python","Some tools were excluded from this evaluation due to various reasons (Supplementary Table S5).",null,"We chose these five methods to represent different categories of algorithms, ranging from first-generation enrichment analysis (ORA) to secondgeneration approaches (GSEA and GSVA), machine learning tools (CIBERSORT) and network-based neighbor voting approaches (METANEIGHBOR).",null,"The six mainstream single-cell assignment strategies were selected based on their popularity and effectiveness in the field. The paper also excluded methods that require prior marker gene information or do not offer the unassigned option for cell type identification.",null,"\" The methods were chosen from the scRNA-tools database [43], representing a wide range of popular clustering algorithms used for cell clustering and the number of cell type estimation from scRNA-seq data (Fig. 1a). \"",null,"We evaluate Secuer against three extensively used methods for scRNA -seq clustering including Louvain, Leiden, and k-means,","Representative","Among them, PhenoGraph, Xshift, FlowSOM, and flowMeans are the best-performance tools in a previous comparison of unsupervised tools by Weber and Robinson [25], DEPECHE is a novel clustering tool [19] for cytometry data that had not been analyzed by Weber et al. [25], and LDA and ACDC are top-performance tools in a previous comparison of semi-supervised tools by Abdelaal et al.","Fourteen clustering methods, publicly available as R packages or scripts, were evaluated in this study (see Table 2 for an overview). We included general-purpose clustering methods, such as hierarchical clustering and K-means, as well as methods developed specifically for scRNA-seq data, such as Seurat and SC3, and methods developed for other types of highthroughput single-cell data (FlowSOM). The collection of methods include representatives for most types of algorithms commonly used to cluster scRNA-seq data.",null,null,null,null,"Seven unsupervised methods (Accense, Xshift, PhenoGraph, FlowSOM, flowMeans, DEPECHE, and kmeans) and two semi-supervised methods (Automated Cell-type Discovery and Classification and linear discriminant analysis (LDA))","To benchmark the clustering performance of scMINER, we considered the three widely-used methods: Seurat10 and Scanpy12, representing graph-based approaches, and SC311 representing k-means-based algorithms.","The four baselines are in three variants of clustering algorithms that cover the entire representative clustering approaches: (i) Algorithms that require the number of clusters as input, such as K-means [17]andSC[19]; (ii) Algorithms such as Greedy [48] that do not require the number of clusters, however, hyperparameter settings should be given; iii) Algorithms that estimate the number of clusters automatically, i.e. FINCH [49]. Three tools for analyzing scRNA-Seq data are SEURAT [11, 12], SINCERA [14] and SNN-Cliq [13].",null,null,"We exemplify the protocol by conducting a comparative study between three clustering algorithms",null,"For the sake of code compatibility and transparency, we restricted our choice to the methods implemented in the R programming language.","Although there are more currently available scRNA-seq clustering methods in addition to those tested in this study, I only selected particular methods, which were implemented in the R environment. Given the rather limited computational resources in my lab, the low memory requirements of R platform allowed me to conduct robust analyses in this study.",null,"We focused on analysis methods available in the R language, as this is one of the most commonly used programming languages for scRNA-seq data analysis.",null,null,"Although there are more currently available scRNA-seq clustering methods in addition to those tested in this study, I only selected particular methods, which were implemented in the R environment.","\"The collection of methods include representatives for most types of algorithms commonly used to cluster scRNA-seq data.\"","Louvain and Leiden with default resolution, two widely-used methods in scCAS data analysis, and scCCESS and scLCA (Cheng et al., 2019), two of the best methods in the most recent benchmark study for scRNA-seq data (Yu et al., 2022)",null,null,"“For the sake of code compatibility and transparency, we restricted our choice to the methods implemented in the R programming language.”","benchmark study of available single-cell Hi-C data clustering methods","we benchmarked 46 combinations between ten BEC methods (ZINB-WaVE16, MNN17, scMerge18, Seurat v319, limma_BEC10, scVI20, scGen21, Scanorama22, RISC23 and ComBat24), covariate models, three meta-analysis methods (weighted Fisher (wFisher)12, fixed effects model (FEM)11 and random effects model (REM)11), observation weights of ZINB-WaVE25, pseudobulk data26 and seven DE methods (DESeq29, edgeR27, edgeR_DetRate28, limmavoom10, limmatrend29, MAST30 and Wilcoxon test). These combinations are denoted as DE workflows in this article.","representative five","The paper selected differential expression testing methods or their multiple group equivalent based on previously reported performance, ability to consider multiple groups, or whether they served as foundation for the scBT and multiple group LRT (LRT multiple) tests developed.",null,"\"compared five representative stat- istical methods that detect DE genes between single-cell popu- lations\"","\"Representative (We include a broad range of methods developed specifically for scRNA-seq DE analysis) \"","Some of the popular tools","Our fit-for-purpose tests were benchmarked to existing dif- ferential expression testing methods or their multiple group equivalent based on previously reported performance, abil- ity to consider multiple groups, or whether they served as foundation for the scBT and multiple group LRT (LRT multiple) tests developed here.",null,"\"As of October 2018, we have identified eight software tools, We also considered tools designed for bulk RNAseq data that are widely used [4, 43] (edgeR, and DESeq2) or can apply to multimodal data [31] (EMDomics)\"","representative methods, designed for bulk, meta genomic differentiation, commonly used nonparametric method",null,null,null,null,"Four methods developed for differential expression analysis of genes between single-cell populations: MAST, SCDE, Monocle and D3E.\nTwo widely used DE methods originally developed for bulk RNA-seq data, DESeq and edgeR.","These nine methods are selected to cover most statistical models used in recent DE analysis. Regarding other DE methods that are not included in this study, they use similar approach comparing to the nine selected methods.",null,"recommended from another paper",null,null,"11 common dimensionality reduction methods",null,null,null,"dimensionality reduction methods in scRNA-seq data to date","To our knowledge, about 10 methods are now available to obtain a low-dimensional representation for scRNA-seq data","we introduce and perform a systematic evaluation of popular DR methods, including t-SNE, art-SNE, UMAP, PaCMAP, TriMap and ForceAtlas2","9 jDR approaches representing 8 mathematical formulations, focusing on methods able to combine more than two omics, implemented in R or Python, and with software readily available and documented",null,"Here, we report an in-depth comparison of nine representative state-of-the-art multiomics joint Dimension Reduction (jDR) approaches, in the context of cancer data analysis.","\"We ran five representative DR methods\"","In the present study, we have collected and compiled a list of 18 popular and widely used dimensionality reduction methods in the field of scRNA-seq analysis.",null,null,"Most common nonlinear dimensionality reduction in single-cell analysis and the autoencoder neural network techniques","five popular and representative DR methods","only dimensionality reduction methods with open-source implementations available in Python 3 were selected for inclusion. Methods were excluded if they did not possess sufficient documentation, raised difficult to fix errors at runtime or only exposed a command-line interface.",null,null,"\"In this study, we investigated the performance of five commonly-used network methods originally developed for bulk sample data, plus three single cell-specific network methods, for reconstructing gene regulatory networks from single cell gene expression data\"","Available in python and R, excluded methods that use pseduo-time","They selected \"general methods\" to compare against sc methods. \"In this study, we investigated the performance of five commonly-used network methods originally de- veloped for bulk sample data, plus three single cell-specific network methods, for reconstructing gene regulatory networks from single cell gene expression\"","They talk about 3 types of methods but didn't really talk about selection criteria.  Its an implicit criteria rather than explicit.","Starting from methods included in past large-scale benchmarks, filtered to ones that did not need pseudotime, with algorithm provided in R or Python code. (One more method excluded as it took too long to run for the scale of the benchmark)",null,"In the sections that follow, we go into detail about the comparisons between two popular approaches that only consider gene expression, GENIE342 and GRNBoost243, as well as prior-based approaches like NetREX-CF, MerlinP7, NetREX2, LassoStARS6, the original CF18, the summation of all prior knowledge (PriorSum), and a technique that assigns a random confidence score (uniformly distributed between 0 and 1; hereafter, a random method)","Last 4 years and can run smooth ... shown in Table 1 (Page 2 column 1)","We ignored methods that did not assign weights or ranks to the interactions, required additional datasets or supervision, or sought to discover cell-type-specific networks. We selected 12 algorithms using these criteria","we present an expanded benchmarking study of eleven recent network inference methods",null,"\"Here, we review 15 computational methods that were specifically developed for GRN inference using single-cell data.\"","They benchmarked measures of association like spearman, kendell, mutual information etc....",null,"Use the combination of 4 sample-specific network reconstruction methods and 4 representative structural control methods","we tested and compared nine of the most recent computational tools for LR-based CCI prediction",null,"“similarity measures that are widely used by various clustering algorithms”","\"We here focus on four state-of-the-art metrics, henceforth called baseline metrics,\"",null,"We benchmark 16 cell-cell interaction methods by integrating scRNA-seq with ST data",null,"They selected 'recent' scRNA-seq GRN inference methods that have available and executable software. Others were excluded","we developed a framework for the in-depth evaluation of the performance of 17 proximity metrics with respect to core structural properties of scRNA-seq data, including sparsity, dimensionality, cell-population distribution and rarity",null,null,"\"Commonly used metrics in clustering. k-means was selected as it is the simplest clustering algorithms widely used. SIMLR was a state-of-the-art methods recently developed. Also published in Nature methods\"","we additionally benchmarked our explainable AI approach against popular ‘average’ network prediction algorithms.","recently published methods that span a variety of modeling formalisms","selected representative methods",null,"\"ranging from measures that are broadly used in analysis of gene expression data to measures that have not, to our knowledge, previously been applied in this setting.\"","they just pick some common GRN inference methods",null,"widely-used",null,"They collected seven widely-used pathway activity transformation algorithms",null,null,"\"We included both bulk- and single-cell-based tools that esti- mate either TF or pathway activities from gene expression data and for which well-defined benchmark data exist. \"",null,null,"Currently only a few methods for predicting the epitope-specificity of TCRs have been published, including TCRdist [10] and a random forest (RF) classifier by De Neuter et al.",null,null,null,null,"To evaluate the imputation performance, we compared scESI with some competitive nine state-of-the-art methods",null,null,"spatial k-nearest neighboring baseline and other imputation methods designed for single-cell RNA sequencing.",null,null,"popular ones","a total of 21 state-of-the-art imputation methods designed specifically for scRNA-seq data imputation, ... We attempted to implement all the methods, but only 12 out of them were performed successfully,","\"These include all of the published imputation methods, at present\"","we select several state-of-the-art scRNA-seq imputation methods (including MAGIC, SAVER, scImpute, deepImpute, PRIME, bayNorm and knn-smoothing) in recent years to infer dropout peaks in scATAC-seq data, and perform a systematic evaluation of these methods through several downstream analyses.","four latest imputation approaches","included methods up date, only those with available software, and methods that do not require any external information to be run successfully besides scRNA-seq data","Here, we evaluate BayesSpace’s ability to identify distinct layer-specific expression profiles and compare its performance to other spatial and non-spatial clustering methods.","we applied the scRNA- seq pooled canonicalization method, which has previously been proved to be superior to other methods","popular methods","16 representative and open-source models",null,null,"Methods can be conceptually distinguished in marker-gene-based approaches (M) and deconvolution-based approaches (D).  All methods focus on immuno-oncology (so its a subset of all other methods)",null,null,"\"popular data integration tools\"",null,null,"(1) The original models of the selected methods have open-source code, and \n(2) the original models used multi-omics fusion and the data fusion part of the original model can be extracted separately so that we can evaluate the data fusion on unified datasets and tasks",null,null,null,null,null,"“We benchmarked 16 popular data integration methods”",null,null,"Given the large variety of different integration approaches, we performed a systematic evaluation of ten integration methods by selecting and grouping approaches according to two categories: early integration approaches and intermediate integration approaches.","Here, we present a comprehensive evaluation of the performance of 14 deconvolution methods","In this study, we compare 12 cell type quantification methods and evaluate their performance using each of 10 separate reference profiles",null,"\"We first benchmarked GLUE against multiple popular unpaired multi-omics integration method\"",null,null,"\"popular batch effect removal methods\"",null,"No strong justification just \" using four methods: our SMNN, published MNN, Seurat v3 and LIGER\" as these are current popular methods for batch effect correction","We focus this benchmark on methods that provide cell-type signatures for immuno-oncology",null,"Availability in R and/or Python environment","\" either widely used or have been recently developed\"",null,null,"The aim was to present a variety of metrics, with the implicit understanding that the ones used were the more popular ones.",null,"Current, state-of-the-art without defining what those subjective terms meant.",null,"we compared DeepST with existing state-of-the-art methods\non the above-benchmarking dataset. Specifically, DeepST\nwas compared with two non-spatial algorithms (K-means\nand Seurat (24)) and four recently published spatial clustering\nalgorithms (stLearn (6), SpaGCN (7), SEDR (8), and\nBayesSpace (5))",null,"We evaluate eight transcriptome deconvolution approaches",null,"we selected three representative baseline strategies; six representative intermediate methods","They selected \"state-of-the-art methods\". They excluded a method (MIA) that did not have code, and another method (DSTG) that could not output cell type information. Other than that, no detailed selection criteria were provided.",null,"we selected widely employed methods, and where possible included methods developed for brain data","We benchmark CLAIRE against eight state-of-the-art batch correction methods on six real datasets. Three classical methods, Seurat, Scanorama and Harmony, and four CL-based methods, INSCT, MAT2, SMILE and CLEAR are included for comparison. In addition, we also include iSMNN",null,null,null,"we benchmark 18 existing methods resolving a cellular deconvolution task",null,"comprehensive, included all batch corrected methods at the time","We have selected 12 commonly used cell type quantification tools. The tools included in this benchmark fall into two basic categories based on their approach to cell type quantification. The first category is deconvolution tool. ;Also included in this study are four signature quantification tools.",null,"“We benchmarked Portal and existing representative methods, including Harmony, Seurat v3, online iNMF, VIPCCA, scVI, fastMNN, Scanorama and BBKNN, following a recent benchmarking study”","comparison algorithms on scRNA-seq data sets","\"commonly used differential analysis methods\"","existing tools for the functional interpretation of scRNA-seq dat",null,"\"commonly used differential analysis methods\"","We compared scHPF's predictive performance to that of PCA, NMF, and factor analysis (FA), as well as two methods developed specifically for scRNA-seq: zero-inflated factor analysis (ZIFA; Pierson & Yau, 2015) and zero-inflated negative binomial-based wanted variation extraction (ZINB-WaVE; Risso et al, 2018). These methods have been used for expression program discovery without a predefined structure across cells (Patel et al, 2014; Pierson & Yau, 2015; Tirosh et al, 2016a; Chung et al, 2017; Puram et al, 2017).",null,null,null,"commonly used normalization methods",null,"We compared seven existing normalization approaches that aim to reduce noise or bias specifically for scRNA-seq data.",null,null,null,null,"Methods were excluded from the evaluation based on several criteria",null,"“Methods were excluded from the evaluation based on several criteria. (1) not freely available; (2) no code available; (3) superseded by another method; (4) requires data types other than expression; (5) no programming interface;\n(6) unresolved errors during wrapping; (7) too slow (requires more than 1 h on\na 100 × 100 dataset); (8) does not return an ordering; and (9) requires additional user input during the algorithm (other than prior information). ”",null,null,null,null,"\"five widely used quantification tools\"",null,null,null,null,"All four potency models use a notion of entropy to quantify differentiation potency","\"We tried to compare the performance of SparseDC to the time-variant clustering (TVC) algorithm presented by Huang et al.(26), which is the only other algorithm currently available for DC analysis\"",null,"two alternative methods for clustering-free differential abundance analysis","including SR/SCENT (Teschendorff and Enver, 2017) and CytoTRACE (Gulati et al., 2020), two of the best performing methods in a recent comparative study",null,null,"\"nine cutting-edge computational doublet-detection methods\"",null,null,"“five popular isoform quantification tools.”",null,"comparison to \"state-of-the-art\"",null,null,null,"Algorithms for detecting rare cell transcriptomes are scarce. Prominent among these are rare cell-type identification (RaceID)15 and GiniClust16. Besides GiniClust and RaceID, we compare FiRE with a rare event detection algorithm called local outlier factor (LOF).","\"An extensive literature review was conducted and a total of 12 published single-cell simulation methods with implementation available in R and Python was found\"",null,"We include it here as a clear contrast with the state-of-art methods",null,null,null,null,null,null,"We included all methods that i) could be installed and run after at most minor manual adjustment(s); and, ii) were reference-based, i.e., supported parameter estimation from a real dataset.",null,"\"We included all methods that ...\"","To the best of our knowledge, there are two power assessment methods for scRNA-seq data: powsimR (Vieth et al., 2017) and scDesign","The software packages that we compare here are aimed at generating synthetic bulk RNA-seq data and take as input either a combination of parameters, a count dataset or both. They have been developed with a focus on the following three major goals:...",null,"two state-of-the-art tools, namely Splat, v1.13.0 (splat, splat-dropout and kersplat) and scDesign2, v0.1.0, on 13 datasets.",null,null,null,null,null,null,null,null,null,"baseline methods",null,null,"\"most frequently used framework\" , \"have been widely used to detect variants \", \"have also been sporadically used \"","\"Since DENDRO is currently the only method for SNA-based subclone detection using scRNA-seq data alone, we benchmarked against more straightforward approaches such as hierarchical clustering based on mutation allele frequencies and genotypes respectively.\"",null,null,"These three clustering methods are model based algorithms that have been used in recent single-cell studies","We initially consider 13 popular interactive analysis and visualization tools and give an overview of their features. We then select those tools that provide web sharing functionality and benchmark them against each other by means of their performance on datasets of different sizes","We select those tools that provide web sharing functionality and benchmark them against each other by means of their performance on datasets of different sizes (from 5000 to 2 million cells).We also evaluated user experience (UX) features of these tools.",null,"standard methods, such as t-stochastic neighbor embedding (t-SNE), are not scalable to datasets with millions of cells and the resulting visualizations cannot be generalized to analyze new datasets"],"Speed_measured_":["Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes"],"Overall_comparison":["No","No","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","No"],"Multi_omics_integration__unmatched____others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"partially matched case: MERFISH with some cells, 10x with other cells from the same region",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"Batch_correction___others":[null,null,null,"removal of any known batch effect in the reduced feature space",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"batch effects in UMAP plot",null,null,null,null,null,null,null,"iLISI",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"impact of HVG selection or scaling--recovery of cell states or modules--conserving trajectory --conserving HVG genes--dealing with varying compositions--integration performance on scRNA-seq data--integration performance on scATAC-seq data","number of batches--number of cells in each batch",null,null,null,"effect of simulated mislabelling on the batch correction",null,null,null,null,null,null,null,null,null,null,null,null,null,"impact of number of cells",null,null,null,null,null,"downstream analysis on DE",null,null,"qualitative and quantitative nature of batch effects",null,null,"big data--multiple batches",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"cross species batch correction",null,null,"maintaining rare cell population--integration across multiple species--preserving gradual change of cells",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"If_you_paper_category_is_not_covered__please_write_down_what_criteria_you_paper_looked_at__thanks_":[null,null,null,null,null,null,"- cell type consistency (silhouette width) \n- protein quantification consistency (pearson correlations)\n- clustering \n","Clustering performance on simulated datasets--Clustering performance on real datasets--Effect of keeping the first PC vs removing the first PC on clustering--Clustering performance when running methods using default setting versus using optimised settings--The effect of excluding regions using the ENCODE blacklist annotation--Rare cell type-specific peak detection--Frequency-based peak selection vs intensity-based peak selection",null,null,null,"ability to batch correct--ability to deconvolves the heterogeneity of chromatin states",null,null,null,"single-cell proteomics(SCP) data analysis",null,"clustering accuracy","TPR--FDR--pAUC--MCC--deviance in library size estimates (RMSE)--computational run time",null,null,"overlap in genes detected--correlation of test statistic for overlapping genes--detecting 2 groups with different spatial patterns","data acquisition and preprocessing pipeline, different normalisation methods","Data communication, visualization and interactive resource.",null,null,"a cloud-based framework for analysis pipeline","correlation between expressed gene and HVG-CNV concordance",null,"Correlation of segmented MERFISH with scRNAseq","average segmentation precision","number of cells detected--fraction of molecules assigned to cells--mutual information with paper--correlation between parts of cells between 2 methods--density","Chromatin Conformation Reconstruction\nDistance of predicted structure to simulated structure.","Chromatin Interactions\n\nPrecision and recall to bulk chromatin interactions itdentfied on same cell type.","Imputation",null,null,null,"imputation: distance from truth (L1 distance, cosine similarity, RMSE)","comparison of alignment statistics--median gene count--impact of reference choice and alignment pipeline on alignment statistics",null,"the accuracy of gene counts","correlation of alignments from STAR and Kallisto workflows--whether the alignment methods preserve the variability of gene expression similar to the RNA Fish method--comparison of alignment method on bulk RNA-Seq data, to STAR and Kallisto on gene detection","Read mapping rate--number of genes per cell--runtime--concordance of log-fold changes","gene counts per cell--gene expression level--cell counts",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"segmentation accuracy--Pearson correlation between cell-type-by-gene-expression matrices",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Dimension reduction",null,"global and local structure preservation in dimensionality reduction transformations","performance in terms of tsne/umap (clarity of the cluster structures)--clustering accuracy--higher PCs (whether higher PCs, which correspond to lower eigenvalues, are accurately calculated)--outlier robustness (outlier cells)--magnitude of eignevalues",null,"- how well the low-dimensional space allows simple methods to recover simulated cell types (k-means based and kNN based) \n- the extent to which clusters are separable in the latent space (average silhouette score)",null,"dimensionality reduction","Dimension reduction (DR) algorithms",null,null,null,"silhouette score--AMI after k-means clustering in the representation space","normalized mutual information (NMI) and adjusted rand index (ARI) values for comparing the true cell labels and inferred cell labels obtained by clustering methods based on the low-dimensional components-- evaluated by Kendall correlation in the downstream trajectory inference analysis","Preservation of unique distances following dimension reduction (Pearson correlation)-- Structural alteration of the cell distance distribution (EMD distance)--Preservation of Knn graph before and after dimensional embedding",null,null,null,"internal validation metrics--clustering accuracy--structure preservation","compared the cross product between the PCs from this gold standard","- ‘unsupervised’ (visualization and clustering)\n- ‘supervised’ (classification)",null,null,null,"Cell cell communication",null,null,null,null,null,"network inference methods",null,null,"measures of association",null,"Sample-Specific network Control (SSC) analysis","cell–cell interaction",null,"Similarity metrics--clustering performance (through NMI, ARI, FM, Jaccard)--concordance among the the evaluation measures--modification of clustering algorithm using the proposed similarity metric--effect of library preparation protocols on clustering",null,null,"cell-cell interaction",null,null,"quality of single-cell clustering depends on the proximity metric",null,null,"performance on clustering (ARI, NMI, FM, Jaccard)",null,null,"AUPRC ratio--early precision--EPR activation--EPR inhibition--stability across multiple datasets--stability in the presence of dropouts--effect of the number of genes--effect of including all significantly varying TFs",null,"measures of association--Functional coherence --impact of the number of cells profiled on functional coherence--impact of the proportion of dropouts on functional coherence--reproducibility across datasets",null,"- Pair Sets Index \n- Adjusted Rand Index \n- Adjusted Mutual Information","separation between cell types based on PAS based on mean silhouette widths--separation between cell types using clustering--separation between cell types through classification--correlation between two metrics--correlation between runs of drop=out events--correlation between different sequencing technologies",null,null,null,"comparison of the percentage of unregulated and down regulated signatures between tumor and normal cells--detection sensitivity--detection specificity--how changes in gene count affect score stability--how dropouts affect score --consensus between methods","Robustness of bulk-based TF and pathway analysis tools against low gene coverage--Performance on simulated data (AUROC)--Performance on real data (AUROC)","V-(D)-J gene overlap between the methods--Effect of coverage and read length on BCR assembly, productivity and sensitivity--Effect of somatic hypermutations ( SHMs ) on BCR assembly, production and accuracy",null,"- cross-validated ROC curves on epitope specificity prediction\n- impact of diversity\n- impact of number of samples \n- impact of using different CDRs \n- evaluation on independent study",null,"Imputation",null,null,"imputation",null,null,null,"similarity to original value--preservation of biological structure",null,"- dropout imputation followed by clustering accuracy ( ARI, Jaccard, Purity, NMI, DBI, connectivity, Dunn index) \n- cell - cell similarities","data imputation","gene correlation between the lowly expressed DE genes--impact of parameter choice on the imputation result--Accuracy of identifying DE genes between the groups--Number of false positive DE genes introduced by imputation--Reproducibility of markers in imputed and un-imputed data--Distortions of gene expression values","Imputation","imputation",null,null,null,"- clustering performance (Adjusted Rand Index (ARI)52, Jaccard Index (JI)53, and Purity Index (PI)) \n- Preservation of the transcriptome landscape ( distance correlation index (dCor)) \n- Similarity between imputed and ground truth ( mean absolute error (MAE) and correlation )",null,"deconvolution",null,null,null,null,null,null,null,null,null,null,null,"Impact of data transformation on deconvolution results--Performance of different combinations of normalization and deconvolution methods--Impact of the markers used in bulk deconvolution methods--Effect of removing cell types from the reference matrix--Deconvolution of real bulk heterogeneous samples","deconvolution",null,null,"how well integration methods can predict spatial distribution of undetected transcripts in ST data--how well integration methods can do cell-type deconvolution on spots of histology slides--tenfold crossvalidation, and used various performance metrics (PCC, SSIM, RMSE and JS), which combine into what they call \\accuracy score\\\"",null,"root mean square error--pearson correlation coefficient--jensen-shannon divergence.","deconvolution method",null,null,null,"different magnitude of batch effect (mild, severe, mixed)--cell composition imbalance--non-overlap cell groups-multiple cell groups--variable library size--variable dropout rate",null,"overlapping cell types but different cell proportions and magnitude of batch effects between two batches--nonoverlapping cell types between two batches--scenarios with multiple batches--scenarios with variable dropout rate and library size",null,null,"deconvolution",null,null,"prediction - spatial distribution of RNA transcripts",null,null,"deconvolution: MSE between true and predicted proportions of cells",null,null,null,null,null,null,"evaluate on its performance in trajectory inference using TI correlation score--perturbation response using AUC--balanced accuracy and F1 score--disease classification using AUC--balanced accuracy and F1 score","Examined the stability of each method",null,"deconvolution",null,null,"- the performance of deconvolution using Jensen–Shannon divergence (JSD) score, root-mean-square error (RMSE) and Pearson correlation coefficient (PCC)","Data preprocess/processing pipeline","cellular deconvolution of spatial transcriptomics",null,null,"deconvolution",null,null,null,"correlation of the top genes selected on rare cell types--the area under the receiver operating characteristic curve based on either the P-values or ranks of the genes by all the differential statistics",null,"feature selection","differential stability genes--global ranks of the differential statistics (correlation)--gene set enrichment on the top ranked genes",null,"- activity GEP deconvolution success rate\n- detection of genes in activity program",null,null,"clustering correctness (Rand index)--clustering compactness (Dunn index)--clustering robustness (Jaccard index, robustness analysis using different sets of samples and genes)--impact of ERCC for clustering",null,null,null,null,null,null,null,"total UMI amount assigned to exotic and intronic targets by each method--distribution of UMI counts between spliced and unspliced target--inferred velocities between quantification methods--performance on negative control datasets (with no strong systematic dynamic)",null,null,null,null,null,"Concordance between methods of total read counts assigned to exons and to introns--Concordance of velocity projections--Higher proportion of cells with basically 0 velocity in negative control",null,null,null,"potency model--ability to discriminate single cells (and bulk samples) that ought to differ substantially in terms of their potencies--ability to adjust for cell-cycle phase (which is a confounding factor)--prediction of differentiation time points",null,null,null,null,"discrimination accuracy (AUC)",null,null,null,null,"Isoform detection F1 score--isoform abundance","isoform quantification--the ability of each method to detect the presence of an isoform (precision and recall)--estimation of isoform abundance (Spearman’s rho and the normalized root mean square error (NRMSE)) --impact of removing drop-outs on the performance of isoform quantification tools","Demultiplexing\n\nAccuracy in identifying known substitution errors.\nF-score against substitution error rate","Empty droplet detection\n\nUse of cell hashing and genetic variation to provide ground truth\nAccuracy, ROC curve, downstream analysis",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"FDR--overlap in genes--proportion of true positives--covariates","null case with no SVGs--different spatial patterns to determine detection of true genes","detection of true genes","ARI","HVG detection--clustering performance--accuracy + precision + sensitivity--number of HVGs detected--reproducibility of detected HVGs--overlaps between methods--dependence with mean--computational runtime","Accuracy, precision and sensitivity using simulated data set--Number of detected HVGs--Overlap within methods--Overlap between methods--Dependence with the mean",null,"scDNA-seq Copy Number Estimation\nF1 score of estimates compared to simulated values.","Deviation of single-cell RNA estimated copy number from whole genome sequencing ground truth--High expression of popular histopathology markers in clusters of cells with copy number events.","performance of variant calling on real data--performance on simulated data in high-confidence regions--impact of parameters on performance --impact of genomic contexts on calling performance--impact of functional regions on calling performance--impact of variant allele frequencies on calling performance",null,"number of inserted mutations as true and the detected SVNs as positive","F1 score to evaluate copy number--F1 score to evaluate cell labels.","- comparison with dimension reduction (clustering accuracy, purity, adjusted rand index, linear error, quadratic discrimination error, nearest neighbour error, normalised mutual information) \n- comparison with clustering  \n- comparison with matrix completion methods","UX of interactive visualisation tools - zoom in/out--ease of selection--how fast the web page loads",null,"Dimension Reduction","Visualisation"],"Multi_omics_integration__unmatched_":[null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","others (please specify in the next question)",null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"cases when the cell types are unknown, others (please specify in the next question)",null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"cases when the cell types are not entirely the same across the modes, cases when the cell types are unknown, method can preserve biology when batch and biology are associated",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)","integration of more than two modes, batch correction within each mode",null,null,null,null,null,null,null,null,null,null,"when cell types are unknown, others (please specify in the next question)",null,null,"when cell types are different across the modes",null,null,null,null,"cases when the cell types are not entirely the same across the modes, cases when the cell types are unknown, method can preserve biology when batch and biology are associated, method can preserve biology when biology is not associated with unwanted factors",null,null,"batch correction within each mode",null,null,null,null,null,null,null,null,null,null,"when cell types are different across the modes, preservation of biology when batch and biology are associated",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"others (please specify in the next question)",null,null],"Multi_omics_integration__matched____others":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Correlation between predicted and ground truth gene expression",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Prediction of spatial distribution of RNA transcripts through integration of spatial transcriptomics and scRNA-seq",null,null,null,null,null,null,null,null,null,"Trajectory downstream analysis: test the correlation between the reference and inferred “pseudo” times--perturbation based disease stats classification: test the classification accuracy.",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"spatial distribution, cell type deconvolution",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"Details_on_prior_knowledge":["This suggests a migration of Th and Tc cells towards beta cells (Fig. 4B), specifically during the early disease stages, reiter- ating the results presented by Damond et al.",null,null,"To enable process single-cell epigenomic data to assist the biological interpretation of chromatin landscapes within cell populations",null,null,null,null,null,"With histopathological knowledge, we separated the whole section into the tumor region on the left side, the basement membrane in the middle, and an adjacent liver region to the right of the section.",null,null,"To assign cell types to clusters, they used canonical marker genes and cell type annotation.",null,null,"Cell types are defined based on the known labels provided with the data",null,null,"Identified known marker genes as DE",null,"algorithm identified known luminal–basal cell interactions in healthy mammary ducts and alveoli","confirm known marker genes",null,null,null,"These studies compare methods using either experimental data where cell type labels are available or simulated datasets.","To test Cumulus and compare it with other tools, we relied on an scRNA-seq dataset of 274,182 cells (Methods), which were profiled from the bone marrow of eight donors as part of the Human Cell Atlas Census of Immune Cells dataset",null,"A human breast cancer cell line (HCC1395; sample A) and a B lymphocyte cell line (HCC1395BL; sample B) derived from the same donor. A total of 20 scRNA-seq datasets were generated from the two cell lines, processed either separately or as mixtures of different ratios of both cell lines.","The mapped cell types follow spatial\ndistributions of high-resolution cell types of this region previously\nannotated through clustering and marker gene annotation",null,null,"\"We plotted the 3D conformations of chromatin of entire cells. Most of the chromosomes are well segregated. These findings support the hypothesis of non-randomness of the chromatin conformation.\"","\"SnapHiC detected previously verified long-range interactions at Sox2, Wnt6 and Mtnr1a loci from scHi-C data of as few as 75 cells\"","All algorithms are used on each gene expression matrix to measure the difference between the imputation values and actual values of the masked entries using two accuracy metrics: Pearson correlation coefficient and mean-squared error (MSE)","Figure 13 MAPK pathway",null,null,"Not surprisingly, we identified several AD-related transcription factors (TFs) and target genes that have been reported to be involved in the development of AD.","recover known DE marker","Du et al. 2020 [15] reported that Kallisto was even faster than STARsolo, a finding that is consistent with our results because Kallisto had overall the shortest runtime across all mappers.",null,null,"1. \" The highest (fold-change) correlation can be observed in STARsolo and Alevin-fry. These results are largely consistent with the results from Kaminow et al.\"\n2. Du et al. reported that Kallisto was even faster than STARsolo, a finding that is consistent with our results because Kallisto had overall the shortest runtime across all mappers.\n3. \"Overall, Cell Ranger and STARsolo had almost identical results regarding the cell count\nand the genes per cell, which is expected from the similarity of both tools.\" One is a reimplementation of the other.",null,"Finally, we validate SODA robustness on known biological examples. Analyse the coupling at the cell membrane between two well-characterised cargos, transferrin (Tf) and interleukin 2 receptors (IL-2R), with proteins implicated in different endocytic pathways",null,"macrophages  consistent with the positive correlation between the PD-L1 expression in macrophages and the response to PD-1 blocked in previous reports","defined by known marker genes",null,null,null,null,null,null,null,null,null,null,"In order to classify input cells CHETAH requires scRNA-seq reference data along with cell type labels for each reference cell",null,null,null,null,"To train a new model with Garnett, the user must specify a cell hierarchy of cell types and subtypes, which may be organized into a tree of arbitrary depth","The characteristics of a dataset can have a critical effect on the performance of a machine learning method. A standard scRNA-seq training dataset for cell phenotype classification consists of a genomic data matrix and class label for each cell. In the genomic data matrix, each record (row) represents a cell and each column represents a single gene’s expression level",null,null,null,"\"the annotations derived by scClassify were very closely related to the original labels\"",null,"The cell types from the datasets are annotated in the literatures by unsupervised clustering and known marker gene expression.","dentifying subpopulations of tumors and normal phenotypes/TP53 mutation/AD/",null,null,null,"In our experiments, incorporating prior knowledge in the form of marker genes does not improve the performance.",null,null,null,null,null,null,null,null,null,null,null,null,null,"We further found that in both the Samusik01 and Levine13dim (Additional file 1: Figure S6B) datasets, DEPECHE failed to recognize the characteristics of some small cell types such as basophil cells, eosinophil cells, nature killer cells, and subtypes of dendritic cell",null,null,null,null,null,"the ability to precisely and robustly reproduce manual labels makes semi-supervised tools suitable for situations where researchers focus on the known cell types with reliable prior knowledge","Previous studies have shown that the exhausted CD8+ T cells contain heterogeneous subpopulations with differential capability to respond to anti-PD1 therapy27–33. We performed scMINER clustering on scRNA-seq profiles of exhausted CD8+ T cells in mice chronically infected with lymphocytic choriomeningitis virus (LCMV) Clone 13 (Cl13) at a late stage (day 28)32. We recapitulated the 3 major subpopulations of exhausted CD8+ T cells defined by TCF-1 (encoded by Tcf7 gene), CX3CR1, and TIM3 (encoded by Hacvr2)+: TCF-1+ exhaustion progenitor (Tpex), CX3CR1+ effector-like (Teff-like, effector-like Tex), and CX3CR1−TIM3+ terminal exhausted T cells (Tex) (Fig. 5a).",null,null,null,"The original study authors assigned a high-confidence true label (cell type) to each cell in their respective datasets",null,null,null,null,null,null,null,null,null,null,null,null,null,"the opportunity to study how genome structure varies among different cell types","we demonstrate that differential expression analysis for a specific cell type outperforms that of large-scale bulk sample data in prioritizing disease-related genes",null,null,null,null,null,null,"Visualization of significantly enriched terms identified enriched functions associated with growth factor and immune cell signaling in addition to expected terms such as xenobiotic metabolism and nuclear receptors in- volved in lipid metabolism",null,"\"As expected, many of these tools recovered, at high significance, several terms strongly related to stem cell biology, including development, differen- tiation, morphogenesis, multicellular, and adhesion as well as many others.\"","which has been reported to show differential expression patterns",null,null,"we used the already-published top 1000\nDE genes that are validated through qRT-PCR experi-\nments [47] as a gold standard gene set [21, 40, 42]",null,null,null,null,null,null,null,null,null,null,null,null,null,"These datasets were derived from purified PBMC subsets, and therefore their labels are reliable as ground truth",null,null,null,null,null,null,null,"A new algorithm, called uniform manifold approximation and projection (UMAP) has been\nrecently published and is claimed to preserve as much of the local and more of the global data structure than t-SNE, with a shorter run time.",null,null,null,"Among them, the Wnt signaling pathway is known to play an important role in regulating pluripotency (16)","\"Protein-Protein Interaction (PPI) networks were extracted from the STRING database and used as a reference network to compare the reconstructed networks. These networks represent potential interactions that are derived from evidence based on experimental results, metabolic and signal transduction pathway databases, text-mining and other sources of information\"",null,null,null,null,null,"known TFs and regulatory links",null,null,"Addition of prior biological knowledge and the estimation of transcription factor activities resulted in the best overall performance with the Inferelator and MERLIN methods that use prior knowledge outperforming methods that use expression alone",null,null,null,null,"This study includes simulation evaluation of representative biological networks, personalized driver genes prioritization on multiple cancer bulk expression datasets with matched patient samples from TCGA, and cell marker genes and key time point identification related to cell differentiation on single-cell RNA-seq datasets","we summarized the similarities and differences of these tools in terms of both LR prediction and CCI inference between cell types",null,null,null,"e ChIP-seq signals of the identified HC CREs associated with NR3C1, a well-known marker of early events after treatment (19). As expected, HC CREs have no NR3C1 signals at 0 hours but the highest NR3C1 signals at 1 hour, consistent with the role of NR3C1 as an early activation marker after treatment (Fig. 3E) (39).","Spatial transcriptomics (ST) dataset is used","Using the genetic loss of function analysis, we confirmed that several transcription factors identified by IReNA were critical for retinal regeneration, including hmga1a and yap1 in zebrafish and Nfia/b/x in mice",null,"For each dataset provided cell-type annotations were used as the ground truth to evaluate clustering performance",null,"Collectively, these results demonstrate that the maxATAC models are learning both canonical and noncanonical TF motifs",null,"\"Genes of T3 are well known to be associated with TP53.\"","Most of the methods recovered cross regulatory interactions between Nanog, Sox2, and Pou5f1, which are master regulators for establishing the ESC state",null,"To validate the links between TFs and peaks, we used publicly available ChIP-seq\n365 data for Olig233, Neurog234, Eomes34, and Tbr135, TFs that play key roles in embryonic\n366 brain development (Table 1). The Olig2 ChIP-seq data were generated in three types of\n","\"Known CVD disease genes were coexpressed in cell types classically involved in CVD pathology and the integrity of the blood–brain barrier, including microvessel endothelial cells and pericytes\"","\"we correctly model reported changes in phenotype that occur as a result of transcription factor perturbation\"",null,null,"known TF activity levels and known pathways associated with specific cell types","These algorithms combined prior knowledge (biological pathways or functional gene sets) with a statistic method to aggregate gene-level matrix into PAS-level matrix.",null,null,null,"\"Remarkably, all tools showed consistent se- quence (dis)similarities when compared to the ground truth, except for MiXCR, which assembled 1-B10 HC with two additional mutations. This is in line with the previous study (45) showing a high frequency of gene mishits during the alignment step of MiXCR and the resulting negative impact on the BCR sequence assembly.\"","Thus, it\nis essential to assess the performance of each algorithm and\nquantitatively understand how sensitive it is in reconstruct-\ning BCRs when compared to the ‘ground truth’ (defined as\nthe original variable domain sequence obtained by classical\nSanger sequencing).","we received 6 different phenotypes that were similar to the phenotypes described by Zheng et al.",null,null,null,null,"The violin plots (Figure 6B) indicated expression levels, and the t-SNE maps confirmed the relative distribution of cell type-specific marker genes across all clusters. The expression profile of some marker genes demonstrated scESI’s ability of preserving information of marker genes in raw data.",null,null,"Among them were several well-known tumor marker genes, including SFRP114, ATP1A115, GNAS16, NDRG117, and SPP1.\nFor example, SFRP1, a tumor marker gene, showed elevated expression values in the black\nregion.",null,null,null,null,null,null,null,null,null,null,null,null,null,"\" We find that the tra- jectory associated with scMerge is most consistent with the full Liver data collection (Fig. 3B) and agrees with current literature (18)\" \n\n\"we observe clear overlap of many cells from human and mouse blastocysts and inner cell mass regions across the pseudotime trajectory, confirming current understanding of a high similarity between early human and mouse embryonic development and lineage specification.\"","immunological genes","cell type marker genes",null,null,null,"We also examined the predicted results of the spatial distribution of COL17A1 in dataset 42 (ST; 10X Chromium; human squamous carcinoma). COL17A1 is a known marker gene for basal cells of squamous carcinoma",null,null,null,null,null,"For example, the peptidergic cells were correctly\nmapped to the hypothalamus region; the granule neurons were correctly mapped to the\ndentate gyrus region, and the medium spiny neurons were correctly mapped to the\nbasal ganglia (Additional file 1: Figure S2a, b).",null,null,null,null,"essentuially recapitulate known marker genes","For each reference matrix used, we designated all genes present in that matrix as marker genes",null,"\"we observed highly significant marker overlap \"",null,null,null,"Cell types were known",null,"We used a single-cell RNA-seq dataset of ∼11 000 cells from the TME to simulate bulk samples of known cell type proportions, and validated the results using independent, publicly available gold-standard estimates.",null,null,null,null,"\"As expected, all factors associated with a neuronal subtype were significantly (FDR<5%) spatially autocorrelated.\"",null,"Finally, the cytotoxic CD8 T-cell cluster, hT1, and the Treg cluster, hT3, invaded the same regions of the tissue, consistent with their demonstrated tendency to traffic together",null,null,null,null,"We evaluate eight transcriptome deconvolution approaches and nine cell-type signatures",null,"we found that all predicted trajectories captured the major cell state transitions supported by the literature. For example, mouse embryonic cell cycle predicted trajectories included the cyclical transition through the proliferative phases of the cell cycle",null,null,null,null,null,"For each sequencing-based dataset (ST, Visium, Slide-seqV2, and stereo-seq), we chose several cell types and their known marker genes to compare the PCC between the spatial distribution of cell-type proportion and marker-gene expression, which was used to quantify the performance of deconvolution without any ground truth",null,"The prior assumption of integrating these two modalities is to share the same cell types between scRNA-seq and spatial transcriptomics data",null,"\"We find that the trajectory associated with scMerge is most consistent with the full Liver data collection (Fig. 3B) and agrees with current literature (18) \"",null,null,"\"We then visualized the expression of known spermatogenesis markers in each cluster and found that these clusters generated by Portal’s alignment clearly captured the key transcriptomic features for each stage of spermatogenesis\"",null,"\"We further confirmed the gene expression of spatial markers were in agreement with prior biological knowledge (Supplementary Fig. 18c), high- lighting the attribute of Cepo to assign high-confidence cell identi- ties at defined locations.\"",". We then applied DrivAER to identify TFs that are relevant for erythrocyte and monocyte differentiation using the entire collection of motif gene sets contained in MolSigDB [31]. Among all 495 gene sets included in the analysis, DrivAER identified the GATA TF family as the most relevant in the erythrocyte trajectory (Fig. 2E, Supplementary Table S1). The DCA embedding derived from the “GATA_C” gene set showed strong clustering by pseudotime, demonstrating that GATA target gene expression is highly coordinated along this trajectory (Fig. 2F). Indeed, expression levels of GATA targets showed strong association with both pseudotime and DCA coordinates",null,null,null,"The initial report of this data confirmed that organoids contain excitatory cell types homologous to those in the cerebral cortex and retina",null,null,null,null,null,null,null,"marker gene","marker gene",null,"\"Indeed, all methods showed generally higher R2 values for the negative control data sets\"",null,"marker gene","marker gene","marker gene","gene marker","Differentiated tissues have cells with little RNA velocity compared to tissues undergoing active development.","In addition, Prox1 is a DE gene associated with a granule cell fate. This gene was previously identified by velocyto and experimentally validated as being necessary for granule cell formation; moreover, the deletion of Prox1 leads to adoption of the pyramidal neuron fate","marker gene",null,"\"These findings are highly consistent with those of a recent study, which showed that suppression of transcription of ribosomal genes and upregulation of lineage- specific factors in hematopoiesis coordinately control lineage differentiation\"",null,"\"This mirrors the findings of Shalek et al.(45), who found that the core anti-viral response genes were detectable in only some LPS cells early on but turned on in most cells between 2 and 4 h.\"","This result means that the identified subtypes using the klDD-clustering model are consistent with existing tumor subtypes.","recovered the previously reported accumulation of intertypical TECs with age; all clusters identified as differentially abundant among cirrhotic or uninjured tissue in the original study","CCAT identifies a multipotent root-state in scRNA-Seq data",null,null,null,null,null,null,null,null,null,"marker gene","high expression of antigen presentation genes (major histocompatibility complex (MHC) class II [H2-Aa, H2-Ab1, H2-Eb1], Cd74) confidently demonstrated the functionality of these rare cells",null,null,null,null,null,null,null,null,"As expected, the ADO rate tends to decrease with γ and c (Fig. 1C), while the genome coverage consistently increases with γ and c",null,null,null,null,"As expected, more cells lead to improved power for both forms of DE",null,null,null,null,null,null,"recover known marker genes that are important in the tissue","In the primary analysis, surprisingly, Moffitt et al.8 discovered spatially restricted cell populations with higher proliferation rates. Consistent with these findings, our method identified a substantial proportion of the genes assayed as SV (N = 91 (65% of all genes); FDR < 0.05; 29 HVGs, with overlap of 24 genes), including six of the seven genes highlighted as differentially expressed between proliferating and resting subpopulations (for example, THBS1 and CENPF; Supplementary Fig. 13c).","For example, Nrgn, one of the top 5 SVgenes in the 10X Brain dataset, is responsible for the transcription of neurogranin protein that encodes transcription and signal synapse transduction.","Our identification of KRT17 and MMP11 as the two positive genes for the cancer region agrees well with pancreatic cancer biology.",null,null,null,"\"All findings were highly concordant with previously reported results.\"","Estimated tumour purity of pancreatic cancer was low (6 to 18%), consistent with histopathology. For anaplastic thyroid cancer, copy number events cell cluster had high KRT8, a marker than histopathologists use the identify cells under a microscope.",null,"Clone with highest mutation burden has best immunotherapy response.",null,"Primary cancer is genomically very similar to matched metastasis.","we were able to identify each cell type based on the top 5 gene markers",null,"Precalculated clustering assignments and tSNE coordinates and metadata",null,"net-SNE learns visualizations that are similar to those of t-SNE on 14 scRNA-seq datasets of various cell types and data sizes up to 1.3 million cells."],"Downstream_analysis":["Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Not sure","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Not sure","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes"],"PMID":["35438129","34906205","31133762","33177523","32873325","33349700","36633424","31739806","34471111","36549467","34020539","33177523","34906205","36873935","34020539","36633424","31604912","31739806","31604912","32873325","28783155","33685491","33349700","33179754","32719530","31133762","32719530","33531477","33531477","34057817","33318659","34650268","34793453","34446921","34553763","34553763","36859400","35349708","33767197","34521337","35084033","32220951","32220951","35084033","34521337","29449608","37324550","36440766","36496406","31566233","36579851","34986249","33106496","34158507","32567229","33359678","31500660","36661313","36583014","31226206","31500660","36583014","31675098","31829268","31501545","35281805","33932523","32062421","35281805","32567229","36526433","34503564","34764492","30744683","34503564","31508207","31500660","32062421","33359678","33611343","31675098","34037791","31508207","36869836","33127686","28351406","35135612","34625546","36469543","30228881","31870419","30271584","34096998","31870278","36410733","36151714","31870419","36747874","33003206","35135612","35561178","33508103","36410733","31921297","31646845","30908236","30228881","28346451","30700040","31646845","30271584","36610708","33508103","36198490","31921297","36151714","36944632","27373736","35061903","36944632","27373736","29481549","28588607","35061903","35743881","30658573","35656596","35743881","29481549","30658573","35880426","28588607","32010190","32010190","35880426","30963075","31823809","32375029","31955711","35853932","30963075","34374742","33833778","35853932","33402734","33833778","33402734","32831127","31823809","32375029","30531897","30531897","32831127","34374742","31955711","28977434","29914350","33828580","29914350","36253792","33828580","36810839","36418514","35822343","31907445","36626328","35578363","34020546","30962620","36451086","33956788","35822343","33956788","30137247","35157031","35648859","36253792","36325073","34020546","36151725","36204155","36719906","30137247","36629274","36626328","31907445","36055233","30962620","36755098","36151725","33209207","32051003","33209207","35212622","35212622","32051003","35855325","35855325","33764977","30906525","35591853","32955565","29994128","35512331","35260632","32854757","36376296","29994128","35524568","31068142","35524568","30906525","34718405","31392316","32854757","34083791","34718405","35177662","35945544","36063455","31028141","31510660","33159064","35058621","34949812","33524142","35577954","35945544","33833776","32322368","34949812","33159064","33971932","34949812","36847692","35577954","36064614","36515467","34346485","35577957","35501393","35048125","35048125","33524142","35048125","33839756","31510660","35788983","31948481","32322368","35577954","31740819","33758076","35046414","36250795","31948481","36250636","31400197","35292647","35058621","36064614","36515467","36600199","35292647","31400197","34850092","36941264","36847692","36941264","31948481","31028141","34346485","35577955","s43588-022-00251-y","33097004","s43588-021-00172-2","33301553","35277963","s43588-021-00172-2","30796088","31282856","36253828","31022373","28468817","32117453","32117453","28468817","31022373","36396800","35640613","30936559","33428615","30936559","34244782","32343706","35758781","34888622","33428615","36449617","30890159","36124665","30289442","30289442","29140455","35151228","34594043","33244588","35814628","33338399","33338399","36342203","30404663","30404663","34215187","34430929","31315641","34252956","34234139","30413715","34824223","34824223","33624750","34911537","33471102","34168133","32614380","31584615","29036287","36991470","33663563","36991470","32614380","36553629","31510652","36575569","34034771","31919373","33320934","34154649","29553579","34165490","34711970","29481632","29481632","36831270","35801503","33462507","31744515","31937348","31744515","36841879","28263960","32766548","32766548","29936184","29936184"],"journal_name":["Bioinformatics (Oxford, England)","Genome biology","Nature methods","Nature communications","Genome biology","Nature biotechnology","Current protocols","Genome biology","Nature communications","Genomics, proteomics & bioinformatics","Briefings in bioinformatics","Nature communications","Genome biology","Frontiers in genetics","Briefings in bioinformatics","Current protocols","Nature communications","Genome biology","Nature communications","Genome biology","Nature methods","Genome biology","Nature biotechnology","Nucleic acids research","Nature methods","Nature methods","Nature methods","Scientific data","Scientific data","Molecular systems biology","Nature methods","Nature biotechnology","PLoS computational biology","Nature methods","Briefings in bioinformatics","Briefings in bioinformatics","Nature communications","Nucleic acids research","Nature communications","BMC genomics","GigaScience","G3 (Bethesda, Md.)","G3 (Bethesda, Md.)","GigaScience","BMC genomics","Nature communications","Proceedings of SPIE--the International Society for Optical Engineering","Nucleic acids research","Nature communications","Nucleic acids research","Journal of chemical information and modeling","Nucleic acids research","Nature communications","Nature communications","Molecular systems biology","Genomics, proteomics & bioinformatics","Genome biology","Bioinformatics (Oxford, England)","Frontiers in genetics","Nucleic acids research","Genome biology","Frontiers in genetics","Briefings in bioinformatics","Genome biology","Nature methods","Frontiers in genetics","Genomics","iScience","Frontiers in genetics","Molecular systems biology","Genome research","Genome biology","Nature biotechnology","Genome biology","Genome biology","F1000Research","Genome biology","iScience","Genomics, proteomics & bioinformatics","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","F1000Research","Briefings in bioinformatics","Science advances","Genome biology","Genome biology","Nature communications","PLoS computational biology","F1000Research","Genome Biology","F1000Research","Bioinformatics (Oxford, England)","BMC bioinformatics","Briefings in bioinformatics","Briefings in bioinformatics","Genome Biology","Research square","Briefings in bioinformatics","Genome biology","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Frontiers in genetics","Statistical applications in genetics and molecular biology","IEEE/ACM transactions on computational biology and bioinformatics","F1000Research","Nature methods","Genes","Statistical applications in genetics and molecular biology","F1000Research","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Genome research","Frontiers in genetics","Briefings in bioinformatics","Nature communications","Briefings in bioinformatics","Nucleic acids research","Nature communications","Briefings in bioinformatics","Nature methods","Frontiers in genetics","Nucleic acids research","Life (Basel, Switzerland)","BMC bioinformatics","Statistics in medicine","Life (Basel, Switzerland)","Nature methods","BMC bioinformatics","Briefings in bioinformatics","Frontiers in genetics","Frontiers in genetics","Frontiers in genetics","Briefings in bioinformatics","Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","Genome biology","Cell reports","Genome biology","Communications biology","Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","Briefings in bioinformatics","Frontiers in genetics","Communications biology","Nature communications","Frontiers in genetics","Nature communications","Genome biology","Genome biology","Cell reports","Nature biotechnology","Nature biotechnology","Genome biology","Briefings in bioinformatics","Genome biology","Nucleic acids research","BMC bioinformatics","Frontiers in genetics","BMC bioinformatics","Genome biology","Frontiers in genetics","Nature communications","Communications biology","Briefings in functional genomics","Nature methods","G3 (Bethesda, Md.)","Genome biology","Briefings in bioinformatics","Nature methods","BMC genomics","PLoS computational biology","Briefings in functional genomics","PLoS computational biology","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Science advances","Genome biology","iScience","Briefings in bioinformatics","Briefings in bioinformatics","Cell genomics","PLoS computational biology","Briefings in bioinformatics","Nucleic acids research","G3 (Bethesda, Md.)","Nature methods","Cell systems","Nature methods","Nature","Briefings in bioinformatics","Computational and structural biotechnology journal","Genome biology","Computational and structural biotechnology journal","eLife","eLife","Genome biology","NAR genomics and bioinformatics","NAR genomics and bioinformatics","PLoS computational biology","F1000Research","Frontiers in genetics","Nucleic acids research","IEEE/ACM transactions on computational biology and bioinformatics","Briefings in bioinformatics","Scientific reports","Genome biology","Nature communications","IEEE/ACM transactions on computational biology and bioinformatics","Nucleic acids research","BMC genomics","Nucleic acids research","F1000Research","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Genome biology","Nature biotechnology","Briefings in bioinformatics","Scientific reports","Genome biology","Bioinformatics (Oxford, England)","Proceedings of the National Academy of Sciences of the United States of America","Bioinformatics (Oxford, England)","Nature communications","Nature biotechnology","Nature methods","Nucleic acids research","Nature methods","Genome biology","Frontiers in genetics","Computational and structural biotechnology journal","Nature methods","Nature communications","Genome biology","Nature methods","Briefings in bioinformatics","Nature methods","Genome biology","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Nature methods","Nature biotechnology","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","Briefings in bioinformatics","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Genetic epidemiology","Genome biology","Computational and structural biotechnology journal",null,"Nature methods","Life science alliance","Nature communications","Bioinformatics (Oxford, England)","Genome biology","Nucleic acids research","Bioinformatics (Oxford, England)","Nature communications","Nature biotechnology","Genome biology","Bioinformatics (Oxford, England)","BMC bioinformatics","Nature communications","Bioinformatics (Oxford, England)","Nucleic acids research","Nature communications","Briefings in bioinformatics","Nature communications","Genome biology","Proceedings of the National Academy of Sciences of the United States of America","Briefings in bioinformatics","Nature methods","Nature computational science","BMC bioinformatics","Nature computational science","GigaScience","GigaScience","Nature computational science","Molecular systems biology","eLife","Genome biology","Cell systems","G3 (Bethesda, Md.)","Frontiers in genetics","Frontiers in genetics","G3 (Bethesda, Md.)","Cell systems","Communications biology","Nucleic acids research","Nature biotechnology","PLoS computational biology","Nature biotechnology","Nucleic acids research","PLoS computational biology","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","PLoS computational biology","Science advances","Genome biology","Nucleic acids research","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","Briefings in bioinformatics","Nature biotechnology","Bioinformatics (Oxford, England)","F1000Research","Cell systems","Cell systems","Bioinformatics (Oxford, England)","Genome biology","Genome biology","BMC bioinformatics","Patterns (New York, N.Y.)","Genome biology","Bioinformatics (Oxford, England)","Nature communications","Nature communications","Nature communications","Nature communications","Bioinformatics (Oxford, England)","Genome biology","Bioinformatics (Oxford, England)","Nature communications","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Genome biology","Genome biology","Genome biology","Bioinformatics (Oxford, England)","Genes","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Genome biology","Nature communications","Briefings in bioinformatics","Genome biology","Nature methods","Bioinformatics (Oxford, England)","Nature methods","Briefings in bioinformatics","Briefings in bioinformatics","Cells","Briefings in bioinformatics","Nature biotechnology","Genome biology","Genome biology","Genome biology","Nature communications","Nature methods","NAR genomics and bioinformatics","NAR genomics and bioinformatics","Cell systems","Cell systems"],"publication_date":["2022 May 26","2021 Dec 14","2019 May 27","2020 Nov 11","2020 Sep 1","2021 Sep","2023 Jan 12","2019 Nov 18","2021 Sep 1","2022 Dec 19","2021 May 20","2020 Nov 11","2021 Dec 14","2023 Feb 17","2021 May 20","2023 Jan 12","2019 Oct 11","2019 Nov 18","2019 Oct 11","2020 Sep 1","2017 Aug 7","2021 Mar 8","2021 Sep","2021 Jan 8","2020 Jul 27","2019 May 27","2020 Jul 27","2021 Feb 2","2021 Feb 2","2021 May 31","2020 Dec 14","2021 Oct 14","2021 Nov 18","2021 Aug 26","2022 Jan 17","2022 Jan 17","2023 Mar 1","2022 Jul 8","2021 Mar 25","2021 Sep 14","2022 Jan 27","2020 May 4","2020 May 4","2022 Jan 27","2021 Sep 14","2018 Feb 15","2023 Apr 7","2022 Nov 28","2022 Dec 10","2019 Dec 16","2023 Jan 9","2022 May 6","2020 Oct 26","2021 Jun 22","2020 Jun 22","2020 Dec 24","2019 Sep 9","2023 Feb 3","2022 Dec 13","2019 Sep 19","2019 Sep 9","2022 Dec 13","2020 Sep 25","2019 Dec 12","2019 Sep 9","2022 Feb 23","2021 Apr 28","2020 Mar 27","2022 Feb 23","2020 Jun 22","2022 Dec 16","2021 Sep 9","2021 Nov 11","2019 Feb 11","2021 Sep 9","2019 Oct 14","2019 Sep 9","2020 Mar 27","2020 Dec 24","2021 Sep 2","2020 Sep 25","2021 Aug 20","2019 Oct 14","2023 Mar 19","2020 Oct 30","2017 Mar 28","2022 Feb 8","2021 Oct 8","2022 Dec 5","2018 Aug 15","2019 Dec 23","2018 Jul 26","2021 Nov 5","2019 Dec 24","2023 Jan 19","2022 Nov 19","2019 Dec 23","2023 Jan 27","2021 Jul 20","2022 Feb 8","2022 May 13","2021 Jan 28","2023 Jan 19","2019 Dec 11","2019 Aug 14","2020 Sep 1","2018 Aug 15","2017 Mar 27","2019 Jan 29","2019 Aug 14","2018 Jul 26","2023 Jan 1","2021 Jan 28","2022 Oct 5","2019 Dec 11","2022 Nov 19","2023 Mar 21","2017 Sep 1","2022 May 6","2023 Mar 21","2017 Sep 1","2018 Feb 26","2017 May 23","2022 May 6","2022 Jun 7","2019 Jan 18","2022 Aug 15","2022 Jun 7","2018 Feb 26","2019 Jan 18","2022 Sep 20","2017 May 23","2020 Jan 17","2020 Jan 17","2022 Sep 20","2019 Mar 14","2019 Dec 10","2020 May 5","2020 Jan 20","2022 Jul 19","2019 Mar 14","2021 Nov 5","2021 Mar 23","2022 Jul 19","2021 Jan 5","2021 Mar 23","2021 Jan 5","2020 Aug 24","2019 Dec 10","2020 May 5","2018 Dec 3","2018 Dec 3","2020 Aug 24","2021 Nov 5","2020 Jan 20","2017 Nov 2","2018 Jun 19","2021 Mar 22","2018 Jun 19","2022 Oct 17","2021 Mar 22","2023 Feb 21","2022 Nov 23","2022 Sep 16","2020 Jan 6","2023 Mar 9","2022 May 16","2021 May 20","2019 Apr 8","2022 Nov 30","2021 May 6","2022 Sep 16","2021 May 6","2019 Nov 27","2022 Apr 12","2022 Jun 3","2022 Oct 17","2022 Nov 18","2021 May 20","2022 Nov 19","2022 Sep 14","2023 Jan 31","2019 Nov 27","2023 Feb 28","2023 Mar 9","2020 Jan 6","2022 Sep 21","2019 Apr 8","2023 Feb 8","2022 Nov 19","2020 Oct 15","2020 Feb 12","2020 Oct 15","2022 Feb 25","2022 Feb 25","2020 Feb 12","2022 Jul 13","2022 Jul 13","2021 Mar 25","2018 Nov 2","2022 May 3","2020 Oct 9","2020 Mar 1","2022 Sep 20","2022 Mar 8","2020 Aug 27","2022 Nov 14","2020 Mar 1","2022 May 20","2019 May 8","2022 May 20","2018 Nov 2","2022 Jan 17","2020 Feb 1","2020 Aug 27","2021 Jun 3","2022 Jan 17","2022 Feb 17","2022 Aug 9","2022 Oct 31","2019 May 14","2019 Jul 15","2020 Nov 6","2022 Jan 20","2021 Dec 23","2021 Apr 19","2022 May 16","2022 Aug 9","2021 Mar 23","2020 mar 30","2021 Dec 23","2020 Nov 6","2021 May 10","2021 Dec 23","2023 Mar 19","2022 May 16","2022 Sep 5","2023 Jan 1","2021 Nov 5","2022 May 16","2022 May 2","2022 Mar 10","2022 Mar 10","2021 Apr 19","2022 Mar 10","2021 Sep 2","2019 Jul 15","2022 Jul 5","2020 Jan 16","2020 mar 30","2022 May 16","2019 Nov 18","2021 Mar 23","2022 Jan 19","2022 Nov 30","2020 Jan 16","2022 Dec 9","2020 Feb 1","2022 Mar 15","2022 Jan 20","2022 Sep 5","2023 Jan 1","2023 Jan 4","2022 Mar 15","2020 Feb 1","2022 Jan 25","2023 Mar 21","2023 Mar 19","2023 Mar 21","2020 Jan 16","2019 May 14","2021 Nov 5","2022 May 16","2022 May 30","2020 Oct 23","2021 Dec 20","2020 Dec 10","2022 Mar 12","2021 Dec 20","2019 Feb 22","2019 Jul 8","2022 Oct 17","2019 Apr 24","2017 Jul 5","2020 Feb 7","2020 Feb 7","2017 Jul 5","2019 Apr 24","2022 Nov 17","2022 Sep 9","2019 Apr 1","2021 Jan 11","2019 Apr 1","2021 Aug 20","2020 Apr 28","2022 Jun 24","2022 Feb 7","2021 Jan 11","2022 Dec 2","2019 Mar 19","2022 Nov 28","2020 Jan 17","2020 Jan 17","2018 Feb 16","2022 Mar 10","2021 Sep 30","2021 Jul 12","2022 May 16","2021 Feb 17","2021 Feb 17","2023 Jan 1","2018 Nov 7","2018 Nov 7","2021 Jul 2","2021 Aug 13","2019 Jul 17","2021 Jul 12","2021 Jul 7","2018 Nov 9","2021 Nov 25","2021 Nov 25","2021 Aug 25","2021 Dec 15","2021 Jul 19","2021 Jun 24","2020 Dec 8","2020 Feb 15","2017 Nov 1","2023 Mar 29","2021 Mar 4","2023 Mar 29","2020 Dec 8","2022 Dec 14","2019 Jul 15","2023 Jan 19","2021 May 25","2020 Jan 9","2021 Jul 20","2021 Jun 21","2018 Mar 19","2021 Dec 7","2021 Oct 28","2019 Jul 19","2019 Jul 19","2023 Feb 13","2022 Jul 18","2021 Jan 18","2019 Nov 19","2020 Jan 14","2019 Nov 19","2023 Feb 25","2017 Mar 6","2020 Jul 29","2020 Jul 29","2018 Aug 22","2018 Aug 22"],"impact_factor":[6.931,17.906,47.99,17,17.906,68.164,7,17.906,17,6.409,13.994,17,17.906,4.772,13.994,7,17,17.906,17,17.906,47.99,17.906,68.164,19.16,47.99,47.99,47.99,8.501,8.501,12.744,47.99,68.164,4.779,47.99,13.994,13.994,17,19.16,17,4.558,7.658,null,null,7.658,4.558,17,7,19.16,17,19.16,7,19.16,17,17,12.744,6.409,17.906,6.931,4.772,19.16,17.906,4.772,13.994,17.906,47.99,4.772,7,6.107,4.772,12.744,9.438,17.906,68.164,17.906,17.906,0.939,17.906,6.107,6.409,13.994,13.994,19.16,0.939,13.994,14.98,17.906,17.906,17,4.779,0.939,7,0.939,6.931,3.327,13.994,13.994,7,7,13.994,17.906,6.931,6.931,13.994,4.772,7,3.702,0.939,47.99,7,7,0.939,6.931,6.931,9.438,4.772,13.994,17,13.994,19.16,17,13.994,47.99,4.772,19.16,7,3.327,7,7,47.99,3.327,13.994,4.772,4.772,4.772,13.994,7,17.906,9.995,17.906,7,7,13.994,4.772,7,17,4.772,17,17.906,17.906,9.995,68.164,68.164,17.906,13.994,17.906,19.16,3.327,4.772,3.327,17.906,4.772,17,7,7,47.99,3.542,17.906,13.994,47.99,4.558,4.779,7,4.779,13.994,6.931,14.98,17.906,6.107,13.994,13.994,null,4.779,13.994,19.16,3.542,47.99,11.091,47.99,69.504,13.994,6.155,17.906,6.155,8.713,8.713,17.906,null,null,4.779,0.939,4.772,19.16,3.702,13.994,4.996,17.906,17,3.702,19.16,4.558,19.16,0.939,13.994,6.931,17.906,68.164,13.994,4.996,17.906,6.931,12.779,6.931,17,68.164,47.99,19.16,47.99,17.906,4.772,6.155,47.99,17,17.906,47.99,13.994,47.99,17.906,6.931,13.994,47.99,68.164,13.994,13.994,19.16,13.994,13.994,6.931,7,17.906,6.155,7,47.99,null,17,6.931,17.906,19.16,6.931,17,68.164,17.906,6.931,3.327,17,6.931,19.16,17,13.994,17,17.906,12.779,13.994,47.99,null,3.327,null,7.658,7.658,null,12.744,8.713,17.906,11.091,null,4.772,4.772,null,11.091,7,19.16,68.164,4.779,68.164,19.16,4.779,6.931,6.931,4.779,14.98,17.906,19.16,13.994,13.994,19.16,13.994,68.164,6.931,0.939,11.091,11.091,6.931,17.906,17.906,3.327,7,17.906,6.931,17,17,17,17,6.931,17.906,6.931,17,6.931,6.931,6.931,17.906,17.906,17.906,6.931,7,6.931,13.994,17.906,17,13.994,17.906,47.99,6.931,47.99,13.994,13.994,7,13.994,68.164,17.906,17.906,17.906,17,47.99,null,null,11.091,11.091],"data_type":["Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell proteomics","Single-cell ATAC-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell proteomics","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Spatial imaging","Spatial transcriptomics","Single-cell Hi-C","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell genomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell Hi-C","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell mass cytometry","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell multiomics","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell sequencing","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell sequencing","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell genomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ChIP-seq","Spatial transcriptomics","Spatial transcriptomics","Spatial transcriptomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq"],"broader_topic":["Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Data","Data","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis"],"finer_topic":["Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Benchmarking datasets","Benchmarking datasets","Cell segmentation","Cell segmentation","Cell segmentation","Chromatin conformation","Chromatin conformation","Multi-task","Multi-task","Multi-task","Multi-task","Multi-task","Alignment","Alignment","Alignment","Alignment","Alignment","Alignment","Assembly","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Regulatory network inference","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Measure of association","Cell cell communication","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Measure of association","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","T/B-cell receptor","T/B-cell receptor","T/B-cell receptor","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Data integration","Deconvolution","Batch correction","Deconvolution","Deconvolution","Data integration","Batch correction","Batch correction","Data integration","Data integration","Data integration","Batch correction","Batch correction","Deconvolution","Deconvolution","Batch correction","Data integration","Data integration","Data integration","Deconvolution","Deconvolution","Data integration","Data integration","Batch correction","Batch correction","Batch correction","Batch correction","Batch correction","Deconvolution","Deconvolution","Batch correction","Batch correction","Data integration","Data integration","Batch correction","Deconvolution","Data integration","Batch correction","Data integration","Batch correction","Deconvolution","Data integration","Data integration","Deconvolution","Data integration","Deconvolution","Batch correction","Data integration","Deconvolution","Data integration","Deconvolution","Batch correction","Batch correction","Deconvolution","Data integration","Data integration","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Standardization","Standardization","Standardization","Standardization","Standardization","Standardization","Standardization","Trajectory","Trajectory","Trajectory","Velocity","Trajectory","Trajectory","Trajectory","Trajectory","Trajectory","Velocity","Velocity","Trajectory","Trajectory","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Doublet detection","Doublet detection","Doublet detection","Doublet detection","Quantification","Quantification","Quantification","Quantification","Rare cell type identification","Rare cell type identification","Rare cell type identification","Rare cell type identification","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Visualisation","Visualisation","Visualisation","Visualisation","Visualisation"],"year":[2022,2021,2019,2020,2020,2021,2023,2019,2021,2022,2021,2020,2021,2023,2021,2023,2019,2019,2019,2020,2017,2021,2021,2021,2020,2019,2020,2021,2021,2021,2020,2021,2021,2021,2022,2022,2023,2022,2021,2021,2022,2020,2020,2022,2021,2018,2023,2022,2022,2019,2023,2022,2020,2021,2020,2020,2019,2023,2022,2019,2019,2022,2020,2019,2019,2022,2021,2020,2022,2020,2022,2021,2021,2019,2021,2019,2019,2020,2020,2021,2020,2021,2019,2023,2020,2017,2022,2021,2022,2018,2019,2018,2021,2019,2023,2022,2019,2023,2021,2022,2022,2021,2023,2019,2019,2020,2018,2017,2019,2019,2018,2023,2021,2022,2019,2022,2023,2017,2022,2023,2017,2018,2017,2022,2022,2019,2022,2022,2018,2019,2022,2017,2020,2020,2022,2019,2019,2020,2020,2022,2019,2021,2021,2022,2021,2021,2021,2020,2019,2020,2018,2018,2020,2021,2020,2017,2018,2021,2018,2022,2021,2023,2022,2022,2020,2023,2022,2021,2019,2022,2021,2022,2021,2019,2022,2022,2022,2022,2021,2022,2022,2023,2019,2023,2023,2020,2022,2019,2023,2022,2020,2020,2020,2022,2022,2020,2022,2022,2021,2018,2022,2020,2020,2022,2022,2020,2022,2020,2022,2019,2022,2018,2022,2020,2020,2021,2022,2022,2022,2022,2019,2019,2020,2022,2021,2021,2022,2022,2021,2020,2021,2020,2021,2021,2023,2022,2022,2023,2021,2022,2022,2022,2022,2021,2022,2021,2019,2022,2020,2020,2022,2019,2021,2022,2022,2020,2022,2020,2022,2022,2022,2023,2023,2022,2020,2022,2023,2023,2023,2020,2019,2021,2022,2022,2020,2021,2020,2022,2021,2019,2019,2022,2019,2017,2020,2020,2017,2019,2022,2022,2019,2021,2019,2021,2020,2022,2022,2021,2022,2019,2022,2020,2020,2018,2022,2021,2021,2022,2021,2021,2023,2018,2018,2021,2021,2019,2021,2021,2018,2021,2021,2021,2021,2021,2021,2020,2020,2017,2023,2021,2023,2020,2022,2019,2023,2021,2020,2021,2021,2018,2021,2021,2019,2019,2023,2022,2021,2019,2020,2019,2023,2017,2020,2020,2018,2018],"biorxiv_post_date":["June 07, 2021","June 17, 2021",null,"July 22, 2019","February 02, 2020",null,null,"August 18, 2019","May 24, 2019","July 24, 2022","February 9, 2020","July 22, 2019","June 17, 2021","November 10, 2022","February 9, 2020",null,"March 19, 2019","August 18, 2019","March 19, 2019","February 02, 2020","February 17, 2017","July 13, 2019",null,null,"October 30, 2019",null,"October 30, 2019","September 21, 2020","September 21, 2020",null,"February 03, 2020",null,null,"August 06, 2022",null,null,"August 03, 2022","December 25, 2021",null,"March 07, 2021","February 16, 2021",null,null,"February 16, 2021","March 07, 2021",null,null,null,"May 27, 2022","April 20, 2018",null,null,"August 24, 2019","September 14, 2020","September 23, 2019","November 1, 2019","May 20, 2019",null,"October 10, 2022","February 22, 2019","May 20, 2019","October 10, 2022",null,"July 15, 2018","February 04, 2019",null,null,null,null,"September 23, 2019","September 26, 2021",null,"June 07, 2020","July 25, 2018",null,"February 28, 2019","May 20, 2019",null,"November 1, 2019",null,null,null,"February 28, 2019",null,null,"August 10, 2016",null,"February 19, 2021",null,null,null,null,"February 07, 2021","September 19, 2019",null,null,null,"January 27, 2023",null,null,null,null,null,null,null,null,null,"January 13, 2016",null,null,null,null,null,null,null,null,null,null,"September 10, 2021",null,null,"May 28, 2017",null,"September 10, 2021",null,null,"March 29, 2021",null,"May 28, 2017",null,"February 19, 2022",null,null,null,"February 19, 2022",null,"May 17, 2019","June 27, 2019","October 2, 2019",null,null,"October 30, 2020",null,null,"January 14, 2020",null,"January 14, 2020",null,"May 17, 2019","June 27, 2019",null,null,null,"October 30, 2020","October 2, 2019",null,null,"November 10, 2020",null,null,"November 10, 2020","November 03, 2021","January 8, 2020",null,"May 20, 2019","June 02, 2021",null,null,null,null,null,null,null,null,null,null,null,"November 23, 2021",null,null,"July 28, 2021","�January 29, 2022",null,null,"June 02, 2021","May 20, 2019","September 24, 2021",null,"February 17, 2020",null,null,null,null,"April 28, 2020","April 28, 2020",null,"March 27, 2022","March 27, 2022","May 24, 2022",null,null,"May 10, 2020","December 31, 2017",null,null,"January 30, 2020",null,"December 31, 2017",null,null,null,null,null,null,"January 30, 2020","September 05, 2020",null,null,null,null,"August 16, 2018","November 06, 2018",null,"January 03, 2021","May 23, 2020","May 27, 2020",null,null,null,null,"May 23, 2020",null,"February 3, 2021","May 23, 2020","September 27, 2022",null,"March 2, 2022",null,null,"March 16, 2021","August 24, 2021",null,null,"May 27, 2020",null,"January 12, 2021","November 06, 2018",null,null,null,null,"November 04, 2018","December 11, 2020","August 05, 2020","February 19, 2022",null,"August 03, 2022","August 21, 2018","June 01, 2020","January 03, 2021","March 2, 2022",null,"June 8, 2022","June 01, 2020","August 21, 2018",null,null,"September 27, 2022",null,null,"August 16, 2018",null,"August 28, 2021","November 19, 2021","June 23, 2019","January 11, 2021","December 05, 2019","February 13, 2021","January 11, 2021","July 11, 2018","April 30, 2018","March 11, 2021","December 16, 2017","July 17, 2016",null,null,"July 17, 2016","December 16, 2017","June 26, 2021","June 12, 2019","March 5, 2018","March 13, 2020","March 5, 2018",null,"October 29, 2018","April 29, 2019","May 03, 2021","March 13, 2020","February 19, 2022","October 25, 2017",null,"January 30, 2018","January 30, 2018",null,null,"November 23, 2020",null,null,null,null,"April 15, 2022","January 17, 2018","January 17, 2018",null,null,"January 09, 2019",null,null,null,"June 02, 2021","June 02, 2021",null,"June 17, 2021","May 30, 2020","February 06, 2020",null,null,"March 15, 2017","March 5, 2018",null,"March 5, 2018",null,null,"October 07, 2018",null,"November 20, 2020","August 13, 2018",null,null,"May 28, 2017","March 24, 2021","December 02, 2020",null,null,null,"February 10, 2022",null,null,"October 30, 2018",null,"November 22, 2021","May 09, 2016","�January 25, 2020","�January 25, 2020",null,null],"biorxiv_number_dataset":["3","10",null,"4","9",null,null,"13","3","33","9","4","10","4","9",null,"5","13","5","9","1","1",null,null,"2",null,"2","20","20",null,"3",null,null,"2",null,null,"4","16",null,"22","3",null,null,"3","22",null,null,null,"4","2",null,null,"16","30","7","6","8",null,"5","5","8","5",null,"5","5",null,null,null,null,"7","2",null,"6","5",null,"3","8",null,"6",null,null,null,"3",null,null,"2",null,"5",null,null,null,null,"6","8",null,null,null,"20",null,null,null,null,null,null,null,null,null,"9",null,null,null,null,null,null,null,null,null,null,"1",null,null,"13",null,"1",null,null,"15",null,"13",null,"0",null,null,null,"0",null,"28","2","4+18",null,null,"55",null,null,"1",null,"1","10","28","2",null,null,"10","55","4+18",null,null,"14",null,null,"14","21","4",null,"10","6",null,null,null,null,null,null,null,null,null,null,null,"3",null,null,"2","6",null,null,"6","10","6",null,"5",null,null,null,null,null,null,null,"4","4","5",null,null,"5","2",null,null,"7",null,"2",null,null,null,null,null,null,"7","3",null,null,null,null,"14","105",null,"3","23","3",null,null,null,null,"23",null,"3","23","19",null,"8",null,null,"2","3",null,null,"3",null,"6","105",null,null,null,null,"3","7","27","4",null,"4","8","420","3","8",null,"4","420","8",null,null,"19",null,null,"14",null,"39","3","6","7","2","2","7","1","2","4","3","120",null,null,"120","3","1","3","110","4","110",null,"1","1","3","4","4","2",null,"11","11",null,null,"3",null,null,null,null,"16","3","3",null,null,"8",null,null,null,"36","36",null,"3","1","1",null,null,"5","16",null,"16",null,null,"18",null,"6","3",null,null,"2","6","5",null,null,null,"9",null,null,"2",null,"5","4","1","1",null,null],"biorxiv_number_method":[0,9,null,3,24,null,null,10,6,2,7,3,9,7,7,null,18,10,18,24,0,0,null,null,2,null,2,0,0,null,2,null,null,3,null,null,5,18,null,2,4,null,null,4,2,null,null,null,5,4,null,null,0,3,11,10,20,null,5,3,20,5,null,0,1,null,null,null,null,11,5,null,0,1,null,4,20,null,10,null,null,null,4,null,null,4,null,1,null,null,null,null,15,4,null,null,null,3,null,null,null,null,null,null,null,null,null,4,null,null,null,null,null,null,null,null,null,null,8,null,null,30,null,8,null,null,7,null,30,null,18,null,null,null,18,null,11,11,21,null,null,33,null,null,9,null,9,5,11,11,null,null,5,33,21,null,null,4,null,null,4,4,5,null,12,11,null,null,null,null,null,null,null,null,null,null,null,1,null,null,0,2,null,null,11,12,null,null,1,null,null,null,null,null,null,null,6,6,5,null,null,3,8,null,null,18,null,8,null,null,null,null,null,null,18,6,null,null,null,null,5,7,null,5,38,8,null,null,null,null,38,null,4,38,5,null,8,null,null,1,4,null,null,8,null,2,7,null,null,null,null,4,9,2,4,null,5,3,7,5,8,null,8,7,3,null,null,5,null,null,5,null,6,6,8,9,1,8,9,3,1,7,10,7,null,null,7,10,1,0,29,12,29,null,0,3,5,12,3,2,null,4,4,null,null,4,null,null,null,null,7,5,5,null,null,2,null,null,null,12,12,null,0,2,3,null,null,4,12,null,12,null,null,4,null,8,2,null,null,1,2,2,null,null,null,4,null,null,3,null,1,8,13,13,null,null],"biorxiv_number_reference":[22,65,null,23,45,null,null,44,30,27,68,23,65,48,68,null,48,44,48,45,26,35,null,null,61,null,61,37,37,null,50,null,null,59,null,null,52,82,null,33,27,null,null,27,33,null,null,null,75,30,null,null,48,57,46,27,40,null,29,53,40,29,null,45,22,null,null,null,null,46,46,null,58,46,null,24,40,null,27,null,null,null,24,null,null,16,null,28,null,null,null,null,22,40,null,null,null,92,null,null,null,null,null,null,null,null,null,48,null,null,null,null,null,null,null,null,null,null,46,null,null,65,null,46,null,null,63,null,65,null,28,null,null,null,28,null,57,44,143,null,null,58,null,null,32,null,32,26,57,44,null,null,26,58,143,null,null,27,null,null,27,43,28,null,41,38,null,null,null,null,null,null,null,null,null,null,null,27,null,null,58,89,null,null,38,41,50,null,79,null,null,null,null,null,null,null,58,58,24,null,null,25,34,null,null,64,null,34,null,null,null,null,null,null,64,35,null,null,null,null,20,35,null,37,66,40,null,null,null,null,66,null,22,66,32,null,78,null,null,53,63,null,null,40,null,26,35,null,null,null,null,23,39,24,42,null,52,47,75,37,78,null,44,75,47,null,null,32,null,null,20,null,44,63,37,39,34,33,39,42,34,46,50,30,null,null,30,50,65,87,105,29,105,null,39,43,15,29,46,51,null,45,45,null,null,47,null,null,null,null,21,54,54,null,null,74,null,null,null,29,29,null,30,10,43,null,null,29,66,null,66,null,null,60,null,107,33,null,null,33,17,27,null,null,null,37,null,null,65,null,37,27,31,31,null,null],"number_reference":[25,77,58,30,67,null,null,40,40,33,75,30,77,48,75,null,53,40,53,67,26,57,null,null,60,58,60,42,42,53,61,45,null,32,null,null,58,78,null,36,46,26,26,46,36,69,null,79,78,37,null,null,46,61,68,39,53,null,29,59,53,29,46,67,23,null,null,35,null,68,68,55,61,57,55,34,53,35,39,null,46,null,34,null,null,26,47,46,null,34,null,54,5,40,37,null,null,75,null,47,null,null,37,50,null,null,34,47,null,null,54,null,null,62,50,null,null,20,46,null,20,52,29,46,null,48,81,null,52,48,32,29,36,36,32,null,85,46,147,null,null,65,null,null,37,null,37,31,85,46,22,22,31,65,147,null,47,31,47,null,31,78,100,null,54,58,57,88,65,null,null,null,null,60,null,60,null,45,88,null,89,100,60,null,58,54,53,65,83,null,57,52,57,27,27,52,46,46,65,35,null,27,34,null,37,86,null,34,null,null,null,35,null,null,86,50,null,null,71,null,56,37,69,44,52,40,80,71,null,35,52,69,24,52,36,80,97,32,null,57,77,37,37,40,37,26,37,null,47,35,80,49,39,51,50,47,52,22,76,44,97,32,48,76,22,null,null,36,null,47,56,null,80,58,44,50,50,50,50,49,60,83,57,27,45,45,27,57,70,65,55,40,55,null,51,32,24,40,73,38,null,51,51,93,null,49,null,null,71,71,20,60,60,null,null,74,null,null,33,35,35,null,34,9,59,36,null,31,113,null,113,36,null,52,null,43,48,null,78,32,17,39,30,30,null,50,47,49,92,49,48,25,34,34,null,null],"topic_year":[2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2020,2020,2020,2016,2016,2016,2016,2016,2016,2016,2012,2012,2012,2012,2012,2012,2015,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2010,2010,2010,2010,2010,2010,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2011,2011,2011,2011,2011,2011,2011,2013,2013,2013,2013,2013,2013,2013,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2011,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2012,2012,2012,2012,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2014,2014,2014,2014,2014,2014,2014,2014,2014,2014,2014,2014,2014,2010,2010,2010,2010,2010],"norm_year":["Year 6","Year 5","Year 3","Year 4","Year 4","Year 5","Year 7","Year 3","Year 5","Year 6","Year 5","Year 4","Year 5","Year 7","Year 5","Year 7","Year 3","Year 3","Year 3","Year 4","Year 1","Year 5","Year 5","Year 5","Year 4","Year 3","Year 4","Year 5","Year 5","Year 1","Year 0","Year 1","Year 5","Year 5","Year 6","Year 6","Year 7","Year 6","Year 5","Year 9","Year 10","Year 8","Year 8","Year 10","Year 9","Year 3","Year 7","Year 6","Year 6","Year 3","Year 7","Year 6","Year 4","Year 5","Year 4","Year 4","Year 3","Year 7","Year 6","Year 3","Year 3","Year 6","Year 4","Year 3","Year 3","Year 6","Year 5","Year 4","Year 6","Year 4","Year 6","Year 5","Year 5","Year 3","Year 5","Year 3","Year 3","Year 4","Year 4","Year 5","Year 4","Year 5","Year 3","Year 7","Year 4","Year 6","Year 11","Year 10","Year 11","Year 7","Year 8","Year 7","Year 10","Year 8","Year 12","Year 11","Year 8","Year 12","Year 10","Year 11","Year 11","Year 10","Year 12","Year 8","Year 8","Year 9","Year 7","Year 6","Year 8","Year 8","Year 7","Year 12","Year 10","Year 11","Year 8","Year 11","Year 10","Year 4","Year 9","Year 10","Year 4","Year 5","Year 4","Year 9","Year 9","Year 6","Year 9","Year 9","Year 5","Year 6","Year 9","Year 4","Year 7","Year 7","Year 9","Year 9","Year 9","Year 10","Year 10","Year 12","Year 9","Year 11","Year 11","Year 12","Year 11","Year 11","Year 11","Year 10","Year 9","Year 10","Year 8","Year 8","Year 10","Year 11","Year 10","Year 7","Year 3","Year 6","Year 3","Year 7","Year 6","Year 8","Year 7","Year 7","Year 5","Year 8","Year 7","Year 6","Year 4","Year 7","Year 6","Year 7","Year 6","Year 4","Year 7","Year 7","Year 7","Year 7","Year 6","Year 7","Year 7","Year 8","Year 4","Year 8","Year 8","Year 5","Year 7","Year 4","Year 8","Year 7","Year 10","Year 10","Year 10","Year 12","Year 12","Year 10","Year 7","Year 7","Year 6","Year 3","Year 7","Year 5","Year 5","Year 7","Year 7","Year 5","Year 7","Year 5","Year 7","Year 4","Year 7","Year 3","Year 7","Year 5","Year 5","Year 6","Year 7","Year 7","Year 7","Year 7","Year 4","Year 4","Year 5","Year 7","Year 6","Year 6","Year 7","Year 7","Year 6","Year 5","Year 6","Year 5","Year 6","Year 6","Year 8","Year 7","Year 7","Year 8","Year 6","Year 7","Year 7","Year 7","Year 7","Year 6","Year 7","Year 6","Year 4","Year 7","Year 5","Year 5","Year 7","Year 4","Year 6","Year 7","Year 7","Year 5","Year 7","Year 5","Year 7","Year 7","Year 7","Year 8","Year 8","Year 7","Year 5","Year 7","Year 8","Year 8","Year 8","Year 5","Year 4","Year 6","Year 7","Year 7","Year 9","Year 10","Year 9","Year 11","Year 10","Year 8","Year 8","Year 9","Year 6","Year 4","Year 7","Year 7","Year 4","Year 6","Year 11","Year 11","Year 8","Year 10","Year 8","Year 10","Year 9","Year 11","Year 11","Year 10","Year 11","Year 8","Year 11","Year 5","Year 5","Year 3","Year 7","Year 6","Year 6","Year 7","Year 6","Year 6","Year 8","Year 6","Year 6","Year 9","Year 9","Year 4","Year 6","Year 6","Year 3","Year 6","Year 6","Year 6","Year 6","Year 6","Year 6","Year 5","Year 5","Year 2","Year 8","Year 6","Year 8","Year 5","Year 7","Year 4","Year 8","Year 6","Year 5","Year 6","Year 7","Year 4","Year 7","Year 7","Year 5","Year 5","Year 9","Year 8","Year 7","Year 5","Year 6","Year 5","Year 9","Year 7","Year 10","Year 10","Year 8","Year 8"],"topic_year2":[2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2020,2020,2020,2016,2016,2016,2016,2016,2016,2016,2017,2017,2017,2017,2017,2017,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2020,2020,2020,2020,2020,2020,2016,2016,2016,2016,2016,2016,2016,2016,2019,2019,2019,2019,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2016,2016,2016,2016,2016,2016,2016,2018,2018,2018,2018,2018,2018,2014,2014,2014,2014,2014],"norm_year2":["Year 6","Year 5","Year 3","Year 4","Year 4","Year 5","Year 7","Year 3","Year 5","Year 6","Year 5","Year 4","Year 5","Year 7","Year 5","Year 7","Year 3","Year 3","Year 3","Year 4","Year 1","Year 5","Year 5","Year 5","Year 4","Year 3","Year 4","Year 5","Year 5","Year 1","Year 0","Year 1","Year 5","Year 5","Year 6","Year 6","Year 7","Year 6","Year 5","Year 4","Year 5","Year 3","Year 3","Year 5","Year 4","Year 0","Year 5","Year 4","Year 4","Year 1","Year 5","Year 4","Year 2","Year 3","Year 2","Year 2","Year 1","Year 5","Year 4","Year 1","Year 1","Year 4","Year 2","Year 1","Year 1","Year 4","Year 3","Year 2","Year 4","Year 2","Year 4","Year 3","Year 3","Year 1","Year 3","Year 1","Year 1","Year 2","Year 2","Year 3","Year 2","Year 3","Year 1","Year 5","Year 2","Year 2","Year 7","Year 6","Year 7","Year 3","Year 4","Year 3","Year 6","Year 4","Year 8","Year 7","Year 4","Year 8","Year 6","Year 7","Year 7","Year 6","Year 8","Year 4","Year 4","Year 5","Year 3","Year 2","Year 4","Year 4","Year 3","Year 8","Year 6","Year 7","Year 4","Year 7","Year 8","Year 2","Year 7","Year 8","Year 2","Year 3","Year 2","Year 7","Year 7","Year 4","Year 7","Year 7","Year 3","Year 4","Year 7","Year 2","Year 5","Year 5","Year 7","Year 4","Year 4","Year 5","Year 5","Year 7","Year 4","Year 6","Year 6","Year 7","Year 6","Year 6","Year 6","Year 5","Year 4","Year 5","Year 3","Year 3","Year 5","Year 6","Year 5","Year 2","Year 2","Year 5","Year 2","Year 6","Year 5","Year 7","Year 6","Year 6","Year 4","Year 7","Year 6","Year 5","Year 3","Year 6","Year 5","Year 6","Year 5","Year 3","Year 6","Year 6","Year 6","Year 6","Year 5","Year 6","Year 6","Year 7","Year 3","Year 7","Year 7","Year 4","Year 6","Year 3","Year 7","Year 6","Year 4","Year 4","Year 4","Year 6","Year 6","Year 4","Year 4","Year 4","Year 3","Year 0","Year 4","Year 2","Year 2","Year 4","Year 4","Year 2","Year 4","Year 2","Year 4","Year 1","Year 4","Year 0","Year 4","Year 2","Year 2","Year 3","Year 4","Year 4","Year 6","Year 6","Year 3","Year 3","Year 4","Year 6","Year 5","Year 5","Year 6","Year 6","Year 5","Year 4","Year 5","Year 4","Year 5","Year 5","Year 7","Year 6","Year 6","Year 7","Year 5","Year 6","Year 6","Year 6","Year 6","Year 5","Year 6","Year 5","Year 3","Year 6","Year 4","Year 4","Year 6","Year 3","Year 5","Year 6","Year 6","Year 4","Year 6","Year 4","Year 6","Year 6","Year 6","Year 7","Year 7","Year 6","Year 4","Year 6","Year 7","Year 7","Year 7","Year 4","Year 3","Year 5","Year 6","Year 6","Year 5","Year 6","Year 5","Year 7","Year 6","Year 4","Year 4","Year 7","Year 4","Year 2","Year 5","Year 5","Year 2","Year 4","Year 7","Year 7","Year 4","Year 6","Year 4","Year 6","Year 5","Year 7","Year 7","Year 6","Year 7","Year 4","Year 7","Year 0","Year 0","Year 0","Year 2","Year 1","Year 1","Year 6","Year 5","Year 5","Year 7","Year 2","Year 2","Year 5","Year 5","Year 0","Year 2","Year 2","Year 0","Year 4","Year 4","Year 4","Year 4","Year 4","Year 4","Year 3","Year 3","Year 0","Year 6","Year 4","Year 6","Year 3","Year 5","Year 2","Year 6","Year 4","Year 3","Year 4","Year 5","Year 2","Year 5","Year 5","Year 3","Year 3","Year 7","Year 4","Year 3","Year 1","Year 2","Year 1","Year 5","Year 3","Year 6","Year 6","Year 4","Year 4"],"deidentified_name":["person1","person2","person3","person4","person4","person5","person3","person3","person6","person7","person2","person3","person4","person6","person8","person4","person2","person9","person9","person2","person1","person10","person11","person12","person5","person4","person4","person2","person4","person13","person13","person13","person6","person6","person4","person14","person2","person2","person15","person10","person2","person2","person3","person6","person2","person1","person16","person17","person7","person2","person18","person18","person2","person18","person3","person10","person3","person18","person3","person4","person2","person2","person4","person4","person4","person4","person18","person4","person5","person17","person18","person4","person17","person4","person2","person6","person4","person18","person2","person11","person2","person18","person2","person18","person18","person2","person3","person13","person2","person3","person3","person2","person2","person2","person10","person14","person4","person16","person2","person2","person2","person4","person2","person2","person5","person2","person2","person2","person2","person11","person3","person3","person2","person2","person3","person4","person4","person19","person20","person2","person3","person3","person3","person3","person11","person3","person3","person5","person2","person21","person3","person4","person3","person19","person21","person2","person5","person7","person3","person5","person3","person7","person11","person4","person11","person5","person15","person3","person20","person3","person6","person4","person7","person3","person22","person3","person15","person16","person16","person14","person15","person16","person16","person14","person2","person4","person16","person15","person16","person16","person4","person4","person5","person3","person15","person16","person4","person16","person16","person4","person16","person16","person21","person15","person3","person3","person16","person3","person15","person3","person3","person10","person4","person19","person3","person3","person3","person21","person3","person4","person2","person23","person5","person13","person22","person9","person7","person8","person5","person3","person11","person3","person4","person13","person16","person2","person5","person3","person8","person13","person3","person14","person21","person3","person3","person2","person22","person20","person13","person20","person2","person3","person13","person24","person5","person25","person2","person15","person4","person26","person15","person5","person3","person3","person20","person23","person4","person13","person27","person3","person5","person28","person28","person15","person6","person9","person7","person23","person4","person17","person11","person13","person13","person2","person23","person23","person3","person11","person4","person24","person17","person2","person11","person3","person23","person3","person23","person13","person29","person23","person3","person23","person19","person3","person5","person19","person8","person9","person30","person30","person2","person3","person3","person30","person30","person30","person30","person6","person23","person30","person30","person3","person4","person3","person3","person17","person3","person31","person2","person3","person31","person2","person3","person31","person31","person30","person30","person30","person2","person3","person2","person3","person3","person3","person3","person3","person3","person3","person2","person3","person3","person3","person11","person3","person3","person3","person3","person3","person10","person10","person10","person10","person10","person3","person7","person6","person6","person3","person6","person8","person6","person3","person22","person4","person2","person4"],"used_for_pilot":[false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,false,true,false,false,false,false,false,false,false,true,false,false,false,true,false,false,true,false,false,false,true,false,false,false,true,false,false,false,false,false,false,true,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,true,false,false,false,false,false,false,false,false,false,true,true,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false]}}]}

</script>

<script type="ojs-define">

{"contents":[{"name":"paper_links","value":{"Paper_title":["A benchmark of batch-effect correction methods for single-cell RNA sequencing data","A benchmark study of deep learning-based multi-omics data fusion methods for cancer","A benchmark study of simulation methods for single-cell RNA sequencing data","A Comparison for Dimensionality Reduction Methods of Single-Cell RNA-seq Data","A comparison framework and guideline of clustering methods for mass cytometry data","A comparison of automatic cell identification methods for single-cell RNA sequencing data","A comparison of methods accounting for batch effects in differential expression analysis of UMI count based single cell RNA sequencing","A comparison of single-cell trajectory inference methods","A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics","A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data","A Framework for Comparison and Assessment of Synthetic RNA-Seq Data","A Hybrid Clustering Algorithm for Identifying Cell Types from Single-Cell RNA-Seq Data","A lineage tree-based hidden Markov model quantifies cellular heterogeneity and plasticity","A model-based constrained deep learning clustering approach for spatially resolved single-cell data","A multi-center cross-platform single-cell RNA sequencing reference dataset","A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples","A multiresolution framework to characterize single-cell state landscapes","A novel method for single-cell data imputation using subspace regression","A Quantitative Framework for Evaluating Single-Cell Data Structure Preservation by Dimensionality Reduction Techniques","A rank-based marker selection method for high throughput scRNA-seq data","A review and performance evaluation of clustering frameworks for single-cell Hi-C data","A sparse differential clustering algorithm for tracing cell type changes via single-cell RNA-sequencing data","A spectral clustering with self-weighted multiple kernel learning method for single-cell RNA-seq data","A statistical simulator scDesign for rational scRNA-seq experimental design","A systematic evaluation of single cell RNA-seq analysis pipelines","A systematic evaluation of single-cell RNA-sequencing imputation methods","A Systematic Evaluation of Supervised Machine Learning Algorithms for Cell Phenotype Classification Using Single-Cell RNA Sequencing Data","A systematic evaluation of the computational tools for ligand-receptor-based cell-cell interaction inference","A systematic performance evaluation of clustering methods for single-cell RNA-seq data","A Variational Algorithm to Detect the Clonal Copy Number Substructure of Tumors from scRNA-seq Data","Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis","Advances in mixed cell deconvolution enable quantification of cell types in spatial transcriptomic data","Adversarial dense graph convolutional networks for single-cell classification","Adversarial domain translation networks for integrating large-scale atlas-level single-cell datasets","AIscEA: unsupervised integration of single-cell gene expression and chromatin accessibility via their biological consistency","Alignment and integration of spatial transcriptomics data","Are dropout imputation methods for scRNA-seq effective for scATAC-seq data?","Assessment of computational methods for the analysis of single-cell ATAC-seq data","Assessment of Single Cell RNA-Seq Normalization Methods","ASTER: accurately estimating the number of cell types in single-cell chromatin accessibility data","Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis","Bayesian information sharing enhances detection of regulatory associations in rare cell types","BBKNN: fast batch alignment of single cell transcriptomes","Benchmark and Parameter Sensitivity Analysis of Single-Cell RNA Sequencing Clustering Methods","Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data","Benchmarking algorithms for pathway activity transformation of single-cell RNA-seq data","Benchmarking and integration of methods for deconvoluting spatial transcriptomic data","Benchmarking atlas-level data integration in single-cell genomics","Benchmarking automated cell type annotation tools for single-cell ATAC-seq data","Benchmarking cell-type clustering methods for spatially resolved transcriptomics data","Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data","Benchmarking computational doublet-detection methods for single-cell RNA sequencing data","Benchmarking computational methods for B-cell receptor reconstruction from single-cell RNA-seq data","Benchmarking integration of single-cell differential expression","Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer","Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data","Benchmarking of a Bayesian single cell RNAseq differential gene expression test for dose–response study designs","Benchmarking of cell type deconvolution pipelines for transcriptomics data","Benchmarking principal component analysis for large-scale single-cell RNA-sequencing","Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments","Benchmarking spatial and single-cell transcriptomics integration methods","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","Benchmarking UMI-based single-cell RNA-seq preprocessing workflows","Bias, robustness and scalability in single-cell differential expression analysis","Built on sand: the shaky foundations of simulating single-cell RNA sequencing data","Cell segmentation in imaging-based spatial transcriptomics","CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing","CellMixS: quantifying and visualizing batch effects in single-cell RNA-seq data","Cellpose: a generalist algorithm for cellular segmentation","CellSIUS provides sensitive and specific detection of rare cell populations from complex single-cell RNA-seq data","CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing","Choice of pre-processing pipeline influences clustering quality of scRNA-seq datasets","ChrNet: A re-trainable chromosome-based 1D convolutional neural network for predicting immune cell types","CIDR: Ultrafast and accurate clustering through imputation for single-cell RNA-seq data","Clustering methods for single-cell RNA-sequencing expression data: performance evaluation with varying sample sizes and cell compositions","ClusterMap for multi-scale clustering analysis of spatial gene expression","Comparative analysis of common alignment tools for single-cell RNA sequencing","Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data","Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data","Comparison of Computational Methods for Imputing Single-Cell RNA-Sequencing Data","Comparison of high-throughput single-cell RNA sequencing data processing pipelines","Comparison of methods to detect differentially expressed genes between single-cell populations","Comparison of visualization tools for single-cell RNAseq data","Comprehensive evaluation of deconvolution methods for human brain gene expression","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology","Cross-species cell-type assignment from single-cell RNA-seq data by a heterogeneous graph neural network","Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq","De novo gene signature identification from single-cell RNA-seq with hierarchical Poisson factorization","Deconvolution analysis of cell-type expression from bulk tissues by integrating with single-cell expression reference","DEEPsc: A Deep Learning-Based Map Connecting Single-Cell Transcriptomics and Spatial Imaging Data","DeepST: identifying spatial domains in spatial transcriptomics by deep learning","DeepVelo: Single-cell transcriptomic deep velocity field learning with neural ordinary differential equations","Delineating Copy Number and Clonal Substructure in Human Tumors from Single-cell Transcriptomes","DENDRO: genetic heterogeneity profiling and subclone detection by single-cell RNA sequencing","Density-based detection of cell transition states to construct disparate and bifurcating trajectories","Destin2: Integrative and cross-modality analysis of single-cell chromatin accessibility data","Detecting critical transition signals from single-cell transcriptomes to infer lineage-determining transcription factors","Differential abundance testing on single-cell data using k-nearest neighbor graphs","Differential expression of single-cell RNA-seq data using Tweedie models","Dimensionality reduction for visualizing single-cell data using UMAP","DIRECT-NET: An efficient method to discover cis-regulatory elements and construct regulatory networks from single-cell multiomics data","Discovery of rare cells from voluminous single cell expression data","Dissecting cell identity via network inference and in silico gene perturbation","Doublet identification in single-cell sequencing data using scDblFinder","DrivAER: Identification of driving transcriptional programs in single-cell RNA sequencing data","EDClust: an EM-MM hybrid method for cell clustering in multiple-subject single-cell RNA sequencing","Efficient prediction of a spatial transcriptomics profile better characterizes breast cancer tissue sections without costly experimentation","EmptyNN: A neural network based on positive and unlabeled learning to remove cell-free droplets and recover lost cells in scRNA-seq data","EpiScanpy: integrated single-cell epigenomic analysis","ESCO: single cell expression simulation incorporating gene co-expression","Evaluating measures of association for single-cell transcriptomics","Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data","Evaluating the reproducibility of single-cell gene regulatory network inference algorithms","Evaluation of Cell Type Annotation R Packages on Single-cell RNA-seq Data","Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information","Evaluation of machine learning approaches for cell-type identification from single-cell transcriptomics data","Evaluation of methods to assign cell type labels to cell clusters from single-cell RNA-sequencing data","Evaluation of single-cell classifiers for single-cell RNA sequencing data sets","Evaluation of some aspects in supervised cell type identification for single-cell RNA-seq: classifier, feature selection, and reference construction","Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment","Evaluation of tools for highly variable gene discovery from single-cell RNA-seq data","False signals induced by single-cell imputation","Fast, sensitive and accurate integration of single-cell data with Harmony","Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench","Functional inference of gene regulation using single-cell multi-omics","GapClust is a light-weight approach distinguishing rare cells from voluminous single cell expression profiles","Generalizable and Scalable Visualization of Single-Cell Data Using Neural Networks","Giotto: a toolbox for integrative analysis and visualization of spatial expression data","HGC: fast hierarchical clustering for large-scale single-cell data","High-throughput single-cell RNA-seq data imputation and characterization with surrogate-assisted automated deep learning","histocat: analysis of cell phenotypes and interactions in multiplex image cytometry data","How does the structure of data impact cell-cell similarity? Evaluating how structural properties influence the performance of proximity metrics in single cell RNA-seq data","IBRAP: integrated benchmarking single-cell RNA-sequencing analytical pipeline","Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq","Identifying phenotype-associated subpopulations by integrating bulk and single-cell sequencing data","Identifying strengths and weaknesses of methods for computational network inference from single-cell RNA-seq data","Impact of similarity metrics on single-cell RNA-seq data clustering","Influence of Cell-type Ratio on Spatially Resolved Single-cell Transcriptomes using the Tangram Algorithm: Based on Implementation on Single-Cell and MxIF Data","Integrating multiple references for single-cell assignment","Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction","Integration of single cell data by disentangled representation learning","Interactive analysis of single-cell epigenomic landscapes with ChromSCape","IReNA: Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles","iSMNN: batch effect correction for single-cell RNA-seq data via iterative supervised mutual nearest neighbor refinement","Joint cell segmentation and cell type annotation for spatial transcriptomics","Joint dimension reduction and clustering analysis of single-cell RNA-seq and spatial transcriptomics data","Latent cellular analysis robustly reveals subtle diversity in large-scale single-cell RNA-seq data","Learning Cell Annotation under Multiple Reference Datasets by Multisource Domain Adaptation","Learning for single-cell assignment","Mapping molecular assemblies with fluorescence microscopy and object-based spatial statistics","maxATAC: Genome-scale transcription-factor binding prediction from ATAC-seq with deep neural networks","Multi-omics single-cell data integration and regulatory inference with graph-linked embedding","NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks","Network embedding-based representation learning for single cell RNA-seq data","NISC: Neural Network-Imputation for Single-Cell RNA Sequencing and Cell Type Clustering","Nonparametric single-cell multiomic characterization of trio relationships between transcription factors, target genes, and cis-regulatory regions","Normalization and de-noising of single-cell Hi-C data with BandNorm and scVI-3D","Normalization Methods on Single-Cell RNA-seq Data: An Empirical Survey","Novel ChIP-seq simulating program with superior versatility: isChIP","Optimal transport improves cell-cell similarity inference in single-cell omics data","PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells","Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics","Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq","Performance assessment of sample-specific network control methods for bulk and single-cell biological data analysis","Perspectives for the reconstruction of 3D chromatin conformation using single cell Hi-C data","Pheniqs 20: accurate, high-performance Bayesian decoding and confidence estimation for combinatorial barcode indexing","pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools","powsimR: power analysis for bulk and single cell RNA-seq experiments","Predicting cell lineages using autoencoders and optimal transport","Predicting recognition between T cell receptors and epitopes with TCRGP","Preprocessing choices affect RNA velocity results for droplet scRNA-seq data","PseudoGA: cell pseudotime reconstruction based on genetic algorithm","psupertime: supervised pseudotime analysis for time-series single-cell RNA-seq data","Quantifying Waddington’s epigenetic landscape: a comparison of single-cell potency measures","Realistic in silico generation and augmentation of single-cell RNA-seq data using generative adversarial networks","Recommendations of scRNA-seq Differential Gene Expression Analysis Based on Comprehensive Benchmarking","Region-specific denoising identifies spatial co-expression patterns and intra-tissue heterogeneity in spatially resolved transcriptomics data","Regulatory analysis of single cell multiome gene expression and chromatin accessibility data with scREG","Reproducibility of Methods to Detect Differentially Expressed Genes from Single-Cell RNA Sequencing","Resolving single-cell copy number profiling for large datasets","Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data","sc3: consensus clustering of single-cell rna-seq data","scAB detects multiresolution cell states with clinical significance by integrating single-cell genomics and bulk sequencing data","SCAN-ATAC-Sim: a scalable and efficient method for simulating single-cell ATAC-seq data from bulk-tissue experiments","scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data","scClassify: sample size estimation and multiscale classification of cells using single and multiple reference","scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured","scESI: evolutionary sparse imputation for single-cell transcriptomes from nearest neighbor cells","scGAD: a new task and end-to-end framework for generalized cell type annotation and discovery","scGCN is a graph convolutional networks algorithm for knowledge transfer in single cell omics","scGNN is a novel graph neural network framework for single-cell RNA-Seq analyses","scHinter: imputing dropout events for single-cell RNA-seq data with limited sample size","scIMC: a platform for benchmarking comparison and visualization analysis of scRNA-seq data imputation methods","scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning","scMAGIC: accurately annotating single cells using two rounds of reference-based classification","scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets","scMINER: a mutual information-based framework for identifying hidden drivers from single-cell omics data","scNPF: an integrative framework assisted by network propagation and network fusion for preprocessing of single-cell RNA-seq data","scPred: accurate supervised method for cell-type classification from single-cell RNA-seq data","scShaper: an ensemble method for fast and accurate linear trajectory inference from single-cell RNA-seq data","SCSsim: an integrated tool for simulating single-cell genome sequencing data","SD2: spatially resolved transcriptomics deconvolution through integration of dropout and spatial information","Secuer: Ultrafast, scalable and accurate clustering of single-cell RNA-seq data","Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data","simATAC: a single-cell ATAC-seq simulation framework","SimCH: simulation of single-cell RNA sequencing data by modeling cellular heterogeneity at gene expression level","Simulation-based benchmarking of isoform quantification in single-cell RNA-seq","Simulation, power evaluation and sample size recommendation for single-cell RNA-seq","SINFONIA: Scalable Identification of Spatially Variable Genes for Deciphering Spatial Domains","Single-cell biological network inference using a heterogeneous graph transformer","Single-cell gene regulatory network prediction by explainable AI","Single-cell multi-omics integration for unpaired data by a siamese network with graph-based contrastive loss","Single-Cell RNA Sequencing Data Interpretation by Evolutionary Multiobjective Clustering","Single-cell RNA-sequencing: assessment of differential expression analysis methods","SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data","SOMDE: a scalable method for identifying spatially variable genes with self-organizing map","SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network","SpaGE: Spatial Gene Enhancement using scRNA-seq","SPARK-X: non-parametric modeling enables scalable and robust detection of spatial expression patterns for large spatial transcriptomic studies","Spatial transcriptomics at subspot resolution with BayesSpace","Spatial-ID: a cell typing method for spatially resolved transcriptomics via transfer learning and spatial embedding","SpatialDE: identification of spatially variable genes","SpatialDWLS: accurate deconvolution of spatial transcriptomic data","Spatially informed clustering, integration, and deconvolution of spatial transcriptomics with GraphST","Spearheading future omics analyses using dyngen, a multi-modal simulator of single cells","spicyR: spatial analysis of in situ cytometry data in R","splatPop: simulating population scale single-cell RNA sequencing data","SRGS: sparse partial least squares-based recursive gene selection for gene regulatory network inference","Supervised application of internal validation measures to benchmark dimensionality reduction methods in scRNA-seq data","Supervised classification enables rapid annotation of cell atlases","Systematic comparative analysis of single-nucleotide variant detection methods from single-cell RNA sequencing data","Systematic evaluation of transcriptomics-based deconvolution methods and references using thousands of clinical samples","The Current State of Single-Cell Proteomics Data Analysis","TISCH: a comprehensive web resource enabling interactive single-cell transcriptome visualization of tumor microenvironment","TIST: Transcriptome and Histopathological Image Integrative Analysis for Spatial Transcriptomics","Towards a comprehensive evaluation of dimension reduction methods for transcriptomic data visualization","Transcriptome analysis method based on differential distribution evaluation","Triku: a feature selection method based on nearest neighbors for single-cell data","Tuning parameters of dimensionality reduction methods for single-cell RNA-seq analysis","Ultra-fast scalable estimation of single-cell differentiation potency from scRNA-Seq data","Uncovering cell identity through differential stability with Cepo","Using single-cell cytometry to illustrate integrated multi-perspective evaluation of clustering algorithms using Pareto fronts","Vaeda computationally annotates doublets in single-cell RNA sequencing data","Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning"],"Link":["https://pubmed.ncbi.nlm.nih.gov/31948481/","https://pubmed.ncbi.nlm.nih.gov/35945544/","https://ideas.repec.org/a/nat/natcom/v12y2021i1d10.1038_s41467-021-27130-w.html","https://pubmed.ncbi.nlm.nih.gov/33833778/","https://pubmed.ncbi.nlm.nih.gov/31870419/","https://pubmed.ncbi.nlm.nih.gov/31500660/","https://pubmed.ncbi.nlm.nih.gov/32322368/","https://pubmed.ncbi.nlm.nih.gov/30936559/","https://pubmed.ncbi.nlm.nih.gov/36941264/","https://pubmed.ncbi.nlm.nih.gov/34020546/","https://pubmed.ncbi.nlm.nih.gov/36553629/","https://pubmed.ncbi.nlm.nih.gov/30700040/","https://pubmed.ncbi.nlm.nih.gov/36396800/","https://pubmed.ncbi.nlm.nih.gov/36198490/","https://pubmed.ncbi.nlm.nih.gov/33531477/","https://pubmed.ncbi.nlm.nih.gov/33349700/","https://pubmed.ncbi.nlm.nih.gov/33106496/","https://pubmed.ncbi.nlm.nih.gov/35177662/","https://pubmed.ncbi.nlm.nih.gov/32375029/","https://pubmed.ncbi.nlm.nih.gov/33097004/","https://pubmed.ncbi.nlm.nih.gov/36151714/","https://pubmed.ncbi.nlm.nih.gov/29140455/","https://pubmed.ncbi.nlm.nih.gov/33003206/","https://pubmed.ncbi.nlm.nih.gov/31510652/","https://pubmed.ncbi.nlm.nih.gov/31604912/","https://pubmed.ncbi.nlm.nih.gov/32854757/","https://pubmed.ncbi.nlm.nih.gov/35281805/","https://pubmed.ncbi.nlm.nih.gov/35822343/","https://pubmed.ncbi.nlm.nih.gov/30271584/","https://pubmed.ncbi.nlm.nih.gov/36841879/","https://pubmed.ncbi.nlm.nih.gov/31823809/","https://pubmed.ncbi.nlm.nih.gov/35046414/","https://pubmed.ncbi.nlm.nih.gov/36661313/","https://pubmed.ncbi.nlm.nih.gov/38177826/","https://pubmed.ncbi.nlm.nih.gov/36250795/","https://pubmed.ncbi.nlm.nih.gov/35577957/","https://pubmed.ncbi.nlm.nih.gov/34718405/","https://pubmed.ncbi.nlm.nih.gov/31739806/","https://pubmed.ncbi.nlm.nih.gov/28468817/","https://pubmed.ncbi.nlm.nih.gov/36610708/","https://pubmed.ncbi.nlm.nih.gov/31870278/","https://pubmed.ncbi.nlm.nih.gov/34252956/","https://pubmed.ncbi.nlm.nih.gov/31400197/","https://pubmed.ncbi.nlm.nih.gov/31921297/","https://pubmed.ncbi.nlm.nih.gov/31907445/","https://pubmed.ncbi.nlm.nih.gov/33209207/","https://pubmed.ncbi.nlm.nih.gov/36515467/","https://pubmed.ncbi.nlm.nih.gov/34949812/","https://pubmed.ncbi.nlm.nih.gov/36583014/","https://pubmed.ncbi.nlm.nih.gov/36410733/","https://pubmed.ncbi.nlm.nih.gov/35135612/","https://pubmed.ncbi.nlm.nih.gov/33338399/","https://pubmed.ncbi.nlm.nih.gov/35855325/","https://pubmed.ncbi.nlm.nih.gov/36944632/","https://pubmed.ncbi.nlm.nih.gov/33402734/","https://pubmed.ncbi.nlm.nih.gov/35880426/","https://pubmed.ncbi.nlm.nih.gov/35061903/","https://pubmed.ncbi.nlm.nih.gov/33159064/","https://pubmed.ncbi.nlm.nih.gov/31955711/","https://pubmed.ncbi.nlm.nih.gov/31133762/","https://pubmed.ncbi.nlm.nih.gov/35577955/","https://pubmed.ncbi.nlm.nih.gov/35577954/","https://pubmed.ncbi.nlm.nih.gov/34906205/","https://pubmed.ncbi.nlm.nih.gov/29481549/","https://pubmed.ncbi.nlm.nih.gov/36991470/","https://pubmed.ncbi.nlm.nih.gov/34650268/","https://pubmed.ncbi.nlm.nih.gov/30744683/","https://pubmed.ncbi.nlm.nih.gov/33758076/","https://pubmed.ncbi.nlm.nih.gov/33318659/","https://pubmed.ncbi.nlm.nih.gov/31315641/","https://pubmed.ncbi.nlm.nih.gov/31226206/","https://pubmed.ncbi.nlm.nih.gov/34521337/","https://pubmed.ncbi.nlm.nih.gov/33932523/","https://pubmed.ncbi.nlm.nih.gov/28351406/","https://pubmed.ncbi.nlm.nih.gov/31646845/","https://pubmed.ncbi.nlm.nih.gov/34625546/","https://pubmed.ncbi.nlm.nih.gov/35084033/","https://pubmed.ncbi.nlm.nih.gov/30658573/","https://pubmed.ncbi.nlm.nih.gov/30228881/","https://pubmed.ncbi.nlm.nih.gov/29994128/","https://pubmed.ncbi.nlm.nih.gov/34020539/","https://pubmed.ncbi.nlm.nih.gov/27373736/","https://pubmed.ncbi.nlm.nih.gov/32766548/","https://pubmed.ncbi.nlm.nih.gov/35292647/","https://pubmed.ncbi.nlm.nih.gov/35048125/","https://pubmed.ncbi.nlm.nih.gov/31510660/","https://pubmed.ncbi.nlm.nih.gov/36526433/","https://pubmed.ncbi.nlm.nih.gov/32719530/","https://pubmed.ncbi.nlm.nih.gov/30796088/","https://pubmed.ncbi.nlm.nih.gov/35788983/","https://pubmed.ncbi.nlm.nih.gov/33833776/","https://pubmed.ncbi.nlm.nih.gov/36250636/","https://pubmed.ncbi.nlm.nih.gov/36449617/","https://pubmed.ncbi.nlm.nih.gov/33462507/","https://pubmed.ncbi.nlm.nih.gov/31937348/","https://pubmed.ncbi.nlm.nih.gov/36124665/","https://pubmed.ncbi.nlm.nih.gov/36873935/","https://pubmed.ncbi.nlm.nih.gov/35640613/","https://pubmed.ncbi.nlm.nih.gov/34594043/","https://pubmed.ncbi.nlm.nih.gov/35656596/","https://pubmed.ncbi.nlm.nih.gov/30531897/","https://pubmed.ncbi.nlm.nih.gov/35648859/","https://pubmed.ncbi.nlm.nih.gov/30413715/","https://pubmed.ncbi.nlm.nih.gov/36755098/","https://pubmed.ncbi.nlm.nih.gov/35814628/","https://pubmed.ncbi.nlm.nih.gov/33301553/","https://pubmed.ncbi.nlm.nih.gov/35561178/","https://pubmed.ncbi.nlm.nih.gov/35260632/","https://pubmed.ncbi.nlm.nih.gov/34430929/","https://pubmed.ncbi.nlm.nih.gov/34471111/","https://pubmed.ncbi.nlm.nih.gov/33624750/","https://pubmed.ncbi.nlm.nih.gov/30962620/","https://pubmed.ncbi.nlm.nih.gov/29914350/","https://pubmed.ncbi.nlm.nih.gov/33828580/","https://pubmed.ncbi.nlm.nih.gov/33359678/","https://pubmed.ncbi.nlm.nih.gov/36253792/","https://pubmed.ncbi.nlm.nih.gov/33611343/","https://pubmed.ncbi.nlm.nih.gov/31508207/","https://pubmed.ncbi.nlm.nih.gov/31675098/","https://pubmed.ncbi.nlm.nih.gov/34503564/","https://pubmed.ncbi.nlm.nih.gov/32220951/","https://pubmed.ncbi.nlm.nih.gov/29481632/","https://pubmed.ncbi.nlm.nih.gov/30906525/","https://pubmed.ncbi.nlm.nih.gov/31740819/","https://pubmed.ncbi.nlm.nih.gov/33524142/","https://pubmed.ncbi.nlm.nih.gov/36204155/","https://pubmed.ncbi.nlm.nih.gov/34234139/","https://pubmed.ncbi.nlm.nih.gov/29936184/","https://pubmed.ncbi.nlm.nih.gov/33685491/","https://pubmed.ncbi.nlm.nih.gov/34096998/","https://pubmed.ncbi.nlm.nih.gov/34553763/","https://pubmed.ncbi.nlm.nih.gov/28783155/","https://pubmed.ncbi.nlm.nih.gov/36151725/","https://pubmed.ncbi.nlm.nih.gov/36847692/","https://pubmed.ncbi.nlm.nih.gov/31282856/","https://pubmed.ncbi.nlm.nih.gov/34764492/","https://pubmed.ncbi.nlm.nih.gov/36626328/","https://pubmed.ncbi.nlm.nih.gov/30137247/","https://pubmed.ncbi.nlm.nih.gov/37324550/","https://pubmed.ncbi.nlm.nih.gov/34037791/","https://pubmed.ncbi.nlm.nih.gov/36064614/","https://pubmed.ncbi.nlm.nih.gov/34850092/","https://pubmed.ncbi.nlm.nih.gov/33177523/","https://pubmed.ncbi.nlm.nih.gov/36325073/","https://pubmed.ncbi.nlm.nih.gov/33839756/","https://pubmed.ncbi.nlm.nih.gov/34057817/","https://pubmed.ncbi.nlm.nih.gov/35349708/","https://pubmed.ncbi.nlm.nih.gov/31566233/","https://pubmed.ncbi.nlm.nih.gov/36579851/","https://pubmed.ncbi.nlm.nih.gov/33127686/","https://pubmed.ncbi.nlm.nih.gov/29449608/","https://pubmed.ncbi.nlm.nih.gov/36719906/","https://pubmed.ncbi.nlm.nih.gov/35501393/","https://pubmed.ncbi.nlm.nih.gov/36418514/","https://pubmed.ncbi.nlm.nih.gov/28977434/","https://pubmed.ncbi.nlm.nih.gov/35591853/","https://pubmed.ncbi.nlm.nih.gov/36055233/","https://pubmed.ncbi.nlm.nih.gov/36253828/","https://pubmed.ncbi.nlm.nih.gov/32117453/","https://pubmed.ncbi.nlm.nih.gov/33320934/","https://pubmed.ncbi.nlm.nih.gov/35157031/","https://pubmed.ncbi.nlm.nih.gov/30890159/","https://pubmed.ncbi.nlm.nih.gov/30963075/","https://pubmed.ncbi.nlm.nih.gov/31022373/","https://pubmed.ncbi.nlm.nih.gov/33956788/","https://pubmed.ncbi.nlm.nih.gov/34793453/","https://pubmed.ncbi.nlm.nih.gov/34215187/","https://pubmed.ncbi.nlm.nih.gov/32873325/","https://pubmed.ncbi.nlm.nih.gov/29036287/","https://pubmed.ncbi.nlm.nih.gov/32343706/","https://pubmed.ncbi.nlm.nih.gov/33764977/","https://pubmed.ncbi.nlm.nih.gov/33428615/","https://pubmed.ncbi.nlm.nih.gov/34244782/","https://pubmed.ncbi.nlm.nih.gov/35758781/","https://pubmed.ncbi.nlm.nih.gov/30289442/","https://pubmed.ncbi.nlm.nih.gov/31919373/","https://pubmed.ncbi.nlm.nih.gov/35743881/","https://pubmed.ncbi.nlm.nih.gov/36376296/","https://pubmed.ncbi.nlm.nih.gov/35578363/","https://pubmed.ncbi.nlm.nih.gov/32010190/","https://pubmed.ncbi.nlm.nih.gov/35801503/","https://pubmed.ncbi.nlm.nih.gov/32051003/","https://pubmed.ncbi.nlm.nih.gov/28346451/","https://pubmed.ncbi.nlm.nih.gov/36440766/","https://pubmed.ncbi.nlm.nih.gov/33471102/","https://pubmed.ncbi.nlm.nih.gov/32062421/","https://pubmed.ncbi.nlm.nih.gov/32567229/","https://pubmed.ncbi.nlm.nih.gov/34034771/","https://pubmed.ncbi.nlm.nih.gov/35512331/","https://pubmed.ncbi.nlm.nih.gov/36869836/","https://pubmed.ncbi.nlm.nih.gov/34158507/","https://pubmed.ncbi.nlm.nih.gov/33767197/","https://pubmed.ncbi.nlm.nih.gov/31392316/","https://pubmed.ncbi.nlm.nih.gov/35524568/","https://pubmed.ncbi.nlm.nih.gov/35058621/","https://pubmed.ncbi.nlm.nih.gov/34986249/","https://pubmed.ncbi.nlm.nih.gov/31028141/","https://pubmed.ncbi.nlm.nih.gov/36747874/","https://pubmed.ncbi.nlm.nih.gov/31068142/","https://pubmed.ncbi.nlm.nih.gov/31829268/","https://pubmed.ncbi.nlm.nih.gov/34888622/","https://pubmed.ncbi.nlm.nih.gov/31584615/","https://pubmed.ncbi.nlm.nih.gov/36063455/","https://pubmed.ncbi.nlm.nih.gov/36469543/","https://pubmed.ncbi.nlm.nih.gov/35212622/","https://pubmed.ncbi.nlm.nih.gov/33663563/","https://pubmed.ncbi.nlm.nih.gov/36575569/","https://pubmed.ncbi.nlm.nih.gov/30404663/","https://pubmed.ncbi.nlm.nih.gov/32614380/","https://pubmed.ncbi.nlm.nih.gov/36831270/","https://pubmed.ncbi.nlm.nih.gov/36810839/","https://pubmed.ncbi.nlm.nih.gov/36629274/","https://pubmed.ncbi.nlm.nih.gov/36600199/","https://pubmed.ncbi.nlm.nih.gov/30908236/","https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2017.00062/full","https://pubmed.ncbi.nlm.nih.gov/34446921/","https://pubmed.ncbi.nlm.nih.gov/34165490/","https://pubmed.ncbi.nlm.nih.gov/34711970/","https://pubmed.ncbi.nlm.nih.gov/32955565/","https://pubmed.ncbi.nlm.nih.gov/34154649/","https://pubmed.ncbi.nlm.nih.gov/34083791/","https://pubmed.ncbi.nlm.nih.gov/36496406/","https://pubmed.ncbi.nlm.nih.gov/29553579/","https://pubmed.ncbi.nlm.nih.gov/33971932/","https://pubmed.ncbi.nlm.nih.gov/36859400/","https://pubmed.ncbi.nlm.nih.gov/34168133/","https://pubmed.ncbi.nlm.nih.gov/35438129/","https://pubmed.ncbi.nlm.nih.gov/34911537/","https://pubmed.ncbi.nlm.nih.gov/36451086/","https://pubmed.ncbi.nlm.nih.gov/34374742/","https://pubmed.ncbi.nlm.nih.gov/31501545/","https://pubmed.ncbi.nlm.nih.gov/31744515/","https://pubmed.ncbi.nlm.nih.gov/34346485/","https://pubmed.ncbi.nlm.nih.gov/36633424/","https://pubmed.ncbi.nlm.nih.gov/33179754/","https://pubmed.ncbi.nlm.nih.gov/36549467/","https://pubmed.ncbi.nlm.nih.gov/35853932/","https://pubmed.ncbi.nlm.nih.gov/35151228/","https://pubmed.ncbi.nlm.nih.gov/35277963/","https://pubmed.ncbi.nlm.nih.gov/32831127/","https://pubmed.ncbi.nlm.nih.gov/33244588/","https://pubmed.ncbi.nlm.nih.gov/38217190/","https://pubmed.ncbi.nlm.nih.gov/33508103/","https://pubmed.ncbi.nlm.nih.gov/36342203/","https://pubmed.ncbi.nlm.nih.gov/28263960/"]}}]}

</script>



</head>

<body class="quarto-dashboard nav-sidebar dashboard-fill fullcontent">

<header id="quarto-dashboard-header">
<nav class="navbar navbar-expand-md slim" data-bs-theme="dark">
  <div class="navbar-container container-fluid">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#dashboard-collapse" aria-controls="dashboard-collapse" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>    


    <div class="navbar-brand-container">       
    <a href="#"><img src="https://www.sydney.edu.au/etc.clientlibs/corporate-commons/clientlibs/foundation/resources/corporate-frontend/assets/img/favicon/favicon-16x16.png" alt="" class="navbar-logo d-inline-block"></a>
      <div class="navbar-title">
        <div class="navbar-title-text"><a href="#">sc bench benchmark dashboard</a></div>
        
        
      </div>
    </div>
<div id="dashboard-collapse" class="navbar-collapse collapse"><ul class="navbar-nav navbar-nav-scroll me-auto" role="tablist"><li class="nav-item" role="presentation"><a id="tab-paper" class="nav-link active" data-bs-toggle="tab" role="tab" data-bs-target="#paper" data-scrolling="false" href="#paper" aria-controls="paper" aria-selected="true"><span class="nav-link-text">Paper</span></a></li><li class="nav-item" role="presentation"><a id="tab-data-overview" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#data-overview" data-scrolling="false" href="#data-overview" aria-controls="data-overview" aria-selected="false"><span class="nav-link-text">Data overview</span></a></li><li class="nav-item" role="presentation"><a id="tab-evaluation-summary" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#evaluation-summary" data-scrolling="false" href="#evaluation-summary" aria-controls="evaluation-summary" aria-selected="false"><span class="nav-link-text">Evaluation Summary</span></a></li><li class="nav-item" role="presentation"><a id="tab-general-information" class="nav-link" data-bs-toggle="tab" role="tab" data-bs-target="#general-information" data-scrolling="false" href="#general-information" aria-controls="general-information" aria-selected="false"><span class="nav-link-text">General Information</span></a></li></ul><div class="quarto-dashboard-links"><a class="quarto-dashboard-link" href="https://github.com/SydneyBioX/sc_bench_benchmark_dashboard/" title="GitHub"><i class="bi bi-github"></i></a><a class="quarto-dashboard-link" href="https://twitter.com/intent/tweet?url=|url|&amp;text=sc%20bench%20benchmark%20dashboard" title="Twitter"><i class="bi bi-twitter"></i></a></div></div></div>
</nav>
</header>
<div id="quarto-search-results"></div>
<!-- content -->
<div id="quarto-content" class="quarto-container page-layout-custom quarto-dashboard-content bslib-gap-spacing html-fill-container bslib-page-fill quarto-dashboard-pages">
<!-- sidebar -->
<!-- margin-sidebar -->
<!-- main -->

<!-- Load data -->
<div class="bslib-grid-item html-fill-item tab-content bslib-grid" data-layout="fill" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div id="paper" class="dashboard-page tab-pane show active html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-paper">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<!-- Papers -->
<div class="card tabset html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header dashboard-card-no-title bslib-navs-card-title">

<ul class="nav nav-tabs card-header-tabs" role="tablist" data-tabsetid="card-tabset-1"><li class="nav-item" role="presentation"><a href="#card-tabset-1-1" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Category" aria-selected="true" class="nav-link active">Category</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-1-2" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Survey Analysis" aria-selected="false" class="nav-link">Survey Analysis</a></li></ul></div>


<div class="tab-content html-fill-item html-fill-container" data-tabset-id="card-tabset-1"><div class="tab-pane active show html-fill-item html-fill-container" role="tabpanel" id="card-tabset-1-1"><div class="card-body html-fill-item html-fill-container" data-title="Category">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 100fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 20fr) minmax(3em, 80fr); grid-auto-columns: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 1fr) minmax(3em, 1fr) minmax(3em, 1fr) minmax(3em, 1fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-warning showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="database-gear" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-database-gear"></i></div>
<div class="value-box-area">
<p class="value-box-title">
Pure benchmark papers
</p>
<p class="value-box-value">
95
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div><div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-info showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="database-gear" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-database-gear"></i></div>
<div class="value-box-area">
<p class="value-box-title">
New method development papers
</p>
<p class="value-box-value">
150
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div><div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-success showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="eyeglasses" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-eyeglasses"></i></div>
<div class="value-box-area">
<p class="value-box-title">
Total number of readers
</p>
<p class="value-box-value">
31
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div><div class="card valuebox cell html-fill-item html-fill-container bslib-card bslib-value-box bg-danger showcase-left-center" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div data-icon="book" class="html-fill-item html-fill-container value-box-grid">
<div class="value-box-showcase html-fill-item html-fill-container">

<i class="bi bi-book"></i></div>
<div class="value-box-area">
<p class="value-box-title">
Total number of reading
</p>
<p class="value-box-value">
356
</p>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 50fr) minmax(3em, 50fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure benchmark papers" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure benchmark papers</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">

</div>
<div class="cell quarto-layout-panel html-fill-item html-fill-container" data-layout-ncol="2" title="Pure benchmark papers">
<div class="quarto-layout-row html-fill-item html-fill-container">
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-d3ac841f6065d328370a" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-d3ac841f6065d328370a">{"x":{"visdat":{"85205b32532f":["function () ","plotlyVisDat"]},"cur_data":"85205b32532f","attrs":{"85205b32532f":{"ids":{},"labels":{},"parents":{},"values":{},"branchvalues":"total","insidetextorientation":"radial","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"sunburst"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"ids":["Benchmark","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell proteomics","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell mass cytometry","Benchmark - Spatial transcriptomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell multiomics","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Single-cell proteomics - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell mass cytometry - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell RNA-seq - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell proteomics - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell RNA-seq - Data - Benchmarking datasets","Benchmark - Single-cell RNA-seq - Initial analysis - Alignment","Benchmark - Single-cell RNA-seq - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell ATAC-seq - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell RNA-seq - Intermediate analysis - Clustering","Benchmark - Single-cell mass cytometry - Intermediate analysis - Clustering","Benchmark - Spatial transcriptomics - Intermediate analysis - Clustering","Benchmark - Single-cell Hi-C - Intermediate analysis - Clustering","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential expression","Benchmark - Single-cell RNA-seq - Intermediate analysis - Dimension reduction","Benchmark - Single-cell multiomics - Intermediate analysis - Dimension reduction","Benchmark - Single-cell RNA-seq - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Downstream analysis - Cell cell communication","Benchmark - Single-cell RNA-seq - Intermediate analysis - Measure of association","Benchmark - Single-cell RNA-seq - Downstream analysis - Pathway enrichment","Benchmark - Single-cell RNA-seq - Downstream analysis - T/B-cell receptor","Benchmark - Single-cell RNA-seq - Initial analysis - Imputation","Benchmark - Single-cell ATAC-seq - Initial analysis - Imputation","Benchmark - Single-cell multiomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Intermediate analysis - Batch correction","Benchmark - Spatial transcriptomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Intermediate analysis - Data integration","Benchmark - Spatial transcriptomics - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Initial analysis - Standardization","Benchmark - Single-cell RNA-seq - Downstream analysis - Trajectory","Benchmark - Single-cell RNA-seq - Downstream analysis - Velocity","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential analysis - others","Benchmark - Single-cell RNA-seq - Initial analysis - Doublet detection","Benchmark - Single-cell RNA-seq - Initial analysis - Quantification","Benchmark - Single-cell RNA-seq - Data - Simulation","Benchmark - Single-cell RNA-seq - Downstream analysis - Highly variable genes","Benchmark - Single-cell RNA-seq - Intermediate analysis - Subclone detection","Benchmark - Single-cell RNA-seq - Initial analysis - Visualisation"],"labels":["Pure benchmarking paper","Single-cell RNA-seq","Single-cell proteomics","Single-cell ATAC-seq","Single-cell mass cytometry","Spatial transcriptomics","Single-cell Hi-C","Single-cell multiomics","Analysis pipelines","Analysis pipelines","Analysis pipelines","Data","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Intermediate analysis","Initial analysis","Initial analysis","Analysis pipelines","Analysis pipelines","Analysis pipelines","Benchmarking datasets","Alignment","Cell type/state identification","Cell type/state identification","Clustering","Clustering","Clustering","Clustering","Differential expression","Dimension reduction","Dimension reduction","Regulatory network inference","Cell cell communication","Measure of association","Pathway enrichment","T/B-cell receptor","Imputation","Imputation","Data integration","Deconvolution","Batch correction","Data integration","Data integration","Deconvolution","Standardization","Trajectory","Velocity","Differential analysis - others","Doublet detection","Quantification","Simulation","Highly variable genes","Subclone detection","Visualisation"],"parents":["","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell proteomics","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell mass cytometry","Benchmark - Spatial transcriptomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell multiomics","Benchmark - Single-cell ATAC-seq","Benchmark - Spatial transcriptomics","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Single-cell proteomics - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell mass cytometry - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell ATAC-seq - Initial analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis"],"values":[192,162,2,6,4,10,2,6,12,2,2,7,32,63,2,4,6,2,48,6,2,4,12,2,2,2,6,14,2,10,4,2,2,16,18,2,12,4,4,6,2,8,2,4,8,13,4,2,4,4,2,2,2,2,2,5,2,2,2],"branchvalues":"total","insidetextorientation":"radial","type":"sunburst","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(255,255,255,1)"}},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-0ffd1c715ba2aaa246d2" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-0ffd1c715ba2aaa246d2">{"x":{"visdat":{"8520528f1b19":["function () ","plotlyVisDat"]},"cur_data":"8520528f1b19","attrs":{"8520528f1b19":{"orientation":"h","node":{"label":["Single-cell ATAC-seq","Single-cell Hi-C","Single-cell mass cytometry","Single-cell multiomics","Single-cell proteomics","Single-cell RNA-seq","Spatial transcriptomics","Data","Initial analysis","Intermediate analysis","Downstream analysis","Analysis pipelines","Benchmarking datasets","Simulation","Alignment","Deconvolution","Doublet detection","Imputation","Quantification","Standardization","Visualisation","Batch correction","Cell type/state identification","Clustering","Data integration","Dimension reduction","Measure of association","Subclone detection","Cell cell communication","Differential analysis - others","Differential expression","Highly variable genes","Pathway enrichment","Regulatory network inference","T/B-cell receptor","Trajectory","Velocity","Analysis pipelines "],"color":["brown","forestgreen","red","orange","darkviolet","steelblue","azure","bisque","darksalmon","darkseagreen","cornsilk","darkgray","#99cccc"],"pad":10,"thickness":10,"line":{"color":"black","width":0.5}},"link":{"source":[0,0,0,1,2,3,3,4,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,6,6,6,7,7,8,8,8,8,8,8,8,8,8,9,9,9,9,9,9,9,9,9,9,9,9,9,9,10,10,10,10,10,10,10,10,10,11,11,11],"target":[8,9,11,9,9,9,9,11,7,7,8,8,8,8,8,8,8,9,9,9,9,9,9,9,10,10,10,10,10,10,10,10,10,11,8,9,9,12,13,14,15,15,16,17,17,18,19,20,21,22,22,23,23,23,23,24,24,24,25,25,26,27,28,29,30,31,32,33,34,35,36,37,37,37],"value":[2,2,2,2,4,2,4,2,2,5,6,8,8,4,2,2,2,14,10,18,4,13,2,2,16,12,4,6,2,2,2,2,2,12,4,2,4,2,5,6,8,4,2,8,2,2,4,2,13,14,2,10,4,2,2,4,4,2,18,2,4,2,4,2,16,2,6,12,2,2,2,12,2,2]},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"sankey"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"orientation":"h","node":{"label":["Single-cell ATAC-seq","Single-cell Hi-C","Single-cell mass cytometry","Single-cell multiomics","Single-cell proteomics","Single-cell RNA-seq","Spatial transcriptomics","Data","Initial analysis","Intermediate analysis","Downstream analysis","Analysis pipelines","Benchmarking datasets","Simulation","Alignment","Deconvolution","Doublet detection","Imputation","Quantification","Standardization","Visualisation","Batch correction","Cell type/state identification","Clustering","Data integration","Dimension reduction","Measure of association","Subclone detection","Cell cell communication","Differential analysis - others","Differential expression","Highly variable genes","Pathway enrichment","Regulatory network inference","T/B-cell receptor","Trajectory","Velocity","Analysis pipelines "],"color":["brown","forestgreen","red","orange","darkviolet","steelblue","azure","bisque","darksalmon","darkseagreen","cornsilk","darkgray","#99cccc"],"pad":10,"thickness":10,"line":{"color":"black","width":0.5}},"link":{"source":[0,0,0,1,2,3,3,4,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,6,6,6,7,7,8,8,8,8,8,8,8,8,8,9,9,9,9,9,9,9,9,9,9,9,9,9,9,10,10,10,10,10,10,10,10,10,11,11,11],"target":[8,9,11,9,9,9,9,11,7,7,8,8,8,8,8,8,8,9,9,9,9,9,9,9,10,10,10,10,10,10,10,10,10,11,8,9,9,12,13,14,15,15,16,17,17,18,19,20,21,22,22,23,23,23,23,24,24,24,25,25,26,27,28,29,30,31,32,33,34,35,36,37,37,37],"value":[2,2,2,2,4,2,4,2,2,5,6,8,8,4,2,2,2,14,10,18,4,13,2,2,16,12,4,6,2,2,2,2,2,12,4,2,4,2,5,6,8,4,2,8,2,2,4,2,13,14,2,10,4,2,2,4,4,2,18,2,4,2,4,2,16,2,6,12,2,2,2,12,2,2]},"type":"sankey","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development papers" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development papers</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">

</div>
<div class="cell quarto-layout-panel html-fill-item html-fill-container" data-layout-ncol="2" title="New method development papers">
<div class="quarto-layout-row html-fill-item html-fill-container">
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-3761859b3c864c0ebe05" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-3761859b3c864c0ebe05">{"x":{"visdat":{"8520791c6099":["function () ","plotlyVisDat"]},"cur_data":"8520791c6099","attrs":{"8520791c6099":{"ids":{},"labels":{},"parents":{},"values":{},"branchvalues":"total","insidetextorientation":"radial","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"sunburst"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"ids":["Benchmark","Benchmark - Spatial imaging","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell multiomics","Benchmark - Spatial transcriptomics","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell Hi-C","Benchmark - Single-cell genomics","Benchmark - Single-cell sequencing","Benchmark - Single-cell ChIP-seq","Benchmark - Spatial imaging - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell genomics - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Spatial imaging - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell multiomics - Downstream analysis","Benchmark - Single-cell ATAC-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell multiomics - Initial analysis","Benchmark - Single-cell Hi-C - Initial analysis","Benchmark - Single-cell sequencing - Downstream analysis","Benchmark - Spatial imaging - Downstream analysis","Benchmark - Single-cell sequencing - Initial analysis","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell ATAC-seq - Data","Benchmark - Single-cell genomics - Data","Benchmark - Single-cell ChIP-seq - Data","Benchmark - Spatial transcriptomics - Downstream analysis","Benchmark - Spatial imaging - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines - Analysis pipelines","Benchmark - Single-cell RNA-seq - Analysis pipelines - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis - Cell segmentation","Benchmark - Spatial transcriptomics - Initial analysis - Cell segmentation","Benchmark - Single-cell Hi-C - Intermediate analysis - Chromatin conformation","Benchmark - Single-cell RNA-seq - Analysis pipelines - Multi-task","Benchmark - Spatial transcriptomics - Analysis pipelines - Multi-task","Benchmark - Single-cell multiomics - Analysis pipelines - Multi-task","Benchmark - Spatial imaging - Initial analysis - Assembly","Benchmark - Single-cell RNA-seq - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell genomics - Intermediate analysis - Cell type/state identification","Benchmark - Spatial transcriptomics - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell multiomics - Intermediate analysis - Cell type/state identification","Benchmark - Single-cell RNA-seq - Intermediate analysis - Clustering","Benchmark - Spatial imaging - Intermediate analysis - Clustering","Benchmark - Single-cell ATAC-seq - Intermediate analysis - Clustering","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential expression","Benchmark - Single-cell RNA-seq - Intermediate analysis - Dimension reduction","Benchmark - Single-cell multiomics - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Intermediate analysis - Measure of association","Benchmark - Single-cell multiomics - Downstream analysis - Cell cell communication","Benchmark - Single-cell ATAC-seq - Downstream analysis - Regulatory network inference","Benchmark - Single-cell RNA-seq - Downstream analysis - T/B-cell receptor","Benchmark - Single-cell RNA-seq - Initial analysis - Imputation","Benchmark - Single-cell multiomics - Initial analysis - Imputation","Benchmark - Spatial transcriptomics - Initial analysis - Imputation","Benchmark - Spatial transcriptomics - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Intermediate analysis - Batch correction","Benchmark - Single-cell multiomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Intermediate analysis - Data integration","Benchmark - Spatial transcriptomics - Intermediate analysis - Data integration","Benchmark - Single-cell RNA-seq - Initial analysis - Deconvolution","Benchmark - Single-cell RNA-seq - Downstream analysis - Marker/signature identification","Benchmark - Single-cell Hi-C - Initial analysis - Standardization","Benchmark - Single-cell RNA-seq - Initial analysis - Standardization","Benchmark - Single-cell sequencing - Downstream analysis - Trajectory","Benchmark - Single-cell RNA-seq - Downstream analysis - Trajectory","Benchmark - Spatial imaging - Downstream analysis - Trajectory","Benchmark - Single-cell RNA-seq - Downstream analysis - Velocity","Benchmark - Single-cell RNA-seq - Downstream analysis - Differential analysis - others","Benchmark - Single-cell multiomics - Initial analysis - Doublet detection","Benchmark - Single-cell RNA-seq - Initial analysis - Doublet detection","Benchmark - Single-cell sequencing - Initial analysis - Quantification","Benchmark - Single-cell RNA-seq - Initial analysis - Quantification","Benchmark - Single-cell RNA-seq - Intermediate analysis - Rare cell type identification","Benchmark - Single-cell RNA-seq - Data - Simulation","Benchmark - Single-cell ATAC-seq - Data - Simulation","Benchmark - Single-cell genomics - Data - Simulation","Benchmark - Single-cell ChIP-seq - Data - Simulation","Benchmark - Spatial transcriptomics - Downstream analysis - Highly variable genes","Benchmark - Single-cell Hi-C - Intermediate analysis - Subclone detection","Benchmark - Single-cell RNA-seq - Intermediate analysis - Subclone detection","Benchmark - Single-cell RNA-seq - Initial analysis - Visualisation"],"labels":["New method development paper","Spatial imaging","Single-cell ATAC-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell Hi-C","Single-cell genomics","Single-cell sequencing","Single-cell ChIP-seq","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Downstream analysis","Downstream analysis","Initial analysis","Data","Data","Data","Data","Downstream analysis","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Cell segmentation","Cell segmentation","Chromatin conformation","Multi-task","Multi-task","Multi-task","Assembly","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Clustering","Clustering","Clustering","Differential expression","Dimension reduction","Regulatory network inference","Regulatory network inference","Measure of association","Cell cell communication","Regulatory network inference","T/B-cell receptor","Imputation","Imputation","Imputation","Deconvolution","Batch correction","Data integration","Data integration","Data integration","Deconvolution","Marker/signature identification","Standardization","Standardization","Trajectory","Trajectory","Trajectory","Velocity","Differential analysis - others","Doublet detection","Doublet detection","Quantification","Quantification","Rare cell type identification","Simulation","Simulation","Simulation","Simulation","Highly variable genes","Subclone detection","Subclone detection","Visualisation"],"parents":["","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark","Benchmark - Spatial imaging","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell multiomics","Benchmark - Spatial transcriptomics","Benchmark - Single-cell RNA-seq","Benchmark - Spatial imaging","Benchmark - Spatial transcriptomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell genomics","Benchmark - Spatial transcriptomics","Benchmark - Single-cell multiomics","Benchmark - Spatial imaging","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell multiomics","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell multiomics","Benchmark - Single-cell Hi-C","Benchmark - Single-cell sequencing","Benchmark - Spatial imaging","Benchmark - Single-cell sequencing","Benchmark - Single-cell RNA-seq","Benchmark - Single-cell ATAC-seq","Benchmark - Single-cell genomics","Benchmark - Single-cell ChIP-seq","Benchmark - Spatial transcriptomics","Benchmark - Spatial imaging - Analysis pipelines","Benchmark - Single-cell ATAC-seq - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Analysis pipelines","Benchmark - Spatial transcriptomics - Analysis pipelines","Benchmark - Single-cell multiomics - Analysis pipelines","Benchmark - Spatial imaging - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell genomics - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial imaging - Intermediate analysis","Benchmark - Single-cell ATAC-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Downstream analysis","Benchmark - Single-cell ATAC-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell multiomics - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Spatial transcriptomics - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell multiomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Spatial transcriptomics - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell Hi-C - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell sequencing - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Spatial imaging - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell RNA-seq - Downstream analysis","Benchmark - Single-cell multiomics - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell sequencing - Initial analysis","Benchmark - Single-cell RNA-seq - Initial analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Data","Benchmark - Single-cell ATAC-seq - Data","Benchmark - Single-cell genomics - Data","Benchmark - Single-cell ChIP-seq - Data","Benchmark - Spatial transcriptomics - Downstream analysis","Benchmark - Single-cell Hi-C - Intermediate analysis","Benchmark - Single-cell RNA-seq - Intermediate analysis","Benchmark - Single-cell RNA-seq - Initial analysis"],"values":[164,7,6,20,18,104,4,2,2,1,2,2,3,3,6,3,7,3,50,1,3,7,1,1,25,8,1,13,2,1,1,1,1,10,2,1,1,5,2,2,2,2,3,2,1,2,3,1,1,1,20,1,1,1,11,1,1,3,1,7,3,2,1,1,1,5,1,3,3,6,6,3,2,1,7,1,2,1,6,1,1,4,1,1,1,1,4,10,2,1,1,5,1,3,3],"branchvalues":"total","insidetextorientation":"radial","type":"sunburst","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(255,255,255,1)"}},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="quarto-layout-cell html-fill-item html-fill-container" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-b31c0c24240dbcb0f8a5" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-b31c0c24240dbcb0f8a5">{"x":{"visdat":{"8520fb662d6":["function () ","plotlyVisDat"]},"cur_data":"8520fb662d6","attrs":{"8520fb662d6":{"orientation":"h","node":{"label":["Single-cell ATAC-seq","Single-cell ChIP-seq","Single-cell genomics","Single-cell Hi-C","Single-cell multiomics","Single-cell RNA-seq","Single-cell sequencing","Spatial imaging","Spatial transcriptomics","Data","Initial analysis","Intermediate analysis","Downstream analysis","Analysis pipelines","Simulation","Assembly","Cell segmentation","Deconvolution","Doublet detection","Imputation","Quantification","Standardization","Visualisation","Batch correction","Cell type/state identification","Chromatin conformation","Clustering","Data integration","Dimension reduction","Measure of association","Rare cell type identification","Subclone detection","Cell cell communication","Differential analysis - others","Differential expression","Highly variable genes","Marker/signature identification","Regulatory network inference","T/B-cell receptor","Trajectory","Velocity","Analysis pipelines ","Multi-task "],"color":["brown","forestgreen","red","orange","darkviolet","steelblue","azure","bisque","darksalmon","darkseagreen","cornsilk","darkgray","#99cccc"],"pad":10,"thickness":10,"line":{"color":"black","width":0.5}},"link":{"source":[0,0,0,0,1,2,2,3,3,3,4,4,4,4,4,4,4,4,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,6,6,7,7,7,7,7,8,8,8,8,8,8,8,8,9,9,9,9,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,12,12,12,12,12,12,12,12,12,12,12,12,12,13,13,13,13,13,13,13,13],"target":[9,11,12,13,9,9,11,10,11,11,10,10,11,11,12,12,13,13,9,10,10,10,10,10,10,11,11,11,11,11,11,11,11,12,12,12,12,12,12,12,13,13,10,12,10,10,11,12,13,10,10,10,11,11,12,13,13,14,14,14,14,15,16,16,17,17,18,18,19,19,19,20,20,21,21,22,23,24,24,24,24,25,26,26,26,27,27,27,28,29,30,31,31,32,33,34,35,36,37,37,37,38,39,39,39,40,41,41,41,41,41,42,42,42],"value":[2,1,1,2,1,1,1,1,2,1,1,1,1,6,7,1,2,1,10,5,1,2,1,1,3,20,11,1,2,6,3,4,3,3,3,1,7,6,1,4,3,3,1,1,2,1,1,1,2,1,3,3,1,2,5,2,1,10,2,1,1,1,2,1,3,1,1,1,5,1,3,1,1,1,2,3,6,20,1,1,1,2,11,1,1,6,3,2,1,2,4,1,3,1,4,3,5,7,7,3,1,1,1,6,1,1,2,2,2,2,3,3,1,1]},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"sankey"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"orientation":"h","node":{"label":["Single-cell ATAC-seq","Single-cell ChIP-seq","Single-cell genomics","Single-cell Hi-C","Single-cell multiomics","Single-cell RNA-seq","Single-cell sequencing","Spatial imaging","Spatial transcriptomics","Data","Initial analysis","Intermediate analysis","Downstream analysis","Analysis pipelines","Simulation","Assembly","Cell segmentation","Deconvolution","Doublet detection","Imputation","Quantification","Standardization","Visualisation","Batch correction","Cell type/state identification","Chromatin conformation","Clustering","Data integration","Dimension reduction","Measure of association","Rare cell type identification","Subclone detection","Cell cell communication","Differential analysis - others","Differential expression","Highly variable genes","Marker/signature identification","Regulatory network inference","T/B-cell receptor","Trajectory","Velocity","Analysis pipelines ","Multi-task "],"color":["brown","forestgreen","red","orange","darkviolet","steelblue","azure","bisque","darksalmon","darkseagreen","cornsilk","darkgray","#99cccc"],"pad":10,"thickness":10,"line":{"color":"black","width":0.5}},"link":{"source":[0,0,0,0,1,2,2,3,3,3,4,4,4,4,4,4,4,4,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,6,6,7,7,7,7,7,8,8,8,8,8,8,8,8,9,9,9,9,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,12,12,12,12,12,12,12,12,12,12,12,12,12,13,13,13,13,13,13,13,13],"target":[9,11,12,13,9,9,11,10,11,11,10,10,11,11,12,12,13,13,9,10,10,10,10,10,10,11,11,11,11,11,11,11,11,12,12,12,12,12,12,12,13,13,10,12,10,10,11,12,13,10,10,10,11,11,12,13,13,14,14,14,14,15,16,16,17,17,18,18,19,19,19,20,20,21,21,22,23,24,24,24,24,25,26,26,26,27,27,27,28,29,30,31,31,32,33,34,35,36,37,37,37,38,39,39,39,40,41,41,41,41,41,42,42,42],"value":[2,1,1,2,1,1,1,1,2,1,1,1,1,6,7,1,2,1,10,5,1,2,1,1,3,20,11,1,2,6,3,4,3,3,3,1,7,6,1,4,3,3,1,1,2,1,1,1,2,1,3,3,1,2,5,2,1,10,2,1,1,1,2,1,3,1,1,1,5,1,3,1,1,1,2,3,6,20,1,1,1,2,11,1,1,6,3,2,1,2,4,1,3,1,4,3,5,7,7,3,1,1,1,6,1,1,2,2,2,2,3,3,1,1]},"type":"sankey","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-1-2"><div class="card-body html-fill-item html-fill-container" data-title="Survey Analysis">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 100fr);
grid-auto-rows: minmax(0, 1fr);">
<!-- Paper - Survey Analysis --><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 15fr) minmax(3em, 12fr) minmax(3em, 73fr); grid-auto-columns: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 25fr) minmax(3em, 25fr) minmax(3em, 25fr) minmax(3em, 25fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-body html-fill-item html-fill-container">
<div class="sourceCode cell-code hidden html-fill-item html-fill-container" id="cb1" data-startfrom="5" data-source-offset="-87"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 4;"><span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>d3 <span class="op">=</span> <span class="pp">require</span>(<span class="st">"d3@7"</span>)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a>jQuery <span class="op">=</span> <span class="pp">require</span>(<span class="st">"jquery"</span>)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>category_list <span class="op">=</span> [<span class="op">...</span><span class="kw">new</span> <span class="bu">Set</span>(paper_data<span class="op">.</span><span class="at">Paper_category</span>)]<span class="op">;</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a>viewof category <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">checkbox</span>(</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a>  category_list<span class="op">,</span> </span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>  {</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a>    <span class="dt">value</span><span class="op">:</span>  category_list<span class="op">,</span> </span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a>    <span class="dt">label</span><span class="op">:</span> <span class="st">"Data category: "</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>  })</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-1-1" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-1-2" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-1-3" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-1-4" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div class="sourceCode cell-code hidden html-fill-item html-fill-container" id="cb2" data-startfrom="2" data-source-offset="-1"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 1;"><span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>datatype_list <span class="op">=</span> [<span class="op">...</span><span class="kw">new</span> <span class="bu">Set</span>(</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>  paper_data<span class="op">.</span><span class="at">Paper_category</span></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>    <span class="op">.</span><span class="fu">map</span>((value<span class="op">,</span> index) <span class="kw">=&gt;</span> category<span class="op">.</span><span class="fu">includes</span>(value) <span class="op">?</span> paper_data<span class="op">.</span><span class="at">data_type</span>[index] <span class="op">:</span> <span class="kw">null</span>)</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>    <span class="op">.</span><span class="fu">filter</span>(value <span class="kw">=&gt;</span> value <span class="op">!==</span> <span class="kw">null</span>)</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>)]<span class="op">;</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a><span class="co">//console.log("datatype_list", datatype_list)</span></span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>viewof data_type <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>  datatype_list<span class="op">,</span> </span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a>  {</span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a>    <span class="dt">value</span><span class="op">:</span> <span class="st">"Single-cell RNA-seq"</span><span class="op">,</span></span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a>    <span class="dt">label</span><span class="op">:</span> <span class="st">"Data type: "</span></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a>  })</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-2-1" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-2-2" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-body html-fill-item html-fill-container">
<div class="sourceCode cell-code hidden html-fill-item html-fill-container" id="cb3" data-startfrom="1" data-source-offset="-1"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>broaderTopic_list <span class="op">=</span> </span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>  paper_data<span class="op">.</span><span class="at">Paper_category</span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>    <span class="op">.</span><span class="fu">map</span>((value<span class="op">,</span> index) <span class="kw">=&gt;</span> category<span class="op">.</span><span class="fu">includes</span>(value) <span class="op">&amp;&amp;</span> data_type<span class="op">.</span><span class="fu">includes</span>(paper_data<span class="op">.</span><span class="at">data_type</span>[index]) <span class="op">?</span> paper_data<span class="op">.</span><span class="at">broader_topic</span>[index] <span class="op">:</span> <span class="kw">null</span>)</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>    <span class="op">.</span><span class="fu">filter</span>(value <span class="kw">=&gt;</span> value <span class="op">!==</span> <span class="kw">null</span>)<span class="op">;</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>viewof broader_topic <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>  broaderTopic_list<span class="op">,</span> </span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>  {</span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>    <span class="dt">value</span><span class="op">:</span> <span class="st">"Intermediate analysis"</span><span class="op">,</span></span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>    <span class="dt">label</span><span class="op">:</span> <span class="st">"Broader topic: "</span><span class="op">,</span></span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>    <span class="dt">unique</span><span class="op">:</span> <span class="kw">true</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>  })</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-3-1" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-3-2" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div class="sourceCode cell-code hidden html-fill-item html-fill-container" id="cb4" data-startfrom="2" data-source-offset="-1"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 1;"><span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>finerTopic_list <span class="op">=</span> [<span class="op">...</span><span class="kw">new</span> <span class="bu">Set</span>(</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>  paper_data<span class="op">.</span><span class="at">Paper_category</span></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>    <span class="op">.</span><span class="fu">map</span>((value<span class="op">,</span> index) <span class="kw">=&gt;</span> category<span class="op">.</span><span class="fu">includes</span>(value) <span class="op">&amp;&amp;</span> </span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>    data_type<span class="op">.</span><span class="fu">includes</span>(paper_data<span class="op">.</span><span class="at">data_type</span>[index]) <span class="op">&amp;&amp;</span> </span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>    broader_topic<span class="op">.</span><span class="fu">includes</span>(paper_data<span class="op">.</span><span class="at">broader_topic</span>[index]) <span class="op">?</span> paper_data<span class="op">.</span><span class="at">finer_topic</span>[index] <span class="op">:</span> <span class="kw">null</span>)<span class="op">.</span><span class="fu">filter</span>(value <span class="kw">=&gt;</span> value <span class="op">!==</span> <span class="kw">null</span>)</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>)]<span class="op">;</span></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>viewof finer_topic <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a> finerTopic_list<span class="op">,</span> </span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>  {</span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>    <span class="dt">value</span><span class="op">:</span> <span class="st">"Cell type/state identification"</span><span class="op">,</span></span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>    <span class="dt">label</span><span class="op">:</span> <span class="st">"Finer topic: "</span></span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>  })</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-4-1" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-4-2" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 100fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div class="sourceCode cell-code hidden html-fill-item html-fill-container" id="cb5" data-startfrom="2" data-source-offset="-1"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 1;"><span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>paperTitle_list <span class="op">=</span> [<span class="op">...</span><span class="kw">new</span> <span class="bu">Set</span>(</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>  paper_data<span class="op">.</span><span class="at">Paper_category</span></span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>    <span class="op">.</span><span class="fu">map</span>((value<span class="op">,</span> index) <span class="kw">=&gt;</span> category<span class="op">.</span><span class="fu">includes</span>(value) <span class="op">&amp;&amp;</span> </span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>    data_type<span class="op">.</span><span class="fu">includes</span>(paper_data<span class="op">.</span><span class="at">data_type</span>[index]) <span class="op">&amp;&amp;</span> </span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>    broader_topic<span class="op">.</span><span class="fu">includes</span>(paper_data<span class="op">.</span><span class="at">broader_topic</span>[index]) <span class="op">&amp;&amp;</span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>    finer_topic<span class="op">.</span><span class="fu">includes</span>(paper_data<span class="op">.</span><span class="at">finer_topic</span>[index]) <span class="op">?</span> </span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>    paper_data<span class="op">.</span><span class="at">Paper_title</span>[index] <span class="op">:</span> <span class="kw">null</span>)<span class="op">.</span><span class="fu">filter</span>(value <span class="kw">=&gt;</span> value <span class="op">!==</span> <span class="kw">null</span>)</span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a>)]<span class="op">;</span></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>viewof paper_title <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(</span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a> paperTitle_list<span class="op">,</span> </span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a>  {</span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>    <span class="dt">label</span><span class="op">:</span> <span class="st">"Paper title: "</span><span class="op">,</span></span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a>    <span class="dt">width</span><span class="op">:</span> <span class="st">"100%"</span></span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a>  })</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-5-1" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-5-2" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 60fr) minmax(3em, 40fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()">
<div class="card-body html-fill-item html-fill-container">
<div class="sourceCode cell-code hidden html-fill-item html-fill-container" id="cb6" data-startfrom="2" data-source-offset="-1"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 1;"><span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>selected_paper_link <span class="op">=</span> paper_links<span class="op">.</span><span class="at">Paper_title</span></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>    <span class="op">.</span><span class="fu">map</span>((value<span class="op">,</span> index) <span class="kw">=&gt;</span> paper_title<span class="op">.</span><span class="fu">includes</span>(value) <span class="op">?</span> </span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>    paper_links<span class="op">.</span><span class="at">Link</span>[index] <span class="op">:</span> <span class="kw">null</span>)<span class="op">.</span><span class="fu">filter</span>(value <span class="kw">=&gt;</span> value <span class="op">!==</span> <span class="kw">null</span>)[<span class="dv">0</span>]<span class="op">;</span></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a><span class="vs">  &lt;iframe src=</span><span class="sc">${</span>selected_paper_link<span class="sc">}</span><span class="vs"> width="100%" height="400px" title="Research paper"&gt;&lt;/iframe&gt;</span></span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a><span class="vs">`</span><span class="op">;</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-6-1" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-6-2" data-nodetype="expression" class="html-fill-item html-fill-container">

</div>
</div>
</div>
</div>
<script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-body html-fill-item html-fill-container">
<div class="sourceCode cell-code hidden html-fill-item html-fill-container" id="cb7" data-startfrom="1" data-source-offset="-1"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>selected_columns <span class="op">=</span> [</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Paper_category"</span><span class="op">,</span></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>  <span class="st">"data_type"</span><span class="op">,</span></span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>  <span class="st">"broader_topic"</span><span class="op">,</span></span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>  <span class="st">"finer_topic"</span><span class="op">,</span></span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>  <span class="st">"publication_date"</span><span class="op">,</span></span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Max_number_of_cells"</span><span class="op">,</span></span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Data_availability"</span><span class="op">,</span></span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Recommendation"</span><span class="op">,</span></span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Applicability"</span><span class="op">,</span></span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Sensitivity_analysis"</span><span class="op">,</span></span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Variability_of_score"</span></span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>]</span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a>filtered_paper_data <span class="op">=</span> <span class="bu">Object</span><span class="op">.</span><span class="fu">keys</span>(paper_data)<span class="op">.</span><span class="fu">reduce</span>((obj<span class="op">,</span> key) <span class="kw">=&gt;</span> {</span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a>  <span class="kw">const</span> index <span class="op">=</span> paper_data<span class="op">.</span><span class="at">Paper_title</span><span class="op">.</span><span class="fu">indexOf</span>(paper_title)<span class="op">;</span></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (index <span class="op">!==</span> <span class="op">-</span><span class="dv">1</span>) {</span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a>    obj[key] <span class="op">=</span> paper_data[key][index]<span class="op">;</span></span>
<span id="cb7-20"><a href="#cb7-20" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb7-21"><a href="#cb7-21" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb7-22"><a href="#cb7-22" aria-hidden="true" tabindex="-1"></a>  <span class="cf">return</span> obj<span class="op">;</span></span>
<span id="cb7-23"><a href="#cb7-23" aria-hidden="true" tabindex="-1"></a>}<span class="op">,</span> {})<span class="op">;</span></span>
<span id="cb7-24"><a href="#cb7-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-25"><a href="#cb7-25" aria-hidden="true" tabindex="-1"></a><span class="co">//console.log("Filtered_paper_data", filtered_paper_data)</span></span>
<span id="cb7-26"><a href="#cb7-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-27"><a href="#cb7-27" aria-hidden="true" tabindex="-1"></a>selected_data <span class="op">=</span> selected_columns<span class="op">.</span><span class="fu">reduce</span>((obj<span class="op">,</span> key) <span class="kw">=&gt;</span> {</span>
<span id="cb7-28"><a href="#cb7-28" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (filtered_paper_data[key] <span class="op">!==</span> <span class="kw">undefined</span>) {</span>
<span id="cb7-29"><a href="#cb7-29" aria-hidden="true" tabindex="-1"></a>    obj[key] <span class="op">=</span> filtered_paper_data[key]<span class="op">;</span></span>
<span id="cb7-30"><a href="#cb7-30" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb7-31"><a href="#cb7-31" aria-hidden="true" tabindex="-1"></a>  <span class="cf">return</span> obj<span class="op">;</span></span>
<span id="cb7-32"><a href="#cb7-32" aria-hidden="true" tabindex="-1"></a>}<span class="op">,</span> {})<span class="op">;</span></span>
<span id="cb7-33"><a href="#cb7-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-34"><a href="#cb7-34" aria-hidden="true" tabindex="-1"></a><span class="co">//console.log("Selected data", selected_data)</span></span>
<span id="cb7-35"><a href="#cb7-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-36"><a href="#cb7-36" aria-hidden="true" tabindex="-1"></a><span class="co">/*html`</span></span>
<span id="cb7-37"><a href="#cb7-37" aria-hidden="true" tabindex="-1"></a><span class="co">  &lt;div style="display: flex; flex-wrap: wrap; gap: 10px;"&gt;</span></span>
<span id="cb7-38"><a href="#cb7-38" aria-hidden="true" tabindex="-1"></a><span class="co">    ${Object.entries(selected_data).map(([key, value]) =&gt; {</span></span>
<span id="cb7-39"><a href="#cb7-39" aria-hidden="true" tabindex="-1"></a><span class="co">      return html`</span></span>
<span id="cb7-40"><a href="#cb7-40" aria-hidden="true" tabindex="-1"></a><span class="co">        &lt;div class="card"&gt;</span></span>
<span id="cb7-41"><a href="#cb7-41" aria-hidden="true" tabindex="-1"></a><span class="co">          &lt;div class="card-body"&gt;</span></span>
<span id="cb7-42"><a href="#cb7-42" aria-hidden="true" tabindex="-1"></a><span class="co">            &lt;h5 class="card-title text-white" style="background-color: #00aa49;"&gt; ${key}&lt;/h5&gt;</span></span>
<span id="cb7-43"><a href="#cb7-43" aria-hidden="true" tabindex="-1"></a><span class="co">            &lt;p class="card-text"&gt;${value}&lt;/p&gt;</span></span>
<span id="cb7-44"><a href="#cb7-44" aria-hidden="true" tabindex="-1"></a><span class="co">        &lt;/div&gt;</span></span>
<span id="cb7-45"><a href="#cb7-45" aria-hidden="true" tabindex="-1"></a><span class="co">      `;</span></span>
<span id="cb7-46"><a href="#cb7-46" aria-hidden="true" tabindex="-1"></a><span class="co">    })}</span></span>
<span id="cb7-47"><a href="#cb7-47" aria-hidden="true" tabindex="-1"></a><span class="co">  &lt;/div&gt;</span></span>
<span id="cb7-48"><a href="#cb7-48" aria-hidden="true" tabindex="-1"></a><span class="co">`*/</span></span>
<span id="cb7-49"><a href="#cb7-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-50"><a href="#cb7-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-51"><a href="#cb7-51" aria-hidden="true" tabindex="-1"></a><span class="co">// Create HTML for cards</span></span>
<span id="cb7-52"><a href="#cb7-52" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`</span></span>
<span id="cb7-53"><a href="#cb7-53" aria-hidden="true" tabindex="-1"></a><span class="vs">  &lt;div style="display: flex; flex-wrap: wrap; gap: 10px;"&gt;</span></span>
<span id="cb7-54"><a href="#cb7-54" aria-hidden="true" tabindex="-1"></a><span class="vs">    </span><span class="sc">${</span><span class="bu">Object</span><span class="op">.</span><span class="fu">entries</span>(selected_data)<span class="op">.</span><span class="fu">map</span>(([key<span class="op">,</span> value]) <span class="kw">=&gt;</span> {</span>
<span id="cb7-55"><a href="#cb7-55" aria-hidden="true" tabindex="-1"></a>      <span class="cf">return</span> <span class="fu">html</span><span class="vs">`</span></span>
<span id="cb7-56"><a href="#cb7-56" aria-hidden="true" tabindex="-1"></a><span class="vs">        &lt;div style="border: 1px solid #ccc; border-radius: 8px; padding: 5px; width: 220px; box-shadow: 2px 2px 5px rgba(0,0,0,0.1);"&gt;</span></span>
<span id="cb7-57"><a href="#cb7-57" aria-hidden="true" tabindex="-1"></a><span class="vs">          &lt;h4 style="margin: 0 0 0 0; font-size: 18px; color: #fff; background-color: #2C5488; padding: 5px; border-radius: 4px;"&gt;</span></span>
<span id="cb7-58"><a href="#cb7-58" aria-hidden="true" tabindex="-1"></a><span class="vs">            </span><span class="sc">${</span>key<span class="op">.</span><span class="fu">charAt</span>(<span class="dv">0</span>)<span class="op">.</span><span class="fu">toUpperCase</span>() <span class="op">+</span> key<span class="op">.</span><span class="fu">slice</span>(<span class="dv">1</span>)<span class="op">.</span><span class="fu">replaceAll</span>(<span class="st">"_"</span><span class="op">,</span> <span class="st">" "</span>)<span class="sc">}</span></span>
<span id="cb7-59"><a href="#cb7-59" aria-hidden="true" tabindex="-1"></a><span class="vs">          &lt;/h4&gt;</span></span>
<span id="cb7-60"><a href="#cb7-60" aria-hidden="true" tabindex="-1"></a><span class="vs">          &lt;p style="margin: 5; font-size: 14px; color: #555;"&gt;</span></span>
<span id="cb7-61"><a href="#cb7-61" aria-hidden="true" tabindex="-1"></a><span class="vs">            </span><span class="sc">${</span>value<span class="sc">}</span></span>
<span id="cb7-62"><a href="#cb7-62" aria-hidden="true" tabindex="-1"></a><span class="vs">          &lt;/p&gt;</span></span>
<span id="cb7-63"><a href="#cb7-63" aria-hidden="true" tabindex="-1"></a><span class="vs">        &lt;/div&gt;</span></span>
<span id="cb7-64"><a href="#cb7-64" aria-hidden="true" tabindex="-1"></a><span class="vs">      `</span><span class="op">;</span></span>
<span id="cb7-65"><a href="#cb7-65" aria-hidden="true" tabindex="-1"></a>    })<span class="sc">}</span></span>
<span id="cb7-66"><a href="#cb7-66" aria-hidden="true" tabindex="-1"></a><span class="vs">  &lt;/div&gt;</span></span>
<span id="cb7-67"><a href="#cb7-67" aria-hidden="true" tabindex="-1"></a><span class="vs">`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-7-1" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-7-2" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-7-3" data-nodetype="declaration" class="html-fill-item html-fill-container">

</div>
</div>
</div>
<div class="cell-output cell-output-display html-fill-item html-fill-container">
<div class="html-fill-item html-fill-container">
<div id="ojs-cell-7-4" data-nodetype="expression" class="html-fill-item html-fill-container">

</div>
</div>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div>
</div>
</div></div></div><bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="data-overview" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-data-overview">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<!-- Data Overview -->
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Result table" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Result table</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<div class="datatables html-widget html-fill-item html-fill-container" id="htmlwidget-ebbf2b469bd06df18c7d" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-ebbf2b469bd06df18c7d">{"x":{"style":"bootstrap","filter":"top","vertical":false,"filterHTML":"<tr>\n  <td><\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Addition - Analysis pipelines&quot;,&quot;Addition - Benchmarking datasets&quot;,&quot;Addition - Cell segmentation&quot;,&quot;Addition - Chromatin conformation&quot;,&quot;Addition - Multi-task&quot;,&quot;Alignment&quot;,&quot;Assembly&quot;,&quot;Classification&quot;,&quot;Clustering&quot;,&quot;Differential Expression&quot;,&quot;Dimensionality Reduction&quot;,&quot;Gene Networks&quot;,&quot;Gene Sets&quot;,&quot;Immune&quot;,&quot;Imputation&quot;,&quot;Integration&quot;,&quot;Marker Genes&quot;,&quot;Normalisation&quot;,&quot;Ordering&quot;,&quot;Perturbations&quot;,&quot;Quality Control&quot;,&quot;Quantification&quot;,&quot;Rare Cells&quot;,&quot;Simulation&quot;,&quot;Variable Genes&quot;,&quot;Variants&quot;,&quot;Visualisation&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;New method development paper&quot;,&quot;Pure benchmarking paper&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"integer\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0\" data-max=\"68\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"72\" data-max=\"4e+07\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Both&quot;,&quot;Experimental data&quot;,&quot;Synthetic data&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"integer\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0\" data-max=\"518\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"integer\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0\" data-max=\"19980\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Bioinformatics (Oxford, England)&quot;,&quot;BMC bioinformatics&quot;,&quot;BMC genomics&quot;,&quot;Briefings in bioinformatics&quot;,&quot;Briefings in functional genomics&quot;,&quot;Cell genomics&quot;,&quot;Cell reports&quot;,&quot;Cell systems&quot;,&quot;Cells&quot;,&quot;Communications biology&quot;,&quot;Computational and structural biotechnology journal&quot;,&quot;Current protocols&quot;,&quot;eLife&quot;,&quot;F1000Research&quot;,&quot;Frontiers in genetics&quot;,&quot;G3 (Bethesda, Md.)&quot;,&quot;Genes&quot;,&quot;Genetic epidemiology&quot;,&quot;Genome biology&quot;,&quot;Genome Biology&quot;,&quot;Genome research&quot;,&quot;Genomics&quot;,&quot;Genomics, proteomics &amp; bioinformatics&quot;,&quot;GigaScience&quot;,&quot;IEEE/ACM transactions on computational biology and bioinformatics&quot;,&quot;iScience&quot;,&quot;Journal of chemical information and modeling&quot;,&quot;Life (Basel, Switzerland)&quot;,&quot;Life science alliance&quot;,&quot;Molecular systems biology&quot;,&quot;NAR genomics and bioinformatics&quot;,&quot;Nature&quot;,&quot;Nature biotechnology&quot;,&quot;Nature communications&quot;,&quot;Nature computational science&quot;,&quot;Nature methods&quot;,&quot;Nucleic acids research&quot;,&quot;Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing&quot;,&quot;Patterns (New York, N.Y.)&quot;,&quot;PLoS computational biology&quot;,&quot;Proceedings of SPIE--the International Society for Optical Engineering&quot;,&quot;Proceedings of the National Academy of Sciences of the United States of America&quot;,&quot;Research square&quot;,&quot;Science advances&quot;,&quot;Scientific data&quot;,&quot;Scientific reports&quot;,&quot;Statistical applications in genetics and molecular biology&quot;,&quot;Statistics in medicine&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Single-cell ATAC-seq&quot;,&quot;Single-cell ChIP-seq&quot;,&quot;Single-cell genomics&quot;,&quot;Single-cell Hi-C&quot;,&quot;Single-cell mass cytometry&quot;,&quot;Single-cell multiomics&quot;,&quot;Single-cell proteomics&quot;,&quot;Single-cell RNA-seq&quot;,&quot;Single-cell sequencing&quot;,&quot;Spatial imaging&quot;,&quot;Spatial transcriptomics&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Analysis pipelines&quot;,&quot;Data&quot;,&quot;Downstream analysis&quot;,&quot;Initial analysis&quot;,&quot;Intermediate analysis&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;Alignment&quot;,&quot;Analysis pipelines&quot;,&quot;Assembly&quot;,&quot;Batch correction&quot;,&quot;Benchmarking datasets&quot;,&quot;Cell cell communication&quot;,&quot;Cell segmentation&quot;,&quot;Cell type/state identification&quot;,&quot;Chromatin conformation&quot;,&quot;Clustering&quot;,&quot;Data integration&quot;,&quot;Deconvolution&quot;,&quot;Differential analysis - others&quot;,&quot;Differential expression&quot;,&quot;Dimension reduction&quot;,&quot;Doublet detection&quot;,&quot;Highly variable genes&quot;,&quot;Imputation&quot;,&quot;Marker/signature identification&quot;,&quot;Measure of association&quot;,&quot;Multi-task&quot;,&quot;Pathway enrichment&quot;,&quot;Quantification&quot;,&quot;Rare cell type identification&quot;,&quot;Regulatory network inference&quot;,&quot;Simulation&quot;,&quot;Standardization&quot;,&quot;Subclone detection&quot;,&quot;T/B-cell receptor&quot;,&quot;Trajectory&quot;,&quot;Velocity&quot;,&quot;Visualisation&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n<\/tr>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356"],["Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Analysis pipelines","Addition - Benchmarking datasets","Addition - Benchmarking datasets","Addition - Cell segmentation","Addition - Cell segmentation","Addition - Cell segmentation","Addition - Chromatin conformation","Addition - Chromatin conformation","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Addition - Multi-task","Alignment","Alignment","Alignment","Alignment","Alignment","Alignment","Assembly","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Classification","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Differential Expression","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Dimensionality Reduction","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Networks","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Gene Sets","Immune","Immune","Immune","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Integration","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Marker Genes","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Normalisation","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Ordering","Perturbations","Perturbations","Perturbations","Perturbations","Perturbations","Perturbations","Quality Control","Quality Control","Quality Control","Quality Control","Quantification","Quantification","Quantification","Quantification","Rare Cells","Rare Cells","Rare Cells","Rare Cells","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variable Genes","Variants","Variants","Variants","Variants","Variants","Variants","Visualisation","Visualisation","Visualisation","Visualisation","Visualisation"],["spicyR: spatial analysis of in situ cytometry data in R","Benchmarking UMI-based single-cell RNA-seq preprocessing workflows","Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments","Interactive analysis of single-cell epigenomic landscapes with ChromSCape","pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools","A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples","The Current State of Single-Cell Proteomics Data Analysis","Assessment of computational methods for the analysis of single-cell ATAC-seq data","EpiScanpy: integrated single-cell epigenomic analysis","TIST: Transcriptome and Histopathological Image Integrative Analysis for Spatial Transcriptomics","Comparison of high-throughput single-cell RNA sequencing data processing pipelines","Interactive analysis of single-cell epigenomic landscapes with ChromSCape","Benchmarking UMI-based single-cell RNA-seq preprocessing workflows","Destin2: Integrative and cross-modality analysis of single-cell chromatin accessibility data","Comparison of high-throughput single-cell RNA sequencing data processing pipelines","The Current State of Single-Cell Proteomics Data Analysis","A systematic evaluation of single cell RNA-seq analysis pipelines","Assessment of computational methods for the analysis of single-cell ATAC-seq data","A systematic evaluation of single cell RNA-seq analysis pipelines","pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools","histocat: analysis of cell phenotypes and interactions in multiplex image cytometry data","Giotto: a toolbox for integrative analysis and visualization of spatial expression data","A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples","TISCH: a comprehensive web resource enabling interactive single-cell transcriptome visualization of tumor microenvironment","Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq","Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments","Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq","A multi-center cross-platform single-cell RNA sequencing reference dataset","A multi-center cross-platform single-cell RNA sequencing reference dataset","Joint cell segmentation and cell type annotation for spatial transcriptomics","Cellpose: a generalist algorithm for cellular segmentation","Cell segmentation in imaging-based spatial transcriptomics","Perspectives for the reconstruction of 3D chromatin conformation using single cell Hi-C data","SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data","High-throughput single-cell RNA-seq data imputation and characterization with surrogate-assisted automated deep learning","High-throughput single-cell RNA-seq data imputation and characterization with surrogate-assisted automated deep learning","Spatially informed clustering, integration, and deconvolution of spatial transcriptomics with GraphST","Joint dimension reduction and clustering analysis of single-cell RNA-seq and spatial transcriptomics data","scGNN is a novel graph neural network framework for single-cell RNA-Seq analyses","Choice of pre-processing pipeline influences clustering quality of scRNA-seq datasets","Comparative analysis of common alignment tools for single-cell RNA sequencing","Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment","Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment","Comparative analysis of common alignment tools for single-cell RNA sequencing","Choice of pre-processing pipeline influences clustering quality of scRNA-seq datasets","Mapping molecular assemblies with fluorescence microscopy and object-based spatial statistics","Influence of Cell-type Ratio on Spatially Resolved Single-cell Transcriptomes using the Tangram Algorithm: Based on Implementation on Single-Cell and MxIF Data","scAB detects multiresolution cell states with clinical significance by integrating single-cell genomics and bulk sequencing data","Spatial-ID: a cell typing method for spatially resolved transcriptomics via transfer learning and spatial embedding","Latent cellular analysis robustly reveals subtle diversity in large-scale single-cell RNA-seq data","Learning Cell Annotation under Multiple Reference Datasets by Multisource Domain Adaptation","scMAGIC: accurately annotating single cells using two rounds of reference-based classification","A multiresolution framework to characterize single-cell state landscapes","scGCN is a graph convolutional networks algorithm for knowledge transfer in single cell omics","scClassify: sample size estimation and multiscale classification of cells using single and multiple reference","Evaluation of Cell Type Annotation R Packages on Single-cell RNA-seq Data","A comparison of automatic cell identification methods for single-cell RNA sequencing data","Adversarial dense graph convolutional networks for single-cell classification","Benchmarking automated cell type annotation tools for single-cell ATAC-seq data","CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing","A comparison of automatic cell identification methods for single-cell RNA sequencing data","Benchmarking automated cell type annotation tools for single-cell ATAC-seq data","Evaluation of single-cell classifiers for single-cell RNA sequencing data sets","scPred: accurate supervised method for cell-type classification from single-cell RNA-seq data","Supervised classification enables rapid annotation of cell atlases","A Systematic Evaluation of Supervised Machine Learning Algorithms for Cell Phenotype Classification Using Single-Cell RNA Sequencing Data","ChrNet: A re-trainable chromosome-based 1D convolutional neural network for predicting immune cell types","scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data","A Systematic Evaluation of Supervised Machine Learning Algorithms for Cell Phenotype Classification Using Single-Cell RNA Sequencing Data","scClassify: sample size estimation and multiscale classification of cells using single and multiple reference","Cross-species cell-type assignment from single-cell RNA-seq data by a heterogeneous graph neural network","Evaluation of some aspects in supervised cell type identification for single-cell RNA-seq: classifier, feature selection, and reference construction","Identifying phenotype-associated subpopulations by integrating bulk and single-cell sequencing data","CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing","Evaluation of some aspects in supervised cell type identification for single-cell RNA-seq: classifier, feature selection, and reference construction","Evaluation of methods to assign cell type labels to cell clusters from single-cell RNA-sequencing data","A comparison of automatic cell identification methods for single-cell RNA sequencing data","scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data","Evaluation of Cell Type Annotation R Packages on Single-cell RNA-seq Data","Evaluation of machine learning approaches for cell-type identification from single-cell transcriptomics data","Evaluation of single-cell classifiers for single-cell RNA sequencing data sets","Integrating multiple references for single-cell assignment","Evaluation of methods to assign cell type labels to cell clusters from single-cell RNA-sequencing data","scGAD: a new task and end-to-end framework for generalized cell type annotation and discovery","Learning for single-cell assignment","CIDR: Ultrafast and accurate clustering through imputation for single-cell RNA-seq data","Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data","ClusterMap for multi-scale clustering analysis of spatial gene expression","Secuer: Ultrafast, scalable and accurate clustering of single-cell RNA-seq data","Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data","A comparison framework and guideline of clustering methods for mass cytometry data","A systematic performance evaluation of clustering methods for single-cell RNA-seq data","HGC: fast hierarchical clustering for large-scale single-cell data","Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis","Benchmarking cell-type clustering methods for spatially resolved transcriptomics data","A review and performance evaluation of clustering frameworks for single-cell Hi-C data","A comparison framework and guideline of clustering methods for mass cytometry data","scMINER: a mutual information-based framework for identifying hidden drivers from single-cell omics data","A spectral clustering with self-weighted multiple kernel learning method for single-cell RNA-seq data","Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data","EDClust: an EM-MM hybrid method for cell clustering in multiple-subject single-cell RNA sequencing","Using single-cell cytometry to illustrate integrated multi-perspective evaluation of clustering algorithms using Pareto fronts","Benchmarking cell-type clustering methods for spatially resolved transcriptomics data","Benchmark and Parameter Sensitivity Analysis of Single-Cell RNA Sequencing Clustering Methods","Clustering methods for single-cell RNA-sequencing expression data: performance evaluation with varying sample sizes and cell compositions","Single-Cell RNA Sequencing Data Interpretation by Evolutionary Multiobjective Clustering","Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data","sc3: consensus clustering of single-cell rna-seq data","A Hybrid Clustering Algorithm for Identifying Cell Types from Single-Cell RNA-Seq Data","Clustering methods for single-cell RNA-sequencing expression data: performance evaluation with varying sample sizes and cell compositions","A systematic performance evaluation of clustering methods for single-cell RNA-seq data","ASTER: accurately estimating the number of cell types in single-cell chromatin accessibility data","Using single-cell cytometry to illustrate integrated multi-perspective evaluation of clustering algorithms using Pareto fronts","A model-based constrained deep learning clustering approach for spatially resolved single-cell data","Benchmark and Parameter Sensitivity Analysis of Single-Cell RNA Sequencing Clustering Methods","A review and performance evaluation of clustering frameworks for single-cell Hi-C data","Benchmarking integration of single-cell differential expression","Comparison of methods to detect differentially expressed genes between single-cell populations","Benchmarking of a Bayesian single cell RNAseq differential gene expression test for dose–response study designs","Benchmarking integration of single-cell differential expression","Comparison of methods to detect differentially expressed genes between single-cell populations","Bias, robustness and scalability in single-cell differential expression analysis","Single-cell RNA-sequencing: assessment of differential expression analysis methods","Benchmarking of a Bayesian single cell RNAseq differential gene expression test for dose–response study designs","Recommendations of scRNA-seq Differential Gene Expression Analysis Based on Comprehensive Benchmarking","Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data","Differential expression of single-cell RNA-seq data using Tweedie models","Recommendations of scRNA-seq Differential Gene Expression Analysis Based on Comprehensive Benchmarking","Bias, robustness and scalability in single-cell differential expression analysis","Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data","Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data","Single-cell RNA-sequencing: assessment of differential expression analysis methods","Reproducibility of Methods to Detect Differentially Expressed Genes from Single-Cell RNA Sequencing","Reproducibility of Methods to Detect Differentially Expressed Genes from Single-Cell RNA Sequencing","Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data","Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics","Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis","A Quantitative Framework for Evaluating Single-Cell Data Structure Preservation by Dimensionality Reduction Techniques","Benchmarking principal component analysis for large-scale single-cell RNA-sequencing","Towards a comprehensive evaluation of dimension reduction methods for transcriptomic data visualization","Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics","Supervised application of internal validation measures to benchmark dimensionality reduction methods in scRNA-seq data","A Comparison for Dimensionality Reduction Methods of Single-Cell RNA-seq Data","Towards a comprehensive evaluation of dimension reduction methods for transcriptomic data visualization","Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer","A Comparison for Dimensionality Reduction Methods of Single-Cell RNA-seq Data","Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer","Tuning parameters of dimensionality reduction methods for single-cell RNA-seq analysis","Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis","A Quantitative Framework for Evaluating Single-Cell Data Structure Preservation by Dimensionality Reduction Techniques","Dimensionality reduction for visualizing single-cell data using UMAP","Dimensionality reduction for visualizing single-cell data using UMAP","Tuning parameters of dimensionality reduction methods for single-cell RNA-seq analysis","Supervised application of internal validation measures to benchmark dimensionality reduction methods in scRNA-seq data","Benchmarking principal component analysis for large-scale single-cell RNA-sequencing","Network embedding-based representation learning for single cell RNA-seq data","Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data","Evaluating the reproducibility of single-cell gene regulatory network inference algorithms","Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data","Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information","Evaluating the reproducibility of single-cell gene regulatory network inference algorithms","Single-cell biological network inference using a heterogeneous graph transformer","NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks","A systematic evaluation of the computational tools for ligand-receptor-based cell-cell interaction inference","Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data","Identifying strengths and weaknesses of methods for computational network inference from single-cell RNA-seq data","Regulatory analysis of single cell multiome gene expression and chromatin accessibility data with scREG","A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data","Evaluating measures of association for single-cell transcriptomics","SRGS: sparse partial least squares-based recursive gene selection for gene regulatory network inference","Performance assessment of sample-specific network control methods for bulk and single-cell biological data analysis","A systematic evaluation of the computational tools for ligand-receptor-based cell-cell interaction inference","Performance assessment of sample-specific network control methods for bulk and single-cell biological data analysis","Impact of similarity metrics on single-cell RNA-seq data clustering","Optimal transport improves cell-cell similarity inference in single-cell omics data","DIRECT-NET: An efficient method to discover cis-regulatory elements and construct regulatory networks from single-cell multiomics data","Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information","IReNA: Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles","A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data","How does the structure of data impact cell-cell similarity? Evaluating how structural properties influence the performance of proximity metrics in single cell RNA-seq data","Functional inference of gene regulation using single-cell multi-omics","maxATAC: Genome-scale transcription-factor binding prediction from ATAC-seq with deep neural networks","Impact of similarity metrics on single-cell RNA-seq data clustering","Single-cell gene regulatory network prediction by explainable AI","Identifying strengths and weaknesses of methods for computational network inference from single-cell RNA-seq data","Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data","Nonparametric single-cell multiomic characterization of trio relationships between transcription factors, target genes, and cis-regulatory regions","Evaluating measures of association for single-cell transcriptomics","Dissecting cell identity via network inference and in silico gene perturbation","How does the structure of data impact cell-cell similarity? Evaluating how structural properties influence the performance of proximity metrics in single cell RNA-seq data","Benchmarking algorithms for pathway activity transformation of single-cell RNA-seq data","Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data","Benchmarking algorithms for pathway activity transformation of single-cell RNA-seq data","Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data","Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data","Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data","Benchmarking computational methods for B-cell receptor reconstruction from single-cell RNA-seq data","Benchmarking computational methods for B-cell receptor reconstruction from single-cell RNA-seq data","Predicting recognition between T cell receptors and epitopes with TCRGP","False signals induced by single-cell imputation","NISC: Neural Network-Imputation for Single-Cell RNA Sequencing and Cell Type Clustering","SpaGE: Spatial Gene Enhancement using scRNA-seq","Comparison of Computational Methods for Imputing Single-Cell RNA-Sequencing Data","scESI: evolutionary sparse imputation for single-cell transcriptomes from nearest neighbor cells","Efficient prediction of a spatial transcriptomics profile better characterizes breast cancer tissue sections without costly experimentation","A systematic evaluation of single-cell RNA-sequencing imputation methods","Region-specific denoising identifies spatial co-expression patterns and intra-tissue heterogeneity in spatially resolved transcriptomics data","Comparison of Computational Methods for Imputing Single-Cell RNA-Sequencing Data","scIMC: a platform for benchmarking comparison and visualization analysis of scRNA-seq data imputation methods","scNPF: an integrative framework assisted by network propagation and network fusion for preprocessing of single-cell RNA-seq data","scIMC: a platform for benchmarking comparison and visualization analysis of scRNA-seq data imputation methods","False signals induced by single-cell imputation","Are dropout imputation methods for scRNA-seq effective for scATAC-seq data?","scHinter: imputing dropout events for single-cell RNA-seq data with limited sample size","A systematic evaluation of single-cell RNA-sequencing imputation methods","Spatial transcriptomics at subspot resolution with BayesSpace","Are dropout imputation methods for scRNA-seq effective for scATAC-seq data?","A novel method for single-cell data imputation using subspace regression","A benchmark study of deep learning-based multi-omics data fusion methods for cancer","SD2: spatially resolved transcriptomics deconvolution through integration of dropout and spatial information","scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets","Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology","Benchmarking of cell type deconvolution pipelines for transcriptomics data","scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning","Benchmarking atlas-level data integration in single-cell genomics","Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","A benchmark study of deep learning-based multi-omics data fusion methods for cancer","DEEPsc: A Deep Learning-Based Map Connecting Single-Cell Transcriptomics and Spatial Imaging Data","A comparison of methods accounting for batch effects in differential expression analysis of UMI count based single cell RNA sequencing","Benchmarking atlas-level data integration in single-cell genomics","Benchmarking of cell type deconvolution pipelines for transcriptomics data","SpatialDWLS: accurate deconvolution of spatial transcriptomic data","Benchmarking atlas-level data integration in single-cell genomics","IBRAP: integrated benchmarking single-cell RNA-sequencing analytical pipeline","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction","Benchmarking and integration of methods for deconvoluting spatial transcriptomic data","Systematic evaluation of transcriptomics-based deconvolution methods and references using thousands of clinical samples","Alignment and integration of spatial transcriptomics data","Multi-omics single-cell data integration and regulatory inference with graph-linked embedding","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench","Comprehensive evaluation of noise reduction methods for single-cell RNA sequencing data","iSMNN: batch effect correction for single-cell RNA-seq data via iterative supervised mutual nearest neighbor refinement","Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology","Deconvolution analysis of cell-type expression from bulk tissues by integrating with single-cell expression reference","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","A comparison of methods accounting for batch effects in differential expression analysis of UMI count based single cell RNA sequencing","Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution","Fast, sensitive and accurate integration of single-cell data with Harmony","CellMixS: quantifying and visualizing batch effects in single-cell RNA-seq data","Advances in mixed cell deconvolution enable quantification of cell types in spatial transcriptomic data","AIscEA: unsupervised integration of single-cell gene expression and chromatin accessibility via their biological consistency","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","DeepST: identifying spatial domains in spatial transcriptomics by deep learning","BBKNN: fast batch alignment of single cell transcriptomes","Comprehensive evaluation of deconvolution methods for human brain gene expression","scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning","Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction","Benchmarking and integration of methods for deconvoluting spatial transcriptomic data","Single-cell multi-omics integration for unpaired data by a siamese network with graph-based contrastive loss","Comprehensive evaluation of deconvolution methods for human brain gene expression","BBKNN: fast batch alignment of single cell transcriptomes","Integration of single cell data by disentangled representation learning","A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics","IBRAP: integrated benchmarking single-cell RNA-sequencing analytical pipeline","A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics","A benchmark of batch-effect correction methods for single-cell RNA sequencing data","scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets","Systematic evaluation of transcriptomics-based deconvolution methods and references using thousands of clinical samples","Benchmarking spatial and single-cell transcriptomics integration methods","Adversarial domain translation networks for integrating large-scale atlas-level single-cell datasets","A rank-based marker selection method for high throughput scRNA-seq data","Uncovering cell identity through differential stability with Cepo","DrivAER: Identification of driving transcriptional programs in single-cell RNA sequencing data","Triku: a feature selection method based on nearest neighbors for single-cell data","Uncovering cell identity through differential stability with Cepo","De novo gene signature identification from single-cell RNA-seq with hierarchical Poisson factorization","Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq","Normalization and de-noising of single-cell Hi-C data with BandNorm and scVI-3D","Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq","Assessment of Single Cell RNA-Seq Normalization Methods","Normalization Methods on Single-Cell RNA-seq Data: An Empirical Survey","Normalization Methods on Single-Cell RNA-seq Data: An Empirical Survey","Assessment of Single Cell RNA-Seq Normalization Methods","Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq","A lineage tree-based hidden Markov model quantifies cellular heterogeneity and plasticity","Detecting critical transition signals from single-cell transcriptomes to infer lineage-determining transcription factors","A comparison of single-cell trajectory inference methods","Preprocessing choices affect RNA velocity results for droplet scRNA-seq data","A comparison of single-cell trajectory inference methods","PseudoGA: cell pseudotime reconstruction based on genetic algorithm","Predicting cell lineages using autoencoders and optimal transport","psupertime: supervised pseudotime analysis for time-series single-cell RNA-seq data","scShaper: an ensemble method for fast and accurate linear trajectory inference from single-cell RNA-seq data","Preprocessing choices affect RNA velocity results for droplet scRNA-seq data","DeepVelo: Single-cell transcriptomic deep velocity field learning with neural ordinary differential equations","PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells","Density-based detection of cell transition states to construct disparate and bifurcating trajectories","Quantifying Waddington’s epigenetic landscape: a comparison of single-cell potency measures","Quantifying Waddington’s epigenetic landscape: a comparison of single-cell potency measures","A sparse differential clustering algorithm for tracing cell type changes via single-cell RNA-sequencing data","Transcriptome analysis method based on differential distribution evaluation","Differential abundance testing on single-cell data using k-nearest neighbor graphs","Ultra-fast scalable estimation of single-cell differentiation potency from scRNA-Seq data","Doublet identification in single-cell sequencing data using scDblFinder","Benchmarking computational doublet-detection methods for single-cell RNA sequencing data","Benchmarking computational doublet-detection methods for single-cell RNA sequencing data","Vaeda computationally annotates doublets in single-cell RNA sequencing data","Simulation-based benchmarking of isoform quantification in single-cell RNA-seq","Simulation-based benchmarking of isoform quantification in single-cell RNA-seq","Pheniqs 20: accurate, high-performance Bayesian decoding and confidence estimation for combinatorial barcode indexing","EmptyNN: A neural network based on positive and unlabeled learning to remove cell-free droplets and recover lost cells in scRNA-seq data","CellSIUS provides sensitive and specific detection of rare cell populations from complex single-cell RNA-seq data","Bayesian information sharing enhances detection of regulatory associations in rare cell types","GapClust is a light-weight approach distinguishing rare cells from voluminous single cell expression profiles","Discovery of rare cells from voluminous single cell expression data","A benchmark study of simulation methods for single-cell RNA sequencing data","A benchmark study of simulation methods for single-cell RNA sequencing data","ESCO: single cell expression simulation incorporating gene co-expression","splatPop: simulating population scale single-cell RNA sequencing data","SCAN-ATAC-Sim: a scalable and efficient method for simulating single-cell ATAC-seq data from bulk-tissue experiments","Spearheading future omics analyses using dyngen, a multi-modal simulator of single cells","Simulation, power evaluation and sample size recommendation for single-cell RNA-seq","SCSsim: an integrated tool for simulating single-cell genome sequencing data","powsimR: power analysis for bulk and single cell RNA-seq experiments","Built on sand: the shaky foundations of simulating single-cell RNA sequencing data","simATAC: a single-cell ATAC-seq simulation framework","Built on sand: the shaky foundations of simulating single-cell RNA sequencing data","Simulation, power evaluation and sample size recommendation for single-cell RNA-seq","A Framework for Comparison and Assessment of Synthetic RNA-Seq Data","A statistical simulator scDesign for rational scRNA-seq experimental design","SimCH: simulation of single-cell RNA sequencing data by modeling cellular heterogeneity at gene expression level","scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured","Realistic in silico generation and augmentation of single-cell RNA-seq data using generative adversarial networks","Novel ChIP-seq simulating program with superior versatility: isChIP","SPARK-X: non-parametric modeling enables scalable and robust detection of spatial expression patterns for large spatial transcriptomic studies","SpatialDE: identification of spatially variable genes","SOMDE: a scalable method for identifying spatially variable genes with self-organizing map","SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network","Evaluation of tools for highly variable gene discovery from single-cell RNA-seq data","Evaluation of tools for highly variable gene discovery from single-cell RNA-seq data","SINFONIA: Scalable Identification of Spatially Variable Genes for Deciphering Spatial Domains","Resolving single-cell copy number profiling for large datasets","Delineating Copy Number and Clonal Substructure in Human Tumors from Single-cell Transcriptomes","Systematic comparative analysis of single-nucleotide variant detection methods from single-cell RNA sequencing data","DENDRO: genetic heterogeneity profiling and subclone detection by single-cell RNA sequencing","Systematic comparative analysis of single-nucleotide variant detection methods from single-cell RNA sequencing data","A Variational Algorithm to Detect the Clonal Copy Number Substructure of Tumors from scRNA-seq Data","Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning","Comparison of visualization tools for single-cell RNAseq data","Comparison of visualization tools for single-cell RNAseq data","Generalizable and Scalable Visualization of Single-Cell Data Using Neural Networks","Generalizable and Scalable Visualization of Single-Cell Data Using Neural Networks"],["New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper","New method development paper","Pure benchmarking paper","New method development paper","Pure benchmarking paper","New method development paper","New method development paper","Pure benchmarking paper","Pure benchmarking paper","New method development paper","New method development paper"],[0,10,27,3,24,21,2,10,11,7,7,3,10,7,7,2,20,10,20,24,5,5,21,0,2,27,2,5,5,1,3,2,2,3,5,5,13,18,9,2,5,2,2,5,2,4,0,5,8,4,2,13,4,4,14,10,22,4,5,3,22,5,9,4,1,13,5,6,13,14,5,9,0,1,9,5,22,2,10,10,9,8,5,7,null,4,14,1,4,12,9,14,20,4,15,8,9,3,7,14,11,3,15,13,11,15,12,5,4,11,14,4,3,7,13,8,46,5,8,46,5,36,6,8,12,11,8,12,36,11,18,6,9,9,18,5,18,11,21,8,5,33,10,8,9,10,9,5,18,11,5,5,5,33,21,3,8,6,8,16,6,4,7,9,12,11,0,14,17,10,16,9,16,4,4,9,16,1,14,17,0,2,4,3,11,12,0,17,4,17,7,7,7,5,5,7,6,6,5,6,14,4,8,9,1,18,8,8,12,8,12,6,7,4,18,7,7,7,16,3,5,7,20,5,68,8,16,16,8,11,68,20,4,68,7,16,10,14,12,3,10,28,28,8,28,4,7,1,14,11,16,4,9,7,7,14,6,6,8,5,10,14,8,8,6,9,18,7,18,14,5,12,16,8,9,9,2,8,9,5,6,8,10,7,8,8,7,10,1,5,45,13,45,6,4,4,6,13,7,4,5,4,4,1,5,4,8,6,9,9,7,5,5,1,4,2,0,4,3,12,12,2,0,2,3,2,2,5,16,0,16,2,5,4,4,8,2,0,3,1,4,5,7,7,3,5,1,7,3,7,2,11,13,13,1,1],["No","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Not sure","No","Yes","No","No","No","No","Yes","Not sure","No","Yes","No","Not sure","No","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","Not sure","No","Yes","Yes","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Not sure","No","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Not sure","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Not sure","Not sure","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Not sure","Yes","Yes","Yes","Yes","No","Yes","Not sure","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","No","Not sure","No","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","Not sure","Yes","Not sure","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","Not sure","Yes","No","Not sure","Not sure","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Not sure","No","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","Yes","No","Yes","Not sure","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes"],["Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","Yes","Yes","Not sure","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No"],["No","Yes","No","No","No","No","No","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","Yes","No","No","Yes","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","No","No","No","Yes","Yes","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No"],[null,10132,4000,null,3918,6425,null,81173,81173,null,null,null,10132,null,null,null,null,81173,null,3918,null,null,6425,null,274182,4000,274182,null,null,null,null,9000,null,null,null,null,100000,33611,null,null,null,800,800,null,null,null,null,null,null,2000000,null,null,null,null,96404,50000,45469,null,43410,null,45469,43410,90000,null,null,8602,null,null,8602,96404,null,null,29888,1300000,null,68579,45469,null,50000,null,90000,null,68579,null,null,10000,10000,null,40000000,8352,80000,6000,400000,null,25551,null,80000,null,null,10000,43603,null,25551,3514,null,null,8352,5000,null,null,6000,68232,null,28000,3514,null,100361,224,null,100361,224,832,200,null,5096,92,null,5096,832,92,null,200,null,null,null,null,10000,null,1306127,1000000,null,27499,50000,1000000,null,50000,null,null,10000,null,200000,200000,null,27499,1306127,1529,null,null,null,null,null,null,null,3005,1000,5520,null,null,null,null,null,3005,null,null,null,null,null,null,null,null,null,null,null,null,5520,1000,null,null,null,null,20000,null,20000,2000,2000,null,72,72,null,null,10000,null,1000,21086,null,100000,null,1000,null,null,null,null,38000,1529,100000,4000,38000,15000,null,null,null,null,10000,1089769,1000000,62337,26252,null,null,null,1000000,10000,null,1000000,null,26252,10000,59651,5052,null,null,null,null,62337,null,null,null,null,833206,null,null,null,80000,null,null,833206,null,524288,null,1089769,10000,59651,null,null,524288,null,53208,null,53208,833206,null,5052,1169213,1100167,null,10000,null,null,10000,9924,null,9230,null,null,2649,2649,null,null,null,null,1000000,null,1000000,null,null,3600,null,null,null,null,null,null,null,50000,null,200000,2000000,25000,10000,10000,null,null,null,null,null,null,null,null,68000,8000,8000,5000,null,1000000,10000,null,null,null,2600,null,2600,null,null,null,1600,null,null,null,10000,null,22860,5665,247,247,null,100000,null,null,null,null,null,68560,2000000,2000000,1283543,1283543],["Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","Yes","No","No","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","No","Yes","No","No","No","No","Yes","No","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","Yes","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","No","No","No","No","No","Yes","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","No","No","No"],["Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","Not sure","No","No","No","No","Yes","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Not sure","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","Not sure","No","No","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Not sure","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","Not sure","No","No","Yes","Yes","No","No","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes"],["No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No"],["Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","No","No","No","No","No","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","No","Yes","No","No","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","No","Yes","No","No","No","Yes","No","Yes","No","No","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No"],["Yes","No","No","Yes","No","Not sure","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Not sure","No","Not sure","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","Yes","No","No","Not sure","Yes","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","Not sure","No","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","Not sure","No","Yes","No","No","No","No","No","No","Not sure","No","No","No","No","No","No","Yes","No","Not sure","No","No","Not sure","No","No","Yes","No","No","No","Yes","Yes","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","Yes","Not sure","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","No","Not sure","Yes","No","No","No","No","Not sure","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Not sure","Not sure","No","No","Yes","Yes","Not sure","Yes","No","No","No","Yes","Not sure","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Yes","Not sure","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Not sure","No","Not sure","Yes","No","No","No","No","Not sure","No","No","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","Not sure","Yes","Yes","No","No","No","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","Yes"],["Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Both","Both","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Experimental data","Both","Both","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Both","Synthetic data","Experimental data","Experimental data","Experimental data","Experimental data","Synthetic data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Synthetic data","Experimental data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Synthetic data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Both","Both","Both","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Experimental data","Experimental data","Both","Both","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Both","Both","Experimental data","Both","Both","Both","Experimental data","Synthetic data","Synthetic data","Both","Experimental data","Experimental data","Experimental data","Experimental data","Both","Experimental data","Experimental data","Both","Experimental data","Both","Experimental data","Both","Both","Experimental data","Experimental data","Both","Experimental data","Both","Both","Experimental data","Both","Both","Both","Experimental data","Experimental data","Synthetic data","Both","Experimental data","Both","Both","Experimental data","Both","Experimental data","Both","Both","Both","Both","Both","Experimental data","Experimental data","Experimental data","Experimental data"],[3,12,14,9,7,20,2,3,4,2,8,9,12,4,8,2,6,3,6,7,1,10,20,76,4,14,4,20,20,2,4,9,2,2,11,11,6,5,4,22,4,3,3,4,22,3,1,3,5,18,8,6,4,30,13,6,27,6,5,9,27,5,3,4,6,43,3,9,43,13,1,10,4,5,10,5,27,9,6,20,3,27,5,10,31,3,2,7,15,6,6,4,4,8,7,6,6,11,11,2,4,4,7,17,null,6,6,12,8,null,4,27,4,49,17,6,9,3,1,9,3,9,2,1,2,2,2,2,9,2,0,2,3,3,0,3,30,4,4,10,3,55,5,10,11,5,11,10,30,4,5,5,10,55,4,3,2,6,2,5,6,13,10,15,5,7,1,null,213,4,9,15,9,22,8,4,5,3,null,2,3,518,22,1,7,5,4,213,5,2,32,2,32,10,10,2,3,3,2,18,3,6,2,12,null,6,12,2,2,8,2,18,8,8,6,5,8,25,6,3,14,5,5,5,23,3,45,6,4,1,23,5,2,23,15,45,10,4,2,5,5,4,4,3,4,2,5,3,9,1,45,6,7,5,1,9,5,6,3,5,10,4,5,3,6,11,7,15,7,9,14,2,null,16,4,9,2,23,9,4,3,4,4,120,4,4,120,4,2,6,110,5,110,4,2,1,1,5,6,8,2,11,11,4,12,3,28,16,16,16,16,null,null,1,4,2,3,9,4,35,35,1,3,1,1,11,1,8,12,7,12,11,1,18,13,6,5,2,6,4,6,7,5,5,15,2,6,2,2,2,3,7,1,1,13,13],[1000,null,14,0,2,null,null,10,null,9,null,0,null,null,null,null,19980,10,19980,2,4,4201,null,null,null,14,null,null,null,1,null,0,1,null,null,null,null,2,4,null,null,null,null,null,null,990,1,null,null,3800,null,null,1,null,0,41,null,null,null,1,null,null,null,10,null,3,null,null,3,0,1,null,3,null,null,null,null,null,41,null,null,null,null,1,null,1,null,null,31,null,null,3,null,0,7,null,null,null,null,null,100,null,7,25,700,null,null,null,null,700,3,21,null,6,25,null,7,3,44,7,3,3,10,44,4,10,16,4,3,10,1280,10,1,1,1280,40,2,2,18,2,40,null,30,2,100,30,100,null,2,2,null,null,null,null,18,null,2,0,2,15,0,null,null,null,420,null,null,100,null,null,null,null,null,0,5,0,15,null,100,30,0,null,0,1,null,420,0,null,null,30,null,30,null,null,null,30,74,74,null,60,60,0,5,6,null,1,null,5,6,null,6,60,null,null,1,null,null,116,3,2,0,200,1000,0,2,6,32,3,null,12,2,1000,1,2,10,32,0,3,2,1,39,28,28,6,28,1,200,1,1,12,32,0,91,null,1,1,null,null,100,0,0,3,null,100,null,2,43,10,43,1,0,2,null,0,20,49,null,1,49,null,20,null,10,null,1,1,null,10,5,null,229,0,229,null,null,1,6,0,null,null,5,null,null,7,1,1172,null,1,112,112,null,3,3,1,null,null,null,null,1,null,null,4,null,1,null,40,13,0,null,21,null,14,2,0,5,30,30,null,98,3001,22,null,1,1,null,9,0,1,9,1,500,4,null,null,null,null],["Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","No","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Not sure","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Not sure","No","Yes","Yes","Yes","Yes","No","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Not sure","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Not sure","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes"],["Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","No","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","No","No","No","No","Yes","No","Not sure","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","No","No","No","Yes","Yes","No","No","No","No","Not sure","No","No","Yes","No","Yes","Yes","No","Yes","No","Not sure","Yes","Not sure","Not sure","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","No","Yes","No","No","No","No","No","Not sure","Yes","No","Yes","No","No","No","Yes","Yes","No","No","No","Yes","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Not sure","Yes","No","Yes","Yes","Not sure","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","Not sure","No","Not sure","Yes","No","No","Not sure","Yes","Yes","No","Yes","No","Not sure","No","Not sure","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Not sure","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Not sure","No","No","No","No","Yes","No","Yes","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Not sure","No","No","No","Yes","No","No","No","No","Yes","Not sure","No","No","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","No","No","Yes","No","No","No","Not sure","Yes","Yes","Yes","No","No","No","No","No","No","No","Yes","No","Yes","No","No","No","Yes","No","Yes","Yes","No","No","No","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","No","Yes"],["Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","Yes","No","No","No","No","No","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","No","No","No","No","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","Yes","Yes","No","Yes","No","No","Yes","Yes","No","No","No","No","Yes","No","Yes","Yes","No","Yes","Yes","No","No","Yes","No","No","No","Yes","No","Yes","Yes","Yes","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","No","Yes","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes"],["No","No","Yes","No","No","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","Yes","No","No","No","No","Yes","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","Yes","No","No","Yes","No","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","Yes","Yes","Yes","No","No","Yes","No","No","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","Yes","Yes","Yes","Yes","No","No","Yes","Yes","Yes","Yes","No","Yes","No","No","Yes","No","No","No","Yes","No","No","No","No","No","Yes","No","No","No","No","Yes","No","Yes","No","No","Yes","No","No","No","No","No","No","Yes","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","No","No","No","No","No","No","No","Yes","No","Yes","No","No","Yes","Yes","No","No","Yes","No","Yes","Yes","No","No","No","Yes","Yes","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","Yes","No","No","No","No","No","No","Yes","No","No","Yes","No","No","No","No","No","Yes","No","No","No","Yes","Yes","No","Yes","No","No","No","Yes","Yes","No","Yes","No","Yes","Yes","No","No","Yes","Yes","Yes","No","No","No","Yes","No","No","Yes","No","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","No","No","Yes","Yes","No","Yes","Yes","Yes","Yes","No","Yes","Yes","Yes","Yes","Yes","No","Yes","No","Yes","No","No","Yes","Yes","Yes","No","No","Yes","Yes","No","No","Yes","Yes","Yes","No","Yes","Yes","No","No","No","Yes","Yes","No","Yes","Yes","No","Yes","No","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes","No","No","No","No","No","No","No","No","Yes","Yes","Yes","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","No","Yes","Yes","No","No","No","No","No","No","Yes","Yes","No","No"],["Bioinformatics (Oxford, England)","Genome biology","Nature methods","Nature communications","Genome biology","Nature biotechnology","Current protocols","Genome biology","Nature communications","Genomics, proteomics &amp; bioinformatics","Briefings in bioinformatics","Nature communications","Genome biology","Frontiers in genetics","Briefings in bioinformatics","Current protocols","Nature communications","Genome biology","Nature communications","Genome biology","Nature methods","Genome biology","Nature biotechnology","Nucleic acids research","Nature methods","Nature methods","Nature methods","Scientific data","Scientific data","Molecular systems biology","Nature methods","Nature biotechnology","PLoS computational biology","Nature methods","Briefings in bioinformatics","Briefings in bioinformatics","Nature communications","Nucleic acids research","Nature communications","BMC genomics","GigaScience","G3 (Bethesda, Md.)","G3 (Bethesda, Md.)","GigaScience","BMC genomics","Nature communications","Proceedings of SPIE--the International Society for Optical Engineering","Nucleic acids research","Nature communications","Nucleic acids research","Journal of chemical information and modeling","Nucleic acids research","Nature communications","Nature communications","Molecular systems biology","Genomics, proteomics &amp; bioinformatics","Genome biology","Bioinformatics (Oxford, England)","Frontiers in genetics","Nucleic acids research","Genome biology","Frontiers in genetics","Briefings in bioinformatics","Genome biology","Nature methods","Frontiers in genetics","Genomics","iScience","Frontiers in genetics","Molecular systems biology","Genome research","Genome biology","Nature biotechnology","Genome biology","Genome biology","F1000Research","Genome biology","iScience","Genomics, proteomics &amp; bioinformatics","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","F1000Research","Briefings in bioinformatics","Science advances","Genome biology","Genome biology","Nature communications","PLoS computational biology","F1000Research","Genome Biology","F1000Research","Bioinformatics (Oxford, England)","BMC bioinformatics","Briefings in bioinformatics","Briefings in bioinformatics","Genome Biology","Research square","Briefings in bioinformatics","Genome biology","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Frontiers in genetics","Statistical applications in genetics and molecular biology","IEEE/ACM transactions on computational biology and bioinformatics","F1000Research","Nature methods","Genes","Statistical applications in genetics and molecular biology","F1000Research","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Genome research","Frontiers in genetics","Briefings in bioinformatics","Nature communications","Briefings in bioinformatics","Nucleic acids research","Nature communications","Briefings in bioinformatics","Nature methods","Frontiers in genetics","Nucleic acids research","Life (Basel, Switzerland)","BMC bioinformatics","Statistics in medicine","Life (Basel, Switzerland)","Nature methods","BMC bioinformatics","Briefings in bioinformatics","Frontiers in genetics","Frontiers in genetics","Frontiers in genetics","Briefings in bioinformatics","Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","Genome biology","Cell reports","Genome biology","Communications biology","Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","Briefings in bioinformatics","Frontiers in genetics","Communications biology","Nature communications","Frontiers in genetics","Nature communications","Genome biology","Genome biology","Cell reports","Nature biotechnology","Nature biotechnology","Genome biology","Briefings in bioinformatics","Genome biology","Nucleic acids research","BMC bioinformatics","Frontiers in genetics","BMC bioinformatics","Genome biology","Frontiers in genetics","Nature communications","Communications biology","Briefings in functional genomics","Nature methods","G3 (Bethesda, Md.)","Genome biology","Briefings in bioinformatics","Nature methods","BMC genomics","PLoS computational biology","Briefings in functional genomics","PLoS computational biology","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Science advances","Genome biology","iScience","Briefings in bioinformatics","Briefings in bioinformatics","Cell genomics","PLoS computational biology","Briefings in bioinformatics","Nucleic acids research","G3 (Bethesda, Md.)","Nature methods","Cell systems","Nature methods","Nature","Briefings in bioinformatics","Computational and structural biotechnology journal","Genome biology","Computational and structural biotechnology journal","eLife","eLife","Genome biology","NAR genomics and bioinformatics","NAR genomics and bioinformatics","PLoS computational biology","F1000Research","Frontiers in genetics","Nucleic acids research","IEEE/ACM transactions on computational biology and bioinformatics","Briefings in bioinformatics","Scientific reports","Genome biology","Nature communications","IEEE/ACM transactions on computational biology and bioinformatics","Nucleic acids research","BMC genomics","Nucleic acids research","F1000Research","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Genome biology","Nature biotechnology","Briefings in bioinformatics","Scientific reports","Genome biology","Bioinformatics (Oxford, England)","Proceedings of the National Academy of Sciences of the United States of America","Bioinformatics (Oxford, England)","Nature communications","Nature biotechnology","Nature methods","Nucleic acids research","Nature methods","Genome biology","Frontiers in genetics","Computational and structural biotechnology journal","Nature methods","Nature communications","Genome biology","Nature methods","Briefings in bioinformatics","Nature methods","Genome biology","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Nature methods","Nature biotechnology","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","Briefings in bioinformatics","Briefings in bioinformatics","Bioinformatics (Oxford, England)","Genetic epidemiology","Genome biology","Computational and structural biotechnology journal",null,"Nature methods","Life science alliance","Nature communications","Bioinformatics (Oxford, England)","Genome biology","Nucleic acids research","Bioinformatics (Oxford, England)","Nature communications","Nature biotechnology","Genome biology","Bioinformatics (Oxford, England)","BMC bioinformatics","Nature communications","Bioinformatics (Oxford, England)","Nucleic acids research","Nature communications","Briefings in bioinformatics","Nature communications","Genome biology","Proceedings of the National Academy of Sciences of the United States of America","Briefings in bioinformatics","Nature methods","Nature computational science","BMC bioinformatics","Nature computational science","GigaScience","GigaScience","Nature computational science","Molecular systems biology","eLife","Genome biology","Cell systems","G3 (Bethesda, Md.)","Frontiers in genetics","Frontiers in genetics","G3 (Bethesda, Md.)","Cell systems","Communications biology","Nucleic acids research","Nature biotechnology","PLoS computational biology","Nature biotechnology","Nucleic acids research","PLoS computational biology","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","PLoS computational biology","Science advances","Genome biology","Nucleic acids research","Briefings in bioinformatics","Briefings in bioinformatics","Nucleic acids research","Briefings in bioinformatics","Nature biotechnology","Bioinformatics (Oxford, England)","F1000Research","Cell systems","Cell systems","Bioinformatics (Oxford, England)","Genome biology","Genome biology","BMC bioinformatics","Patterns (New York, N.Y.)","Genome biology","Bioinformatics (Oxford, England)","Nature communications","Nature communications","Nature communications","Nature communications","Bioinformatics (Oxford, England)","Genome biology","Bioinformatics (Oxford, England)","Nature communications","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Bioinformatics (Oxford, England)","Genome biology","Genome biology","Genome biology","Bioinformatics (Oxford, England)","Genes","Bioinformatics (Oxford, England)","Briefings in bioinformatics","Genome biology","Nature communications","Briefings in bioinformatics","Genome biology","Nature methods","Bioinformatics (Oxford, England)","Nature methods","Briefings in bioinformatics","Briefings in bioinformatics","Cells","Briefings in bioinformatics","Nature biotechnology","Genome biology","Genome biology","Genome biology","Nature communications","Nature methods","NAR genomics and bioinformatics","NAR genomics and bioinformatics","Cell systems","Cell systems"],["2022 May 26","2021 Dec 14","2019 May 27","2020 Nov 11","2020 Sep 1","2021 Sep","2023 Jan 12","2019 Nov 18","2021 Sep 1","2022 Dec 19","2021 May 20","2020 Nov 11","2021 Dec 14","2023 Feb 17","2021 May 20","2023 Jan 12","2019 Oct 11","2019 Nov 18","2019 Oct 11","2020 Sep 1","2017 Aug 7","2021 Mar 8","2021 Sep","2021 Jan 8","2020 Jul 27","2019 May 27","2020 Jul 27","2021 Feb 2","2021 Feb 2","2021 May 31","2020 Dec 14","2021 Oct 14","2021 Nov 18","2021 Aug 26","2022 Jan 17","2022 Jan 17","2023 Mar 1","2022 Jul 8","2021 Mar 25","2021 Sep 14","2022 Jan 27","2020 May 4","2020 May 4","2022 Jan 27","2021 Sep 14","2018 Feb 15","2023 Apr 7","2022 Nov 28","2022 Dec 10","2019 Dec 16","2023 Jan 9","2022 May 6","2020 Oct 26","2021 Jun 22","2020 Jun 22","2020 Dec 24","2019 Sep 9","2023 Feb 3","2022 Dec 13","2019 Sep 19","2019 Sep 9","2022 Dec 13","2020 Sep 25","2019 Dec 12","2019 Sep 9","2022 Feb 23","2021 Apr 28","2020 Mar 27","2022 Feb 23","2020 Jun 22","2022 Dec 16","2021 Sep 9","2021 Nov 11","2019 Feb 11","2021 Sep 9","2019 Oct 14","2019 Sep 9","2020 Mar 27","2020 Dec 24","2021 Sep 2","2020 Sep 25","2021 Aug 20","2019 Oct 14","2023 Mar 19","2020 Oct 30","2017 Mar 28","2022 Feb 8","2021 Oct 8","2022 Dec 5","2018 Aug 15","2019 Dec 23","2018 Jul 26","2021 Nov 5","2019 Dec 24","2023 Jan 19","2022 Nov 19","2019 Dec 23","2023 Jan 27","2021 Jul 20","2022 Feb 8","2022 May 13","2021 Jan 28","2023 Jan 19","2019 Dec 11","2019 Aug 14","2020 Sep 1","2018 Aug 15","2017 Mar 27","2019 Jan 29","2019 Aug 14","2018 Jul 26","2023 Jan 1","2021 Jan 28","2022 Oct 5","2019 Dec 11","2022 Nov 19","2023 Mar 21","2017 Sep 1","2022 May 6","2023 Mar 21","2017 Sep 1","2018 Feb 26","2017 May 23","2022 May 6","2022 Jun 7","2019 Jan 18","2022 Aug 15","2022 Jun 7","2018 Feb 26","2019 Jan 18","2022 Sep 20","2017 May 23","2020 Jan 17","2020 Jan 17","2022 Sep 20","2019 Mar 14","2019 Dec 10","2020 May 5","2020 Jan 20","2022 Jul 19","2019 Mar 14","2021 Nov 5","2021 Mar 23","2022 Jul 19","2021 Jan 5","2021 Mar 23","2021 Jan 5","2020 Aug 24","2019 Dec 10","2020 May 5","2018 Dec 3","2018 Dec 3","2020 Aug 24","2021 Nov 5","2020 Jan 20","2017 Nov 2","2018 Jun 19","2021 Mar 22","2018 Jun 19","2022 Oct 17","2021 Mar 22","2023 Feb 21","2022 Nov 23","2022 Sep 16","2020 Jan 6","2023 Mar 9","2022 May 16","2021 May 20","2019 Apr 8","2022 Nov 30","2021 May 6","2022 Sep 16","2021 May 6","2019 Nov 27","2022 Apr 12","2022 Jun 3","2022 Oct 17","2022 Nov 18","2021 May 20","2022 Nov 19","2022 Sep 14","2023 Jan 31","2019 Nov 27","2023 Feb 28","2023 Mar 9","2020 Jan 6","2022 Sep 21","2019 Apr 8","2023 Feb 8","2022 Nov 19","2020 Oct 15","2020 Feb 12","2020 Oct 15","2022 Feb 25","2022 Feb 25","2020 Feb 12","2022 Jul 13","2022 Jul 13","2021 Mar 25","2018 Nov 2","2022 May 3","2020 Oct 9","2020 Mar 1","2022 Sep 20","2022 Mar 8","2020 Aug 27","2022 Nov 14","2020 Mar 1","2022 May 20","2019 May 8","2022 May 20","2018 Nov 2","2022 Jan 17","2020 Feb 1","2020 Aug 27","2021 Jun 3","2022 Jan 17","2022 Feb 17","2022 Aug 9","2022 Oct 31","2019 May 14","2019 Jul 15","2020 Nov 6","2022 Jan 20","2021 Dec 23","2021 Apr 19","2022 May 16","2022 Aug 9","2021 Mar 23","2020 mar 30","2021 Dec 23","2020 Nov 6","2021 May 10","2021 Dec 23","2023 Mar 19","2022 May 16","2022 Sep 5","2023 Jan 1","2021 Nov 5","2022 May 16","2022 May 2","2022 Mar 10","2022 Mar 10","2021 Apr 19","2022 Mar 10","2021 Sep 2","2019 Jul 15","2022 Jul 5","2020 Jan 16","2020 mar 30","2022 May 16","2019 Nov 18","2021 Mar 23","2022 Jan 19","2022 Nov 30","2020 Jan 16","2022 Dec 9","2020 Feb 1","2022 Mar 15","2022 Jan 20","2022 Sep 5","2023 Jan 1","2023 Jan 4","2022 Mar 15","2020 Feb 1","2022 Jan 25","2023 Mar 21","2023 Mar 19","2023 Mar 21","2020 Jan 16","2019 May 14","2021 Nov 5","2022 May 16","2022 May 30","2020 Oct 23","2021 Dec 20","2020 Dec 10","2022 Mar 12","2021 Dec 20","2019 Feb 22","2019 Jul 8","2022 Oct 17","2019 Apr 24","2017 Jul 5","2020 Feb 7","2020 Feb 7","2017 Jul 5","2019 Apr 24","2022 Nov 17","2022 Sep 9","2019 Apr 1","2021 Jan 11","2019 Apr 1","2021 Aug 20","2020 Apr 28","2022 Jun 24","2022 Feb 7","2021 Jan 11","2022 Dec 2","2019 Mar 19","2022 Nov 28","2020 Jan 17","2020 Jan 17","2018 Feb 16","2022 Mar 10","2021 Sep 30","2021 Jul 12","2022 May 16","2021 Feb 17","2021 Feb 17","2023 Jan 1","2018 Nov 7","2018 Nov 7","2021 Jul 2","2021 Aug 13","2019 Jul 17","2021 Jul 12","2021 Jul 7","2018 Nov 9","2021 Nov 25","2021 Nov 25","2021 Aug 25","2021 Dec 15","2021 Jul 19","2021 Jun 24","2020 Dec 8","2020 Feb 15","2017 Nov 1","2023 Mar 29","2021 Mar 4","2023 Mar 29","2020 Dec 8","2022 Dec 14","2019 Jul 15","2023 Jan 19","2021 May 25","2020 Jan 9","2021 Jul 20","2021 Jun 21","2018 Mar 19","2021 Dec 7","2021 Oct 28","2019 Jul 19","2019 Jul 19","2023 Feb 13","2022 Jul 18","2021 Jan 18","2019 Nov 19","2020 Jan 14","2019 Nov 19","2023 Feb 25","2017 Mar 6","2020 Jul 29","2020 Jul 29","2018 Aug 22","2018 Aug 22"],["Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell proteomics","Single-cell ATAC-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell proteomics","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Spatial imaging","Spatial transcriptomics","Single-cell Hi-C","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell genomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell Hi-C","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell mass cytometry","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell mass cytometry","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell multiomics","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell sequencing","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Spatial imaging","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell multiomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell sequencing","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell genomics","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ATAC-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell ChIP-seq","Spatial transcriptomics","Spatial transcriptomics","Spatial transcriptomics","Spatial transcriptomics","Single-cell RNA-seq","Single-cell RNA-seq","Spatial transcriptomics","Single-cell Hi-C","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq","Single-cell RNA-seq"],["Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Data","Data","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Data","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Downstream analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Intermediate analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis","Initial analysis"],["Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Analysis pipelines","Benchmarking datasets","Benchmarking datasets","Cell segmentation","Cell segmentation","Cell segmentation","Chromatin conformation","Chromatin conformation","Multi-task","Multi-task","Multi-task","Multi-task","Multi-task","Alignment","Alignment","Alignment","Alignment","Alignment","Alignment","Assembly","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Cell type/state identification","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Clustering","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Differential expression","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Dimension reduction","Regulatory network inference","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Cell cell communication","Regulatory network inference","Measure of association","Cell cell communication","Regulatory network inference","Cell cell communication","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Regulatory network inference","Regulatory network inference","Regulatory network inference","Measure of association","Regulatory network inference","Measure of association","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","Pathway enrichment","T/B-cell receptor","T/B-cell receptor","T/B-cell receptor","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Imputation","Data integration","Deconvolution","Batch correction","Deconvolution","Deconvolution","Data integration","Batch correction","Batch correction","Data integration","Data integration","Data integration","Batch correction","Batch correction","Deconvolution","Deconvolution","Batch correction","Data integration","Data integration","Data integration","Deconvolution","Deconvolution","Data integration","Data integration","Batch correction","Batch correction","Batch correction","Batch correction","Batch correction","Deconvolution","Deconvolution","Batch correction","Batch correction","Data integration","Data integration","Batch correction","Deconvolution","Data integration","Batch correction","Data integration","Batch correction","Deconvolution","Data integration","Data integration","Deconvolution","Data integration","Deconvolution","Batch correction","Data integration","Deconvolution","Data integration","Deconvolution","Batch correction","Batch correction","Deconvolution","Data integration","Data integration","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Marker/signature identification","Standardization","Standardization","Standardization","Standardization","Standardization","Standardization","Standardization","Trajectory","Trajectory","Trajectory","Velocity","Trajectory","Trajectory","Trajectory","Trajectory","Trajectory","Velocity","Velocity","Trajectory","Trajectory","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Differential analysis - others","Doublet detection","Doublet detection","Doublet detection","Doublet detection","Quantification","Quantification","Quantification","Quantification","Rare cell type identification","Rare cell type identification","Rare cell type identification","Rare cell type identification","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Simulation","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Highly variable genes","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Subclone detection","Visualisation","Visualisation","Visualisation","Visualisation","Visualisation"]],"container":"<table class=\"table table-striped table-condensed\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>luke_topic<\/th>\n      <th>Paper.title<\/th>\n      <th>Paper.category<\/th>\n      <th>Methods.compared<\/th>\n      <th>Selection.criteria<\/th>\n      <th>Variability.of.score<\/th>\n      <th>Memory.measured<\/th>\n      <th>Max.number.of.cells<\/th>\n      <th>Sensitivity.analysis<\/th>\n      <th>Discovery.<\/th>\n      <th>Website<\/th>\n      <th>Data.availability<\/th>\n      <th>Prior.knowledge<\/th>\n      <th>Types.of.data<\/th>\n      <th>Number.of.experimental.datasets<\/th>\n      <th>Number.of.synthetic.datasets<\/th>\n      <th>Recommendation<\/th>\n      <th>Applicability<\/th>\n      <th>Speed.measured.<\/th>\n      <th>Overall.comparison<\/th>\n      <th>journal_name<\/th>\n      <th>publication_date<\/th>\n      <th>data_type<\/th>\n      <th>broader_topic<\/th>\n      <th>finer_topic<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"bPaginate":false,"dom":"<\"top\"flip>rt","language":{"info":"Showing _TOTAL_ entries"},"pageLength":10,"scrollX":true,"searching":true,"autoWidth":true,"columnDefs":[{"width":"150px","targets":[1,23,24,25]},{"width":"200px","targets":2},{"className":"dt-right","targets":[4,8,15,16]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"luke_topic","targets":1},{"name":"Paper.title","targets":2},{"name":"Paper.category","targets":3},{"name":"Methods.compared","targets":4},{"name":"Selection.criteria","targets":5},{"name":"Variability.of.score","targets":6},{"name":"Memory.measured","targets":7},{"name":"Max.number.of.cells","targets":8},{"name":"Sensitivity.analysis","targets":9},{"name":"Discovery.","targets":10},{"name":"Website","targets":11},{"name":"Data.availability","targets":12},{"name":"Prior.knowledge","targets":13},{"name":"Types.of.data","targets":14},{"name":"Number.of.experimental.datasets","targets":15},{"name":"Number.of.synthetic.datasets","targets":16},{"name":"Recommendation","targets":17},{"name":"Applicability","targets":18},{"name":"Speed.measured.","targets":19},{"name":"Overall.comparison","targets":20},{"name":"journal_name","targets":21},{"name":"publication_date","targets":22},{"name":"data_type","targets":23},{"name":"broader_topic","targets":24},{"name":"finer_topic","targets":25}],"order":[],"orderClasses":false,"orderCellsTop":true},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="evaluation-summary" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-evaluation-summary">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card tabset html-fill-item html-fill-container bslib-card" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header dashboard-card-no-title bslib-navs-card-title">

<ul class="nav nav-tabs card-header-tabs" role="tablist" data-tabsetid="card-tabset-16"><li class="nav-item" role="presentation"><a href="#card-tabset-16-1" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Journal" aria-selected="true" class="nav-link active">Journal</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-16-2" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Publication Year" aria-selected="false" class="nav-link">Publication Year</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-16-3" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Category Frequency" aria-selected="false" class="nav-link">Category Frequency</a></li><li class="nav-item" role="presentation"><a href="#card-tabset-16-4" role="tab" data-toggle="tab" data-bs-toggle="tab" data-value="Statistics" aria-selected="false" class="nav-link">Statistics</a></li></ul></div>




<div class="tab-content html-fill-item html-fill-container" data-tabset-id="card-tabset-16"><div class="tab-pane active show html-fill-item html-fill-container" role="tabpanel" id="card-tabset-16-1"><div class="card-body html-fill-item html-fill-container" data-title="Journal">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 50fr) minmax(3em, 50fr);
grid-auto-rows: minmax(0, 1fr);">
<!-- Evaluation - Journal --><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure Benchmark" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure Benchmark</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-88fe61ed3e5b17b028db" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-88fe61ed3e5b17b028db">{"x":{"data":[{"x":[0,-0.010897298250727605,-0.043417336099766501,-0.097047253123707655,-0.17094127349412447,-0.26393404436762724,-0.37455901419635218,-0.50107156112194673,-0.64147650670446166,-0.79355958107821178,-0.95492234331029824,-1.1230200062468794,-1.2952015693267684,-1.4687516264439155,-1.6409331895238051,-1.809030852460386,-1.9703936146924723,-2.1224766890662226,-2.2628816346487373,-2.3893941815743318,-2.5000191514030563,-2.5930119222765597,-2.6669059426469763,-2.7205358596709175,-2.7530558975199564,-2.763953195770684,-2.7530558975199564,-2.7205358596709175,-2.6669059426469763,-2.5930119222765593,-2.5000191514030563,-2.3893941815743318,-2.2628816346487373,-2.1224766890662221,-1.9703936146924717,-1.809030852460386,-1.6409331895238046,-1.4687516264439153,-1.295201569326768,-1.1230200062468785,-0.95492234331029857,-0.7935595810782119,-0.64147650670446144,-0.50107156112194629,-0.37455901419635174,-0.2639340443676268,-0.17094127349412447,-0.097047253123707655,-0.043417336099766279,-0.010897298250727605,0,0],"y":[0,0.17320759572316616,0.3436836041925756,0.50873951691723462,0.66577230355385231,0.81230546325032948,0.94602808054358611,1.0648312698768878,1.1668414339848632,1.2504498116416116,1.3143378487862616,1.3574979929081792,1.3792495827519409,1.3792495827519409,1.357497992908179,1.3143378487862616,1.2504498116416114,1.1668414339848632,1.0648312698768878,0.94602808054358578,0.8123054632503296,0.6657723035538522,0.50873951691723429,0.34368360419257565,0.17320759572316594,-4.4448316312575701e-16,-0.17320759572316619,-0.34368360419257593,-0.50873951691723507,-0.66577230355385242,-0.81230546325032971,-0.94602808054358611,-1.064831269876888,-1.1668414339848636,-1.2504498116416118,-1.3143378487862616,-1.3574979929081792,-1.3792495827519409,-1.3792495827519409,-1.357497992908179,-1.3143378487862618,-1.2504498116416116,-1.1668414339848632,-1.0648312698768876,-0.94602808054358545,-0.81230546325032871,-0.66577230355385231,-0.5087395169172344,-0.34368360419257521,-0.17320759572316549,-3.3847546452084446e-16,0],"text":"id: 1<br />as.factor(id): 1","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(248,118,109,0.6)","hoveron":"fills","name":"1","legendgroup":"1","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.2567583341910251,2.2478607274279563,2.221308227712576,2.17751958382709,2.1171853688436411,2.0412570893744308,1.950932179744044,1.8476351177348374,1.7329949597213705,1.6088196494782268,1.4770675058266745,1.3398163387770146,1.1992306812234936,1.0575276529675315,0.91694199541401011,0.77969082836435055,0.64793868471279836,0.52376337446965404,0.40912321645618743,0.30582615444698102,0.21550124481659449,0.13957296534738395,0.079238750363934685,0.03545010647844915,0.0088976067630688327,0,0.0088976067630688327,0.03545010647844915,0.079238750363935129,0.13957296534738406,0.21550124481659461,0.30582615444698114,0.40912321645618777,0.52376337446965471,0.64793868471279903,0.77969082836435055,0.91694199541401034,1.0575276529675317,1.1992306812234941,1.3398163387770152,1.4770675058266742,1.6088196494782265,1.732994959721371,1.8476351177348378,1.9509321797440444,2.0412570893744313,2.1171853688436411,2.1775195838270904,2.221308227712576,2.2478607274279563,2.2567583341910251,2.2567583341910251],"y":[0,0.14142340969867698,0.28061648774415587,0.41538407614574507,0.54360080952809653,0.66324463341280659,0.77242869322545737,0.86943109111940753,0.95272204133344573,1.0209879958270949,1.0731523597178851,1.1083924698257788,1.1261525685629523,1.1261525685629523,1.1083924698257785,1.0731523597178851,1.0209879958270947,0.95272204133344562,0.86943109111940753,0.77242869322545715,0.66324463341280671,0.54360080952809642,0.4153840761457448,0.28061648774415587,0.14142340969867678,-3.6291898297212129e-16,-0.14142340969867701,-0.2806164877441561,-0.41538407614574546,-0.54360080952809664,-0.66324463341280682,-0.77242869322545737,-0.86943109111940764,-0.95272204133344596,-1.0209879958270951,-1.0731523597178851,-1.1083924698257788,-1.1261525685629523,-1.1261525685629523,-1.1083924698257785,-1.0731523597178851,-1.0209879958270949,-0.95272204133344562,-0.86943109111940731,-0.77242869322545693,-0.66324463341280593,-0.54360080952809653,-0.41538407614574496,-0.28061648774415549,-0.14142340969867645,-2.763640595091933e-16,0],"text":"id: 2<br />as.factor(id): 2","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(239,127,73,0.6)","hoveron":"fills","name":"2","legendgroup":"2","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.87848726959383272,0.87219571151533548,0.85342025890913609,0.82245701187874631,0.77979427932638246,0.72610487802987644,0.66223552192016788,0.58919346889681068,0.50813063576919104,0.42032543184032051,0.3271625976284438,0.23011136668186169,0.13070229488819721,0.030503122693737346,-0.068905949099927283,-0.16595718004650925,-0.25912001425838588,-0.34692521818725663,-0.42798805131487616,-0.50103010433823336,-0.56489946044794181,-0.61858886174444783,-0.6612515942968118,-0.69221484132720146,-0.71099029393340085,-0.71728185201189809,-0.71099029393340085,-0.69221484132720146,-0.66125159429681157,-0.61858886174444783,-0.5648994604479417,-0.50103010433823325,-0.42798805131487605,-0.34692521818725619,-0.25912001425838543,-0.16595718004650925,-0.068905949099927144,0.030503122693737506,0.13070229488819754,0.23011136668186216,0.32716259762844357,0.42032543184032045,0.50813063576919115,0.58919346889681101,0.66223552192016821,0.72610487802987678,0.77979427932638246,0.82245701187874642,0.8534202589091362,0.87219571151533548,0.87848726959383272,0.87848726959383272],"y":[-1.6163713814713934,-1.5163699294549355,-1.417945560074749,-1.3226504844318279,-1.2319875627955794,-1.1473866035996121,-1.0701818145086088,-1.0015907611664412,-0.94269516545862364,-0.8944238461119921,-0.8575380706685316,-0.83261954984147946,-0.8200612635898813,-0.8200612635898813,-0.83261954984147957,-0.8575380706685316,-0.89442384611199222,-0.94269516545862364,-1.0015907611664412,-1.0701818145086093,-1.1473866035996119,-1.2319875627955796,-1.3226504844318279,-1.417945560074749,-1.5163699294549358,-1.6163713814713936,-1.7163728334878512,-1.8147972028680379,-1.9100922785109593,-2.0007552001472071,-2.0853561593431751,-2.1625609484341779,-2.2311520017763455,-2.2900475974841634,-2.338318916830795,-2.3752046922742549,-2.4001232131013071,-2.4126814993529053,-2.4126814993529053,-2.4001232131013071,-2.3752046922742553,-2.3383189168307945,-2.290047597484163,-2.2311520017763455,-2.1625609484341775,-2.0853561593431742,-2.0007552001472071,-1.9100922785109589,-1.8147972028680375,-1.7163728334878507,-1.6163713814713936,-1.6163713814713934],"text":"id: 3<br />as.factor(id): 3","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(229,135,0,0.6)","hoveron":"fills","name":"3","legendgroup":"3","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5139490783753313,3.4887828460613424,3.4136810356365448,3.2898280475149857,3.1191771173055303,2.9044195121195067,2.6489420876806724,2.3567738755872432,2.0325225430767646,1.6813017273612827,1.3086503905137759,0.92044546672744743,0.52280917955278949,0.12201249077495005,-0.27562379639970847,-0.66382872018603634,-1.0364800570335428,-1.3877008727490259,-1.711952205259504,-2.004120417352933,-2.2595978417917664,-2.4743554469777909,-2.6450063771872463,-2.7688593653088054,-2.843961175733603,-2.8691274080475919,-2.843961175733603,-2.7688593653088054,-2.6450063771872454,-2.4743554469777909,-2.2595978417917664,-2.0041204173529321,-1.7119522052595035,-1.3877008727490241,-1.0364800570335411,-0.66382872018603634,-0.27562379639970791,0.12201249077495069,0.52280917955279083,0.92044546672744931,1.3086503905137752,1.6813017273612825,2.0325225430767651,2.3567738755872449,2.6489420876806733,2.9044195121195076,3.1191771173055303,3.2898280475149866,3.4136810356365457,3.4887828460613424,3.5139490783753313,3.5139490783753313],"y":[4.2440666205653086,4.6440724286311399,5.0377699061518859,5.4189502087235706,5.7816018952685644,6.1200057320524337,6.4288248884164467,6.7031891017851173,6.938771484616387,7.1318567620029132,7.2793998637767556,7.3790739470849642,7.4293070920913564,7.4293070920913564,7.3790739470849633,7.2793998637767556,7.1318567620029132,6.938771484616387,6.7031891017851173,6.4288248884164449,6.1200057320524346,5.7816018952685635,5.4189502087235706,5.0377699061518859,4.644072428631139,4.2440666205653077,3.8440608124994773,3.4503633349787304,3.0691830324070453,2.7065313458620528,2.3681275090781826,2.0593083527141709,1.7849441393454994,1.5493617565142288,1.3562764791277027,1.2087333773538615,1.109059294045653,1.0588261490392603,1.0588261490392603,1.1090592940456534,1.2087333773538611,1.3562764791277035,1.5493617565142297,1.7849441393455003,2.0593083527141727,2.3681275090781848,2.7065313458620528,3.0691830324070466,3.4503633349787322,3.8440608124994786,4.2440666205653077,4.2440666205653086],"text":"id: 4<br />as.factor(id): 4","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(216,144,0,0.6)","hoveron":"fills","name":"4","legendgroup":"4","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5310542537814351,3.5247626957029379,3.5059872430967385,3.4750239960663487,3.4323612635139851,3.3786718622174789,3.3148025061077702,3.2417604530844133,3.1606976199567933,3.0728924160279232,2.9797295818160463,2.882678350869464,2.7832692790757996,2.6830701068813401,2.5836610350876752,2.4866098041410933,2.3934469699292165,2.3056417660003459,2.2245789328727263,2.151536879849369,2.0876675237396607,2.0339781224431546,1.9913153898907907,1.9603521428604012,1.9415766902542018,1.9352851321757045,1.9415766902542018,1.9603521428604012,1.9913153898907909,2.0339781224431546,2.0876675237396607,2.1515368798493695,2.2245789328727263,2.3056417660003463,2.3934469699292169,2.4866098041410933,2.5836610350876752,2.6830701068813401,2.7832692790758,2.8826783508694644,2.9797295818160463,3.0728924160279227,3.1606976199567938,3.2417604530844137,3.3148025061077706,3.3786718622174794,3.4323612635139851,3.4750239960663487,3.5059872430967385,3.5247626957029379,3.5310542537814351,3.5310542537814351],"y":[1.0654291705472203,1.1654306225636781,1.2638549919438646,1.3591500675867858,1.4498129892230343,1.5344139484190018,1.6116187375100046,1.6802097908521725,1.7391053865599901,1.7873767059066217,1.8242624813500821,1.8491810021771342,1.8617392884287325,1.8617392884287325,1.8491810021771342,1.8242624813500821,1.7873767059066215,1.7391053865599901,1.6802097908521725,1.6116187375100046,1.5344139484190018,1.449812989223034,1.3591500675867856,1.2638549919438646,1.1654306225636781,1.0654291705472201,0.96542771853076248,0.86700334915057575,0.77170827350765447,0.68104535187140636,0.59644439267543881,0.51923960358443588,0.45064855024226802,0.39175295453445036,0.34348163518781882,0.30659585974435855,0.2816773389173064,0.26911905266570824,0.26911905266570824,0.28167733891730651,0.30659585974435843,0.34348163518781905,0.39175295453445058,0.45064855024226824,0.51923960358443633,0.59644439267543936,0.68104535187140636,0.77170827350765481,0.8670033491505762,0.96542771853076281,1.0654291705472201,1.0654291705472203],"text":"id: 5<br />as.factor(id): 5","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(201,152,0,0.6)","hoveron":"fills","name":"5","legendgroup":"5","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.7837295178853312,3.7774379598068339,3.7586625072006346,3.7276992601702448,3.6850365276178811,3.631347126321375,3.5674777702116662,3.4944357171883094,3.4133728840606894,3.3255676801318192,3.2324048459199424,3.1353536149733601,3.0359445431796956,2.9357453709852361,2.8363362991915713,2.7392850682449894,2.6461222340331125,2.5583170301042419,2.4772541969766224,2.4042121439532651,2.3403427878435568,2.2866533865470506,2.243990653994687,2.2130274069642972,2.1942519543580978,2.1879603962796006,2.1942519543580978,2.2130274069642972,2.243990653994687,2.2866533865470506,2.3403427878435568,2.4042121439532655,2.4772541969766224,2.5583170301042424,2.646122234033113,2.7392850682449894,2.8363362991915713,2.9357453709852361,3.0359445431796961,3.1353536149733605,3.2324048459199424,3.3255676801318188,3.4133728840606898,3.4944357171883098,3.5674777702116667,3.6313471263213755,3.6850365276178811,3.7276992601702448,3.7586625072006346,3.7774379598068339,3.7837295178853312,3.7837295178853312],"y":[-0.51020857347275317,-0.41020712145629534,-0.31178275207610884,-0.21648767643318761,-0.12582475479693928,-0.04122379560097178,0.035980993490031254,0.10457204683219901,0.16346764254001656,0.21173896188664809,0.24862473733010859,0.27354325815716074,0.2861015444087589,0.2861015444087589,0.27354325815716063,0.24862473733010859,0.21173896188664798,0.16346764254001656,0.10457204683219901,0.035980993490031032,-0.041223795600971724,-0.12582475479693939,-0.21648767643318778,-0.31178275207610884,-0.41020712145629545,-0.51020857347275339,-0.61021002548921111,-0.70863439486939761,-0.803929470512319,-0.89459239214856712,-0.97919335134453478,-1.0563981404355376,-1.1249891937777055,-1.1838847894855231,-1.2321561088321547,-1.269041884275615,-1.2939604051026672,-1.3065186913542652,-1.3065186913542652,-1.293960405102667,-1.269041884275615,-1.2321561088321544,-1.1838847894855229,-1.1249891937777052,-1.0563981404355371,-0.97919335134453411,-0.89459239214856701,-0.80392947051231867,-0.70863439486939728,-0.61021002548921066,-0.51020857347275339,-0.51020857347275317],"text":"id: 6<br />as.factor(id): 6","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(183,159,0,0.6)","hoveron":"fills","name":"6","legendgroup":"6","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.7616417968946236,2.7553502388161264,2.736574786209927,2.7056115391795372,2.6629488066271731,2.6092594053306675,2.5453900492209587,2.4723479961976014,2.3912851630699818,2.3034799591411113,2.2103171249292348,2.1132658939826525,2.0138568221889881,1.9136576499945281,1.8142485782008635,1.7171973472542816,1.624034513042405,1.5362293091135342,1.4551664759859146,1.3821244229625576,1.318255066852849,1.2645656655563431,1.221902933003979,1.1909396859735892,1.1721642333673898,1.1658726752888926,1.1721642333673898,1.1909396859735892,1.2219029330039792,1.2645656655563431,1.3182550668528492,1.3821244229625576,1.4551664759859149,1.5362293091135346,1.6240345130424054,1.7171973472542816,1.8142485782008637,1.9136576499945284,2.0138568221889885,2.113265893982653,2.2103171249292344,2.3034799591411113,2.3912851630699818,2.4723479961976018,2.5453900492209591,2.6092594053306675,2.6629488066271731,2.7056115391795372,2.736574786209927,2.7553502388161264,2.7616417968946236,2.7616417968946236],"y":[-1.735694509773267,-1.6356930577568092,-1.5372686883766227,-1.4419736127337015,-1.3513106910974531,-1.2667097319014857,-1.1895049428104825,-1.1209138894683148,-1.0620182937604974,-1.0137469744138659,-0.97686119897040524,-0.95194267814335309,-0.93938439189175493,-0.93938439189175493,-0.9519426781433532,-0.97686119897040524,-1.0137469744138659,-1.0620182937604974,-1.1209138894683148,-1.1895049428104829,-1.2667097319014855,-1.3513106910974533,-1.4419736127337015,-1.5372686883766227,-1.6356930577568094,-1.7356945097732672,-1.8356959617897248,-1.9341203311699116,-2.0294154068128329,-2.1200783284490807,-2.2046792876450487,-2.2818840767360515,-2.3504751300782192,-2.4093707257860371,-2.4576420451326686,-2.4945278205761285,-2.5194463414031807,-2.532004627654779,-2.532004627654779,-2.5194463414031807,-2.494527820576129,-2.4576420451326682,-2.4093707257860366,-2.3504751300782192,-2.2818840767360511,-2.2046792876450478,-2.1200783284490807,-2.0294154068128325,-1.9341203311699111,-1.8356959617897244,-1.7356945097732672,-1.735694509773267],"text":"id: 7<br />as.factor(id): 7","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(163,165,0,0.6)","hoveron":"fills","name":"7","legendgroup":"7","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.7386794362094156,1.7323878781309183,1.7136124255247189,1.6826491784943292,1.6399864459419651,1.5862970446454592,1.5224276885357506,1.4493856355123933,1.3683228023847738,1.2805175984559032,1.1873547642440265,1.0903035332974444,0.99089446150377991,0.89069528930932007,0.79128621751565542,0.69423498656907345,0.60107215235719691,0.5132669484283261,0.43220411530070657,0.35916206227734937,0.29529270616764092,0.24160330487113491,0.19894057231877094,0.16797732528838127,0.14920187268218188,0.14291031460368464,0.14920187268218188,0.16797732528838127,0.19894057231877116,0.24160330487113491,0.29529270616764103,0.35916206227734948,0.43220411530070668,0.51326694842832654,0.60107215235719735,0.69423498656907345,0.79128621751565564,0.89069528930932029,0.99089446150378024,1.0903035332974449,1.1873547642440263,1.2805175984559032,1.3683228023847738,1.4493856355123937,1.5224276885357511,1.5862970446454594,1.6399864459419651,1.6826491784943292,1.7136124255247189,1.7323878781309183,1.7386794362094156,1.7386794362094156],"y":[-2.9604504438644934,-2.8604489918480356,-2.7620246224678491,-2.6667295468249277,-2.5760666251886795,-2.4914656659927119,-2.4142608769017091,-2.3456698235595415,-2.2867742278517236,-2.238502908505092,-2.2016171330616316,-2.1766986122345795,-2.1641403259829812,-2.1641403259829812,-2.1766986122345795,-2.2016171330616316,-2.238502908505092,-2.2867742278517236,-2.3456698235595415,-2.4142608769017091,-2.4914656659927119,-2.5760666251886795,-2.6667295468249281,-2.7620246224678491,-2.8604489918480356,-2.9604504438644939,-3.0604518958809512,-3.1588762652611377,-3.2541713409040591,-3.3448342625403074,-3.4294352217362749,-3.5066400108272777,-3.5752310641694458,-3.6341266598772632,-3.6823979792238948,-3.7192837546673552,-3.7442022754944073,-3.7567605617460056,-3.7567605617460056,-3.7442022754944073,-3.7192837546673552,-3.6823979792238948,-3.6341266598772632,-3.5752310641694454,-3.5066400108272773,-3.4294352217362745,-3.3448342625403074,-3.2541713409040591,-3.1588762652611373,-3.0604518958809508,-2.9604504438644934,-2.9604504438644934],"text":"id: 8<br />as.factor(id): 8","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(138,171,0,0.6)","hoveron":"fills","name":"8","legendgroup":"8","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-0.50375537361170553,-0.51265298037477436,-0.53920548009015468,-0.58299412397564043,-0.64332833895908947,-0.71925661842829991,-0.80958152805868666,-0.91287859006789318,-1.0275187480813601,-1.1516940583245039,-1.2834462019760562,-1.4206973690257161,-1.561283026579237,-1.7029860548351992,-1.8435717123887205,-1.9808228794383802,-2.1125750230899323,-2.2367503333330765,-2.3513904913465433,-2.4546875533557495,-2.5450124629861364,-2.6209407424553466,-2.681274957438796,-2.7250636013242815,-2.7516161010396618,-2.7605137078027306,-2.7516161010396618,-2.7250636013242815,-2.6812749574387955,-2.6209407424553466,-2.5450124629861359,-2.4546875533557495,-2.3513904913465429,-2.236750333333076,-2.1125750230899314,-1.9808228794383802,-1.8435717123887203,-1.7029860548351989,-1.5612830265792366,-1.4206973690257154,-1.2834462019760564,-1.1516940583245041,-1.0275187480813597,-0.91287859006789274,-0.80958152805868633,-0.71925661842829958,-0.64332833895908947,-0.58299412397564021,-0.53920548009015468,-0.51265298037477436,-0.50375537361170553,-0.50375537361170553],"y":[-2.4978604985898909,-2.3564370888912141,-2.217244010845735,-2.0824764224441457,-1.9542596890617943,-1.8346158651770843,-1.7254318053644335,-1.6284294074704833,-1.5451384572564453,-1.476872502762796,-1.4247081388720058,-1.3894680287641121,-1.3717079300269386,-1.3717079300269386,-1.3894680287641124,-1.4247081388720058,-1.4768725027627962,-1.5451384572564453,-1.6284294074704833,-1.7254318053644337,-1.8346158651770841,-1.9542596890617945,-2.0824764224441461,-2.217244010845735,-2.3564370888912141,-2.4978604985898913,-2.6392839082885677,-2.7784769863340468,-2.9132445747356366,-3.0414613081179875,-3.1611051320026977,-3.2702891918153485,-3.3672915897092985,-3.450582539923337,-3.5188484944169858,-3.571012858307776,-3.6062529684156699,-3.624013067152843,-3.624013067152843,-3.6062529684156694,-3.571012858307776,-3.5188484944169858,-3.4505825399233365,-3.3672915897092981,-3.2702891918153476,-3.1611051320026968,-3.0414613081179875,-2.9132445747356357,-2.7784769863340464,-2.6392839082885673,-2.4978604985898913,-2.4978604985898909],"text":"id: 9<br />as.factor(id): 9","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(107,177,0,0.6)","hoveron":"fills","name":"9","legendgroup":"9","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.3381645541442726,-2.3444561122227698,-2.3632315648289692,-2.394194811859359,-2.4368575444117226,-2.4905469457082288,-2.5544163018179376,-2.6274583548412944,-2.7085211879689144,-2.7963263918977845,-2.8894892261096614,-2.9865404570562437,-3.0859495288499081,-3.1861487010443676,-3.2855577728380325,-3.3826090037846144,-3.4757718379964913,-3.5635770419253618,-3.6446398750529814,-3.7176819280763387,-3.781551284186047,-3.8352406854825531,-3.8779034180349168,-3.9088666650653066,-3.927642117671506,-3.9339336757500032,-3.927642117671506,-3.9088666650653066,-3.8779034180349168,-3.8352406854825531,-3.781551284186047,-3.7176819280763382,-3.6446398750529814,-3.5635770419253614,-3.4757718379964908,-3.3826090037846144,-3.2855577728380325,-3.1861487010443676,-3.0859495288499077,-2.9865404570562433,-2.8894892261096614,-2.796326391897785,-2.7085211879689139,-2.627458354841294,-2.5544163018179371,-2.4905469457082283,-2.4368575444117226,-2.394194811859359,-2.3632315648289692,-2.3444561122227698,-2.3381645541442726,-2.3381645541442726],"y":[-1.2942272891752791,-1.1942258371588212,-1.0958014677786347,-1.0005063921357136,-0.90984347049946512,-0.82524251130349768,-0.74803772221249465,-0.6794466688703269,-0.62055107316250935,-0.57227975381587781,-0.53539397837241731,-0.51047545754536516,-0.497917171293767,-0.497917171293767,-0.51047545754536527,-0.53539397837241731,-0.57227975381587792,-0.62055107316250935,-0.6794466688703269,-0.74803772221249487,-0.82524251130349757,-0.90984347049946535,-1.0005063921357138,-1.0958014677786347,-1.1942258371588212,-1.2942272891752793,-1.3942287411917369,-1.4926531105719236,-1.5879481862148448,-1.678611107851093,-1.7632120670470606,-1.8404168561380634,-1.9090079094802315,-1.9679035051880489,-2.0161748245346804,-2.0530605999781408,-2.077979120805193,-2.0905374070567913,-2.0905374070567913,-2.077979120805193,-2.0530605999781408,-2.0161748245346804,-1.9679035051880489,-1.909007909480231,-1.8404168561380629,-1.7632120670470601,-1.678611107851093,-1.5879481862148446,-1.4926531105719232,-1.3942287411917365,-1.2942272891752793,-1.2942272891752791],"text":"id: 10<br />as.factor(id): 10","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(57,182,0,0.6)","hoveron":"fills","name":"10","legendgroup":"10","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.7498546304827549,-2.7561461885612522,-2.7749216411674515,-2.8058848881978413,-2.848547620750205,-2.9022370220467111,-2.9661063781564199,-3.0391484311797767,-3.1202112643073967,-3.2080164682362668,-3.3011793024481437,-3.398230533394726,-3.4976396051883905,-3.59783877738285,-3.6972478491765148,-3.7942990801230967,-3.8874619143349736,-3.9752671182638442,-4.0563299513914632,-4.1293720044148206,-4.1932413605245298,-4.2469307618210355,-4.2895934943733991,-4.3205567414037889,-4.3393321940099883,-4.3456237520884855,-4.3393321940099883,-4.3205567414037889,-4.2895934943733991,-4.2469307618210355,-4.1932413605245289,-4.1293720044148206,-4.0563299513914632,-3.9752671182638437,-3.8874619143349731,-3.7942990801230967,-3.6972478491765148,-3.59783877738285,-3.49763960518839,-3.3982305333947256,-3.3011793024481437,-3.2080164682362673,-3.1202112643073963,-3.0391484311797763,-2.9661063781564194,-2.9022370220467106,-2.848547620750205,-2.8058848881978413,-2.7749216411674515,-2.7561461885612522,-2.7498546304827549,-2.7498546304827549],"y":[0.24752183779536407,0.3475232898118219,0.44594765919200841,0.54124273483492957,0.63190565647117802,0.71650661566714546,0.7937114047581485,0.86230245810031625,0.9211980538081338,0.96946937315476533,1.0063551485982258,1.031273669425278,1.0438319556768763,1.0438319556768763,1.031273669425278,1.0063551485982258,0.96946937315476522,0.9211980538081338,0.86230245810031625,0.79371140475814828,0.71650661566714557,0.6319056564711778,0.54124273483492946,0.44594765919200841,0.34752328981182179,0.24752183779536382,0.14752038577890619,0.049096016398719577,-0.046199059244201757,-0.13686198088044987,-0.22146294007641748,-0.29866772916742035,-0.36725878250958821,-0.42615437821740587,-0.47442569756403741,-0.51131147300749769,-0.53622999383454983,-0.54878828008614799,-0.54878828008614799,-0.53622999383454972,-0.5113114730074978,-0.47442569756403719,-0.42615437821740565,-0.36725878250958799,-0.2986677291674199,-0.22146294007641687,-0.13686198088044982,-0.046199059244201424,0.049096016398719994,0.14752038577890658,0.24752183779536388,0.24752183779536407],"text":"id: 11<br />as.factor(id): 11","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,186,56,0.6)","hoveron":"fills","name":"11","legendgroup":"11","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.5637675519716572,-2.5763506681286517,-2.6139015733410504,-2.67582806740183,-2.7611535325065573,-2.8685323350995695,-2.9962710473189866,-3.1423551533657008,-3.3044808196209403,-3.4800912274786815,-3.6664168959024348,-3.860519357795599,-4.0593375013829283,-4.2597358457718473,-4.458553989359177,-4.6526564512523407,-4.8389821196760945,-5.0145925275338357,-5.1767181937890747,-5.3228022998357893,-5.450541012055206,-5.5579198146482183,-5.6432452797529464,-5.7051717738137251,-5.7427226790261239,-5.7553057951831184,-5.7427226790261239,-5.7051717738137251,-5.6432452797529455,-5.5579198146482183,-5.450541012055206,-5.3228022998357893,-5.1767181937890747,-5.0145925275338348,-4.8389821196760936,-4.6526564512523407,-4.458553989359177,-4.2597358457718473,-4.0593375013829274,-3.8605193577955981,-3.6664168959024352,-3.4800912274786815,-3.3044808196209399,-3.1423551533657004,-2.9962710473189862,-2.8685323350995686,-2.7611535325065573,-2.6758280674018295,-2.61390157334105,-2.5763506681286517,-2.5637675519716572,-2.5637675519716572],"y":[2.5616700162769597,2.7616729203098753,2.9585216590702483,3.1491118103560907,3.3304376536285876,3.4996395720205227,3.6540491502025283,3.791231256886864,3.9090224483024993,4.0055650869957624,4.0793366378826832,4.1291736795367875,4.154290252039984,4.154290252039984,4.1291736795367875,4.0793366378826832,4.0055650869957624,3.9090224483024993,3.791231256886864,3.6540491502025283,3.4996395720205227,3.3304376536285871,3.1491118103560902,2.9585216590702483,2.7616729203098753,2.5616700162769592,2.361667112244044,2.1648183734836706,1.974228222197828,1.7929023789253318,1.6237004605333967,1.4692908823513908,1.3321087756670551,1.2143175842514198,1.1177749455581567,1.0440033946712362,0.99416635301713185,0.96904978051393553,0.96904978051393553,0.99416635301713208,1.0440033946712359,1.1177749455581572,1.2143175842514202,1.3321087756670555,1.4692908823513917,1.6237004605333978,1.7929023789253318,1.9742282221978287,2.1648183734836715,2.3616671122440449,2.5616700162769592,2.5616700162769597],"text":"id: 12<br />as.factor(id): 12","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,189,95,0.6)","hoveron":"fills","name":"12","legendgroup":"12","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.7951990436603631,1.7774038301342254,1.7242988307034648,1.6367215429324933,1.5160531129655954,1.3641965540271745,1.1835467347664008,0.97695261074798789,0.74767229472105423,0.49932167423476626,0.23581738693166188,-0.038684947167657802,-0.31985626227469977,-0.60326231878662429,-0.88443363389366669,-1.1589359679929858,-1.4224402552960902,-1.6707908757823788,-1.9000711918093121,-2.1066653158277249,-2.2873151350884982,-2.439171694026919,-2.5598401239938178,-2.6474174117647888,-2.7005224111955495,-2.7183176247216871,-2.7005224111955495,-2.6474174117647888,-2.5598401239938169,-2.439171694026919,-2.2873151350884977,-2.1066653158277249,-1.9000711918093116,-1.6707908757823775,-1.4224402552960891,-1.1589359679929858,-0.88443363389366625,-0.60326231878662384,-0.31985626227469888,-0.038684947167656414,0.23581738693166132,0.49932167423476614,0.74767229472105468,0.97695261074798878,1.1835467347664017,1.364196554027175,1.5160531129655954,1.6367215429324937,1.7242988307034648,1.7774038301342254,1.7951990436603631,1.7951990436603631],"y":[-5.6741648741638935,-5.3913180547665398,-5.1129318986755816,-4.843396721872403,-4.5869632551077002,-4.3476756073382798,-4.1293074877129783,-3.9353026919250782,-3.7687207914970022,-3.6321888825097037,-3.5278601547281232,-3.4573799345123359,-3.4218597370379888,-3.4218597370379888,-3.4573799345123364,-3.5278601547281232,-3.6321888825097042,-3.7687207914970022,-3.9353026919250782,-4.1293074877129792,-4.3476756073382798,-4.5869632551077011,-4.8433967218724039,-5.1129318986755816,-5.3913180547665398,-5.6741648741638944,-5.9570116935612472,-6.2353978496522053,-6.5049330264553848,-6.7613664932200868,-7.0006541409895071,-7.2190222606148087,-7.4130270564027088,-7.5796089568307856,-7.7161408658180832,-7.8204695935996638,-7.8909498138154515,-7.9264700112897977,-7.9264700112897977,-7.8909498138154506,-7.8204695935996638,-7.7161408658180832,-7.5796089568307847,-7.4130270564027079,-7.2190222606148069,-7.0006541409895053,-6.7613664932200868,-6.5049330264553831,-6.2353978496522044,-5.9570116935612463,-5.6741648741638944,-5.6741648741638935],"text":"id: 13<br />as.factor(id): 13","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,191,125,0.6)","hoveron":"fills","name":"13","legendgroup":"13","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.4146520322795908,5.4037547340288636,5.3712346961798243,5.3176047791558831,5.2437107587854666,5.150717987911964,5.0400930180832386,4.9135804711576441,4.7731755255751294,4.6210924512013793,4.4597296889692926,4.291632026032711,4.1194504629528224,3.9459004058356753,3.7737188427557857,3.605621179819205,3.4442584175871183,3.2921753432133682,3.1517703976308535,3.025257850705259,2.9146328808765345,2.8216401100030311,2.7477460896326145,2.6941161726086733,2.6615961347596344,2.6506988365089068,2.6615961347596344,2.6941161726086733,2.7477460896326145,2.8216401100030315,2.9146328808765345,3.025257850705259,3.1517703976308535,3.2921753432133687,3.4442584175871191,3.605621179819205,3.7737188427557862,3.9459004058356753,4.1194504629528232,4.2916320260327119,4.4597296889692926,4.6210924512013793,4.7731755255751294,4.9135804711576441,5.0400930180832386,5.150717987911964,5.2437107587854666,5.3176047791558831,5.3712346961798243,5.4037547340288636,5.4146520322795908,5.4146520322795908],"y":[-2.4222599909112805,-2.2490523951881145,-2.078576386718705,-1.9135204739940459,-1.7564876873574282,-1.609954527660951,-1.4762319103676944,-1.3574287210343927,-1.2554185569264174,-1.1718101792696689,-1.1079221421250189,-1.0647619980031013,-1.0430104081593397,-1.0430104081593397,-1.0647619980031016,-1.1079221421250189,-1.1718101792696691,-1.2554185569264174,-1.3574287210343927,-1.4762319103676949,-1.609954527660951,-1.7564876873574282,-1.9135204739940463,-2.078576386718705,-2.2490523951881145,-2.422259990911281,-2.5954675866344465,-2.7659435951038565,-2.9309995078285156,-3.0880322944651328,-3.2345654541616105,-3.3682880714548666,-3.4870912607881683,-3.5891014248961444,-3.6727098025528924,-3.7365978396975423,-3.7797579838194597,-3.8015095736632212,-3.8015095736632212,-3.7797579838194597,-3.7365978396975423,-3.6727098025528919,-3.5891014248961435,-3.4870912607881683,-3.3682880714548658,-3.2345654541616091,-3.0880322944651328,-2.9309995078285151,-2.7659435951038556,-2.5954675866344461,-2.422259990911281,-2.4222599909112805],"text":"id: 14<br />as.factor(id): 14","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,192,151,0.6)","hoveron":"fills","name":"14","legendgroup":"14","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.8357913265194448,2.8313425231379106,2.8180662732802202,2.7961719513374774,2.766004843845753,2.7280407041111476,2.6828782492959542,2.6312297182913511,2.5739096392846177,2.5118219841630456,2.4459459123372698,2.3773203288124396,2.3070275000356792,2.2361759859076979,2.1658831571309376,2.0972575736061074,2.0313815017803316,1.9692938466587593,1.9119737676520261,1.8603252366474228,1.8151627818322296,1.7771986420976242,1.7470315346058998,1.725137212663157,1.7118609628054666,1.7074121594239324,1.7118609628054666,1.725137212663157,1.7470315346058998,1.7771986420976242,1.8151627818322296,1.860325236647423,1.9119737676520261,1.9692938466587597,2.0313815017803316,2.0972575736061074,2.1658831571309376,2.2361759859076979,2.3070275000356792,2.37732032881244,2.4459459123372693,2.5118219841630456,2.5739096392846177,2.6312297182913511,2.6828782492959546,2.7280407041111481,2.766004843845753,2.7961719513374774,2.8180662732802202,2.8313425231379106,2.8357913265194448,2.8357913265194448],"y":[-3.2506211797251967,-3.1799094748758581,-3.110312935853119,-3.0429291416523241,-2.9788207749611484,-2.9189988630187935,-2.8644068331124681,-2.8159056341654929,-2.7742601590584739,-2.7401271818116495,-2.7140449998662541,-2.6964249448123074,-2.6875448954437204,-2.6875448954437204,-2.6964249448123074,-2.7140449998662541,-2.7401271818116495,-2.7742601590584739,-2.8159056341654929,-2.8644068331124681,-2.9189988630187935,-2.9788207749611484,-3.0429291416523245,-3.110312935853119,-3.1799094748758585,-3.2506211797251967,-3.3213328845745353,-3.3909294235972749,-3.4583132177980693,-3.522421584489245,-3.5822434964315999,-3.6368355263379253,-3.6853367252849005,-3.7269822003919195,-3.7611151776387444,-3.7871973595841393,-3.804817414638086,-3.813697464006673,-3.813697464006673,-3.804817414638086,-3.7871973595841393,-3.7611151776387439,-3.7269822003919195,-3.6853367252849005,-3.6368355263379253,-3.5822434964315999,-3.522421584489245,-3.4583132177980693,-3.3909294235972745,-3.3213328845745349,-3.2506211797251967,-3.2506211797251967],"text":"id: 15<br />as.factor(id): 15","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,192,175,0.6)","hoveron":"fills","name":"15","legendgroup":"15","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[10.238248675584332,10.210824409722747,10.128984109193086,9.9940184444237303,9.8080559045726741,9.5740292299820027,9.2956291611107709,8.9772462333532168,8.6239015356728217,8.2411675250298941,7.83508014540649,7.4120436373641123,6.9787295393443491,6.5419714735222678,6.1086573755025038,5.6856208674601261,5.2795334878367219,4.8967994771937944,4.5434547795133993,4.2250718517558461,3.9466717828846147,3.7126451082939433,3.5266825684428857,3.3917169036735313,3.3098766031438709,3.2824523372822845,3.3098766031438709,3.3917169036735313,3.5266825684428866,3.7126451082939433,3.9466717828846152,4.2250718517558461,4.5434547795134002,4.8967994771937962,5.2795334878367237,5.6856208674601261,6.1086573755025046,6.5419714735222687,6.97872953934435,7.4120436373641141,7.83508014540649,8.2411675250298941,8.6239015356728217,8.9772462333532186,9.2956291611107709,9.5740292299820027,9.8080559045726741,9.9940184444237303,10.128984109193086,10.210824409722747,10.238248675584332,10.238248675584332],"y":[2.5021381353393122,2.9380343588863838,3.3670562385963354,3.7824378431408934,4.1776283564794472,4.5463953881412777,4.8829232617463312,5.1819047316958482,5.4386246816058526,5.6490344845091993,5.8098158521213783,5.9184331662290113,5.9731734669037841,5.9731734669037841,5.9184331662290113,5.8098158521213783,5.6490344845091993,5.4386246816058517,5.1819047316958482,4.8829232617463294,4.5463953881412777,4.1776283564794463,3.7824378431408929,3.3670562385963354,2.9380343588863829,2.5021381353393108,2.0662419117922406,1.6372200320822881,1.2218384275377299,0.82664791419917716,0.45788088253734616,0.12135300893229362,-0.17762846101722429,-0.43434841092722909,-0.64475821383057541,-0.80553958144275351,-0.91415689555038737,-0.9688971962251598,-0.9688971962251598,-0.91415689555038693,-0.80553958144275351,-0.64475821383057497,-0.43434841092722776,-0.17762846101722296,0.12135300893229495,0.45788088253734927,0.82664791419917738,1.2218384275377314,1.6372200320822901,2.0662419117922424,2.5021381353393113,2.5021381353393122],"text":"id: 16<br />as.factor(id): 16","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,191,196,0.6)","hoveron":"fills","name":"16","legendgroup":"16","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.894838288967307,6.8885467308888098,6.8697712782826104,6.8388080312522206,6.796145298699857,6.7424558974033513,6.678586541293642,6.6055444882702847,6.5244816551426652,6.4366764512137946,6.3435136170019177,6.2464623860553363,6.1470533142616715,6.0468541420672119,5.9474450702735471,5.8503938393269648,5.7572310051150888,5.6694258011862182,5.5883629680585987,5.5153209150352414,5.4514515589255321,5.3977621576290264,5.3550994250766628,5.324136178046273,5.3053607254400736,5.2990691673615764,5.3053607254400736,5.324136178046273,5.3550994250766628,5.3977621576290264,5.451451558925533,5.5153209150352414,5.5883629680585987,5.6694258011862182,5.7572310051150888,5.8503938393269648,5.9474450702735471,6.0468541420672119,6.1470533142616723,6.2464623860553363,6.3435136170019177,6.4366764512137946,6.5244816551426652,6.6055444882702856,6.6785865412936429,6.7424558974033513,6.796145298699857,6.8388080312522206,6.8697712782826104,6.8885467308888098,6.894838288967307,6.894838288967307],"y":[-1.7218673888467571,-1.6218659368302992,-1.5234415674501127,-1.4281464918071916,-1.3374835701709431,-1.2528826109749756,-1.1756778218839727,-1.1070867685418049,-1.0481911728339872,-0.99991985348735579,-0.96303407804389529,-0.93811555721684314,-0.92555727096524498,-0.92555727096524498,-0.93811555721684325,-0.96303407804389529,-0.9999198534873559,-1.0481911728339872,-1.1070867685418049,-1.1756778218839727,-1.2528826109749756,-1.3374835701709433,-1.4281464918071918,-1.5234415674501127,-1.6218659368302992,-1.7218673888467573,-1.8218688408632149,-1.9202932102434016,-2.0155882858863228,-2.106251207522571,-2.1908521667185386,-2.2680569558095414,-2.3366480091517094,-2.3955436048595269,-2.4438149242061584,-2.4807006996496188,-2.505619220476671,-2.5181775067282692,-2.5181775067282692,-2.505619220476671,-2.4807006996496188,-2.4438149242061584,-2.3955436048595269,-2.336648009151709,-2.2680569558095409,-2.1908521667185381,-2.106251207522571,-2.0155882858863228,-1.9202932102434012,-1.8218688408632144,-1.7218673888467573,-1.7218673888467571],"text":"id: 17<br />as.factor(id): 17","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,188,216,0.6)","hoveron":"fills","name":"17","legendgroup":"17","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.15218839453666,-2.1584799526151572,-2.1772554052213566,-2.2082186522517464,-2.25088138480411,-2.3045707861006162,-2.368440142210325,-2.4414821952336818,-2.5225450283613018,-2.6103502322901719,-2.7035130665020488,-2.8005642974486311,-2.8999733692422955,-3.000172541436755,-3.0995816132304199,-3.1966328441770018,-3.2897956783888787,-3.3776008823177492,-3.4586637154453688,-3.5317057684687261,-3.5955751245784344,-3.6492645258749405,-3.6919272584273042,-3.722890505457694,-3.7416659580638933,-3.7479575161423906,-3.7416659580638933,-3.722890505457694,-3.6919272584273042,-3.6492645258749405,-3.5955751245784344,-3.5317057684687256,-3.4586637154453688,-3.3776008823177488,-3.2897956783888782,-3.1966328441770018,-3.0995816132304199,-3.000172541436755,-2.8999733692422951,-2.8005642974486307,-2.7035130665020488,-2.6103502322901724,-2.5225450283613013,-2.4414821952336814,-2.3684401422103245,-2.3045707861006157,-2.25088138480411,-2.2082186522517464,-2.1772554052213566,-2.1584799526151572,-2.15218839453666,-2.15218839453666],"y":[-3.9026844322746204,-3.8026829802581625,-3.704258610877976,-3.6089635352350546,-3.5183006135988064,-3.4336996544028389,-3.3564948653118361,-3.2879038119696684,-3.2290082162618505,-3.180736896915219,-3.1438511214717586,-3.1189326006447065,-3.1063743143931082,-3.1063743143931082,-3.1189326006447065,-3.1438511214717586,-3.180736896915219,-3.2290082162618505,-3.2879038119696684,-3.3564948653118361,-3.4336996544028389,-3.5183006135988064,-3.6089635352350551,-3.704258610877976,-3.8026829802581625,-3.9026844322746208,-4.0026858842910782,-4.1011102536712647,-4.1964053293141861,-4.2870682509504343,-4.3716692101464023,-4.4488739992374047,-4.5174650525795723,-4.5763606482873902,-4.6246319676340217,-4.6615177430774821,-4.6864362639045343,-4.6989945501561321,-4.6989945501561321,-4.6864362639045343,-4.6615177430774821,-4.6246319676340217,-4.5763606482873902,-4.5174650525795723,-4.4488739992374047,-4.3716692101464014,-4.2870682509504343,-4.1964053293141861,-4.1011102536712647,-4.0026858842910782,-3.9026844322746204,-3.9026844322746204],"text":"id: 18<br />as.factor(id): 18","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,183,233,0.6)","hoveron":"fills","name":"18","legendgroup":"18","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-3.1576389844112556,-3.1639305424897528,-3.1827059950959522,-3.213669242126342,-3.2563319746787056,-3.3100213759752117,-3.3738907320849201,-3.4469327851082774,-3.5279956182358974,-3.6158008221647675,-3.7089636563766444,-3.8060148873232267,-3.9054239591168911,-4.0056231313113511,-4.105032203105015,-4.2020834340515973,-4.2952462682634742,-4.3830514721923448,-4.4641143053199643,-4.5371563583433216,-4.60102571445303,-4.6547151157495357,-4.6973778483019002,-4.72834109533229,-4.7471165479384894,-4.7534081060169866,-4.7471165479384894,-4.72834109533229,-4.6973778483018993,-4.6547151157495357,-4.60102571445303,-4.5371563583433216,-4.4641143053199643,-4.3830514721923439,-4.2952462682634733,-4.2020834340515973,-4.105032203105015,-4.0056231313113511,-3.9054239591168907,-3.8060148873232258,-3.7089636563766444,-3.6158008221647675,-3.5279956182358969,-3.4469327851082774,-3.3738907320849201,-3.3100213759752113,-3.2563319746787056,-3.2136692421263415,-3.1827059950959518,-3.1639305424897528,-3.1576389844112556,-3.1576389844112556],"y":[-2.6635120468531439,-2.5635105948366861,-2.4650862254564996,-2.3697911498135782,-2.27912822817733,-2.1945272689813624,-2.1173224798903596,-2.0487314265481915,-1.9898358308403741,-1.9415645114937425,-1.9046787360502822,-1.87976021522323,-1.8672019289716317,-1.8672019289716317,-1.87976021522323,-1.9046787360502822,-1.9415645114937428,-1.9898358308403741,-2.0487314265481915,-2.1173224798903596,-2.1945272689813624,-2.27912822817733,-2.3697911498135786,-2.4650862254564996,-2.5635105948366861,-2.6635120468531444,-2.7635134988696017,-2.8619378682497882,-2.9572329438927096,-3.0478958655289579,-3.1324968247249254,-3.2097016138159282,-3.2782926671580963,-3.3371882628659137,-3.3854595822125453,-3.4223453576560057,-3.4472638784830578,-3.4598221647346561,-3.4598221647346561,-3.4472638784830578,-3.4223453576560057,-3.3854595822125453,-3.3371882628659137,-3.2782926671580959,-3.2097016138159278,-3.132496824724925,-3.0478958655289579,-2.9572329438927092,-2.8619378682497878,-2.7635134988696013,-2.6635120468531439,-2.6635120468531439],"text":"id: 19<br />as.factor(id): 19","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,176,246,0.6)","hoveron":"fills","name":"19","legendgroup":"19","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.1983386347352467,3.1920470766567495,3.1732716240505501,3.1423083770201603,3.0996456444677967,3.0459562431712905,2.9820868870615822,2.9090448340382249,2.8279820009106049,2.7401767969817348,2.6470139627698579,2.5499627318232756,2.4505536600296112,2.3503544878351517,2.2509454160414868,2.1538941850949049,2.0607313508830281,1.9729261469541575,1.8918633138265379,1.8188212608031806,1.7549519046934723,1.7012625033969662,1.6585997708446023,1.6276365238142128,1.6088610712080134,1.6025695131295161,1.6088610712080134,1.6276365238142128,1.6585997708446025,1.7012625033969662,1.7549519046934723,1.8188212608031809,1.8918633138265379,1.9729261469541579,2.0607313508830285,2.1538941850949049,2.2509454160414868,2.3503544878351517,2.4505536600296116,2.5499627318232765,2.6470139627698579,2.7401767969817348,2.8279820009106054,2.9090448340382249,2.9820868870615822,3.045956243171291,3.0996456444677967,3.1423083770201607,3.1732716240505505,3.1920470766567495,3.1983386347352467,3.1983386347352467],"y":[-4.6065869062516729,-4.5065854542352151,-4.4081610848550286,-4.3128660092121072,-4.2222030875758589,-4.1376021283798918,-4.0603973392888886,-3.9918062859467209,-3.9329106902389031,-3.8846393708922715,-3.8477535954488111,-3.822835074621759,-3.8102767883701607,-3.8102767883701607,-3.822835074621759,-3.8477535954488111,-3.8846393708922715,-3.9329106902389031,-3.9918062859467209,-4.0603973392888886,-4.1376021283798918,-4.2222030875758589,-4.3128660092121072,-4.4081610848550286,-4.5065854542352151,-4.6065869062516729,-4.7065883582681307,-4.8050127276483172,-4.9003078032912386,-4.9909707249274868,-5.0755716841234548,-5.1527764732144572,-5.2213675265566248,-5.2802631222644427,-5.3285344416110743,-5.3654202170545346,-5.3903387378815868,-5.4028970241331846,-5.4028970241331846,-5.3903387378815868,-5.3654202170545346,-5.3285344416110743,-5.2802631222644427,-5.2213675265566248,-5.1527764732144572,-5.0755716841234539,-4.9909707249274868,-4.9003078032912386,-4.8050127276483172,-4.7065883582681307,-4.6065869062516729,-4.6065869062516729],"text":"id: 20<br />as.factor(id): 20","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,167,255,0.6)","hoveron":"fills","name":"20","legendgroup":"20","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-3.933737154441614,-3.9400287125201112,-3.9588041651263106,-3.9897674121567004,-4.032430144709064,-4.0861195460055697,-4.1499889021152789,-4.2230309551386362,-4.3040937882662558,-4.3918989921951264,-4.4850618264070032,-4.5821130573535847,-4.6815221291472495,-4.781721301341709,-4.8811303731353739,-4.9781816040819562,-5.0713444382938322,-5.1591496422227028,-5.2402124753503223,-5.3132545283736796,-5.3771238844833888,-5.4308132857798945,-5.4734760183322582,-5.5044392653626479,-5.5232147179688473,-5.5295062760473446,-5.5232147179688473,-5.5044392653626479,-5.4734760183322582,-5.4308132857798945,-5.377123884483388,-5.3132545283736796,-5.2402124753503223,-5.1591496422227028,-5.0713444382938322,-4.9781816040819562,-4.8811303731353739,-4.781721301341709,-4.6815221291472486,-4.5821130573535847,-4.4850618264070032,-4.3918989921951264,-4.3040937882662558,-4.2230309551386354,-4.149988902115278,-4.0861195460055697,-4.032430144709064,-3.9897674121567004,-3.9588041651263106,-3.9400287125201112,-3.933737154441614,-3.933737154441614],"y":[-1.2691839936512248,-1.169182541634767,-1.0707581722545805,-0.97546309661165931,-0.88480017497541086,-0.80019921577944342,-0.72299442668844038,-0.65440337334627263,-0.59550777763845508,-0.54723645829182355,-0.51035068284836305,-0.4854321620213109,-0.47287387576971274,-0.47287387576971274,-0.48543216202131101,-0.51035068284836305,-0.54723645829182366,-0.59550777763845508,-0.65440337334627263,-0.72299442668844061,-0.80019921577944331,-0.88480017497541108,-0.97546309661165942,-1.0707581722545805,-1.169182541634767,-1.269183993651225,-1.3691854456676826,-1.4676098150478694,-1.5629048906907905,-1.6535678123270388,-1.7381687715230063,-1.8153735606140091,-1.8839646139561772,-1.9428602096639946,-1.9911315290106262,-2.0280173044540866,-2.0529358252811387,-2.065494111532737,-2.065494111532737,-2.0529358252811387,-2.0280173044540866,-1.9911315290106262,-1.9428602096639946,-1.8839646139561768,-1.8153735606140087,-1.7381687715230059,-1.6535678123270388,-1.5629048906907903,-1.4676098150478689,-1.3691854456676822,-1.269183993651225,-1.2691839936512248],"text":"id: 21<br />as.factor(id): 21","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(97,156,255,0.6)","hoveron":"fills","name":"21","legendgroup":"21","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-4.6333543120143768,-4.6422519187774451,-4.6688044184928259,-4.7125930623783114,-4.7729272773617604,-4.848855556830971,-4.9391804664613579,-5.0424775284705641,-5.1571176864840309,-5.2812929967271751,-5.4130451403787276,-5.5502963074283871,-5.6908819649819078,-5.8325849932378704,-5.9731706507913911,-6.1104218178410514,-6.2421739614926031,-6.3663492717357473,-6.4809894297492141,-6.5842864917584203,-6.6746114013888072,-6.7505396808580178,-6.8108738958414667,-6.8546625397269523,-6.881215039442333,-6.8901126462054014,-6.881215039442333,-6.8546625397269523,-6.8108738958414667,-6.7505396808580178,-6.6746114013888072,-6.5842864917584203,-6.4809894297492141,-6.3663492717357473,-6.2421739614926022,-6.1104218178410514,-5.9731706507913911,-5.8325849932378704,-5.6908819649819078,-5.5502963074283862,-5.4130451403787276,-5.2812929967271751,-5.1571176864840309,-5.0424775284705641,-4.939180466461357,-4.848855556830971,-4.7729272773617604,-4.7125930623783114,-4.6688044184928259,-4.6422519187774451,-4.6333543120143768,-4.6333543120143768],"y":[0.35850200289818829,0.49992541259686529,0.63911849064234416,0.77388607904393336,0.90210281242628487,1.0217466363109948,1.1309306961236456,1.2279330940175959,1.3112240442316341,1.3794899987252831,1.4316543626160734,1.466894472723967,1.4846545714611405,1.4846545714611405,1.4668944727239668,1.4316543626160734,1.3794899987252829,1.3112240442316339,1.2279330940175959,1.1309306961236454,1.021746636310995,0.90210281242628465,0.77388607904393303,0.63911849064234416,0.49992541259686507,0.3585020028981879,0.21707859319951128,0.077885515154032192,-0.056882073247557174,-0.18509880662990835,-0.30474263051461853,-0.41392669032726909,-0.51092908822121941,-0.59422003843525761,-0.66248599292890686,-0.71465035681969691,-0.74989046692759054,-0.76765056566476408,-0.76765056566476408,-0.74989046692759032,-0.71465035681969691,-0.66248599292890664,-0.59422003843525739,-0.51092908822121896,-0.41392669032726864,-0.30474263051461764,-0.18509880662990824,-0.056882073247556675,0.077885515154032803,0.21707859319951184,0.35850200289818801,0.35850200289818829],"text":"id: 22<br />as.factor(id): 22","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(149,144,255,0.6)","hoveron":"fills","name":"22","legendgroup":"22","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.8949672631368832,5.8840699648861552,5.8515499270371167,5.7979200100131756,5.724025989642759,5.6310332187692556,5.5204082489405311,5.3938957020149365,5.2534907564324218,5.1014076820586718,4.940044919826585,4.7719472568900034,4.5997656938101148,4.4262156366929677,4.2540340736130782,4.0859364106764975,3.9245736484444107,3.7724905740706607,3.632085628488146,3.5055730815625514,3.3949481117338269,3.3019553408603235,3.2280613204899069,3.1744314034659658,3.1419113656169269,3.1310140673661992,3.1419113656169269,3.1744314034659658,3.2280613204899069,3.3019553408603239,3.3949481117338269,3.5055730815625514,3.632085628488146,3.7724905740706611,3.9245736484444116,4.0859364106764975,4.2540340736130782,4.4262156366929677,4.5997656938101157,4.7719472568900043,4.940044919826585,5.1014076820586709,5.2534907564324218,5.3938957020149374,5.5204082489405319,5.6310332187692564,5.724025989642759,5.7979200100131756,5.8515499270371167,5.8840699648861552,5.8949672631368832,5.8949672631368832],"y":[-5.1441590609389483,-4.9709514652157818,-4.8004754567463728,-4.6354195440217136,-4.4783867573850955,-4.3318535976886192,-4.1981309803953621,-4.0793277910620604,-3.9773176269540853,-3.8937092492973369,-3.8298212121526864,-3.7866610680307691,-3.7649094781870076,-3.7649094781870076,-3.7866610680307691,-3.8298212121526864,-3.8937092492973369,-3.9773176269540853,-4.0793277910620604,-4.1981309803953621,-4.3318535976886183,-4.4783867573850964,-4.6354195440217136,-4.8004754567463728,-4.9709514652157827,-5.1441590609389491,-5.3173666566621147,-5.4878426651315237,-5.6528985778561829,-5.809931364492801,-5.9564645241892782,-6.0901871414825344,-6.2089903308158361,-6.3110004949238121,-6.3946088725805605,-6.4584969097252101,-6.5016570538471274,-6.5234086436908889,-6.5234086436908889,-6.5016570538471274,-6.4584969097252101,-6.3946088725805597,-6.3110004949238112,-6.2089903308158361,-6.0901871414825335,-5.9564645241892773,-5.809931364492801,-5.6528985778561829,-5.4878426651315237,-5.3173666566621138,-5.1441590609389483,-5.1441590609389483],"text":"id: 23<br />as.factor(id): 23","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(185,131,255,0.6)","hoveron":"fills","name":"23","legendgroup":"23","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.4777118863275907,-2.4943577843624136,-2.5440329627110967,-2.6259540141365676,-2.7388289947205822,-2.8808777985910856,-3.0498602312551983,-3.2431113388045949,-3.457583435830605,-3.6898941692437077,-3.9363798600022735,-4.193153281519626,-4.4561649635494271,-4.7212670547505065,-4.9842787367803085,-5.2410521582976601,-5.4875378490562259,-5.7198485824693295,-5.9343206794953387,-6.1275717870447357,-6.296554219708848,-6.4386030235793514,-6.551478004163366,-6.6333990555888374,-6.6830742339375195,-6.6997201319723434,-6.6830742339375195,-6.6333990555888374,-6.551478004163366,-6.4386030235793514,-6.2965542197088471,-6.1275717870447348,-5.9343206794953387,-5.7198485824693286,-5.487537849056225,-5.2410521582976601,-4.9842787367803085,-4.7212670547505056,-4.4561649635494263,-4.1931532815196242,-3.9363798600022744,-3.6898941692437077,-3.4575834358306046,-3.243111338804594,-3.0498602312551979,-2.8808777985910847,-2.7388289947205822,-2.6259540141365671,-2.5440329627110967,-2.4943577843624136,-2.4777118863275907,-2.4777118863275907],"y":[6.2435137097986644,6.5080926825795702,6.7684990869079042,7.020626158228163,7.2604977020122128,7.4843308007222653,7.6885954726802517,7.8700703419875939,8.025893441547483,8.1536073479956492,8.2511979367348189,8.3171261458827832,8.3503522481976731,8.3503522481976731,8.3171261458827832,8.2511979367348189,8.1536073479956492,8.0258934415474812,7.8700703419875939,7.6885954726802517,7.4843308007222662,7.2604977020122128,7.0206261582281622,6.7684990869079042,6.5080926825795702,6.2435137097986635,5.9789347370177586,5.7185283326894236,5.4664012613691657,5.226529717585116,5.0026966188750626,4.7984319469170771,4.6169570776097348,4.4611339780498458,4.3334200716016777,4.2358294828625089,4.1699012737145456,4.1366751713996557,4.1366751713996557,4.1699012737145456,4.235829482862508,4.3334200716016786,4.4611339780498467,4.6169570776097348,4.7984319469170771,5.0026966188750643,5.226529717585116,5.4664012613691666,5.7185283326894254,5.9789347370177595,6.2435137097986635,6.2435137097986644],"text":"id: 24<br />as.factor(id): 24","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(211,118,255,0.6)","hoveron":"fills","name":"24","legendgroup":"24","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.53945220754255052,0.52280630950772755,0.47313113115904448,0.3912100797335738,0.27833509914955923,0.13628629527905556,-0.032696137385057167,-0.22594724493445373,-0.44041934196046384,-0.67273007537356666,-0.91921576613213252,-1.1759891876494846,-1.4390008696792864,-1.7041029608803651,-1.9671146429101674,-2.223888064427519,-2.4703737551860847,-2.7026844885991883,-2.9171565856251975,-3.1104076931745945,-3.2793901258387068,-3.4214389297092103,-3.5343139102932253,-3.6162349617186957,-3.6659101400673788,-3.6825560381022018,-3.6659101400673788,-3.6162349617186957,-3.5343139102932248,-3.4214389297092103,-3.2793901258387064,-3.1104076931745941,-2.9171565856251975,-2.702684488599187,-2.4703737551860834,-2.223888064427519,-1.9671146429101669,-1.7041029608803648,-1.4390008696792855,-1.1759891876494832,-0.91921576613213296,-0.67273007537356677,-0.4404193419604634,-0.22594724493445306,-0.032696137385056501,0.13628629527905622,0.27833509914955923,0.39121007973357402,0.47313113115904448,0.52280630950772755,0.53945220754255052,0.53945220754255052],"y":[9.1968295508412439,9.4614085236221506,9.7218149279504846,9.9739419992707425,10.213813543054792,10.437646641764845,10.641911313722831,10.823386183030173,10.979209282590062,11.106923189038231,11.2045137777774,11.270441986925363,11.303668089240253,11.303668089240253,11.270441986925363,11.2045137777774,11.106923189038229,10.979209282590062,10.823386183030173,10.641911313722831,10.437646641764845,10.213813543054792,9.9739419992707425,9.7218149279504846,9.4614085236221488,9.1968295508412439,8.9322505780603372,8.6718441737320031,8.4197171024117452,8.1798455586276955,7.9560124599176421,7.7517477879596566,7.5702729186523143,7.4144498190924253,7.2867359126442572,7.1891453239050884,7.123217114757125,7.0899910124422352,7.0899910124422352,7.123217114757125,7.1891453239050875,7.2867359126442581,7.4144498190924262,7.5702729186523143,7.7517477879596566,7.9560124599176438,8.1798455586276955,8.4197171024117452,8.6718441737320049,8.932250578060339,9.1968295508412439,9.1968295508412439],"text":"id: 25<br />as.factor(id): 25","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(231,107,243,0.6)","hoveron":"fills","name":"25","legendgroup":"25","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.6920105562359922,2.6831129494729229,2.656560449757543,2.612771805872057,2.5524375908886081,2.4765093114193975,2.386184401789011,2.2828873397798044,2.1682471817663376,2.0440718715231934,1.9123197278716413,1.7750685608219814,1.6344829032684605,1.4927798750124983,1.3521942174589769,1.2149430504093175,1.0831909067577652,0.95901559651462087,0.84437543850115426,0.74107837649194785,0.65075346686156132,0.57482518739235078,0.51449097240890151,0.47070232852341598,0.44414982880803566,0.43525222204496683,0.44414982880803566,0.47070232852341598,0.51449097240890196,0.57482518739235089,0.65075346686156144,0.74107837649194797,0.8443754385011546,0.95901559651462154,1.0831909067577659,1.2149430504093175,1.3521942174589772,1.4927798750124985,1.6344829032684609,1.7750685608219821,1.9123197278716411,2.0440718715231934,2.1682471817663376,2.2828873397798048,2.386184401789011,2.4765093114193979,2.5524375908886081,2.6127718058720575,2.656560449757543,2.6831129494729229,2.6920105562359922,2.6920105562359922],"y":[8.3818266387707272,8.5232500484694036,8.6624431265148836,8.7972107149164724,8.925427448298823,9.0450712721835345,9.1542553319961844,9.2512577298901348,9.3345486801041737,9.4028146345978225,9.4549789984886132,9.4902191085965057,9.5079792073336797,9.5079792073336797,9.4902191085965057,9.4549789984886132,9.4028146345978225,9.3345486801041737,9.2512577298901348,9.1542553319961844,9.0450712721835345,8.925427448298823,8.7972107149164724,8.6624431265148836,8.5232500484694036,8.3818266387707272,8.2404032290720508,8.1012101510265708,7.966442562624982,7.8382258292426306,7.7185820053579199,7.60939794554527,7.5123955476513196,7.4291045974372816,7.3608386429436319,7.3086742790528421,7.2734341689449487,7.2556740702077747,7.2556740702077747,7.2734341689449487,7.3086742790528421,7.3608386429436319,7.4291045974372816,7.5123955476513196,7.60939794554527,7.7185820053579217,7.8382258292426306,7.966442562624982,8.1012101510265708,8.2404032290720508,8.3818266387707272,8.3818266387707272],"text":"id: 26<br />as.factor(id): 26","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(245,100,227,0.6)","hoveron":"fills","name":"26","legendgroup":"26","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.8473212062001068,3.8410296481216095,3.8222541955154101,3.7912909484850204,3.7486282159326567,3.6949388146361506,3.6310694585264418,3.5580274055030849,3.476964572375465,3.3891593684465948,3.2959965342347179,3.1989453032881356,3.0995362314944712,2.9993370593000117,2.8999279875063468,2.802876756559765,2.7097139223478881,2.6219087184190175,2.540845885291398,2.4678038322680407,2.4039344761583323,2.3502450748618262,2.3075823423094626,2.2766190952790728,2.2578436426728734,2.2515520845943762,2.2578436426728734,2.2766190952790728,2.3075823423094626,2.3502450748618262,2.4039344761583323,2.4678038322680411,2.540845885291398,2.621908718419018,2.7097139223478885,2.802876756559765,2.8999279875063468,2.9993370593000117,3.0995362314944717,3.1989453032881361,3.2959965342347179,3.3891593684465944,3.4769645723754654,3.5580274055030854,3.6310694585264423,3.694938814636151,3.7486282159326567,3.7912909484850204,3.8222541955154101,3.8410296481216095,3.8473212062001068,3.8473212062001068],"y":[7.1559087418882612,7.255910193904719,7.3543345632849055,7.4496296389278269,7.5402925605640752,7.6248935197600423,7.7020983088510455,7.7706893621932132,7.829584957901031,7.8778562772476626,7.914742052691123,7.9396605735181751,7.9522188597697729,7.9522188597697729,7.9396605735181751,7.914742052691123,7.8778562772476626,7.829584957901031,7.7706893621932132,7.7020983088510455,7.6248935197600423,7.5402925605640752,7.4496296389278269,7.3543345632849055,7.255910193904719,7.1559087418882612,7.0559072898718034,6.9574829204916169,6.8621878448486955,6.7715249232124473,6.6869239640164793,6.6097191749254769,6.5411281215833093,6.4822325258754914,6.4339612065288598,6.3970754310853994,6.3721569102583473,6.3595986240067495,6.3595986240067495,6.3721569102583473,6.3970754310853994,6.4339612065288598,6.4822325258754914,6.5411281215833093,6.6097191749254769,6.6869239640164801,6.7715249232124473,6.8621878448486955,6.9574829204916169,7.0559072898718034,7.1559087418882612,7.1559087418882612],"text":"id: 27<br />as.factor(id): 27","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,97,209,0.6)","hoveron":"fills","name":"27","legendgroup":"27","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.0559733774009894,6.047075770637921,6.0205232709225402,5.9767346270370547,5.9164004120536058,5.8404721325843951,5.7501472229540083,5.6468501609448021,5.5322100029313352,5.408034692688191,5.2762825490366385,5.1390313819869791,4.9984457244334584,4.8567426961774958,4.7161570386239751,4.5789058715743147,4.4471537279227631,4.3229784176796189,4.208338259666152,4.1050411976569459,4.014716288026559,3.9387880085573483,3.8784537935738994,3.8346651496884139,3.8081126499730331,3.7992150432099647,3.8081126499730331,3.8346651496884139,3.8784537935738994,3.9387880085573483,4.014716288026559,4.1050411976569459,4.208338259666152,4.3229784176796189,4.447153727922764,4.5789058715743147,4.7161570386239751,4.8567426961774958,4.9984457244334584,5.13903138198698,5.2762825490366385,5.408034692688191,5.5322100029313352,5.6468501609448021,5.7501472229540092,5.8404721325843951,5.9164004120536058,5.9767346270370547,6.0205232709225402,6.047075770637921,6.0559733774009894,6.0559733774009894],"y":[6.7281049190968796,6.8695283287955569,7.0087214068410351,7.1434889952426248,7.2717057286249762,7.3913495525096859,7.5005336123223367,7.5975360102162872,7.6808269604303252,7.749092914923974,7.8012572788147647,7.8364973889226581,7.8542574876598321,7.8542574876598321,7.8364973889226581,7.8012572788147647,7.749092914923974,7.6808269604303252,7.5975360102162872,7.5005336123223367,7.3913495525096859,7.2717057286249762,7.1434889952426239,7.0087214068410351,6.869528328795556,6.7281049190968796,6.5866815093982023,6.4474884313527232,6.3127208429511343,6.1845041095687829,6.0648602856840732,5.9556762258714224,5.8586738279774719,5.775382877763434,5.7071169232697843,5.6549525593789944,5.619712449271101,5.601952350533927,5.601952350533927,5.619712449271101,5.6549525593789944,5.7071169232697851,5.775382877763434,5.8586738279774719,5.9556762258714224,6.0648602856840732,6.1845041095687829,6.3127208429511343,6.4474884313527241,6.5866815093982032,6.7281049190968796,6.7281049190968796],"text":"id: 28<br />as.factor(id): 28","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(255,98,188,0.6)","hoveron":"fills","name":"28","legendgroup":"28","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.8188590744133615,6.8125675163348642,6.7937920637286648,6.7628288166982751,6.7201660841459114,6.6664766828494049,6.6026073267396965,6.5295652737163392,6.4485024405887197,6.3606972366598491,6.2675344024479731,6.1704831715013908,6.0710740997077259,5.9708749275132664,5.8714658557196016,5.7744146247730193,5.6812517905611433,5.5934465866322718,5.5123837535046523,5.4393417004812958,5.3754723443715875,5.3217829430750809,5.2791202105227173,5.2481569634923275,5.2293815108861281,5.2230899528076309,5.2293815108861281,5.2481569634923275,5.2791202105227173,5.3217829430750809,5.3754723443715875,5.4393417004812958,5.5123837535046531,5.5934465866322727,5.6812517905611433,5.7744146247730193,5.8714658557196016,5.9708749275132664,6.0710740997077268,6.1704831715013908,6.2675344024479722,6.3606972366598491,6.4485024405887197,6.5295652737163401,6.6026073267396974,6.6664766828494058,6.7201660841459114,6.7628288166982751,6.7937920637286648,6.8125675163348642,6.8188590744133615,6.8188590744133615],"y":[-3.3158266883353265,-3.2158252363188686,-3.1174008669386821,-3.0221057912957607,-2.9314428696595125,-2.846841910463545,-2.7696371213725421,-2.7010460680303741,-2.6421504723225566,-2.5938791529759251,-2.5569933775324647,-2.5320748567054125,-2.5195165704538143,-2.5195165704538143,-2.5320748567054125,-2.5569933775324647,-2.5938791529759255,-2.6421504723225566,-2.7010460680303741,-2.7696371213725421,-2.846841910463545,-2.9314428696595125,-3.0221057912957612,-3.1174008669386821,-3.2158252363188686,-3.3158266883353269,-3.4158281403517843,-3.5142525097319708,-3.6095475853748922,-3.7002105070111404,-3.784811466207108,-3.8620162552981108,-3.9306073086402789,-3.9895029043480963,-4.0377742236947283,-4.0746599991381878,-4.0995785199652399,-4.1121368062168386,-4.1121368062168386,-4.0995785199652399,-4.0746599991381887,-4.0377742236947274,-3.9895029043480963,-3.9306073086402784,-3.8620162552981103,-3.7848114662071075,-3.7002105070111404,-3.6095475853748917,-3.5142525097319703,-3.4158281403517838,-3.3158266883353265,-3.3158266883353265],"text":"id: 29<br />as.factor(id): 29","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(255,103,164,0.6)","hoveron":"fills","name":"29","legendgroup":"29","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[7.4596427459943673,7.45335118791587,7.4345757353096706,7.4036124882792809,7.3609497557269172,7.3072603544304116,7.2433909983207023,7.170348945297345,7.0892861121697255,7.0014809082408549,6.908318074028978,6.8112668430823966,6.7118577712887317,6.6116585990942722,6.5122495273006074,6.4151982963540251,6.3220354621421491,6.2342302582132785,6.1531674250856589,6.0801253720623016,6.0162560159525924,5.9625666146560867,5.9199038821037231,5.8889406350733333,5.8701651824671339,5.8638736243886367,5.8701651824671339,5.8889406350733333,5.9199038821037231,5.9625666146560867,6.0162560159525933,6.0801253720623016,6.1531674250856589,6.2342302582132785,6.3220354621421491,6.4151982963540251,6.5122495273006074,6.6116585990942722,6.7118577712887326,6.8112668430823966,6.908318074028978,7.0014809082408549,7.0892861121697255,7.1703489452973459,7.2433909983207032,7.3072603544304116,7.3609497557269172,7.4036124882792809,7.4345757353096706,7.45335118791587,7.4596427459943673,7.4596427459943673],"y":[-4.7772901246571156,-4.6772886726406577,-4.5788643032604712,-4.4835692276175498,-4.3929063059813016,-4.3083053467853345,-4.2311005576943312,-4.1625095043521636,-4.1036139086443457,-4.0553425892977142,-4.0184568138542538,-3.9935382930272016,-3.9809800067756034,-3.9809800067756034,-3.9935382930272016,-4.0184568138542538,-4.0553425892977142,-4.1036139086443457,-4.1625095043521636,-4.2311005576943312,-4.3083053467853345,-4.3929063059813016,-4.4835692276175498,-4.5788643032604712,-4.6772886726406577,-4.7772901246571156,-4.8772915766735734,-4.9757159460537599,-5.0710110216966813,-5.1616739433329295,-5.2462749025288975,-5.3234796916198999,-5.3920707449620675,-5.4509663406698854,-5.4992376600165169,-5.5361234354599773,-5.5610419562870295,-5.5736002425386273,-5.5736002425386273,-5.5610419562870295,-5.5361234354599773,-5.4992376600165169,-5.4509663406698854,-5.3920707449620675,-5.3234796916198999,-5.2462749025288966,-5.1616739433329295,-5.0710110216966813,-4.9757159460537599,-4.8772915766735734,-4.7772901246571156,-4.7772901246571156],"text":"id: 30<br />as.factor(id): 30","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(254,110,138,0.6)","hoveron":"fills","name":"30","legendgroup":"30","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-1.381976597885342,1.1283791670955126,0.080602708790967315,0.32241083516386992,2.7331696929785698,2.9858449570824659,1.9637572360917581,0.94079487540655005,-1.6321345407072181,-3.1360491149471379,-3.5477391912856202,-4.1595366735773878,-0.46155929053066191,4.0326754343942488,2.2716017429716886,6.7603505064333085,6.0969537281644417,-2.9500729553395253,-3.9555235452141209,2.4004540739323814,-4.7316217152444793,-5.7617334791098891,4.5129906652515412,-4.5887160091499668,-1.5715519152798256,1.5636313891404794,3.0494366453972415,4.9275942103054771,6.0209745136104962,6.661758185191502],"y":[0,0,-1.6163713814713934,4.2440666205653086,1.0654291705472203,-0.51020857347275317,-1.735694509773267,-2.9604504438644934,-2.4978604985898909,-1.2942272891752791,0.24752183779536407,2.5616700162769597,-5.6741648741638935,-2.4222599909112805,-3.2506211797251967,2.5021381353393122,-1.7218673888467571,-3.9026844322746204,-2.6635120468531439,-4.6065869062516729,-1.2691839936512248,0.35850200289818829,-5.1441590609389483,6.2435137097986644,9.1968295508412439,8.3818266387707272,7.1559087418882612,6.7281049190968796,-3.3158266883353265,-4.7772901246571156],"text":["Bioinformatics (Oxford, England)","BMC bioinformatics","BMC genomics","Briefings in bioinformatics","Briefings in functional genomics","Cell reports","Cell systems","Communications biology","Computational and structural biotechnology journal","Current protocols","eLife","F1000Research","Frontiers in genetics","G3 (Bethesda, Md.)","Genes","Genome biology","Genome Biology","Genomics, proteomics & bioinformatics","GigaScience","IEEE/ACM transactions on computational biology and bioinformatics","Life (Basel, Switzerland)","NAR genomics and bioinformatics","Nature biotechnology","Nature communications","Nature methods","Nucleic acids research","Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","PLoS computational biology","Scientific data","Statistical applications in genetics and molecular biology"],"hovertext":["x: -1.38197660<br />y:  0.0000000<br />Freq:  6<br />Var1: Bioinformatics (Oxford, England)","x:  1.12837917<br />y:  0.0000000<br />Freq:  4<br />Var1: BMC bioinformatics","x:  0.08060271<br />y: -1.6163714<br />Freq:  2<br />Var1: BMC genomics","x:  0.32241084<br />y:  4.2440666<br />Freq: 32<br />Var1: Briefings in bioinformatics","x:  2.73316969<br />y:  1.0654292<br />Freq:  2<br />Var1: Briefings in functional genomics","x:  2.98584496<br />y: -0.5102086<br />Freq:  2<br />Var1: Cell reports","x:  1.96375724<br />y: -1.7356945<br />Freq:  2<br />Var1: Cell systems","x:  0.94079488<br />y: -2.9604504<br />Freq:  2<br />Var1: Communications biology","x: -1.63213454<br />y: -2.4978605<br />Freq:  4<br />Var1: Computational and structural biotechnology journal","x: -3.13604911<br />y: -1.2942273<br />Freq:  2<br />Var1: Current protocols","x: -3.54773919<br />y:  0.2475218<br />Freq:  2<br />Var1: eLife","x: -4.15953667<br />y:  2.5616700<br />Freq:  8<br />Var1: F1000Research","x: -0.46155929<br />y: -5.6741649<br />Freq: 16<br />Var1: Frontiers in genetics","x:  4.03267543<br />y: -2.4222600<br />Freq:  6<br />Var1: G3 (Bethesda, Md.)","x:  2.27160174<br />y: -3.2506212<br />Freq:  1<br />Var1: Genes","x:  6.76035051<br />y:  2.5021381<br />Freq: 38<br />Var1: Genome biology","x:  6.09695373<br />y: -1.7218674<br />Freq:  2<br />Var1: Genome Biology","x: -2.95007296<br />y: -3.9026844<br />Freq:  2<br />Var1: Genomics, proteomics & bioinformatics","x: -3.95552355<br />y: -2.6635120<br />Freq:  2<br />Var1: GigaScience","x:  2.40045407<br />y: -4.6065869<br />Freq:  2<br />Var1: IEEE/ACM transactions on computational biology and bioinformatics","x: -4.73162172<br />y: -1.2691840<br />Freq:  2<br />Var1: Life (Basel, Switzerland)","x: -5.76173348<br />y:  0.3585020<br />Freq:  4<br />Var1: NAR genomics and bioinformatics","x:  4.51299067<br />y: -5.1441591<br />Freq:  6<br />Var1: Nature biotechnology","x: -4.58871601<br />y:  6.2435137<br />Freq: 14<br />Var1: Nature communications","x: -1.57155192<br />y:  9.1968296<br />Freq: 14<br />Var1: Nature methods","x:  1.56363139<br />y:  8.3818266<br />Freq:  4<br />Var1: Nucleic acids research","x:  3.04943665<br />y:  7.1559087<br />Freq:  2<br />Var1: Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing","x:  4.92759421<br />y:  6.7281049<br />Freq:  4<br />Var1: PLoS computational biology","x:  6.02097451<br />y: -3.3158267<br />Freq:  2<br />Var1: Scientific data","x:  6.66175819<br />y: -4.7772901<br />Freq:  2<br />Var1: Statistical applications in genetics and molecular biology"],"textfont":{"size":[11.727201007145998,10.787687040669919,9.4231066277142883,17.937654683763181,9.4231066277142883,9.4231066277142883,9.4231066277142883,9.4231066277142883,10.787687040669919,9.4231066277142883,9.4231066277142883,12.490871843537086,14.778495571279521,11.727201007145998,7.559055118110237,18.897637795275593,9.4231066277142883,9.4231066277142883,9.4231066277142883,9.4231066277142883,9.4231066277142883,10.787687040669919,11.727201007145998,14.279988416093698,14.279988416093698,10.787687040669919,9.4231066277142883,10.787687040669919,9.4231066277142883,9.4231066277142883],"color":"rgba(0,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":16,"r":0,"b":0,"l":0},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-7.7465307122948879,11.094666741673819],"tickmode":"array","ticktext":["-5","0","5","10"],"tickvals":[-5,0,5,10],"categoryorder":"array","categoryarray":["-5","0","5","10"],"nticks":null,"ticks":"outside","tickcolor":null,"ticklen":0,"tickwidth":0,"showticklabels":false,"tickfont":{"color":null,"family":null,"size":0},"tickangle":-0,"showline":true,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"scaleanchor":"y","scaleratio":1,"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-8.8879769163162994,12.265174994266754],"tickmode":"array","ticktext":["-5","0","5","10"],"tickvals":[-5,0,5.0000000000000018,9.9999999999999982],"categoryorder":"array","categoryarray":["-5","0","5","10"],"nticks":null,"ticks":"outside","tickcolor":null,"ticklen":0,"tickwidth":0,"showticklabels":false,"tickfont":{"color":null,"family":null,"size":0},"tickangle":-0,"showline":true,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"scaleanchor":"x","scaleratio":1,"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"8520fe61c4d":{"x":{},"y":{},"fill":{},"type":"scatter"},"852014c021f":{"x":{},"y":{},"size":{},"label":{}}},"cur_data":"8520fe61c4d","visdat":{"8520fe61c4d":["function (y) ","x"],"852014c021f":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development paper" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development paper</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-f0412714cdfc8f391979" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-f0412714cdfc8f391979">{"x":{"data":[{"x":[0,-0.02179459650145521,-0.086834672199533003,-0.19409450624741531,-0.34188254698824894,-0.52786808873525448,-0.74911802839270436,-1.0021431222438935,-1.2829530134089233,-1.5871191621564236,-1.9098446866205965,-2.2460400124937587,-2.5904031386535369,-2.9375032528878311,-3.2818663790476101,-3.6180617049207719,-3.9407872293849446,-4.2449533781324451,-4.5257632692974745,-4.7787883631486636,-5.0000383028061126,-5.1860238445531195,-5.3338118852939527,-5.441071719341835,-5.5061117950399128,-5.527906391541368,-5.5061117950399128,-5.441071719341835,-5.3338118852939527,-5.1860238445531186,-5.0000383028061126,-4.7787883631486636,-4.5257632692974745,-4.2449533781324442,-3.9407872293849433,-3.6180617049207719,-3.2818663790476093,-2.9375032528878307,-2.590403138653536,-2.246040012493757,-1.9098446866205971,-1.5871191621564238,-1.2829530134089229,-1.0021431222438926,-0.74911802839270347,-0.52786808873525359,-0.34188254698824894,-0.19409450624741531,-0.086834672199532559,-0.02179459650145521,0,0],"y":[0,0.34641519144633232,0.6873672083851512,1.0174790338344692,1.3315446071077046,1.624610926500659,1.8920561610871722,2.1296625397537756,2.3336828679697263,2.5008996232832232,2.6286756975725232,2.7149959858163584,2.7584991655038817,2.7584991655038817,2.7149959858163579,2.6286756975725232,2.5008996232832228,2.3336828679697263,2.1296625397537756,1.8920561610871716,1.6246109265006592,1.3315446071077044,1.0174790338344686,0.68736720838515131,0.34641519144633187,-8.8896632625151402e-16,-0.34641519144633237,-0.68736720838515186,-1.0174790338344701,-1.3315446071077048,-1.6246109265006594,-1.8920561610871722,-2.1296625397537761,-2.3336828679697272,-2.5008996232832237,-2.6286756975725232,-2.7149959858163584,-2.7584991655038817,-2.7584991655038817,-2.7149959858163579,-2.6286756975725236,-2.5008996232832232,-2.3336828679697263,-2.1296625397537752,-1.8920561610871709,-1.6246109265006574,-1.3315446071077046,-1.0174790338344688,-0.68736720838515042,-0.34641519144633098,-6.7695092904168893e-16,0],"text":"id: 1<br />as.factor(id): 1","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(248,118,109,0.6)","hoveron":"fills","name":"1","legendgroup":"1","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.2567583341910251,2.2478607274279563,2.221308227712576,2.17751958382709,2.1171853688436411,2.0412570893744308,1.950932179744044,1.8476351177348374,1.7329949597213705,1.6088196494782268,1.4770675058266745,1.3398163387770146,1.1992306812234936,1.0575276529675315,0.91694199541401011,0.77969082836435055,0.64793868471279836,0.52376337446965404,0.40912321645618743,0.30582615444698102,0.21550124481659449,0.13957296534738395,0.079238750363934685,0.03545010647844915,0.0088976067630688327,0,0.0088976067630688327,0.03545010647844915,0.079238750363935129,0.13957296534738406,0.21550124481659461,0.30582615444698114,0.40912321645618777,0.52376337446965471,0.64793868471279903,0.77969082836435055,0.91694199541401034,1.0575276529675317,1.1992306812234941,1.3398163387770152,1.4770675058266742,1.6088196494782265,1.732994959721371,1.8476351177348378,1.9509321797440444,2.0412570893744313,2.1171853688436411,2.1775195838270904,2.221308227712576,2.2478607274279563,2.2567583341910251,2.2567583341910251],"y":[0,0.14142340969867698,0.28061648774415587,0.41538407614574507,0.54360080952809653,0.66324463341280659,0.77242869322545737,0.86943109111940753,0.95272204133344573,1.0209879958270949,1.0731523597178851,1.1083924698257788,1.1261525685629523,1.1261525685629523,1.1083924698257785,1.0731523597178851,1.0209879958270947,0.95272204133344562,0.86943109111940753,0.77242869322545715,0.66324463341280671,0.54360080952809642,0.4153840761457448,0.28061648774415587,0.14142340969867678,-3.6291898297212129e-16,-0.14142340969867701,-0.2806164877441561,-0.41538407614574546,-0.54360080952809664,-0.66324463341280682,-0.77242869322545737,-0.86943109111940764,-0.95272204133344596,-1.0209879958270951,-1.0731523597178851,-1.1083924698257788,-1.1261525685629523,-1.1261525685629523,-1.1083924698257785,-1.0731523597178851,-1.0209879958270949,-0.95272204133344562,-0.86943109111940731,-0.77242869322545693,-0.66324463341280593,-0.54360080952809653,-0.41538407614574496,-0.28061648774415549,-0.14142340969867645,-2.763640595091933e-16,0],"text":"id: 2<br />as.factor(id): 2","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(241,125,81,0.6)","hoveron":"fills","name":"2","legendgroup":"2","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.1331589268822047,1.1268673688037074,1.108091916197508,1.0771286691671182,1.0344659366147546,0.98077653531824849,0.91690717920853992,0.84386512618518272,0.76280229305756309,0.6749970891286925,0.58183425491681584,0.48478302397023376,0.38537395217656922,0.28517477998210938,0.18576570818844476,0.088714477241862794,-0.0044483569700138337,-0.09225356089888459,-0.17331639402650412,-0.24635844704986132,-0.31022780315956977,-0.36391720445607578,-0.40657993700843975,-0.43754318403882941,-0.45631863664502881,-0.46261019472352605,-0.45631863664502881,-0.43754318403882941,-0.40657993700843953,-0.36391720445607578,-0.31022780315956966,-0.24635844704986121,-0.17331639402650401,-0.092253560898884146,-0.0044483569700133896,0.088714477241862794,0.1857657081884449,0.28517477998210955,0.38537395217656956,0.48478302397023421,0.58183425491681562,0.6749970891286925,0.7628022930575632,0.84386512618518306,0.91690717920854026,0.98077653531824882,1.0344659366147546,1.0771286691671185,1.1080919161975082,1.1268673688037074,1.1331589268822047,1.1331589268822047],"y":[-1.7554135478633106,-1.6554120958468528,-1.5569877264666663,-1.4616926508237451,-1.3710297291874967,-1.2864287699915291,-1.2092239809005263,-1.1406329275583584,-1.0817373318505408,-1.0334660125039092,-0.99658023706044885,-0.9716617162333967,-0.95910342998179854,-0.95910342998179854,-0.97166171623339681,-0.99658023706044885,-1.0334660125039095,-1.0817373318505408,-1.1406329275583584,-1.2092239809005263,-1.2864287699915291,-1.3710297291874969,-1.4616926508237453,-1.5569877264666663,-1.6554120958468528,-1.7554135478633108,-1.8554149998797684,-1.9538393692599552,-2.0491344449028763,-2.1397973665391246,-2.2243983257350921,-2.3016031148260949,-2.370194168168263,-2.4290897638760804,-2.477361083222712,-2.5142468586661724,-2.5391653794932245,-2.5517236657448228,-2.5517236657448228,-2.5391653794932245,-2.5142468586661724,-2.477361083222712,-2.4290897638760804,-2.3701941681682626,-2.3016031148260945,-2.2243983257350917,-2.1397973665391246,-2.0491344449028759,-1.9538393692599547,-1.855414999879768,-1.7554135478633108,-1.7554135478633106],"text":"id: 3<br />as.factor(id): 3","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(233,132,41,0.6)","hoveron":"fills","name":"3","legendgroup":"3","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.7756611683411543,2.7602500613617504,2.7142597827867534,2.6384156267825518,2.5339137009964348,2.4024020632244736,2.2459547305555763,2.0670389708830377,1.8684763926159889,1.6533984462292355,1.4251970394186337,1.1874710446904995,0.94396954299232183,0.69853269846662691,0.45503119676844889,0.21730520204031512,-0.010896204770286522,-0.22597415115704045,-0.42453672942408904,-0.60345248909662741,-0.75989982176552429,-0.89141145953748591,-0.99591338532360318,-1.0717575413278047,-1.1177478199028017,-1.1331589268822051,-1.1177478199028017,-1.0717575413278047,-0.99591338532360274,-0.89141145953748568,-0.75989982176552406,-0.60345248909662719,-0.4245367294240886,-0.22597415115703956,-0.010896204770285411,0.21730520204031512,0.45503119676844928,0.69853269846662736,0.94396954299232261,1.1874710446905006,1.4251970394186333,1.6533984462292353,1.8684763926159893,2.0670389708830386,2.2459547305555767,2.402402063224474,2.5339137009964348,2.6384156267825523,2.7142597827867538,2.7602500613617504,2.7756611683411543,2.7756611683411543],"y":[3.0674519891027421,3.3124045200804799,3.5534940033171494,3.7869183136422317,4.0089962102409773,4.2162253920211175,4.405337730993268,4.5733508126016016,4.7176149701830061,4.8358550717931914,4.9262064003965511,4.9872440615677442,5.0180054549279696,5.0180054549279696,4.9872440615677442,4.9262064003965511,4.8358550717931905,4.7176149701830061,4.5733508126016016,4.405337730993268,4.2162253920211175,4.0089962102409773,3.7869183136422313,3.5534940033171494,3.3124045200804795,3.0674519891027416,2.8224994581250042,2.5814099748883343,2.3479856645632515,2.1259077679645069,1.918678586184366,1.729566247212216,1.5615531656038819,1.4172890080224776,1.2990489064122925,1.2086975778089328,1.1476599166377397,1.116898523277515,1.116898523277515,1.1476599166377399,1.2086975778089326,1.299048906412293,1.4172890080224783,1.5615531656038826,1.7295662472122169,1.9186785861843676,2.1259077679645069,2.3479856645632524,2.5814099748883357,2.8224994581250051,3.0674519891027416,3.0674519891027421],"text":"id: 4<br />as.factor(id): 4","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(223,139,0,0.6)","hoveron":"fills","name":"4","legendgroup":"4","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.9703141687938897,2.9658653654123555,2.9525891155546651,2.9306947936119223,2.9005276861201978,2.8625635463855925,2.8174010915703991,2.765752560565796,2.7084324815590626,2.6463448264374905,2.5804687546117142,2.5118431710868845,2.4415503423101241,2.3706988281821428,2.3004059994053825,2.2317804158805523,2.1659043440547765,2.1038166889332044,2.046496609926471,1.9948480789218677,1.9496856241066745,1.9117214843720691,1.8815543768803447,1.8596600549376019,1.8463838050799115,1.8419350016983773,1.8463838050799115,1.8596600549376019,1.8815543768803447,1.9117214843720691,1.9496856241066745,1.9948480789218679,2.046496609926471,2.1038166889332044,2.1659043440547769,2.2317804158805523,2.3004059994053825,2.3706988281821428,2.4415503423101241,2.5118431710868849,2.5804687546117142,2.6463448264374905,2.7084324815590626,2.765752560565796,2.8174010915703995,2.8625635463855925,2.9005276861201978,2.9306947936119223,2.9525891155546651,2.9658653654123555,2.9703141687938897,2.9703141687938897],"y":[1.1100250546943573,1.1807367595436957,1.2503332985664353,1.3177170927672299,1.3818254594584056,1.4416473714007605,1.4962394013070859,1.5447406002540611,1.5863860753610801,1.6205190526079047,1.6466012345532999,1.6642212896072466,1.6731013389758336,1.6731013389758336,1.6642212896072466,1.6466012345532999,1.6205190526079045,1.5863860753610801,1.5447406002540611,1.4962394013070859,1.4416473714007607,1.3818254594584056,1.3177170927672297,1.2503332985664353,1.1807367595436957,1.1100250546943571,1.0393133498450189,0.96971681082227923,0.90233301662148457,0.83822464993030898,0.7784027379879539,0.72381070808162862,0.67530950913465349,0.63366403402763427,0.59953105678080976,0.57344887483541473,0.55582881978146792,0.54694877041288115,0.54694877041288115,0.55582881978146803,0.57344887483541473,0.59953105678080987,0.63366403402763449,0.67530950913465371,0.72381070808162884,0.77840273798795434,0.83822464993030898,0.9023330166214848,0.96971681082227956,1.0393133498450191,1.1100250546943571,1.1100250546943573],"text":"id: 5<br />as.factor(id): 5","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(212,146,0,0.6)","hoveron":"fills","name":"5","legendgroup":"5","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.7528934595755485,4.742945632795907,4.7132590356278392,4.6643018434426011,4.5968461404065621,4.5119557432526838,4.410969424255148,4.2954797979908506,4.1673082048561358,4.0284759874407374,3.8811726127476436,3.7277211429905344,3.5705415995149443,3.412112797615992,3.2549332541404015,3.1014817843832927,2.9541784096901988,2.8153461922748004,2.6871745991400857,2.5716849728757873,2.4706986538782525,2.3858082567243746,2.3183525536883351,2.2693953615030971,2.2397087643350297,2.2297609375553877,2.2397087643350297,2.2693953615030971,2.3183525536883351,2.3858082567243746,2.4706986538782529,2.5716849728757878,2.6871745991400862,2.8153461922748009,2.9541784096901993,3.1014817843832927,3.2549332541404019,3.412112797615992,3.5705415995149448,3.7277211429905348,3.8811726127476431,4.0284759874407374,4.1673082048561358,4.2954797979908506,4.4109694242551489,4.5119557432526838,4.5968461404065621,4.6643018434426011,4.7132590356278392,4.742945632795907,4.7528934595755485,4.7528934595755485],"y":[-0.35821187849523706,-0.20009569964721466,-0.044473107393652456,0.10620163702118041,0.24955230286913266,0.3833181645162449,0.50538965441649009,0.61384163225216759,0.70696374554673769,0.78328740294509092,0.84160893477651211,0.88100857564442592,0.90086496967614016,0.90086496967614016,0.8810085756444257,0.84160893477651211,0.78328740294509069,0.70696374554673747,0.61384163225216759,0.50538965441648975,0.38331816451624512,0.24955230286913255,0.10620163702118013,-0.044473107393652456,-0.20009569964721485,-0.35821187849523745,-0.51632805734325948,-0.67195064959682194,-0.82262539401165502,-0.96597605985960677,-1.0997419215067192,-1.2218134114069641,-1.3302653892426419,-1.423387502537212,-1.4997111599355653,-1.5580326917669862,-1.5974323326349,-1.6172887266666143,-1.6172887266666143,-1.5974323326348998,-1.5580326917669864,-1.499711159935565,-1.4233875025372116,-1.3302653892426415,-1.2218134114069636,-1.0997419215067183,-0.96597605985960677,-0.82262539401165435,-0.67195064959682127,-0.51632805734325893,-0.35821187849523739,-0.35821187849523706],"text":"id: 6<br />as.factor(id): 6","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(200,152,0,0.6)","hoveron":"fills","name":"6","legendgroup":"6","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5839167024656375,2.5794678990841033,2.566191649226413,2.5442973272836702,2.5141302197919457,2.4761660800573404,2.431003625242147,2.3793550942375439,2.3220350152308105,2.2599473601092384,2.1940712882834621,2.1254457047586324,2.055152875981872,1.9843013618538907,1.9140085330771301,1.8453829495523004,1.7795068777265244,1.7174192226049521,1.6600991435982189,1.6084506125936155,1.5632881577784223,1.525324018043817,1.4951569105520925,1.4732625886093498,1.4599863387516594,1.4555375353701252,1.4599863387516594,1.4732625886093498,1.4951569105520925,1.525324018043817,1.5632881577784223,1.6084506125936158,1.6600991435982189,1.7174192226049525,1.7795068777265246,1.8453829495523004,1.9140085330771304,1.9843013618538909,2.055152875981872,2.1254457047586328,2.1940712882834621,2.2599473601092384,2.3220350152308105,2.3793550942375439,2.4310036252421474,2.4761660800573404,2.5141302197919457,2.5442973272836702,2.566191649226413,2.5794678990841033,2.5839167024656375,2.5839167024656375],"y":[-1.4388494724731233,-1.3681377676237849,-1.2985412286010454,-1.2311574344002507,-1.167049067709075,-1.1072271557667199,-1.0526351258603945,-1.0041339269134195,-0.96248845180640052,-0.9283554745595759,-0.90227329261418077,-0.88465323756023395,-0.87577318819164718,-0.87577318819164718,-0.88465323756023406,-0.90227329261418077,-0.92835547455957601,-0.96248845180640052,-1.0041339269134195,-1.0526351258603948,-1.1072271557667199,-1.1670490677090752,-1.2311574344002509,-1.2985412286010454,-1.3681377676237849,-1.4388494724731236,-1.5095611773224618,-1.5791577163452013,-1.6465415105459962,-1.7106498772371717,-1.7704717891795267,-1.8250638190858521,-1.8735650180328272,-1.9152104931398464,-1.9493434703866708,-1.9754256523320659,-1.9930457073860128,-2.0019257567545994,-2.0019257567545994,-1.9930457073860126,-1.9754256523320659,-1.9493434703866708,-1.9152104931398461,-1.8735650180328269,-1.8250638190858517,-1.7704717891795263,-1.7106498772371717,-1.6465415105459957,-1.5791577163452011,-1.5095611773224615,-1.4388494724731236,-1.4388494724731233],"text":"id: 7<br />as.factor(id): 7","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(186,158,0,0.6)","hoveron":"fills","name":"7","legendgroup":"7","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.3941101215023997,2.3878185634239024,2.369043110817703,2.3380798637873133,2.2954171312349496,2.2417277299384435,2.1778583738287347,2.1048163208053778,2.0237534876777579,1.9359482837488875,1.8427854495370108,1.7457342185904288,1.6463251467967643,1.5461259746023044,1.4467169028086397,1.3496656718620579,1.2565028376501812,1.1686976337213104,1.0876348005936909,1.0145927475703336,0.95072339146062523,0.89703399016411922,0.85437125761175525,0.82340801058136559,0.8046325579751662,0.79834099989666896,0.8046325579751662,0.82340801058136559,0.85437125761175547,0.89703399016411922,0.95072339146062534,1.0145927475703338,1.0876348005936909,1.1686976337213109,1.2565028376501817,1.3496656718620579,1.44671690280864,1.5461259746023046,1.6463251467967646,1.7457342185904292,1.8427854495370106,1.9359482837488875,2.0237534876777583,2.1048163208053783,2.1778583738287352,2.2417277299384439,2.2954171312349496,2.3380798637873133,2.369043110817703,2.3878185634239024,2.3941101215023997,2.3941101215023997],"y":[-2.7334121032079635,-2.6334106511915056,-2.5349862818113191,-2.4396912061683977,-2.3490282845321495,-2.264427325336182,-2.1872225362451791,-2.1186314829030115,-2.0597358871951936,-2.0114645678485621,-1.9745787924051017,-1.9496602715780496,-1.9371019853264513,-1.9371019853264513,-1.9496602715780496,-1.9745787924051017,-2.0114645678485621,-2.0597358871951936,-2.1186314829030115,-2.1872225362451791,-2.264427325336182,-2.3490282845321495,-2.4396912061683982,-2.5349862818113191,-2.6334106511915056,-2.7334121032079639,-2.8334135552244213,-2.9318379246046078,-3.0271330002475292,-3.1177959218837774,-3.202396881079745,-3.2796016701707478,-3.3481927235129159,-3.4070883192207333,-3.4553596385673648,-3.4922454140108252,-3.5171639348378774,-3.5297222210894756,-3.5297222210894756,-3.5171639348378774,-3.4922454140108252,-3.4553596385673648,-3.4070883192207333,-3.3481927235129154,-3.2796016701707473,-3.2023968810797445,-3.1177959218837774,-3.0271330002475292,-2.9318379246046073,-2.8334135552244208,-2.7334121032079635,-2.7334121032079635],"text":"id: 8<br />as.factor(id): 8","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(170,163,0,0.6)","hoveron":"fills","name":"8","legendgroup":"8","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.85338407324789267,0.84893526986635826,0.8356590200086681,0.81376469806592522,0.78359759057420075,0.74563345083959542,0.7004709960244021,0.6488224650197989,0.59150238601306548,0.52941473089149338,0.46353865906571734,0.39491307554088739,0.32462024676412693,0.2537687326361458,0.1834759038593852,0.11485032033455539,0.048974248508779322,-0.013113406612792866,-0.07043348561952617,-0.12208201662412937,-0.16724447143932264,-0.20520861117392791,-0.23537571866565254,-0.25727004060839531,-0.27054629046608547,-0.27499509384761989,-0.27054629046608547,-0.25727004060839531,-0.23537571866565232,-0.20520861117392786,-0.16724447143932258,-0.12208201662412932,-0.070433485619526004,-0.013113406612792533,0.048974248508779628,0.11485032033455539,0.18347590385938531,0.25376873263614591,0.32462024676412715,0.39491307554088778,0.46353865906571723,0.52941473089149338,0.59150238601306548,0.64882246501979912,0.70047099602440221,0.74563345083959565,0.78359759057420075,0.81376469806592533,0.8356590200086681,0.84893526986635826,0.85338407324789267,0.85338407324789267],"y":[-3.116708012592142,-3.0459963077428034,-2.9763997687200643,-2.9090159745192694,-2.8449076078280937,-2.7850856958857388,-2.7304936659794135,-2.6819924670324382,-2.6403469919254192,-2.6062140146785948,-2.5801318327331995,-2.5625117776792528,-2.5536317283106658,-2.5536317283106658,-2.5625117776792528,-2.5801318327331995,-2.6062140146785948,-2.6403469919254192,-2.6819924670324382,-2.7304936659794135,-2.7850856958857388,-2.8449076078280937,-2.9090159745192699,-2.9763997687200643,-3.0459963077428038,-3.116708012592142,-3.1874197174414807,-3.2570162564642202,-3.3244000506650146,-3.3885084173561903,-3.4483303292985452,-3.5029223592048706,-3.5514235581518458,-3.5930690332588648,-3.6272020105056897,-3.6532841924510846,-3.6709042475050313,-3.6797842968736183,-3.6797842968736183,-3.6709042475050313,-3.6532841924510846,-3.6272020105056892,-3.5930690332588648,-3.5514235581518458,-3.5029223592048706,-3.4483303292985452,-3.3885084173561903,-3.3244000506650146,-3.2570162564642198,-3.1874197174414802,-3.116708012592142,-3.116708012592142],"text":"id: 9<br />as.factor(id): 9","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(151,168,0,0.6)","hoveron":"fills","name":"9","legendgroup":"9","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-0.1671765425453462,-0.17162534592688061,-0.18490159578457077,-0.20679591772731365,-0.23696302521903817,-0.27492716495364339,-0.32008961976883676,-0.37173815077344002,-0.42905822978017344,-0.49114588490174543,-0.55702195672752153,-0.62564754025235148,-0.69594036902911194,-0.76679188315709301,-0.8370847119338537,-0.90571029545868353,-0.97158636728445957,-1.0336740224060317,-1.0909941014127651,-1.1426426324173682,-1.1878050872325616,-1.2257692269671667,-1.2559363344588914,-1.2778306564016342,-1.2911069062593243,-1.2955557096408588,-1.2911069062593243,-1.2778306564016342,-1.2559363344588912,-1.2257692269671667,-1.1878050872325614,-1.1426426324173682,-1.0909941014127649,-1.0336740224060315,-0.97158636728445924,-0.90571029545868353,-0.83708471193385359,-0.7667918831570929,-0.69594036902911172,-0.62564754025235114,-0.55702195672752164,-0.49114588490174549,-0.42905822978017333,-0.3717381507734398,-0.3200896197688366,-0.27492716495364322,-0.23696302521903817,-0.20679591772731354,-0.18490159578457077,-0.17162534592688061,-0.1671765425453462,-0.1671765425453462],"y":[-2.6353603479807033,-2.5646486431313646,-2.4950521041086255,-2.4276683099078307,-2.363559943216655,-2.3037380312743001,-2.2491460013679747,-2.2006448024209995,-2.1589993273139805,-2.1248663500671556,-2.0987841681217607,-2.081164113067814,-2.072284063699227,-2.072284063699227,-2.081164113067814,-2.0987841681217607,-2.1248663500671561,-2.1589993273139805,-2.2006448024209995,-2.2491460013679747,-2.3037380312743001,-2.363559943216655,-2.4276683099078307,-2.4950521041086255,-2.5646486431313651,-2.6353603479807033,-2.7060720528300419,-2.7756685918527815,-2.8430523860535759,-2.9071607527447516,-2.9669826646871069,-3.0215746945934319,-3.0700758935404071,-3.1117213686474261,-3.1458543458942509,-3.1719365278396459,-3.1895565828935926,-3.1984366322621796,-3.1984366322621796,-3.1895565828935926,-3.1719365278396459,-3.1458543458942509,-3.1117213686474261,-3.0700758935404071,-3.0215746945934319,-2.966982664687106,-2.9071607527447516,-2.8430523860535759,-2.775668591852781,-2.7060720528300415,-2.6353603479807033,-2.6353603479807033],"text":"id: 10<br />as.factor(id): 10","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(130,173,0,0.6)","hoveron":"fills","name":"10","legendgroup":"10","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.3429945504725955,4.3352889969828938,4.3122938576953951,4.2743717796932943,4.2221208168002358,4.1563649979142552,4.0781413315798067,3.9886834517435377,3.889402162610013,3.7818631894166366,3.6677624860113354,3.5488994886472685,3.4271487377981797,3.3044303155353321,3.1826795646862429,3.063816567322176,2.9497158639168752,2.8421768907234983,2.7428956015899741,2.6534377217537051,2.5752140554192566,2.5094582365332756,2.4572072736402171,2.4192851956381163,2.396290056350618,2.3885845028609163,2.396290056350618,2.4192851956381163,2.4572072736402175,2.509458236533276,2.5752140554192566,2.6534377217537051,2.7428956015899741,2.8421768907234988,2.9497158639168761,3.063816567322176,3.1826795646862434,3.3044303155353325,3.4271487377981797,3.5488994886472689,3.6677624860113354,3.7818631894166361,3.889402162610013,3.9886834517435377,4.0781413315798067,4.1563649979142561,4.2221208168002358,4.2743717796932952,4.312293857695396,4.3352889969828938,4.3429945504725955,4.3429945504725955],"y":[-2.59346066971526,-2.4709844042263911,-2.3504396626080566,-2.233727507445515,-2.1226885591461424,-2.0190739682560723,-1.9245177987699971,-1.84051125796583,-1.768379179175128,-1.7092591283700354,-1.6640834640683555,-1.6335646334827589,-1.6181839368026465,-1.6181839368026465,-1.6335646334827589,-1.6640834640683555,-1.7092591283700356,-1.768379179175128,-1.84051125796583,-1.9245177987699971,-2.0190739682560723,-2.1226885591461424,-2.2337275074455154,-2.3504396626080566,-2.4709844042263915,-2.5934606697152605,-2.7159369352041289,-2.8364816768224639,-2.9531938319850051,-3.0642327802843776,-3.1678473711744481,-3.262403540660523,-3.3464100814646902,-3.4185421602553925,-3.4776622110604847,-3.5228378753621645,-3.5533567059477611,-3.5687374026278738,-3.5687374026278738,-3.5533567059477611,-3.522837875362165,-3.4776622110604847,-3.418542160255392,-3.3464100814646898,-3.2624035406605225,-3.1678473711744473,-3.0642327802843776,-2.9531938319850046,-2.8364816768224634,-2.7159369352041285,-2.5934606697152605,-2.59346066971526],"text":"id: 11<br />as.factor(id): 11","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(103,177,0,0.6)","hoveron":"fills","name":"11","legendgroup":"11","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.6659330063086779,3.6614842029271437,3.6482079530694533,3.6263136311267106,3.5961465236349861,3.5581823839003808,3.5130199290851873,3.4613713980805842,3.4040513190738508,3.3419636639522787,3.2760875921265029,3.2074620086016727,3.1371691798249124,3.0663176656969311,2.9960248369201707,2.9273992533953406,2.8615231815695648,2.7994355264479927,2.7421154474412592,2.6904669164366561,2.6453044616214627,2.6073403218868574,2.5771732143951329,2.5552788924523901,2.5420026425946998,2.5375538392131656,2.5420026425946998,2.5552788924523901,2.5771732143951329,2.6073403218868574,2.6453044616214627,2.6904669164366561,2.7421154474412592,2.7994355264479927,2.8615231815695648,2.9273992533953406,2.9960248369201707,3.0663176656969311,3.1371691798249124,3.2074620086016732,3.2760875921265025,3.3419636639522787,3.4040513190738508,3.4613713980805842,3.5130199290851878,3.5581823839003812,3.5961465236349861,3.6263136311267106,3.6482079530694533,3.6614842029271437,3.6659330063086779,3.6659330063086779],"y":[1.9984810061234295,2.0691927109727679,2.1387892499955075,2.2061730441963019,2.2702814108874776,2.3301033228298329,2.3846953527361583,2.4331965516831331,2.4748420267901525,2.5089750040369769,2.5350571859823718,2.552677241036319,2.5615572904049055,2.5615572904049055,2.552677241036319,2.5350571859823718,2.5089750040369769,2.4748420267901521,2.4331965516831331,2.3846953527361583,2.3301033228298329,2.2702814108874776,2.2061730441963019,2.1387892499955075,2.0691927109727679,1.9984810061234293,1.9277693012740911,1.8581727622513515,1.7907889680505567,1.7266806013593812,1.6668586894170261,1.6122666595107007,1.5637654605637257,1.5221199854567065,1.4879870082098821,1.4619048262644869,1.44428477121054,1.4354047218419534,1.4354047218419534,1.4442847712105402,1.4619048262644869,1.4879870082098821,1.5221199854567067,1.5637654605637259,1.6122666595107011,1.6668586894170265,1.7266806013593812,1.7907889680505571,1.8581727622513518,1.9277693012740913,1.9984810061234293,1.9984810061234295],"text":"id: 12<br />as.factor(id): 12","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(63,181,0,0.6)","hoveron":"fills","name":"12","legendgroup":"12","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.6104713365464063,4.6060225331648716,4.5927462833071813,4.5708519613644381,4.5406848538727136,4.5027207141381087,4.4575582593229157,4.4059097283183117,4.3485896493115783,4.2865019941900071,4.2206259223642304,4.1520003388394011,4.0817075100626399,4.0108559959346595,3.9405631671578987,3.8719375836330685,3.8060615118072927,3.7439738566857206,3.6866537776789872,3.6350052466743841,3.5898427918591906,3.5518786521245853,3.5217115446328608,3.4998172226901181,3.4865409728324277,3.4820921694508935,3.4865409728324277,3.4998172226901181,3.5217115446328608,3.5518786521245853,3.5898427918591906,3.6350052466743841,3.6866537776789872,3.7439738566857206,3.8060615118072931,3.8719375836330685,3.9405631671578987,4.0108559959346595,4.0817075100626408,4.1520003388394011,4.2206259223642304,4.2865019941900071,4.3485896493115792,4.4059097283183126,4.4575582593229157,4.5027207141381087,4.5406848538727136,4.5708519613644389,4.5927462833071813,4.6060225331648716,4.6104713365464063,4.6104713365464063],"y":[1.3811586531773474,1.4518703580266858,1.5214668970494254,1.58885069125022,1.6529590579413957,1.7127809698837506,1.767372999790076,1.8158741987370512,1.8575196738440702,1.8916526510908949,1.91773483303629,1.9353548880902367,1.9442349374588237,1.9442349374588237,1.9353548880902367,1.91773483303629,1.8916526510908946,1.8575196738440702,1.8158741987370512,1.767372999790076,1.7127809698837508,1.6529590579413957,1.5888506912502198,1.5214668970494254,1.4518703580266858,1.3811586531773472,1.310446948328009,1.2408504093052695,1.1734666151044748,1.1093582484132991,1.049536336470944,0.99494430656461874,0.94644310761764361,0.90479763251062439,0.87066465526379988,0.84458247331840486,0.82696241826445804,0.81808236889587127,0.81808236889587127,0.82696241826445815,0.84458247331840486,0.87066465526379999,0.90479763251062462,0.94644310761764383,0.99494430656461896,1.0495363364709445,1.1093582484132991,1.173466615104475,1.2408504093052697,1.3104469483280092,1.3811586531773472,1.3811586531773474],"text":"id: 13<br />as.factor(id): 13","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,185,41,0.6)","hoveron":"fills","name":"13","legendgroup":"13","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.77439276113146782,0.75774686309664485,0.70807168474796178,0.6261506333224911,0.51327565273847653,0.37122684886797286,0.20224441620386013,0.0089933086544635721,-0.20547878837154654,-0.43778952178464936,-0.68427521254321522,-0.94104863406056727,-1.2040603160903691,-1.4691624072914478,-1.7321740893212501,-1.9889475108386017,-2.2354332015971674,-2.4677439350102706,-2.6822160320362807,-2.8754671395856768,-3.0444495722497891,-3.1864983761202934,-3.299373356704308,-3.3812944081297784,-3.4309695864784615,-3.4476154845132845,-3.4309695864784615,-3.3812944081297784,-3.2993733567043075,-3.186498376120293,-3.0444495722497891,-2.8754671395856768,-2.6822160320362798,-2.4677439350102697,-2.2354332015971661,-1.9889475108386017,-1.7321740893212496,-1.4691624072914475,-1.2040603160903682,-0.94104863406056594,-0.68427521254321566,-0.43778952178464947,-0.20547878837154609,0.0089933086544642382,0.2022444162038608,0.37122684886797352,0.51327565273847653,0.62615063332249132,0.70807168474796178,0.75774686309664485,0.77439276113146782,0.77439276113146782],"y":[-5.2411883490944611,-4.9766093763135553,-4.7162029719852212,-4.4640759006649624,-4.2242043568809127,-4.0003712581708601,-3.7961065862128733,-3.6146317169055315,-3.4588086173456434,-3.3310947108974753,-3.2335041221583056,-3.1675759130103422,-3.1343498106954519,-3.1343498106954519,-3.1675759130103422,-3.2335041221583056,-3.3310947108974753,-3.4588086173456434,-3.6146317169055315,-3.7961065862128738,-4.0003712581708593,-4.2242043568809127,-4.4640759006649633,-4.7162029719852212,-4.9766093763135553,-5.241188349094462,-5.5057673218753669,-5.7661737262037018,-6.0183007975239597,-6.2581723413080095,-6.4820054400180629,-6.6862701119760484,-6.8677449812833906,-7.0235680808432797,-7.1512819872914477,-7.2488725760306165,-7.3148007851785799,-7.3480268874934698,-7.3480268874934698,-7.3148007851785799,-7.2488725760306174,-7.1512819872914468,-7.0235680808432788,-6.8677449812833906,-6.6862701119760484,-6.4820054400180611,-6.2581723413080095,-6.0183007975239589,-5.7661737262037001,-5.505767321875366,-5.241188349094462,-5.2411883490944611],"text":"id: 14<br />as.factor(id): 14","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,188,79,0.6)","hoveron":"fills","name":"14","legendgroup":"14","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.8341826295042756,-2.8404741875827728,-2.8592496401889722,-2.890212887219362,-2.9328756197717256,-2.9865650210682317,-3.0504343771779405,-3.1234764302012974,-3.2045392633289174,-3.2923444672577875,-3.3855073014696644,-3.4825585324162467,-3.5819676042099111,-3.6821667764043706,-3.7815758481980355,-3.8786270791446174,-3.9717899133564942,-4.0595951172853653,-4.1406579504129848,-4.2137000034363412,-4.2775693595460496,-4.3312587608425561,-4.3739214933949198,-4.4048847404253095,-4.4236601930315089,-4.4299517511100062,-4.4236601930315089,-4.4048847404253095,-4.3739214933949198,-4.3312587608425561,-4.2775693595460496,-4.2137000034363412,-4.1406579504129839,-4.0595951172853644,-3.9717899133564938,-3.8786270791446174,-3.7815758481980355,-3.6821667764043706,-3.5819676042099107,-3.4825585324162462,-3.3855073014696644,-3.292344467257788,-3.2045392633289169,-3.1234764302012969,-3.0504343771779401,-2.9865650210682313,-2.9328756197717256,-2.890212887219362,-2.8592496401889722,-2.8404741875827728,-2.8341826295042756,-2.8341826295042756],"y":[-3.4544270576526657,-3.3544256056362078,-3.2560012362560213,-3.1607061606130999,-3.0700432389768517,-2.9854422797808842,-2.9082374906898814,-2.8396464373477137,-2.7807508416398958,-2.7324795222932643,-2.6955937468498039,-2.6706752260227518,-2.6581169397711535,-2.6581169397711535,-2.6706752260227518,-2.6955937468498039,-2.7324795222932643,-2.7807508416398958,-2.8396464373477137,-2.9082374906898814,-2.9854422797808842,-3.0700432389768517,-3.1607061606131004,-3.2560012362560213,-3.3544256056362078,-3.4544270576526661,-3.5544285096691235,-3.65285287904931,-3.7481479546922314,-3.8388108763284796,-3.9234118355244472,-4.00061662461545,-4.0692076779576176,-4.1281032736654355,-4.176374593012067,-4.2132603684555274,-4.2381788892825796,-4.2507371755341774,-4.2507371755341774,-4.2381788892825796,-4.2132603684555274,-4.176374593012067,-4.1281032736654355,-4.0692076779576176,-4.00061662461545,-3.9234118355244467,-3.8388108763284796,-3.7481479546922314,-3.6528528790493096,-3.554428509669123,-3.4544270576526657,-3.4544270576526657],"text":"id: 15<br />as.factor(id): 15","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,190,107,0.6)","hoveron":"fills","name":"15","legendgroup":"15","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-4.1831904321204991,-4.1876392355020338,-4.2009154853597241,-4.2228098073024665,-4.2529769147941918,-4.2909410545287967,-4.3361035093439897,-4.3877520403485937,-4.4450721193553271,-4.5071597744768983,-4.573035846302675,-4.6416614298275043,-4.7119542586042655,-4.7828057727322459,-4.8530986015090072,-4.9217241850338365,-4.9876002568596132,-5.0496879119811853,-5.1070079909879187,-5.1586565219925218,-5.2038189768077148,-5.2417831165423197,-5.271950224034045,-5.2938445459767873,-5.3071207958344777,-5.3115695992160123,-5.3071207958344777,-5.2938445459767873,-5.2719502240340441,-5.2417831165423197,-5.2038189768077148,-5.1586565219925218,-5.1070079909879178,-5.0496879119811844,-4.9876002568596123,-4.9217241850338365,-4.8530986015090072,-4.7828057727322459,-4.7119542586042646,-4.6416614298275043,-4.573035846302675,-4.5071597744768983,-4.4450721193553262,-4.3877520403485928,-4.3361035093439897,-4.2909410545287967,-4.2529769147941918,-4.2228098073024665,-4.2009154853597241,-4.1876392355020338,-4.1831904321204991,-4.1831904321204991],"y":[-2.6725554081516205,-2.6018437033022819,-2.5322471642795428,-2.4648633700787479,-2.4007550033875722,-2.3409330914452173,-2.2863410615388919,-2.2378398625919167,-2.1961943874848977,-2.1620614102380733,-2.1359792282926779,-2.1183591732387312,-2.1094791238701442,-2.1094791238701442,-2.1183591732387312,-2.1359792282926779,-2.1620614102380733,-2.1961943874848977,-2.2378398625919167,-2.2863410615388919,-2.3409330914452173,-2.4007550033875722,-2.4648633700787483,-2.5322471642795428,-2.6018437033022823,-2.6725554081516205,-2.7432671130009592,-2.8128636520236987,-2.8802474462244931,-2.9443558129156688,-3.0041777248580237,-3.0587697547643491,-3.1072709537113243,-3.1489164288183433,-3.1830494060651682,-3.2091315880105631,-3.2267516430645098,-3.2356316924330968,-3.2356316924330968,-3.2267516430645098,-3.2091315880105631,-3.1830494060651677,-3.1489164288183433,-3.1072709537113243,-3.0587697547643491,-3.0041777248580237,-2.9443558129156688,-2.8802474462244931,-2.8128636520236983,-2.7432671130009587,-2.6725554081516205,-2.6725554081516205],"text":"id: 16<br />as.factor(id): 16","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,191,130,0.6)","hoveron":"fills","name":"16","legendgroup":"16","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-4.9621869550307869,-4.9666357584123215,-4.9799120082700119,-5.0018063302127551,-5.0319734377044796,-5.0699375774390845,-5.1151000322542775,-5.1667485632588814,-5.2240686422656148,-5.2861562973871861,-5.3520323692129628,-5.4206579527377921,-5.4909507815145533,-5.5618022956425337,-5.6320951244192949,-5.7007207079441242,-5.7665967797699009,-5.828684434891473,-5.8860045138982064,-5.9376530449028095,-5.9828154997180025,-6.0207796394526074,-6.0509467469443328,-6.0728410688870751,-6.0861173187447655,-6.0905661221263001,-6.0861173187447655,-6.0728410688870751,-6.0509467469443319,-6.0207796394526074,-5.9828154997180025,-5.9376530449028095,-5.8860045138982056,-5.8286844348914721,-5.7665967797699,-5.7007207079441242,-5.6320951244192949,-5.5618022956425337,-5.4909507815145524,-5.4206579527377921,-5.3520323692129628,-5.2861562973871861,-5.224068642265614,-5.1667485632588805,-5.1151000322542775,-5.0699375774390845,-5.0319734377044796,-5.0018063302127542,-4.9799120082700119,-4.9666357584123215,-4.9621869550307869,-4.9621869550307869],"y":[-1.8562197161542393,-1.7855080113049009,-1.7159114722821613,-1.6485276780813667,-1.584419311390191,-1.5245973994478361,-1.4700053695415107,-1.4215041705945355,-1.3798586954875165,-1.3457257182406919,-1.3196435362952967,-1.30202348124135,-1.293143431872763,-1.293143431872763,-1.30202348124135,-1.3196435362952967,-1.3457257182406921,-1.3798586954875165,-1.4215041705945355,-1.4700053695415107,-1.5245973994478359,-1.584419311390191,-1.6485276780813669,-1.7159114722821613,-1.7855080113049009,-1.8562197161542395,-1.9269314210035777,-1.9965279600263173,-2.0639117542271119,-2.1280201209182876,-2.1878420328606429,-2.2424340627669679,-2.2909352617139431,-2.3325807368209621,-2.366713714067787,-2.3927958960131819,-2.4104159510671286,-2.4192960004357156,-2.4192960004357156,-2.4104159510671286,-2.3927958960131819,-2.366713714067787,-2.3325807368209621,-2.2909352617139431,-2.2424340627669679,-2.187842032860642,-2.1280201209182876,-2.0639117542271119,-1.9965279600263171,-1.9269314210035775,-1.8562197161542395,-1.8562197161542393],"text":"id: 17<br />as.factor(id): 17","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,192,151,0.6)","hoveron":"fills","name":"17","legendgroup":"17","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-5.4573276736180141,-5.4636192316965113,-5.4823946843027107,-5.5133579313331005,-5.5560206638854641,-5.6097100651819698,-5.673579421291679,-5.7466214743150363,-5.8276843074426559,-5.9154895113715265,-6.0086523455834033,-6.1057035765299847,-6.2051126483236496,-6.3053118205181091,-6.404720892311774,-6.5017721232583563,-6.5949349574702323,-6.6827401613991029,-6.7638029945267224,-6.8368450475500797,-6.9007144036597889,-6.9544038049562946,-6.9970665375086583,-7.028029784539048,-7.0468052371452474,-7.0530967952237447,-7.0468052371452474,-7.028029784539048,-6.9970665375086583,-6.9544038049562946,-6.9007144036597881,-6.8368450475500797,-6.7638029945267224,-6.6827401613991029,-6.5949349574702323,-6.5017721232583563,-6.404720892311774,-6.3053118205181091,-6.2051126483236487,-6.1057035765299847,-6.0086523455834033,-5.9154895113715265,-5.8276843074426559,-5.7466214743150354,-5.6735794212916781,-5.6097100651819698,-5.5560206638854641,-5.5133579313331005,-5.4823946843027107,-5.4636192316965113,-5.4573276736180141,-5.4573276736180141],"y":[-0.70554838898279271,-0.60554693696633488,-0.50712256758614838,-0.41182749194322715,-0.32116457030697881,-0.23656361111101132,-0.15935882202000828,-0.090767768677840532,-0.031872172970022983,0.016399146376608553,0.053284921820069053,0.078203442647121202,0.090761728898719363,0.090761728898719363,0.078203442647121091,0.053284921820069053,0.016399146376608442,-0.031872172970022983,-0.090767768677840532,-0.15935882202000851,-0.23656361111101126,-0.32116457030697892,-0.41182749194322732,-0.50712256758614838,-0.60554693696633499,-0.70554838898279293,-0.80554984099925053,-0.90397421037943726,-0.99926928602235854,-1.0899322076586067,-1.1745331668545742,-1.2517379559455772,-1.3203290092877449,-1.3792246049955628,-1.4274959243421943,-1.4643816997856545,-1.4893002206127066,-1.5018585068643047,-1.5018585068643047,-1.4893002206127064,-1.4643816997856547,-1.4274959243421939,-1.3792246049955623,-1.3203290092877449,-1.2517379559455768,-1.1745331668545735,-1.0899322076586067,-0.99926928602235821,-0.90397421037943682,-0.8055498409992502,-0.70554838898279293,-0.70554838898279271],"text":"id: 18<br />as.factor(id): 18","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,193,170,0.6)","hoveron":"fills","name":"18","legendgroup":"18","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-5.4662124269064671,-5.4706612302880018,-5.4839374801456922,-5.5058318020884354,-5.5359989095801598,-5.5739630493147647,-5.6191255041299577,-5.6707740351345617,-5.7280941141412951,-5.7901817692628663,-5.856057841088643,-5.9246834246134723,-5.9949762533902335,-6.065827767518214,-6.1361205962949752,-6.2047461798198045,-6.2706222516455812,-6.3327099067671533,-6.3900299857738867,-6.4416785167784898,-6.4868409715936828,-6.5248051113282877,-6.554972218820013,-6.5768665407627553,-6.5901427906204457,-6.5945915940019804,-6.5901427906204457,-6.5768665407627553,-6.5549722188200121,-6.5248051113282877,-6.4868409715936828,-6.4416785167784898,-6.3900299857738858,-6.3327099067671524,-6.2706222516455803,-6.2047461798198045,-6.1361205962949752,-6.065827767518214,-5.9949762533902327,-5.9246834246134723,-5.856057841088643,-5.7901817692628663,-5.7280941141412942,-5.6707740351345608,-5.6191255041299577,-5.5739630493147647,-5.5359989095801598,-5.5058318020884345,-5.4839374801456922,-5.4706612302880018,-5.4662124269064671,-5.4662124269064671],"y":[0.63784520111228515,0.70855690596162368,0.77815344498436312,0.84553723918515766,0.90964560587633336,0.96946751781868845,1.0240595477250138,1.0725607466719889,1.1142062217790081,1.1483391990258327,1.1744213809712276,1.1920414360251745,1.2009214853937613,1.2009214853937613,1.1920414360251743,1.1744213809712276,1.1483391990258325,1.1142062217790079,1.0725607466719889,1.0240595477250136,0.96946751781868845,0.90964560587633336,0.84553723918515755,0.77815344498436312,0.70855690596162357,0.63784520111228493,0.56713349626294662,0.49753695724020708,0.43015316303941242,0.36604479634823683,0.30622288440588175,0.25163085449955647,0.20312965555258133,0.16148418044556218,0.12735120319873761,0.10126902125334258,0.083648966199395769,0.074768916830808996,0.074768916830808996,0.08364896619939588,0.10126902125334258,0.12735120319873772,0.16148418044556234,0.2031296555525815,0.25163085449955669,0.30622288440588219,0.36604479634823689,0.43015316303941264,0.49753695724020741,0.56713349626294696,0.63784520111228504,0.63784520111228515],"text":"id: 19<br />as.factor(id): 19","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,192,188,0.6)","hoveron":"fills","name":"19","legendgroup":"19","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-4.8383885925086423,-4.846094145998344,-4.8690892852858427,-4.9070113632879435,-4.959262326181002,-5.0250181450669826,-5.1032418114014311,-5.1926996912377001,-5.2919809803712248,-5.3995199535646012,-5.513620656969902,-5.6324836543339698,-5.7542344051830581,-5.8769528274459057,-5.9987035782949949,-6.1175665756590618,-6.2316672790643626,-6.339206252257739,-6.4384875413912637,-6.5279454212275327,-6.6061690875619812,-6.6719249064479618,-6.7241758693410212,-6.7620979473431211,-6.7850930866306198,-6.7927986401203215,-6.7850930866306198,-6.7620979473431211,-6.7241758693410203,-6.6719249064479618,-6.6061690875619812,-6.5279454212275327,-6.4384875413912637,-6.339206252257739,-6.2316672790643617,-6.1175665756590618,-5.9987035782949949,-5.8769528274459057,-5.7542344051830581,-5.6324836543339689,-5.5136206569699029,-5.3995199535646012,-5.2919809803712248,-5.1926996912377001,-5.1032418114014311,-5.0250181450669817,-4.959262326181002,-4.9070113632879426,-4.8690892852858418,-4.846094145998344,-4.8383885925086423,-4.8383885925086423],"y":[2.1641985970812039,2.2866748625700728,2.4072196041884073,2.5239317593509489,2.6349707076503215,2.7385852985403916,2.8331414680264668,2.9171480088306341,2.9892800876213359,3.0484001384264285,3.0935758027281084,3.124094633313705,3.1394753299938172,3.1394753299938172,3.124094633313705,3.0935758027281084,3.0484001384264285,2.9892800876213359,2.9171480088306341,2.8331414680264668,2.738585298540392,2.6349707076503215,2.5239317593509485,2.4072196041884073,2.2866748625700724,2.1641985970812034,2.041722331592335,1.921177589974,1.8044654348114586,1.6934264865120863,1.5898118956220157,1.4952557261359409,1.4112491853317737,1.3391171065410716,1.2799970557359792,1.2348213914342994,1.2043025608487028,1.1889218641685904,1.1889218641685904,1.2043025608487028,1.2348213914342991,1.2799970557359792,1.3391171065410719,1.4112491853317741,1.4952557261359414,1.5898118956220166,1.6934264865120863,1.804465434811459,1.9211775899740007,2.0417223315923354,2.1641985970812034,2.1641985970812039],"text":"id: 20<br />as.factor(id): 20","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,190,205,0.6)","hoveron":"fills","name":"20","legendgroup":"20","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-3.8819767420986273,-3.8864255454801615,-3.8997017953378519,-3.9215961172805947,-3.9517632247723191,-3.9897273645069244,-4.0348898193221174,-4.0865383503267214,-4.1438584293334548,-4.205946084455026,-4.2718221562808028,-4.340447739805632,-4.4107405685823933,-4.4815920827103737,-4.5518849114871349,-4.6205104950119642,-4.6863865668377409,-4.748474221959313,-4.8057943009660464,-4.8574428319706495,-4.9026052867858425,-4.9405694265204474,-4.9707365340121727,-4.9926308559549151,-5.0059071058126055,-5.0103559091941401,-5.0059071058126055,-4.9926308559549151,-4.9707365340121719,-4.9405694265204474,-4.9026052867858425,-4.8574428319706495,-4.8057943009660455,-4.7484742219593121,-4.68638656683774,-4.6205104950119642,-4.5518849114871349,-4.4815920827103737,-4.4107405685823924,-4.340447739805632,-4.2718221562808028,-4.205946084455026,-4.1438584293334539,-4.0865383503267205,-4.0348898193221174,-3.9897273645069244,-3.9517632247723191,-3.9215961172805947,-3.8997017953378519,-3.8864255454801615,-3.8819767420986273,-3.8819767420986273],"y":[2.8717056508638334,2.9424173557131721,3.0120138947359112,3.0793976889367061,3.1435060556278818,3.2033279675702366,3.257919997476562,3.3064211964235373,3.3480666715305563,3.3821996487773811,3.408281830722776,3.4259018857767227,3.4347819351453097,3.4347819351453097,3.4259018857767227,3.408281830722776,3.3821996487773807,3.3480666715305563,3.3064211964235373,3.257919997476562,3.2033279675702366,3.1435060556278818,3.0793976889367061,3.0120138947359112,2.9424173557131716,2.8717056508638334,2.8009939460144948,2.7313974069917553,2.6640136127909608,2.5999052460997851,2.5400833341574298,2.4854913042511049,2.4369901053041296,2.3953446301971106,2.3612116529502858,2.3351294710048909,2.3175094159509442,2.3086293665823572,2.3086293665823572,2.3175094159509442,2.3351294710048909,2.3612116529502858,2.3953446301971106,2.4369901053041296,2.4854913042511049,2.5400833341574307,2.5999052460997851,2.6640136127909608,2.7313974069917557,2.8009939460144953,2.8717056508638334,2.8717056508638334],"text":"id: 21<br />as.factor(id): 21","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,187,220,0.6)","hoveron":"fills","name":"21","legendgroup":"21","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.8257559115909627,-2.8302047149724969,-2.8434809648301873,-2.8653752867729301,-2.8955423942646545,-2.9335065339992599,-2.9786689888144533,-3.0303175198190564,-3.0876375988257898,-3.1497252539473619,-3.2156013257731377,-3.2842269092979679,-3.3545197380747283,-3.4253712522027095,-3.4956640809794699,-3.5642896645043001,-3.6301657363300759,-3.692253391451648,-3.7495734704583814,-3.8012220014629845,-3.8463844562781779,-3.8843485960127833,-3.9145157035045077,-3.9364100254472505,-3.9496862753049409,-3.9541350786864751,-3.9496862753049409,-3.9364100254472505,-3.9145157035045077,-3.8843485960127833,-3.8463844562781779,-3.8012220014629845,-3.7495734704583814,-3.692253391451648,-3.6301657363300759,-3.5642896645043001,-3.4956640809794699,-3.4253712522027095,-3.3545197380747283,-3.2842269092979675,-3.2156013257731382,-3.1497252539473619,-3.0876375988257898,-3.0303175198190564,-2.9786689888144529,-2.9335065339992594,-2.8955423942646545,-2.8653752867729301,-2.8434809648301873,-2.8302047149724969,-2.8257559115909627,-2.8257559115909627],"y":[3.2687410421539047,3.3394527470032433,3.4090492860259825,3.4764330802267773,3.540541446917953,3.6003633588603079,3.6549553887666333,3.7034565877136085,3.7451020628206275,3.7792350400674524,3.8053172220128473,3.822937277066794,3.831817326435381,3.831817326435381,3.822937277066794,3.8053172220128473,3.7792350400674519,3.7451020628206275,3.7034565877136085,3.6549553887666333,3.6003633588603079,3.540541446917953,3.4764330802267773,3.4090492860259825,3.3394527470032429,3.2687410421539047,3.1980293373045661,3.1284327982818265,3.0610490040810321,2.9969406373898564,2.9371187254475011,2.8825266955411761,2.8340254965942009,2.7923800214871819,2.758247044240357,2.7321648622949621,2.7145448072410154,2.7056647578724284,2.7056647578724284,2.7145448072410154,2.7321648622949621,2.758247044240357,2.7923800214871819,2.8340254965942009,2.8825266955411761,2.937118725447502,2.9969406373898564,3.0610490040810321,3.128432798281827,3.1980293373045665,3.2687410421539047,3.2687410421539047],"text":"id: 22<br />as.factor(id): 22","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,183,233,0.6)","hoveron":"fills","name":"22","legendgroup":"22","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-0.8918617468148764,-0.90075935357794523,-0.92731185329332555,-0.97110049717881131,-1.0314347121622602,-1.1073629916314709,-1.1976879012618575,-1.3009849632710639,-1.4156251212845308,-1.539800431527675,-1.6715525751792271,-1.808803742228887,-1.9493893997824079,-2.0910924280383703,-2.2316780855918914,-2.3689292526415509,-2.5006813962931029,-2.6248567065362476,-2.739496864549714,-2.8427939265589206,-2.933118836189307,-3.0090471156585177,-3.0693813306419671,-3.1131699745274526,-3.1397224742428325,-3.1486200810059017,-3.1397224742428325,-3.1131699745274526,-3.0693813306419662,-3.0090471156585172,-2.933118836189307,-2.8427939265589206,-2.7394968645497135,-2.6248567065362467,-2.5006813962931025,-2.3689292526415509,-2.231678085591891,-2.0910924280383698,-1.9493893997824074,-1.8088037422288863,-1.6715525751792273,-1.539800431527675,-1.4156251212845308,-1.3009849632710635,-1.1976879012618573,-1.1073629916314705,-1.0314347121622602,-0.97110049717881108,-0.92731185329332555,-0.90075935357794523,-0.8918617468148764,-0.8918617468148764],"y":[4.2630741532470839,4.4044975629457612,4.5436906409912394,4.6784582293928292,4.8066749627751806,4.9263187866598903,5.0355028464725411,5.1325052443664916,5.2157961945805296,5.2840621490741793,5.3362265129649691,5.3714666230728625,5.3892267218100365,5.3892267218100365,5.3714666230728625,5.3362265129649691,5.2840621490741784,5.2157961945805296,5.1325052443664916,5.0355028464725411,4.9263187866598903,4.8066749627751806,4.6784582293928292,4.5436906409912394,4.4044975629457603,4.2630741532470839,4.1216507435484067,3.982457665502928,3.8476900771013387,3.7194733437189873,3.5998295198342771,3.4906454600216268,3.3936430621276763,3.3103521119136379,3.2420861574199886,3.1899217935291988,3.1546816834213054,3.1369215846841314,3.1369215846841314,3.1546816834213054,3.1899217935291988,3.2420861574199891,3.3103521119136383,3.3936430621276767,3.4906454600216268,3.599829519834278,3.7194733437189873,3.8476900771013391,3.9824576655029285,4.1216507435484075,4.2630741532470839,4.2630741532470839],"text":"id: 23<br />as.factor(id): 23","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,177,244,0.6)","hoveron":"fills","name":"23","legendgroup":"23","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.5908980809873925,1.5864492776058579,1.5731730277481679,1.5512787058054249,1.5211115983137005,1.4831474585790951,1.4379850037639019,1.3863364727592986,1.3290163937525652,1.2669287386309931,1.2010526668052171,1.1324270832803871,1.0621342545036265,0.99128274037564557,0.92098991159888488,0.85236432807405516,0.78648825624827901,0.72440060112670679,0.6670805221199736,0.61543199111537028,0.57026953630017707,0.53230539656557174,0.50213828907384717,0.4802439671311044,0.46696771727341424,0.46251891389187982,0.46696771727341424,0.4802439671311044,0.50213828907384739,0.53230539656557185,0.57026953630017707,0.61543199111537039,0.66708052211997371,0.72440060112670723,0.78648825624827934,0.85236432807405516,0.92098991159888499,0.99128274037564568,1.062134254503627,1.1324270832803875,1.2010526668052168,1.2669287386309931,1.3290163937525652,1.3863364727592988,1.4379850037639019,1.4831474585790954,1.5211115983137005,1.5512787058054252,1.5731730277481679,1.5864492776058579,1.5908980809873925,1.5908980809873925],"y":[-3.9707060413505131,-3.8999943365011744,-3.8303977974784353,-3.7630140032776405,-3.6989056365864648,-3.6390837246441099,-3.5844916947377845,-3.5359904957908093,-3.4943450206837903,-3.4602120434369654,-3.4341298614915705,-3.4165098064376238,-3.4076297570690368,-3.4076297570690368,-3.4165098064376238,-3.4341298614915705,-3.4602120434369659,-3.4943450206837903,-3.5359904957908093,-3.5844916947377845,-3.6390837246441099,-3.6989056365864648,-3.7630140032776405,-3.8303977974784353,-3.8999943365011749,-3.9707060413505131,-4.0414177461998513,-4.1110142852225913,-4.1783980794233857,-4.242506446114561,-4.3023283580569167,-4.3569203879632417,-4.4054215869102169,-4.4470670620172363,-4.4812000392640607,-4.5072822212094561,-4.5249022762634024,-4.5337823256319894,-4.5337823256319894,-4.5249022762634024,-4.5072822212094561,-4.4812000392640607,-4.4470670620172363,-4.4054215869102169,-4.3569203879632417,-4.3023283580569158,-4.242506446114561,-4.1783980794233857,-4.1110142852225913,-4.0414177461998513,-3.9707060413505131,-3.9707060413505131],"text":"id: 24<br />as.factor(id): 24","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(0,170,254,0.6)","hoveron":"fills","name":"24","legendgroup":"24","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.9395492632049063,3.9269661470479118,3.889415241835513,3.8274887477747335,3.7421632826700062,3.634784480076994,3.5070457678575768,3.3609616618108626,3.1988359955556231,3.023225587697882,2.8368999192741287,2.6427974573809645,2.4439793137936356,2.2435809694047157,2.0447628258173864,1.8506603639242225,1.6643346955004694,1.4887242876427278,1.3265986213874887,1.1805145153407743,1.0527758031213574,0.94539700052834541,0.86007153542361747,0.79814504136283815,0.76059413615043936,0.74801101999344488,0.76059413615043936,0.79814504136283815,0.86007153542361792,0.94539700052834541,1.0527758031213577,1.1805145153407746,1.326598621387489,1.4887242876427287,1.6643346955004703,1.8506603639242225,2.0447628258173869,2.2435809694047162,2.4439793137936361,2.6427974573809654,2.8368999192741282,3.023225587697882,3.1988359955556236,3.3609616618108631,3.5070457678575773,3.6347844800769948,3.7421632826700062,3.8274887477747339,3.8894152418355135,3.9269661470479118,3.9395492632049063,3.9395492632049063],"y":[-5.6826475976307549,-5.4826446935978392,-5.2857959548374662,-5.0952058035516234,-4.913879960279127,-4.7446780418871919,-4.5902684637051863,-4.453086357020851,-4.3352951656052152,-4.2387525269119521,-4.1649809760250314,-4.1151439343709271,-4.0900273618677305,-4.0900273618677305,-4.1151439343709271,-4.1649809760250314,-4.2387525269119521,-4.3352951656052152,-4.453086357020851,-4.5902684637051863,-4.7446780418871919,-4.913879960279127,-5.0952058035516243,-5.2857959548374662,-5.4826446935978392,-5.6826475976307558,-5.8826505016636705,-6.0794992404240435,-6.2700893917098863,-6.4514152349823828,-6.6206171533743179,-6.7750267315563235,-6.9122088382406597,-7.0300000296562946,-7.1265426683495576,-7.2003142192364784,-7.2501512608905827,-7.2752678333937792,-7.2752678333937792,-7.2501512608905827,-7.2003142192364784,-7.1265426683495576,-7.0300000296562946,-6.9122088382406588,-6.7750267315563226,-6.620617153374317,-6.4514152349823828,-6.2700893917098863,-6.0794992404240427,-5.8826505016636697,-5.6826475976307549,-5.6826475976307549],"text":"id: 25<br />as.factor(id): 25","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(48,162,255,0.6)","hoveron":"fills","name":"25","legendgroup":"25","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[8.5449647916457376,8.526090117410245,8.4697637595916468,8.3768740185004784,8.2488858208433875,8.0878176169538687,7.8962095486247428,7.6770833895546717,7.4338948901718123,7.1704792783852014,6.8909907757495708,6.5998370829098247,6.301609867528831,6.0010123509454516,5.7027851355644579,5.4116314427247119,5.1321429400890821,4.8687273283024703,4.6255388289196118,4.4064126698495398,4.2148046015204148,4.053736397630896,3.9257481999738046,3.8328584588826358,3.7765321010640376,3.7576574268285459,3.7765321010640376,3.8328584588826358,3.9257481999738051,4.0537363976308969,4.2148046015204148,4.4064126698495407,4.6255388289196118,4.8687273283024712,5.1321429400890839,5.4116314427247119,5.7027851355644588,6.0010123509454525,6.3016098675288328,6.5998370829098265,6.8909907757495708,7.1704792783852014,7.4338948901718132,7.6770833895546726,7.8962095486247446,8.0878176169538705,8.2488858208433875,8.3768740185004784,8.4697637595916486,8.526090117410245,8.5449647916457376,8.5449647916457376],"y":[-4.4917646312283344,-4.1917602751789609,-3.8964871670384014,-3.6106019401096376,-3.338613175200893,-3.0848102976129903,-2.8531959303399814,-2.6474227703134781,-2.4707359831900253,-2.3259220251501307,-2.2152646988197495,-2.1405091363385931,-2.1028342775837983,-2.1028342775837983,-2.1405091363385931,-2.2152646988197495,-2.3259220251501311,-2.4707359831900257,-2.6474227703134781,-2.8531959303399819,-3.0848102976129903,-3.3386131752008934,-3.6106019401096381,-3.8964871670384014,-4.1917602751789609,-4.4917646312283352,-4.7917689872777078,-5.0870420954182674,-5.3729273223470315,-5.6449160872557762,-5.8987189648436793,-6.1303333321166873,-6.3361064921431911,-6.5127932792666439,-6.6576072373065385,-6.7682645636369188,-6.8430201261180752,-6.8806949848728705,-6.8806949848728705,-6.8430201261180752,-6.7682645636369196,-6.6576072373065376,-6.512793279266643,-6.3361064921431902,-6.1303333321166864,-5.8987189648436766,-5.6449160872557762,-5.3729273223470306,-5.0870420954182665,-4.7917689872777069,-4.4917646312283352,-4.4917646312283344],"text":"id: 26<br />as.factor(id): 26","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(114,153,255,0.6)","hoveron":"fills","name":"26","legendgroup":"26","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.49994037017148274,0.49223481668178093,0.46923967739428241,0.43131759939218162,0.37906663649912309,0.31331081761314239,0.23508715127869373,0.14562927144242455,0.046347982308900026,-0.061190990884476659,-0.17529169428977748,-0.29415469165384461,-0.41590544250293343,-0.53862386476578084,-0.66037461561486988,-0.77923761297893679,-0.89333831638423766,-1.0008772895776146,-1.100158578711139,-1.1896164585474081,-1.2678401248818565,-1.3335959437678373,-1.385846906660896,-1.4237689846629966,-1.4467641239504951,-1.4544696774401968,-1.4467641239504951,-1.4237689846629966,-1.3858469066608956,-1.3335959437678371,-1.2678401248818565,-1.1896164585474081,-1.1001585787111385,-1.0008772895776141,-0.893338316384237,-0.77923761297893679,-0.66037461561486976,-0.53862386476578061,-0.41590544250293304,-0.294154691653844,-0.1752916942897777,-0.061190990884476715,0.046347982308900249,0.14562927144242488,0.23508715127869406,0.31331081761314272,0.37906663649912309,0.43131759939218173,0.46923967739428252,0.49223481668178093,0.49994037017148274,0.49994037017148274],"y":[5.6958019868562086,5.8182782523450776,5.9388229939634121,6.0555351491259533,6.1665740974253263,6.2701886883153968,6.3647448578014716,6.4487513986056388,6.5208834773963407,6.5800035282014333,6.6251791925031132,6.6556980230887097,6.6710787197688219,6.6710787197688219,6.6556980230887097,6.6251791925031132,6.5800035282014333,6.5208834773963407,6.4487513986056388,6.3647448578014716,6.2701886883153968,6.1665740974253263,6.0555351491259533,5.9388229939634121,5.8182782523450776,5.6958019868562086,5.5733257213673397,5.4527809797490052,5.3360688245864631,5.225029876287091,5.1214152853970205,5.0268591159109457,4.9428525751067784,4.8707204963160766,4.811600445510984,4.7664247812093041,4.7359059506237076,4.7205252539435953,4.7205252539435953,4.7359059506237076,4.7664247812093041,4.811600445510984,4.8707204963160766,4.9428525751067784,5.0268591159109457,5.1214152853970214,5.225029876287091,5.336068824586464,5.4527809797490052,5.5733257213673406,5.6958019868562086,5.6958019868562086],"text":"id: 27<br />as.factor(id): 27","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(152,143,255,0.6)","hoveron":"fills","name":"27","legendgroup":"27","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.2335001527458243,4.21808904576642,4.1720987671914234,4.0962546111872218,3.9917526854011047,3.8602410476291431,3.7037937149602458,3.5248779552877076,3.3263153770206584,3.111237430633905,2.8830360238233035,2.6453100290951692,2.4018085273969914,2.1563716828712964,1.9128701811731186,1.6751441864449848,1.4469427796343832,1.2318648332476292,1.0333022549805806,0.85438649530804223,0.69793916263914535,0.56642752486718373,0.46192559908106645,0.38608144307686509,0.34009116450186805,0.32468005752246443,0.34009116450186805,0.38608144307686509,0.4619255990810669,0.56642752486718395,0.69793916263914557,0.85438649530804245,1.033302254980581,1.2318648332476301,1.4469427796343841,1.6751441864449848,1.912870181173119,2.1563716828712969,2.4018085273969922,2.6453100290951701,2.883036023823303,3.111237430633905,3.3263153770206588,3.5248779552877081,3.7037937149602467,3.8602410476291436,3.9917526854011047,4.0962546111872218,4.1720987671914234,4.21808904576642,4.2335001527458243,4.2335001527458243],"y":[6.6942384499206256,6.9391909808983634,7.1802804641350324,7.4137047744601157,7.6357826710588608,7.8430118528390009,8.0321241918111514,8.200137273419486,8.3444014310008896,8.4626415326110749,8.5529928612144346,8.6140305223856277,8.6447919157458522,8.6447919157458522,8.6140305223856277,8.5529928612144346,8.4626415326110749,8.3444014310008896,8.200137273419486,8.0321241918111514,7.8430118528390018,7.6357826710588608,7.4137047744601148,7.1802804641350324,6.9391909808983625,6.6942384499206247,6.4492859189428877,6.2081964357062178,5.9747721253811354,5.7526942287823903,5.5454650470022493,5.3563527080300997,5.1883396264217652,5.0440754688403615,4.9258353672301762,4.8354840386268165,4.7744463774556234,4.743684984095399,4.743684984095399,4.7744463774556234,4.8354840386268165,4.9258353672301762,5.0440754688403615,5.188339626421766,5.3563527080301006,5.5454650470022511,5.7526942287823903,5.9747721253811363,6.2081964357062187,6.4492859189428886,6.6942384499206247,6.6942384499206256],"text":"id: 28<br />as.factor(id): 28","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(181,132,255,0.6)","hoveron":"fills","name":"28","legendgroup":"28","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[7.9994499747486785,7.9805753005131868,7.9242489426945886,7.8313592016034193,7.7033710039463283,7.5423028000568095,7.3506947317276836,7.1315685726576126,6.8883800732747531,6.6249644614881422,6.3454759588525116,6.0543222660127656,5.7560950506317718,5.4554975340483924,5.1572703186673987,4.8661166258276527,4.586628123192023,4.3232125114054112,4.0800240120225526,3.8608978529524807,3.6692897846233556,3.5082215807338373,3.3802333830767455,3.2873436419855766,3.2310172841669784,3.2121426099314867,3.2310172841669784,3.2873436419855766,3.3802333830767459,3.5082215807338377,3.6692897846233556,3.8608978529524811,4.0800240120225526,4.3232125114054121,4.5866281231920247,4.8661166258276527,5.1572703186673996,5.4554975340483933,5.7560950506317736,6.0543222660127674,6.3454759588525116,6.6249644614881422,6.888380073274754,7.1315685726576135,7.3506947317276854,7.5423028000568104,7.7033710039463283,7.8313592016034193,7.9242489426945895,7.9805753005131868,7.9994499747486785,7.9994499747486785],"y":[3.8944734340088334,4.1944777900582073,4.4897508981987659,4.7756361251275301,5.0476248900362748,5.301427767624177,5.5330421348971868,5.7388152949236897,5.9155020820471425,6.0603160400870371,6.1709733664174182,6.2457289288985747,6.2834037876533699,6.2834037876533699,6.2457289288985747,6.1709733664174182,6.0603160400870362,5.9155020820471425,5.7388152949236897,5.5330421348971859,5.3014277676241779,5.0476248900362748,4.7756361251275292,4.4897508981987659,4.1944777900582064,3.8944734340088325,3.5944690779594599,3.2991959698189,3.0133107428901358,2.741321977981392,2.4875191003934889,2.2559047331204805,2.0501315730939766,1.8734447859705239,1.7286308279306293,1.6179735016002486,1.5432179391190921,1.5055430803642973,1.5055430803642973,1.5432179391190921,1.6179735016002481,1.7286308279306297,1.8734447859705248,2.0501315730939775,2.2559047331204818,2.4875191003934907,2.741321977981392,3.0133107428901371,3.2991959698189013,3.5944690779594612,3.8944734340088329,3.8944734340088334],"text":"id: 29<br />as.factor(id): 29","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(204,122,255,0.6)","hoveron":"fills","name":"29","legendgroup":"29","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.5268607945447084,5.5224119911631737,5.5091357413054833,5.487241419362741,5.4570743118710157,5.4191101721364108,5.3739477173212178,5.3222991863166138,5.2649791073098804,5.2028914521883092,5.1370153803625325,5.0683897968377032,4.9980969680609419,4.9272454539329615,4.8569526251562003,4.788327041631371,4.7224509698055943,4.6603633146840222,4.6030432356772888,4.5513947046726857,4.5062322498574927,4.4682681101228878,4.4381010026311625,4.4162066806884201,4.4029304308307298,4.3984816274491951,4.4029304308307298,4.4162066806884201,4.4381010026311634,4.4682681101228878,4.5062322498574927,4.5513947046726857,4.6030432356772897,4.6603633146840231,4.7224509698055952,4.788327041631371,4.8569526251562003,4.9272454539329615,4.9980969680609428,5.0683897968377032,5.1370153803625325,5.2028914521883092,5.2649791073098813,5.3222991863166147,5.3739477173212178,5.4191101721364108,5.4570743118710157,5.487241419362741,5.5091357413054833,5.5224119911631737,5.5268607945447084,5.5268607945447084],"y":[0.72277433835450666,0.79348604320384519,0.86308258222658463,0.93046637642737917,0.99457474311855498,1.0543966550609101,1.1089886849672355,1.1574898839142105,1.1991353590212295,1.2332683362680541,1.2593505182134492,1.2769705732673962,1.2858506226359827,1.2858506226359827,1.2769705732673959,1.2593505182134492,1.2332683362680541,1.1991353590212295,1.1574898839142105,1.1089886849672352,1.0543966550609101,0.99457474311855487,0.93046637642737906,0.86308258222658463,0.79348604320384508,0.72277433835450644,0.65206263350516813,0.58246609448242859,0.51508230028163393,0.45097393359045834,0.39115202164810325,0.33655999174177798,0.28805879279480284,0.24641331768778368,0.21228034044095911,0.18619815849556409,0.16857810344161728,0.1596980540730305,0.1596980540730305,0.16857810344161739,0.18619815849556409,0.21228034044095923,0.24641331768778385,0.28805879279480301,0.3365599917417782,0.3911520216481037,0.4509739335904584,0.51508230028163415,0.58246609448242892,0.65206263350516847,0.72277433835450655,0.72277433835450666],"text":"id: 30<br />as.factor(id): 30","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(222,112,249,0.6)","hoveron":"fills","name":"30","legendgroup":"30","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[7.2377889907321036,7.227841163952462,7.1981545667843942,7.1491973745991562,7.0817416715631172,6.9968512744092388,6.8958649554117031,6.7803753291474056,6.6522037360126909,6.5133715185972925,6.3660681439041991,6.212616674147089,6.0554371306714998,5.8970083287725465,5.7398287852969565,5.5863773155398473,5.4390739408467539,5.3002417234313555,5.1720701302966408,5.0565805040323424,4.9555941850348075,4.8707037878809292,4.8032480848448902,4.7542908926596521,4.7246042954915843,4.7146564687119428,4.7246042954915843,4.7542908926596521,4.8032480848448902,4.8707037878809292,4.9555941850348084,5.0565805040323433,5.1720701302966408,5.3002417234313564,5.4390739408467539,5.5863773155398473,5.7398287852969574,5.8970083287725474,6.0554371306714998,6.2126166741470898,6.3660681439041982,6.5133715185972925,6.6522037360126909,6.7803753291474056,6.895864955411704,6.9968512744092388,7.0817416715631172,7.1491973745991562,7.1981545667843942,7.227841163952462,7.2377889907321036,7.2377889907321036],"y":[-0.79580977944312115,-0.63769360059509872,-0.48207100834153654,-0.33139626392670368,-0.18804559807875143,-0.054279736431639192,0.067791753468605997,0.1762437313042835,0.2693658445988536,0.34568950199720683,0.40401103382862802,0.44341067469654183,0.46326706872825607,0.46326706872825607,0.44341067469654161,0.40401103382862802,0.34568950199720661,0.26936584459885338,0.1762437313042835,0.067791753468605664,-0.05427973643163897,-0.18804559807875154,-0.33139626392670396,-0.48207100834153654,-0.63769360059509894,-0.79580977944312159,-0.95392595829114357,-1.109548550544706,-1.2602232949595391,-1.4035739608074909,-1.5373398224546033,-1.6594113123548482,-1.767863290190526,-1.8609854034850961,-1.9373090608834493,-1.9956305927148703,-2.0350302335827841,-2.0548866276144984,-2.0548866276144984,-2.0350302335827841,-1.9956305927148705,-1.9373090608834491,-1.8609854034850957,-1.7678632901905256,-1.6594113123548477,-1.5373398224546024,-1.4035739608074909,-1.2602232949595384,-1.1095485505447054,-0.95392595829114302,-0.79580977944312148,-0.79580977944312115],"text":"id: 31<br />as.factor(id): 31","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(236,104,238,0.6)","hoveron":"fills","name":"31","legendgroup":"31","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[8.0349358476692192,8.0304870442876854,8.017210794429996,7.9953164724872519,7.9651493649955274,7.9271852252609225,7.8820227704457295,7.8303742394411255,7.7730541604343921,7.7109665053128209,7.6450904334870442,7.5764648499622149,7.5061720211854537,7.4353205070574733,7.365027678280712,7.2964020947558828,7.230526022930106,7.1684383678085339,7.1111182888018005,7.0594697577971974,7.0143073029820044,6.9763431632473996,6.9461760557556742,6.9242817338129319,6.9110054839552415,6.9065566805737069,6.9110054839552415,6.9242817338129319,6.9461760557556751,6.9763431632473996,7.0143073029820044,7.0594697577971974,7.1111182888018014,7.1684383678085348,7.2305260229301069,7.2964020947558828,7.365027678280712,7.4353205070574733,7.5061720211854546,7.5764648499622149,7.6450904334870442,7.7109665053128209,7.773054160434393,7.8303742394411264,7.8820227704457295,7.9271852252609225,7.9651493649955274,7.9953164724872527,8.017210794429996,8.0304870442876854,8.0349358476692192,8.0349358476692192],"y":[-1.8445155575636125,-1.7738038527142741,-1.7042073136915346,-1.6368235194907399,-1.5727151527995642,-1.5128932408572093,-1.4583012109508839,-1.4098000120039087,-1.3681545368968897,-1.3340215596500651,-1.3079393777046699,-1.2903193226507232,-1.2814392732821362,-1.2814392732821362,-1.2903193226507232,-1.3079393777046699,-1.3340215596500653,-1.3681545368968897,-1.4098000120039087,-1.4583012109508839,-1.5128932408572091,-1.5727151527995642,-1.6368235194907401,-1.7042073136915346,-1.7738038527142741,-1.8445155575636127,-1.9152272624129509,-1.9848238014356905,-2.0522075956364851,-2.1163159623276608,-2.1761378742700161,-2.2307299041763411,-2.2792311031233163,-2.3208765782303353,-2.3550095554771602,-2.3810917374225551,-2.3987117924765018,-2.4075918418450888,-2.4075918418450888,-2.3987117924765018,-2.3810917374225551,-2.3550095554771602,-2.3208765782303353,-2.2792311031233163,-2.2307299041763411,-2.1761378742700153,-2.1163159623276608,-2.0522075956364851,-1.9848238014356903,-1.9152272624129507,-1.8445155575636127,-1.8445155575636125],"text":"id: 32<br />as.factor(id): 32","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(246,99,225,0.6)","hoveron":"fills","name":"32","legendgroup":"32","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[7.7667260876929447,7.7604345296144475,7.7416590770082481,7.7106958299778583,7.6680330974254947,7.614343696128989,7.5504743400192798,7.4774322869959224,7.3963694538683029,7.3085642499394323,7.2154014157275554,7.118350184780974,7.0189411129873092,6.9187419407928497,6.8193328689991848,6.7222816380526025,6.6291188038407265,6.5413135999118559,6.4602507667842364,6.3872087137608791,6.3233393576511698,6.2696499563546642,6.2269872238023005,6.1960239767719107,6.1772485241657114,6.1709569660872141,6.1772485241657114,6.1960239767719107,6.2269872238023005,6.2696499563546642,6.3233393576511707,6.3872087137608791,6.4602507667842364,6.5413135999118559,6.6291188038407265,6.7222816380526025,6.8193328689991848,6.9187419407928497,7.01894111298731,7.118350184780974,7.2154014157275554,7.3085642499394323,7.3963694538683029,7.4774322869959233,7.5504743400192806,7.614343696128989,7.6680330974254947,7.7106958299778583,7.7416590770082481,7.7604345296144475,7.7667260876929447,7.7667260876929447],"y":[1.008641830042337,1.1086432820587948,1.2070676514389813,1.3023627270819025,1.3930256487181509,1.4776266079141185,1.5548313970051213,1.6234224503472892,1.6823180460551068,1.7305893654017384,1.7674751408451987,1.7923936616722509,1.8049519479238492,1.8049519479238492,1.7923936616722509,1.7674751408451987,1.7305893654017381,1.6823180460551068,1.6234224503472892,1.5548313970051213,1.4776266079141185,1.3930256487181507,1.3023627270819023,1.2070676514389813,1.1086432820587948,1.0086418300423368,0.90864037802587916,0.81021600864569243,0.71492093300277115,0.62425801136652304,0.53965705217055548,0.46245226307955256,0.3938612097373847,0.33496561402956704,0.2866942946829355,0.24980851923947522,0.22488999841242308,0.21233171216082491,0.21233171216082491,0.22488999841242319,0.24980851923947511,0.28669429468293572,0.33496561402956726,0.39386120973738492,0.46245226307955301,0.53965705217055604,0.62425801136652304,0.71492093300277149,0.81021600864569288,0.90864037802587949,1.0086418300423368,1.008641830042337],"text":"id: 33<br />as.factor(id): 33","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,97,210,0.6)","hoveron":"fills","name":"33","legendgroup":"33","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-3.6977090619034922,-3.7021578652850264,-3.7154341151427168,-3.7373284370854596,-3.7674955445771841,-3.8054596843117894,-3.8506221391269828,-3.9022706701315859,-3.9595907491383193,-4.021678404259891,-4.0875544760856677,-4.156180059610497,-4.2264728883872582,-4.2973244025152386,-4.3676172312919999,-4.4362428148168291,-4.5021188866426058,-4.5642065417641779,-4.6215266207709114,-4.6731751517755145,-4.7183376065907074,-4.7563017463253123,-4.7864688538170377,-4.80836317575978,-4.8216394256174704,-4.826088228999005,-4.8216394256174704,-4.80836317575978,-4.7864688538170368,-4.7563017463253123,-4.7183376065907074,-4.6731751517755145,-4.6215266207709105,-4.5642065417641771,-4.502118886642605,-4.4362428148168291,-4.3676172312919999,-4.2973244025152386,-4.2264728883872573,-4.156180059610497,-4.0875544760856677,-4.021678404259891,-3.9595907491383193,-3.9022706701315859,-3.8506221391269824,-3.8054596843117894,-3.7674955445771841,-3.7373284370854596,-3.7154341151427168,-3.7021578652850264,-3.6977090619034922,-3.6977090619034922],"y":[3.9849374177348054,4.0556491225841436,4.1252456616068836,4.192629455807678,4.2567378224988532,4.316559734441209,4.3711517643475339,4.4196529632945092,4.4612984384015286,4.495431415648353,4.5215135975937475,4.5391336526476946,4.5480137020162816,4.5480137020162816,4.5391336526476946,4.5215135975937475,4.495431415648353,4.4612984384015277,4.4196529632945092,4.3711517643475339,4.316559734441209,4.2567378224988532,4.192629455807678,4.1252456616068836,4.0556491225841436,3.9849374177348054,3.9142257128854667,3.8446291738627272,3.7772453796619327,3.713137012970757,3.6533151010284017,3.5987230711220768,3.5502218721751015,3.5085763970680826,3.4744434198212577,3.4483612378758628,3.4307411828219161,3.4218611334533291,3.4218611334533291,3.4307411828219161,3.4483612378758628,3.4744434198212577,3.5085763970680826,3.5502218721751015,3.5987230711220768,3.6533151010284026,3.713137012970757,3.7772453796619327,3.8446291738627276,3.9142257128854672,3.9849374177348054,3.9849374177348054],"text":"id: 34<br />as.factor(id): 34","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(255,97,193,0.6)","hoveron":"fills","name":"34","legendgroup":"34","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.7865403494009753,-2.7942459028906774,-2.8172410421781757,-2.8551631201802765,-2.907414083073335,-2.9731699019593156,-3.0513935682937645,-3.1408514481300336,-3.2401327372635582,-3.3476717104569351,-3.4617724138622359,-3.5806354112263028,-3.7023861620753915,-3.8251045843382392,-3.9468553351873283,-4.0657183325513948,-4.1798190359566956,-4.2873580091500729,-4.3866392982835976,-4.4760971781198666,-4.5543208444543151,-4.6200766633402957,-4.6723276262333542,-4.710249704235455,-4.7332448435229537,-4.7409503970126554,-4.7332448435229537,-4.710249704235455,-4.6723276262333542,-4.6200766633402957,-4.5543208444543151,-4.4760971781198666,-4.3866392982835967,-4.287358009150072,-4.1798190359566956,-4.0657183325513948,-3.9468553351873279,-3.8251045843382387,-3.7023861620753911,-3.5806354112263024,-3.4617724138622359,-3.3476717104569351,-3.2401327372635578,-3.1408514481300331,-3.0513935682937641,-2.9731699019593156,-2.907414083073335,-2.8551631201802765,-2.8172410421781757,-2.7942459028906774,-2.7865403494009753,-2.7865403494009753],"y":[5.4436151247458202,5.5660913902346891,5.6866361318530236,5.8033482870155648,5.9143872353149378,6.0180018262050083,6.1125579956910832,6.1965645364952504,6.2686966152859522,6.3278166660910449,6.3729923303927247,6.4035111609783213,6.4188918576584335,6.4188918576584335,6.4035111609783213,6.3729923303927247,6.3278166660910449,6.2686966152859522,6.1965645364952504,6.1125579956910832,6.0180018262050083,5.9143872353149378,5.8033482870155648,5.6866361318530236,5.5660913902346891,5.4436151247458202,5.3211388592569513,5.2005941176386168,5.0838819624760747,4.9728430141767026,4.8692284232866321,4.7746722538005573,4.69066571299639,4.6185336342056882,4.5594135834005955,4.5142379190989157,4.4837190885133191,4.4683383918332069,4.4683383918332069,4.4837190885133191,4.5142379190989157,4.5594135834005955,4.6185336342056882,4.69066571299639,4.7746722538005573,4.869228423286633,4.9728430141767026,5.0838819624760756,5.2005941176386168,5.3211388592569522,5.4436151247458202,5.4436151247458202],"text":"id: 35<br />as.factor(id): 35","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(255,100,175,0.6)","hoveron":"fills","name":"35","legendgroup":"35","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-1.3730707321407181,-1.3793622902192153,-1.3981377428254147,-1.4291009898558045,-1.4717637224081681,-1.5254531237046742,-1.5893224798143828,-1.6623645328377399,-1.7434273659653596,-1.8312325698942302,-1.9243954041061069,-2.0214466350526892,-2.1208557068463536,-2.2210548790408131,-2.320463950834478,-2.4175151817810598,-2.5106780159929367,-2.5984832199218073,-2.6795460530494268,-2.7525881060727841,-2.8164574621824925,-2.8701468634789986,-2.9128095960313622,-2.943772843061752,-2.9625482956679514,-2.9688398537464487,-2.9625482956679514,-2.943772843061752,-2.9128095960313622,-2.8701468634789986,-2.8164574621824925,-2.7525881060727837,-2.6795460530494268,-2.5984832199218069,-2.5106780159929363,-2.4175151817810598,-2.320463950834478,-2.2210548790408131,-2.1208557068463532,-2.0214466350526887,-1.9243954041061071,-1.8312325698942302,-1.7434273659653594,-1.6623645328377397,-1.5893224798143826,-1.5254531237046738,-1.4717637224081681,-1.4291009898558042,-1.3981377428254145,-1.3793622902192153,-1.3730707321407181,-1.3730707321407181],"y":[6.2271729041095067,6.3271743561259646,6.4255987255061511,6.5208938011490725,6.6115567227853207,6.6961576819812878,6.7733624710722911,6.8419535244144587,6.9008491201222766,6.9491204394689081,6.9860062149123685,7.0109247357394207,7.0234830219910185,7.0234830219910185,7.0109247357394207,6.9860062149123685,6.9491204394689081,6.9008491201222766,6.8419535244144587,6.7733624710722911,6.6961576819812878,6.6115567227853207,6.5208938011490725,6.4255987255061511,6.3271743561259646,6.2271729041095067,6.1271714520930489,6.0287470827128624,5.933452007069941,5.8427890854336928,5.7581881262377248,5.6809833371467224,5.6123922838045548,5.5534966880967369,5.5052253687501054,5.468339593306645,5.4434210724795928,5.430862786227995,5.430862786227995,5.4434210724795928,5.468339593306645,5.5052253687501054,5.5534966880967369,5.6123922838045548,5.6809833371467224,5.7581881262377257,5.8427890854336928,5.933452007069941,6.0287470827128624,6.1271714520930489,6.2271729041095067,6.2271729041095067],"text":"id: 36<br />as.factor(id): 36","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(255,105,155,0.6)","hoveron":"fills","name":"36","legendgroup":"36","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-3.4064606534720023,-3.4109094568535365,-3.4241857067112269,-3.4460800286539697,-3.4762471361456941,-3.5142112758802995,-3.5593737306954929,-3.611022261700096,-3.6683423407068294,-3.7304299958284015,-3.7963060676541778,-3.8649316511790075,-3.9352244799557679,-4.0060759940837487,-4.0763688228605099,-4.1449944063853392,-4.2108704782111159,-4.272958133332688,-4.3302782123394215,-4.3819267433440245,-4.4270891981592175,-4.4650533378938224,-4.4952204453855478,-4.5171147673282901,-4.5303910171859805,-4.5348398205675151,-4.5303910171859805,-4.5171147673282901,-4.4952204453855469,-4.4650533378938224,-4.4270891981592175,-4.3819267433440245,-4.3302782123394206,-4.2729581333326871,-4.210870478211115,-4.1449944063853392,-4.0763688228605099,-4.0060759940837487,-3.9352244799557679,-3.8649316511790071,-3.7963060676541778,-3.7304299958284015,-3.6683423407068294,-3.611022261700096,-3.5593737306954925,-3.5142112758802995,-3.4762471361456941,-3.4460800286539697,-3.4241857067112269,-3.4109094568535365,-3.4064606534720023,-3.4064606534720023],"y":[-4.7737479038410671,-4.7030361989917289,-4.6334396599689889,-4.5660558657681944,-4.5019474990770192,-4.4421255871346634,-4.3875335572283385,-4.3390323582813632,-4.2973868831743438,-4.2632539059275194,-4.237171723982124,-4.2195516689281778,-4.2106716195595908,-4.2106716195595908,-4.2195516689281778,-4.237171723982124,-4.2632539059275194,-4.2973868831743438,-4.3390323582813632,-4.3875335572283385,-4.4421255871346634,-4.5019474990770192,-4.5660558657681944,-4.6334396599689889,-4.7030361989917289,-4.7737479038410671,-4.8444596086904053,-4.9140561477131453,-4.9814399419139397,-5.0455483086051149,-5.1053702205474707,-5.1599622504537956,-5.2084634494007709,-5.2501089245077903,-5.2842419017546147,-5.3103240837000101,-5.3279441387539563,-5.3368241881225433,-5.3368241881225433,-5.3279441387539563,-5.3103240837000101,-5.2842419017546147,-5.2501089245077903,-5.2084634494007709,-5.1599622504537956,-5.1053702205474698,-5.0455483086051149,-4.9814399419139397,-4.9140561477131453,-4.8444596086904053,-4.7737479038410671,-4.7737479038410671],"text":"id: 37<br />as.factor(id): 37","type":"scatter","mode":"lines","line":{"width":1.8897637795275593,"color":"rgba(128,128,128,0.6)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,111,133,0.6)","hoveron":"fills","name":"37","legendgroup":"37","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.763953195770684,1.1283791670955126,0.33527436607933936,0.82125112072947448,2.4061245852461335,3.4913271985654681,2.0197271189178814,1.5962255606995344,0.28919448970013639,-0.73136612609310248,3.3657895266667559,3.1017434227609217,4.0462817529986497,-1.3366113616909083,-3.6320671903071409,-4.7473800156682557,-5.5263765385785435,-6.2552122344208794,-6.0304020104542237,-5.8155936163144819,-4.4461663256463835,-3.3899454951387189,-2.020240913910389,1.0267084974396361,2.3437801415991757,6.1513111092371417,-0.47726465363435711,2.2790901051341441,5.6057962923400826,4.9626712109969517,5.9762227297220232,7.4707462641214635,6.9688415268900794,-4.2618986454512484,-3.7637453732068153,-2.1709552929435834,-3.9706502370197585],"y":[0,0,-1.7554135478633106,3.0674519891027421,1.1100250546943573,-0.35821187849523706,-1.4388494724731233,-2.7334121032079635,-3.116708012592142,-2.6353603479807033,-2.59346066971526,1.9984810061234295,1.3811586531773474,-5.2411883490944611,-3.4544270576526657,-2.6725554081516205,-1.8562197161542393,-0.70554838898279271,0.63784520111228515,2.1641985970812039,2.8717056508638334,3.2687410421539047,4.2630741532470839,-3.9707060413505131,-5.6826475976307549,-4.4917646312283344,5.6958019868562086,6.6942384499206256,3.8944734340088334,0.72277433835450666,-0.79580977944312115,-1.8445155575636125,1.008641830042337,3.9849374177348054,5.4436151247458202,6.2271729041095067,-4.7737479038410671],"text":["Bioinformatics (Oxford, England)","BMC bioinformatics","BMC genomics","Briefings in bioinformatics","Cell genomics","Cell systems","Cells","Communications biology","eLife","F1000Research","Frontiers in genetics","Genes","Genetic epidemiology","Genome biology","Genome research","Genomics","Genomics, proteomics & bioinformatics","GigaScience","IEEE/ACM transactions on computational biology and bioinformatics","iScience","Journal of chemical information and modeling","Life science alliance","Molecular systems biology","Nature","Nature biotechnology","Nature communications","Nature computational science","Nature methods","Nucleic acids research","Patterns (New York, N.Y.)","PLoS computational biology","Proceedings of SPIE--the International Society for Optical Engineering","Proceedings of the National Academy of Sciences of the United States of America","Research square","Science advances","Scientific reports","Statistics in medicine"],"hovertext":["x: -2.7639532<br />y:  0.0000000<br />Freq: 24<br />Var1: Bioinformatics (Oxford, England)","x:  1.1283792<br />y:  0.0000000<br />Freq:  4<br />Var1: BMC bioinformatics","x:  0.3352744<br />y: -1.7554135<br />Freq:  2<br />Var1: BMC genomics","x:  0.8212511<br />y:  3.0674520<br />Freq: 12<br />Var1: Briefings in bioinformatics","x:  2.4061246<br />y:  1.1100251<br />Freq:  1<br />Var1: Cell genomics","x:  3.4913272<br />y: -0.3582119<br />Freq:  5<br />Var1: Cell systems","x:  2.0197271<br />y: -1.4388495<br />Freq:  1<br />Var1: Cells","x:  1.5962256<br />y: -2.7334121<br />Freq:  2<br />Var1: Communications biology","x:  0.2891945<br />y: -3.1167080<br />Freq:  1<br />Var1: eLife","x: -0.7313661<br />y: -2.6353603<br />Freq:  1<br />Var1: F1000Research","x:  3.3657895<br />y: -2.5934607<br />Freq:  3<br />Var1: Frontiers in genetics","x:  3.1017434<br />y:  1.9984810<br />Freq:  1<br />Var1: Genes","x:  4.0462818<br />y:  1.3811587<br />Freq:  1<br />Var1: Genetic epidemiology","x: -1.3366114<br />y: -5.2411883<br />Freq: 14<br />Var1: Genome biology","x: -3.6320672<br />y: -3.4544271<br />Freq:  2<br />Var1: Genome research","x: -4.7473800<br />y: -2.6725554<br />Freq:  1<br />Var1: Genomics","x: -5.5263765<br />y: -1.8562197<br />Freq:  1<br />Var1: Genomics, proteomics & bioinformatics","x: -6.2552122<br />y: -0.7055484<br />Freq:  2<br />Var1: GigaScience","x: -6.0304020<br />y:  0.6378452<br />Freq:  1<br />Var1: IEEE/ACM transactions on computational biology and bioinformatics","x: -5.8155936<br />y:  2.1641986<br />Freq:  3<br />Var1: iScience","x: -4.4461663<br />y:  2.8717057<br />Freq:  1<br />Var1: Journal of chemical information and modeling","x: -3.3899455<br />y:  3.2687410<br />Freq:  1<br />Var1: Life science alliance","x: -2.0202409<br />y:  4.2630742<br />Freq:  4<br />Var1: Molecular systems biology","x:  1.0267085<br />y: -3.9707060<br />Freq:  1<br />Var1: Nature","x:  2.3437801<br />y: -5.6826476<br />Freq:  8<br />Var1: Nature biotechnology","x:  6.1513111<br />y: -4.4917646<br />Freq: 18<br />Var1: Nature communications","x: -0.4772647<br />y:  5.6958020<br />Freq:  3<br />Var1: Nature computational science","x:  2.2790901<br />y:  6.6942384<br />Freq: 12<br />Var1: Nature methods","x:  5.6057963<br />y:  3.8944734<br />Freq: 18<br />Var1: Nucleic acids research","x:  4.9626712<br />y:  0.7227743<br />Freq:  1<br />Var1: Patterns (New York, N.Y.)","x:  5.9762227<br />y: -0.7958098<br />Freq:  5<br />Var1: PLoS computational biology","x:  7.4707463<br />y: -1.8445156<br />Freq:  1<br />Var1: Proceedings of SPIE--the International Society for Optical Engineering","x:  6.9688415<br />y:  1.0086418<br />Freq:  2<br />Var1: Proceedings of the National Academy of Sciences of the United States of America","x: -4.2618986<br />y:  3.9849374<br />Freq:  1<br />Var1: Research square","x: -3.7637454<br />y:  5.4436151<br />Freq:  3<br />Var1: Science advances","x: -2.1709553<br />y:  6.2271729<br />Freq:  2<br />Var1: Scientific reports","x: -3.9706502<br />y: -4.7737479<br />Freq:  1<br />Var1: Statistics in medicine"],"textfont":{"size":[18.897637795275593,11.654069963232464,9.9233130412770691,15.400411556879524,7.559055118110237,12.287570964443903,7.559055118110237,9.9233130412770691,7.559055118110237,7.559055118110237,10.902620738000817,7.559055118110237,7.559055118110237,16.083508288510249,9.9233130412770691,7.559055118110237,7.559055118110237,9.9233130412770691,7.559055118110237,10.902620738000817,7.559055118110237,7.559055118110237,11.654069963232464,7.559055118110237,13.814293618023729,17.307140261530527,10.902620738000817,15.400411556879524,17.307140261530527,7.559055118110237,12.287570964443903,7.559055118110237,9.9233130412770691,7.559055118110237,10.902620738000817,9.9233130412770691,7.559055118110237],"color":"rgba(0,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":16,"r":0,"b":0,"l":0},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-7.8329998745672187,9.3248678709892125],"tickmode":"array","ticktext":["-5","0","5"],"tickvals":[-5,0,4.9999999999999991],"categoryorder":"array","categoryarray":["-5","0","5"],"nticks":null,"ticks":"outside","tickcolor":null,"ticklen":0,"tickwidth":0,"showticklabels":false,"tickfont":{"color":null,"family":null,"size":0},"tickangle":-0,"showline":true,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"scaleanchor":"y","scaleratio":1,"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-8.147667827655436,9.4444328559078183],"tickmode":"array","ticktext":["-5","0","5"],"tickvals":[-5,0,5],"categoryorder":"array","categoryarray":["-5","0","5"],"nticks":null,"ticks":"outside","tickcolor":null,"ticklen":0,"tickwidth":0,"showticklabels":false,"tickfont":{"color":null,"family":null,"size":0},"tickangle":-0,"showline":true,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"scaleanchor":"x","scaleratio":1,"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"85207c90c1b":{"x":{},"y":{},"fill":{},"type":"scatter"},"852012b05403":{"x":{},"y":{},"size":{},"label":{}}},"cur_data":"85207c90c1b","visdat":{"85207c90c1b":["function (y) ","x"],"852012b05403":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-16-2"><div class="card-body html-fill-item html-fill-container" data-title="Publication Year">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 50fr) minmax(3em, 50fr);
grid-auto-rows: minmax(0, 1fr);">
<!-- Evaluation - Pub year --><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure Benchmark" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure Benchmark</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-a64528f7d948646da243" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-a64528f7d948646da243">{"x":{"data":[{"orientation":"v","width":[0.17999999999999972,0.22499999999999964,0.22499999999999964,0.17999999999999972],"base":[0,0,0,0],"x":[4.6399999999999997,2.6624999999999996,3.6624999999999996,6.6399999999999997],"y":[6,6,2,2],"text":["factor(year): 2021<br />size:  6<br />broader_topic: Analysis pipelines","factor(year): 2019<br />size:  6<br />broader_topic: Analysis pipelines","factor(year): 2020<br />size:  2<br />broader_topic: Analysis pipelines","factor(year): 2023<br />size:  2<br />broader_topic: Analysis pipelines"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Analysis pipelines","legendgroup":"Analysis pipelines","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.17999999999999972,0.17999999999999972,0.22499999999999964],"base":[0,0,0],"x":[4.8200000000000003,6.8200000000000003,5.6624999999999996],"y":[4,2,1],"text":["factor(year): 2021<br />size:  4<br />broader_topic: Data","factor(year): 2023<br />size:  2<br />broader_topic: Data","factor(year): 2022<br />size:  1<br />broader_topic: Data"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(163,165,0,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Data","legendgroup":"Data","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.17999999999999972,0.44999999999999996,0.29999999999999982,0.22499999999999964,0.22499999999999964,0.22499999999999964,0.17999999999999972],"base":[0,0,0,0,0,0,0],"x":[7,0.77499999999999991,1.6999999999999997,5.8875000000000002,2.8875000000000002,3.8875000000000002,5],"y":[4,4,4,12,6,10,8],"text":["factor(year): 2023<br />size:  4<br />broader_topic: Downstream analysis","factor(year): 2017<br />size:  4<br />broader_topic: Downstream analysis","factor(year): 2018<br />size:  4<br />broader_topic: Downstream analysis","factor(year): 2022<br />size: 12<br />broader_topic: Downstream analysis","factor(year): 2019<br />size:  6<br />broader_topic: Downstream analysis","factor(year): 2020<br />size: 10<br />broader_topic: Downstream analysis","factor(year): 2021<br />size:  8<br />broader_topic: Downstream analysis"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(0,191,125,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Downstream analysis","legendgroup":"Downstream analysis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.17999999999999972,0.22499999999999964,0.22499999999999964,0.29999999999999982,0.22499999999999964,0.17999999999999972,0.45000000000000018],"base":[0,0,0,0,0,0,0],"x":[5.1799999999999997,6.1124999999999998,4.1124999999999998,2,3.1124999999999998,7.1799999999999997,1.2250000000000001],"y":[6,8,12,4,2,4,2],"text":["factor(year): 2021<br />size:  6<br />broader_topic: Initial analysis","factor(year): 2022<br />size:  8<br />broader_topic: Initial analysis","factor(year): 2020<br />size: 12<br />broader_topic: Initial analysis","factor(year): 2018<br />size:  4<br />broader_topic: Initial analysis","factor(year): 2019<br />size:  2<br />broader_topic: Initial analysis","factor(year): 2023<br />size:  4<br />broader_topic: Initial analysis","factor(year): 2017<br />size:  2<br />broader_topic: Initial analysis"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(0,176,246,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Initial analysis","legendgroup":"Initial analysis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.22499999999999964,0.22499999999999964,0.22499999999999964,0.17999999999999972,0.29999999999999982,0.17999999999999972],"base":[0,0,0,0,0,0],"x":[4.3375000000000004,3.3375000000000004,6.3375000000000004,5.3600000000000003,2.3000000000000003,7.3600000000000003],"y":[15,19,23,16,6,4],"text":["factor(year): 2020<br />size: 15<br />broader_topic: Intermediate analysis","factor(year): 2019<br />size: 19<br />broader_topic: Intermediate analysis","factor(year): 2022<br />size: 23<br />broader_topic: Intermediate analysis","factor(year): 2021<br />size: 16<br />broader_topic: Intermediate analysis","factor(year): 2018<br />size:  6<br />broader_topic: Intermediate analysis","factor(year): 2023<br />size:  4<br />broader_topic: Intermediate analysis"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(231,107,243,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Intermediate analysis","legendgroup":"Intermediate analysis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":27.68949771689498,"r":7.3059360730593621,"b":41.643835616438366,"l":37.260273972602747},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,7.5999999999999996],"tickmode":"array","ticktext":["2017","2018","2019","2020","2021","2022","2023"],"tickvals":[1,2,3,4.0000000000000009,5,6,7],"categoryorder":"array","categoryarray":["2017","2018","2019","2020","2021","2022","2023"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":{"text":"Publication Year","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1.1500000000000001,24.149999999999999],"tickmode":"array","ticktext":["0","5","10","15","20"],"tickvals":[0,5,10,14.999999999999998,20],"categoryorder":"array","categoryarray":["0","5","10","15","20"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716894984},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"x","title":{"text":"Numbers","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"title":{"text":"broader_topic","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"852013a92d3b":{"x":{},"y":{},"fill":{},"type":"bar"}},"cur_data":"852013a92d3b","visdat":{"852013a92d3b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development paper" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development paper</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-f5d4726f5fe43da5e84e" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-f5d4726f5fe43da5e84e">{"x":{"data":[{"orientation":"v","width":[0.22499999999999964,0.17999999999999972,0.17999999999999972,0.17999999999999972,0.22500000000000009],"base":[0,0,0,0,0],"x":[5.6624999999999996,3.6399999999999997,4.6399999999999997,6.6399999999999997,0.66249999999999987],"y":[5,4,4,2,1],"text":["factor(year): 2022<br />size:  5<br />broader_topic: Analysis pipelines","factor(year): 2020<br />size:  4<br />broader_topic: Analysis pipelines","factor(year): 2021<br />size:  4<br />broader_topic: Analysis pipelines","factor(year): 2023<br />size:  2<br />broader_topic: Analysis pipelines","factor(year): 2017<br />size:  1<br />broader_topic: Analysis pipelines"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Analysis pipelines","legendgroup":"Analysis pipelines","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.17999999999999972,0.17999999999999972,0.22500000000000009,0.22499999999999964,0.17999999999999972],"base":[0,0,0,0,0],"x":[4.8200000000000003,3.8199999999999998,0.88749999999999996,2.6624999999999996,6.8200000000000003],"y":[7,4,1,1,1],"text":["factor(year): 2021<br />size:  7<br />broader_topic: Data","factor(year): 2020<br />size:  4<br />broader_topic: Data","factor(year): 2017<br />size:  1<br />broader_topic: Data","factor(year): 2019<br />size:  1<br />broader_topic: Data","factor(year): 2023<br />size:  1<br />broader_topic: Data"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(163,165,0,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Data","legendgroup":"Data","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.22499999999999964,0.17999999999999972,0.17999999999999972,0.17999999999999972,0.22499999999999964,0.29999999999999982],"base":[0,0,0,0,0,0],"x":[5.8875000000000002,7,5,4,2.8875000000000002,1.6999999999999997],"y":[19,5,9,3,3,2],"text":["factor(year): 2022<br />size: 19<br />broader_topic: Downstream analysis","factor(year): 2023<br />size:  5<br />broader_topic: Downstream analysis","factor(year): 2021<br />size:  9<br />broader_topic: Downstream analysis","factor(year): 2020<br />size:  3<br />broader_topic: Downstream analysis","factor(year): 2019<br />size:  3<br />broader_topic: Downstream analysis","factor(year): 2018<br />size:  2<br />broader_topic: Downstream analysis"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(0,191,125,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Downstream analysis","legendgroup":"Downstream analysis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.17999999999999972,0.17999999999999972,0.29999999999999982,0.22499999999999964,0.22499999999999964,0.17999999999999972,0.22500000000000009],"base":[0,0,0,0,0,0,0],"x":[5.1799999999999997,4.1799999999999997,2,6.1124999999999998,3.1124999999999998,7.1799999999999997,1.1125],"y":[6,3,3,10,3,1,1],"text":["factor(year): 2021<br />size:  6<br />broader_topic: Initial analysis","factor(year): 2020<br />size:  3<br />broader_topic: Initial analysis","factor(year): 2018<br />size:  3<br />broader_topic: Initial analysis","factor(year): 2022<br />size: 10<br />broader_topic: Initial analysis","factor(year): 2019<br />size:  3<br />broader_topic: Initial analysis","factor(year): 2023<br />size:  1<br />broader_topic: Initial analysis","factor(year): 2017<br />size:  1<br />broader_topic: Initial analysis"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(0,176,246,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Initial analysis","legendgroup":"Initial analysis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.17999999999999972,0.17999999999999972,0.22499999999999964,0.22499999999999964,0.17999999999999972,0.22500000000000009,0.29999999999999982],"base":[0,0,0,0,0,0,0],"x":[5.3600000000000003,7.3600000000000003,6.3375000000000004,3.3375000000000004,4.3600000000000003,1.3375000000000001,2.3000000000000003],"y":[15,8,16,13,10,3,1],"text":["factor(year): 2021<br />size: 15<br />broader_topic: Intermediate analysis","factor(year): 2023<br />size:  8<br />broader_topic: Intermediate analysis","factor(year): 2022<br />size: 16<br />broader_topic: Intermediate analysis","factor(year): 2019<br />size: 13<br />broader_topic: Intermediate analysis","factor(year): 2020<br />size: 10<br />broader_topic: Intermediate analysis","factor(year): 2017<br />size:  3<br />broader_topic: Intermediate analysis","factor(year): 2018<br />size:  1<br />broader_topic: Intermediate analysis"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(231,107,243,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Intermediate analysis","legendgroup":"Intermediate analysis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":27.68949771689498,"r":7.3059360730593621,"b":41.643835616438366,"l":37.260273972602747},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,7.5999999999999996],"tickmode":"array","ticktext":["2017","2018","2019","2020","2021","2022","2023"],"tickvals":[1,2,3,4.0000000000000009,5,6,7],"categoryorder":"array","categoryarray":["2017","2018","2019","2020","2021","2022","2023"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":{"text":"Publication Year","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.95000000000000007,19.949999999999999],"tickmode":"array","ticktext":["0","5","10","15"],"tickvals":[0,5,10,15],"categoryorder":"array","categoryarray":["0","5","10","15"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716894984},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"x","title":{"text":"Numbers","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"title":{"text":"broader_topic","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"8520737d5538":{"x":{},"y":{},"fill":{},"type":"bar"}},"cur_data":"8520737d5538","visdat":{"8520737d5538":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-16-3"><div class="card-body html-fill-item html-fill-container" data-title="Category Frequency">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 50fr) minmax(3em, 50fr);
grid-auto-rows: minmax(0, 1fr);">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<!-- Evaluation - category freq -->
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Pure Benchmark" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Pure Benchmark</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - category freq - Pure benchmark-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="New method development paper" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">New method development paper</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<img src="index_files/figure-html/Evaluation - category freq - New method development paper-1.png" class="img-fluid quarto-dashboard-img-contain">
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div><div class="tab-pane html-fill-item html-fill-container" role="tabpanel" id="card-tabset-16-4"><div class="card-body html-fill-item html-fill-container" data-title="Statistics">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-columns: minmax(3em, 100fr);
grid-auto-rows: minmax(0, 1fr);">
<!-- Evaluation - Statistis --><div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, 1fr); grid-auto-columns: minmax(0, 1fr);">
<div class="card cell html-fill-item html-fill-container bslib-card" data-title="Statistics of attributes" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-header"><div class="card-title">Statistics of attributes</div></div>
<div class="card-body html-fill-item html-fill-container">
<div class="cell-output-display html-fill-item html-fill-container">
<div class="plotly html-widget html-fill-item html-fill-container" id="htmlwidget-8fc0d5735be34ffad88c" style="width:100%;height:406px;"></div>
<script type="application/json" data-for="htmlwidget-8fc0d5735be34ffad88c">{"x":{"data":[{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[53,111],"x":[111,81],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 111<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  81<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[63,94],"x":[101,98],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 101<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  98<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[45,3],"x":[119,189],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 119<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value: 189<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x3","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[147,122],"x":[17,70],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  17<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  70<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x4","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[87,110],"x":[77,82],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  77<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  82<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x5","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[35,64],"x":[129,128],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 129<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value: 128<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x6","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[49,87],"x":[115,105],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 115<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value: 105<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x7","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[147,135],"x":[17,57],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  17<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  57<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x8","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[77,58],"x":[87,134],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  87<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value: 134<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x9","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[115,173],"x":[49,19],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  49<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  19<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x10","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[67,110],"x":[97,82],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  97<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  82<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x11","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[43,65],"x":[121,127],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 121<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value: 127<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x12","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[76,132],"x":[88,60],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  88<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  60<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x13","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[95,132],"x":[69,60],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  69<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value:  60<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x14","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[9,9],"x":[155,183],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 155<br />Answers: No","Paper.category: Pure benchmarking paper<br />Value: 183<br />Answers: No"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(127,201,127,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"No","legendgroup":"No","showlegend":false,"xaxis":"x15","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[46,99],"x":[7,12],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   7<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:  12<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[63,94],"x":[0,0],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   0<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[37,1],"x":[8,2],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   8<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   2<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x3","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[146,119],"x":[1,3],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   1<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   3<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x4","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[85,108],"x":[2,2],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   2<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   2<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x5","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[35,64],"x":[0,0],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   0<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x6","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[49,87],"x":[0,0],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   0<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x7","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[147,135],"x":[0,0],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   0<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x8","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[67,46],"x":[10,12],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  10<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:  12<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x9","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[110,170],"x":[5,3],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   5<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   3<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x10","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[57,102],"x":[10,8],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  10<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   8<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x11","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[43,65],"x":[0,0],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   0<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x12","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[76,132],"x":[0,0],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   0<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x13","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[95,131],"x":[0,1],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   1<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x14","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[9,9],"x":[0,0],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   0<br />Answers: Not sure","Paper.category: Pure benchmarking paper<br />Value:   0<br />Answers: Not sure"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(190,174,212,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Not sure","legendgroup":"Not sure","showlegend":false,"xaxis":"x15","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[46,99],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  46<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:  99<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[63,94],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  63<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:  94<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[37,1],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  37<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:   1<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x3","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[146,119],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 146<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value: 119<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x4","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[85,108],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  85<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value: 108<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x5","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[35,64],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  35<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:  64<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x6","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[49,87],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  49<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:  87<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x7","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[147,135],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 147<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value: 135<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x8","yaxis":"y2","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[67,46],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  67<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:  46<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x9","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[110,170],"y":[1,2],"text":["Paper.category: New method development paper<br />Value: 110<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value: 170<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x10","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[57,102],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  57<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value: 102<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x11","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[43,65],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  43<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:  65<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x12","yaxis":"y3","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[76,132],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  76<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value: 132<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x13","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[95,131],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:  95<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value: 131<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x14","yaxis":"y4","hoverinfo":"text","frame":null},{"orientation":"h","width":[0.89999999999999991,0.90000000000000013],"base":[0,0],"x":[9,9],"y":[1,2],"text":["Paper.category: New method development paper<br />Value:   9<br />Answers: Yes","Paper.category: Pure benchmarking paper<br />Value:   9<br />Answers: Yes"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(253,192,134,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Yes","legendgroup":"Yes","showlegend":false,"xaxis":"x15","yaxis":"y4","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":56.913242009132418,"r":7.3059360730593621,"b":43.300616536043236,"l":189.22374429223754},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"title":{"text":"Stacked Bar Chart by Attributes","font":{"color":"rgba(0,0,0,1)","family":"","size":17.534246575342465},"x":0,"xref":"paper"},"xaxis":{"domain":[0,0.24906249999999999],"automargin":true,"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"annotations":[{"text":"Size","x":0.5,"y":0,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"top","annotationType":"axis","yshift":-23.574589138782954},{"text":"Paper.category","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-173.88127853881281},{"text":"Applicability","x":0.12453125,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Data.availability","x":0.375,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Discovery.","x":0.625,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Downstream.analysis","x":0.87546875000000002,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Future.directions","x":0.12453125,"y":0.72894079972391035,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Memory.measured","x":0.375,"y":0.72894079972391035,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Overall.comparison","x":0.625,"y":0.72894079972391035,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Package.availability","x":0.87546875000000002,"y":0.72894079972391035,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Prior.knowledge","x":0.12453125,"y":0.47894079972391029,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Recommendation","x":0.375,"y":0.47894079972391029,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Selection.criteria","x":0.625,"y":0.47894079972391029,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Sensitivity.analysis","x":0.87546875000000002,"y":0.47894079972391029,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Speed.measured.","x":0.12453125,"y":0.22894079972391029,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Variability.of.score","x":0.375,"y":0.22894079972391029,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Website","x":0.625,"y":0.22894079972391029,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"yaxis":{"domain":[0.77105920027608965,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["New method development paper","Pure benchmarking paper"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["New method development paper","Pure benchmarking paper"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716894984},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0.77105920027608965,"y1":1},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0,"y1":23.37899543378996,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0.77105920027608965,"y1":1},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0,"y1":23.37899543378996,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0.77105920027608965,"y1":1},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0,"y1":23.37899543378996,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.75093750000000004,"x1":1,"y0":0.77105920027608965,"y1":1},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.75093750000000004,"x1":1,"y0":0,"y1":23.37899543378996,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0.52105920027608965,"y1":0.72894079972391035},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0,"y1":23.37899543378996,"yanchor":0.72894079972391035,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0.52105920027608965,"y1":0.72894079972391035},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0,"y1":23.37899543378996,"yanchor":0.72894079972391035,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0.52105920027608965,"y1":0.72894079972391035},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0,"y1":23.37899543378996,"yanchor":0.72894079972391035,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.75093750000000004,"x1":1,"y0":0.52105920027608965,"y1":0.72894079972391035},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.75093750000000004,"x1":1,"y0":0,"y1":23.37899543378996,"yanchor":0.72894079972391035,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0.27105920027608971,"y1":0.47894079972391029},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0,"y1":23.37899543378996,"yanchor":0.47894079972391029,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0.27105920027608971,"y1":0.47894079972391029},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0,"y1":23.37899543378996,"yanchor":0.47894079972391029,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0.27105920027608971,"y1":0.47894079972391029},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0,"y1":23.37899543378996,"yanchor":0.47894079972391029,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.75093750000000004,"x1":1,"y0":0.27105920027608971,"y1":0.47894079972391029},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.75093750000000004,"x1":1,"y0":0,"y1":23.37899543378996,"yanchor":0.47894079972391029,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0,"y1":0.22894079972391029},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.24906249999999999,"y0":0,"y1":23.37899543378996,"yanchor":0.22894079972391029,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0,"y1":0.22894079972391029},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.25093749999999998,"x1":0.49906250000000002,"y0":0,"y1":23.37899543378996,"yanchor":0.22894079972391029,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0,"y1":0.22894079972391029},{"type":"rect","fillcolor":"rgba(211,211,211,1)","line":{"color":"rgba(0,0,0,1)","width":1.32835201328352,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.50093750000000004,"x1":0.74906249999999996,"y0":0,"y1":23.37899543378996,"yanchor":0.22894079972391029,"ysizemode":"pixel"}],"xaxis2":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.25093749999999998,0.49906250000000002],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis3":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.50093750000000004,0.74906249999999996],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis4":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.75093750000000004,1],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis5":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0,0.24906249999999999],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y2","title":"","hoverformat":".2f"},"yaxis2":{"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["New method development paper","Pure benchmarking paper"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["New method development paper","Pure benchmarking paper"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716894984},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.52105920027608965,0.72894079972391035],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"xaxis6":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.25093749999999998,0.49906250000000002],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y2","title":"","hoverformat":".2f"},"xaxis7":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.50093750000000004,0.74906249999999996],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y2","title":"","hoverformat":".2f"},"xaxis8":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.75093750000000004,1],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y2","title":"","hoverformat":".2f"},"xaxis9":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0,0.24906249999999999],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y3","title":"","hoverformat":".2f"},"yaxis3":{"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["New method development paper","Pure benchmarking paper"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["New method development paper","Pure benchmarking paper"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716894984},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.27105920027608971,0.47894079972391029],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"xaxis10":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.25093749999999998,0.49906250000000002],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y3","title":"","hoverformat":".2f"},"xaxis11":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.50093750000000004,0.74906249999999996],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y3","title":"","hoverformat":".2f"},"xaxis12":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.75093750000000004,1],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y3","title":"","hoverformat":".2f"},"xaxis13":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0,0.24906249999999999],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y4","title":"","hoverformat":".2f"},"yaxis4":{"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["New method development paper","Pure benchmarking paper"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["New method development paper","Pure benchmarking paper"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716894984},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0,0.22894079972391029],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"xaxis14":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.25093749999999998,0.49906250000000002],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y4","title":"","hoverformat":".2f"},"xaxis15":{"type":"linear","autorange":false,"range":[-9.6000000000000014,201.59999999999999],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.6529680365296811,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-20,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"domain":[0.50093750000000004,0.74906249999999996],"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y4","title":"","hoverformat":".2f"},"showlegend":true,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"title":{"text":"Answers","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"85205806d6b":{"x":{},"y":{},"fill":{},"type":"bar"}},"cur_data":"85205806d6b","visdat":{"85205806d6b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
</div></div></div><bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>
</div>
</div>
<div id="general-information" class="dashboard-page tab-pane grid-skip html-fill-item html-fill-container" data-orientation="rows" aria-labelledby="tab-general-information">
<div class="html-fill-item html-fill-container bslib-grid" style="display: grid; grid-template-rows: minmax(3em, max-content); grid-auto-columns: minmax(0, 1fr);">
<div class="card html-fill-item html-fill-container bslib-card" data-fill="false" data-bslib-card-init="" data-require-bs-caller="card()" data-full-screen="false">
<div class="card-body html-fill-item html-fill-container">
<!-- Information -->
<div style="text-align: center;" class="html-fill-item html-fill-container">
<h3>
<a href="https://www.biorxiv.org/content/10.1101/2023.12.19.572303v1" target="_blank"> <strong>The current landscape and emerging challenges of benchmarking single-cell methods</strong> </a>
</h3>
</div>
<strong>Introduction</strong>
<p>A comprehensive evaluation strategy is critical for single-cell methodological development to assess the applicability of the method and to examine under what circumstances it works or fails.Informative evaluation results can drive the direction of method refinement. As a result, method development and evaluation should be considered as an iterative process.&nbsp;</p>
<p>Such evaluation should be distinguished from benchmarking purely based on evaluation metrics, as it aims to not only provide the method developers with a better understanding of their own methods but also demonstrate to the biologists how they can use the developed methods to gain new biological insights, which is what all the computational methods should be developed for. Due to the complexity of single-cell omics data, several key aspects should be taken into account while evaluating single-cell computational methods, including:&nbsp;</p>
<strong>Accuracy</strong>
<p>Whether the method does what it is designed to do. For example, data integration methods should remove the dataset effect and other unwanted variations while retaining the biological signals; and cell type classification methods should accurately classify the cell types and reveal the novel cell types that are not in the reference data. This aspect can usually be quantified by evaluation metrics, such as silhouette coefficients, adjusted rand index, Local Inverse Simpson’s Index (LISI) <em>harmony</em> and kBET<em>buttner2017assessment</em> for data integration methods; and classification accuracy and F1-score for cell type classification methods.</p>
<strong>Scalability</strong>
<p>Whether the method can handle large-scale single-cell data in a computationally efficient manner. This requires benchmarking both the computational time and memory usage of the method.</p>
<strong>Stability</strong>
<ul>
<li><p>Generalisability and Applicability (Stability of input): whether the method can be applied to data with different properties. This involves evaluating the developed method on data from diverse technologies (low- and high-throughput methods), tissues, organisms, stages (developmental and well-differentiated stages), disease, and sample sizes (tissue-specific and atlas-scale).</p></li>
<li><p>Robustness (Stability of model): whether the method is robust to noise in the data and the choice of hyper parameter. It can be evaluated by whether the method’s performance is significantly impacted when (i) only a subset of data is used; (ii) simulated noise is introduced into the data and (iii) models are run using different hyper parameter settings.</p></li>
<li><p>Reproducibility (Stability of output): technical vs broader. Whether the method produce same result when repeated with the same setting&nbsp;</p></li>
</ul>
<strong>Interpretability</strong>
<ul>
<li><p>Impact on downstream analysis: whether applying the method improves or hinders the performance of downstream analysis that is described in Section <em>downstream</em>. For example, for data integration of developmental data, it is important that the integrated data preserve the continuous signals so that the trajectory analysis can be applied to infer the pseudotime. For example, batch correction methods that return adjusted matrix would be more useful than methods that return embedding.&nbsp;</p></li>
<li><p>Biological impact: whether the developed method can help biologists to gain new biological insights. For example, data integration of multi-omics that is able to reveal rare and novel cell types, which could not be identified using single omics, would allow biologists to examine the cell characteristics of such rare cell types.</p></li>
</ul>
</div>
<bslib-tooltip placement="auto" bsoptions="[]" data-require-bs-version="5" data-require-bs-caller="tooltip()">
    <template>Expand</template>
    <span class="bslib-full-screen-enter badge rounded-pill">
        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24" style="height:1em;width:1em;" aria-hidden="true" role="img"><path d="M20 5C20 4.4 19.6 4 19 4H13C12.4 4 12 3.6 12 3C12 2.4 12.4 2 13 2H21C21.6 2 22 2.4 22 3V11C22 11.6 21.6 12 21 12C20.4 12 20 11.6 20 11V5ZM4 19C4 19.6 4.4 20 5 20H11C11.6 20 12 20.4 12 21C12 21.6 11.6 22 11 22H3C2.4 22 2 21.6 2 21V13C2 12.4 2.4 12 3 12C3.6 12 4 12.4 4 13V19Z"></path></svg>
    </span>
</bslib-tooltip><script data-bslib-card-init="">bslib.Card.initializeAllCards();</script></div>


<div id="3ade8a4a-fb1d-4a6c-8409-ac45482d5fc9" class="hidden html-fill-item html-fill-container">

</div>
</div>
</div>
</div>


<script type="ojs-module-contents">
{"contents":[{"methodName":"interpret","cellName":"ojs-cell-1","inline":false,"source":"\n//d3 = require(\"d3@7\")\n//topojson = require(\"topojson\")\n//jQuery = require(\"jQuery\")\n\nd3 = require(\"d3@7\")\njQuery = require(\"jquery\")\n\ncategory_list = [...new Set(paper_data.Paper_category)];\n\nviewof category = Inputs.checkbox(\n  category_list, \n  {\n    value:  category_list, \n    label: \"Data category: \"\n  })\n\n"},{"methodName":"interpret","cellName":"ojs-cell-2","inline":false,"source":"\ndatatype_list = [...new Set(\n  paper_data.Paper_category\n    .map((value, index) => category.includes(value) ? paper_data.data_type[index] : null)\n    .filter(value => value !== null)\n)];\n\n//console.log(\"datatype_list\", datatype_list)\n\nviewof data_type = Inputs.select(\n  datatype_list, \n  {\n    value: \"Single-cell RNA-seq\",\n    label: \"Data type: \"\n  })\n\n"},{"methodName":"interpret","cellName":"ojs-cell-3","inline":false,"source":"\nbroaderTopic_list = \n  paper_data.Paper_category\n    .map((value, index) => category.includes(value) && data_type.includes(paper_data.data_type[index]) ? paper_data.broader_topic[index] : null)\n    .filter(value => value !== null);\n\nviewof broader_topic = Inputs.select(\n  broaderTopic_list, \n  {\n    value: \"Intermediate analysis\",\n    label: \"Broader topic: \",\n    unique: true\n  })\n\n"},{"methodName":"interpret","cellName":"ojs-cell-4","inline":false,"source":"\nfinerTopic_list = [...new Set(\n  paper_data.Paper_category\n    .map((value, index) => category.includes(value) && \n    data_type.includes(paper_data.data_type[index]) && \n    broader_topic.includes(paper_data.broader_topic[index]) ? paper_data.finer_topic[index] : null).filter(value => value !== null)\n)];\n\nviewof finer_topic = Inputs.select(\n finerTopic_list, \n  {\n    value: \"Cell type/state identification\",\n    label: \"Finer topic: \"\n  })\n\n"},{"methodName":"interpret","cellName":"ojs-cell-5","inline":false,"source":"\npaperTitle_list = [...new Set(\n  paper_data.Paper_category\n    .map((value, index) => category.includes(value) && \n    data_type.includes(paper_data.data_type[index]) && \n    broader_topic.includes(paper_data.broader_topic[index]) &&\n    finer_topic.includes(paper_data.finer_topic[index]) ? \n    paper_data.Paper_title[index] : null).filter(value => value !== null)\n)];\n\nviewof paper_title = Inputs.select(\n paperTitle_list, \n  {\n    label: \"Paper title: \",\n    width: \"100%\"\n  })\n\n"},{"methodName":"interpret","cellName":"ojs-cell-6","inline":false,"source":"\nselected_paper_link = paper_links.Paper_title\n    .map((value, index) => paper_title.includes(value) ? \n    paper_links.Link[index] : null).filter(value => value !== null)[0];\n\nhtml`\n  <iframe src=${selected_paper_link} width=\"100%\" height=\"400px\" title=\"Research paper\"></iframe>\n`;\n\n"},{"methodName":"interpret","cellName":"ojs-cell-7","inline":false,"source":"\nselected_columns = [\n  \"Paper_category\",\n  \"data_type\",\n  \"broader_topic\",\n  \"finer_topic\",\n  \"publication_date\",\n  \"Max_number_of_cells\",\n  \"Data_availability\",\n  \"Recommendation\",\n  \"Applicability\",\n  \"Sensitivity_analysis\",\n  \"Variability_of_score\"\n]\n\nfiltered_paper_data = Object.keys(paper_data).reduce((obj, key) => {\n  const index = paper_data.Paper_title.indexOf(paper_title);\n  \n  if (index !== -1) {\n    obj[key] = paper_data[key][index];\n  }\n  \n  return obj;\n}, {});\n\n//console.log(\"Filtered_paper_data\", filtered_paper_data)\n\nselected_data = selected_columns.reduce((obj, key) => {\n  if (filtered_paper_data[key] !== undefined) {\n    obj[key] = filtered_paper_data[key];\n  }\n  return obj;\n}, {});\n\n//console.log(\"Selected data\", selected_data)\n\n/*html`\n  <div style=\"display: flex; flex-wrap: wrap; gap: 10px;\">\n    ${Object.entries(selected_data).map(([key, value]) => {\n      return html`\n        <div class=\"card\">\n          <div class=\"card-body\">\n            <h5 class=\"card-title text-white\" style=\"background-color: #00aa49;\"> ${key}</h5>\n            <p class=\"card-text\">${value}</p>\n        </div>\n      `;\n    })}\n  </div>\n`*/\n\n\n// Create HTML for cards\nhtml`\n  <div style=\"display: flex; flex-wrap: wrap; gap: 10px;\">\n    ${Object.entries(selected_data).map(([key, value]) => {\n      return html`\n        <div style=\"border: 1px solid #ccc; border-radius: 8px; padding: 5px; width: 220px; box-shadow: 2px 2px 5px rgba(0,0,0,0.1);\">\n          <h4 style=\"margin: 0 0 0 0; font-size: 18px; color: #fff; background-color: #2C5488; padding: 5px; border-radius: 4px;\">\n            ${key.charAt(0).toUpperCase() + key.slice(1).replaceAll(\"_\", \" \")}\n          </h4>\n          <p style=\"margin: 5; font-size: 14px; color: #555;\">\n            ${value}\n          </p>\n        </div>\n      `;\n    })}\n  </div>\n`\n\n/*first_indices = paper_data.Paper_category.map((c, index) => c === \"New method development paper\" ? index : -1).filter(index => index !== -1);\n\nselected_paper_category = \"New method development paper\"\n\nconsole.log(\"First indices\", first_indices)\n\nfirst_filtered_data = paper_data.filter(p => category.includes(p.paper_category))\n\nconsole.log(\"First indices length\", first_filtered_data.length) \nconsole.log(\"First indices\", first_filtered_data)\n\n\nsecondFilter = first_indices.map(index => paper_data.data_type[index])\n\nconsole.log(\"Second filter\", secondFilter)\n\nselected_data_type = [\"Single-cell Hi-C\", \"Single-cell mass cytometry\"]\n\nsecond_filtered_data = first_filtered_data.filter(p => selected_data_type.includes(p.data_type))\n\nconsole.log(\"Second indices length\", second_filtered_data.length)\nconsole.log(\"Second indices\", second_filtered_data)\n\nfiltered_data = paper_data.filter(p => category.includes(p.Paper_category))\n\nconsole.log(filtered_data)\n\nviewof data_type = Inputs.checkbox(\n  [...new Set(paper_data.data_type)], \n  {\n    value:  [...new Set(paper_data.data_type)], \n    label: \"Data type: \"\n  })*/\n\n\n\n/*category_list = [...new Set(paper_data.Paper_category)];\n\nconsole.log(\"category_list: \", category_list)\n\ndatatype_list = [...new Set(paper_data.data_type)];\n\nconsole.log(\"datatype list: \", datatype_list)\n\nbroader_topic_list = [...new Set(paper_data.broader_topic)];\n\nconsole.log(\"broader_topic: \", broader_topic_list)\n\nfiner_topic_list = [...new Set(paper_data.finer_topic)];\n\nconsole.log(\"finer_topic: \", finer_topic_list)\n\nPaper_title_list = [...new Set(paper_data.Paper_title)];\n\nconsole.log(\"paper title: \", Paper_title_list)*/\n\n"},{"methodName":"interpretQuiet","source":"shinyInput('category')"},{"methodName":"interpretQuiet","source":"shinyInput('data_type')"},{"methodName":"interpretQuiet","source":"shinyInput('broader_topic')"},{"methodName":"interpretQuiet","source":"shinyInput('finer_topic')"},{"methodName":"interpretQuiet","source":"shinyInput('paper_title')"}]}
</script>
<script type="module">
if (window.location.protocol === "file:") { alert("The OJS runtime does not work with file:// URLs. Please use a web server to view this document."); }
window._ojs.paths.runtimeToDoc = "../..";
window._ojs.paths.runtimeToRoot = "../..";
window._ojs.paths.docToRoot = "";
window._ojs.selfContained = false;
window._ojs.runtime.interpretFromScriptTags();
</script>
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/sydneybiox\.github\.io\/sc_bench_benchmark_dashboard\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>© Sydney Precision Data Science Centre 2024</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>

<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (_event) {
  if (window.bslib.Card) {
    window.bslib.Card.initializeAllCards();
  }
}); 
</script>
  



</body></html>